{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "2f4e6dbd",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import os\n",
    "import pickle\n",
    "\n",
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "86a04792",
   "metadata": {},
   "outputs": [],
   "source": [
    "TREC_COVID_DATASET_URL=\"https://huggingface.co/datasets/BeIR/trec-covid/resolve/main/corpus.jsonl.gz\"\n",
    "TREC_COVID_DATASET_FILENAME=os.path.basename(TREC_COVID_DATASET_URL)\n",
    "\n",
    "TREC_COVID_QUERIES_URL=\"https://huggingface.co/datasets/BeIR/trec-covid/resolve/main/queries.jsonl.gz\"\n",
    "TREC_COVID_QUERIES_FILENAME=os.path.basename(TREC_COVID_QUERIES_URL)\n",
    "\n",
    "TREC_COVID_QRELS_URL=\"https://huggingface.co/datasets/BeIR/trec-covid-qrels/raw/main/test.tsv\"\n",
    "TREC_COVID_QRELS_FILENAME=os.path.basename(TREC_COVID_QRELS_URL)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "cbd9eec3",
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option('display.max_colwidth', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "eec118dc",
   "metadata": {},
   "outputs": [],
   "source": [
    "def read_line_json(which_filename):\n",
    "    line_json_entries = []\n",
    "\n",
    "    with open(which_filename) as inputFile:\n",
    "        for line in inputFile:\n",
    "            line_json_entries.append(json.loads(line))\n",
    "            \n",
    "    return line_json_entries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "13d3f202",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2023-04-02 10:30:49--  https://huggingface.co/datasets/BeIR/trec-covid/resolve/main/corpus.jsonl.gz\n",
      "Resolving huggingface.co (huggingface.co)... 2600:1f18:147f:e850:727f:50a8:8fb4:8ed2, 2600:1f18:147f:e800:fb72:485f:7796:fcdf, 2600:1f18:147f:e850:e8b8:bf4c:ea70:44c3, ...\n",
      "Connecting to huggingface.co (huggingface.co)|2600:1f18:147f:e850:727f:50a8:8fb4:8ed2|:443... connected.\n",
      "HTTP request sent, awaiting response... 302 Found\n",
      "Location: https://cdn-lfs.huggingface.co/repos/a8/10/a810e88b0e7b233be82b89c1fa6ec2d75efc6d55784c2ada9dcac8434a634f3a/e9e97686e3138eaff989f67c04cd32e8f8f4c0d4857187e3f180275b23e24e85?response-content-disposition=attachment%3B+filename*%3DUTF-8%27%27corpus.jsonl.gz%3B+filename%3D%22corpus.jsonl.gz%22%3B&response-content-type=application%2Fgzip&Expires=1680701450&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9jZG4tbGZzLmh1Z2dpbmdmYWNlLmNvL3JlcG9zL2E4LzEwL2E4MTBlODhiMGU3YjIzM2JlODJiODljMWZhNmVjMmQ3NWVmYzZkNTU3ODRjMmFkYTlkY2FjODQzNGE2MzRmM2EvZTllOTc2ODZlMzEzOGVhZmY5ODlmNjdjMDRjZDMyZThmOGY0YzBkNDg1NzE4N2UzZjE4MDI3NWIyM2UyNGU4NT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE2ODA3MDE0NTB9fX1dfQ__&Signature=m9DVzQ2CfDv-afOZp%7EIwjM%7Ed71JCvEmthZhWy1VpumZuyxl3M3UN95gU51X0rJ0Q89OcfhEMDV2Mqg0FROggIzXRuQ9qZtSzavvyzBz9hBJI8F-rs-JR1RDK-nP5krIQYQO7cnIQWEzvFdWQnkS6nfWgT2wRcudkG-bPVvpDu3vcGWS9bfAtCSG%7EEQ-9TiWJU4JRn6wvmYDj%7Ey-UwFBP2uaqZXVFHZxRilDOLTFAL7TLnvxwsBMlFLEFOC2dMitwuwa6VIHs6rDu0NeOZfD4oAy%7Ezm08Ef8dkc%7Ejhb4IzpwMvM%7E6p9iDF4EOUHA9q01u8MnTXSlob6TFqPBC4fT8XA__&Key-Pair-Id=KVTP0A1DKRTAX [following]\n",
      "--2023-04-02 10:30:49--  https://cdn-lfs.huggingface.co/repos/a8/10/a810e88b0e7b233be82b89c1fa6ec2d75efc6d55784c2ada9dcac8434a634f3a/e9e97686e3138eaff989f67c04cd32e8f8f4c0d4857187e3f180275b23e24e85?response-content-disposition=attachment%3B+filename*%3DUTF-8%27%27corpus.jsonl.gz%3B+filename%3D%22corpus.jsonl.gz%22%3B&response-content-type=application%2Fgzip&Expires=1680701450&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9jZG4tbGZzLmh1Z2dpbmdmYWNlLmNvL3JlcG9zL2E4LzEwL2E4MTBlODhiMGU3YjIzM2JlODJiODljMWZhNmVjMmQ3NWVmYzZkNTU3ODRjMmFkYTlkY2FjODQzNGE2MzRmM2EvZTllOTc2ODZlMzEzOGVhZmY5ODlmNjdjMDRjZDMyZThmOGY0YzBkNDg1NzE4N2UzZjE4MDI3NWIyM2UyNGU4NT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE2ODA3MDE0NTB9fX1dfQ__&Signature=m9DVzQ2CfDv-afOZp%7EIwjM%7Ed71JCvEmthZhWy1VpumZuyxl3M3UN95gU51X0rJ0Q89OcfhEMDV2Mqg0FROggIzXRuQ9qZtSzavvyzBz9hBJI8F-rs-JR1RDK-nP5krIQYQO7cnIQWEzvFdWQnkS6nfWgT2wRcudkG-bPVvpDu3vcGWS9bfAtCSG%7EEQ-9TiWJU4JRn6wvmYDj%7Ey-UwFBP2uaqZXVFHZxRilDOLTFAL7TLnvxwsBMlFLEFOC2dMitwuwa6VIHs6rDu0NeOZfD4oAy%7Ezm08Ef8dkc%7Ejhb4IzpwMvM%7E6p9iDF4EOUHA9q01u8MnTXSlob6TFqPBC4fT8XA__&Key-Pair-Id=KVTP0A1DKRTAX\n",
      "Resolving cdn-lfs.huggingface.co (cdn-lfs.huggingface.co)... 2600:9000:24f5:4600:11:f807:5180:93a1, 2600:9000:24f5:b200:11:f807:5180:93a1, 2600:9000:24f5:6200:11:f807:5180:93a1, ...\n",
      "Connecting to cdn-lfs.huggingface.co (cdn-lfs.huggingface.co)|2600:9000:24f5:4600:11:f807:5180:93a1|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 73452199 (70M) [application/gzip]\n",
      "Saving to: ‘corpus.jsonl.gz’\n",
      "\n",
      "corpus.jsonl.gz     100%[===================>]  70,05M  22,4MB/s    in 3,6s    \n",
      "\n",
      "2023-04-02 10:30:54 (19,6 MB/s) - ‘corpus.jsonl.gz’ saved [73452199/73452199]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget {TREC_COVID_DATASET_URL}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "36045660",
   "metadata": {},
   "outputs": [],
   "source": [
    "!gunzip {TREC_COVID_DATASET_FILENAME}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "903a4e3a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2023-04-02 10:40:58--  https://huggingface.co/datasets/BeIR/trec-covid/resolve/main/queries.jsonl.gz\n",
      "Resolving huggingface.co (huggingface.co)... 2600:1f18:147f:e800:f843:4174:6d04:4c66, 2600:1f18:147f:e850:df5c:b7d3:ef4d:663d, 2600:1f18:147f:e850:727f:50a8:8fb4:8ed2, ...\n",
      "Connecting to huggingface.co (huggingface.co)|2600:1f18:147f:e800:f843:4174:6d04:4c66|:443... connected.\n",
      "HTTP request sent, awaiting response... 302 Found\n",
      "Location: https://cdn-lfs.huggingface.co/repos/a8/10/a810e88b0e7b233be82b89c1fa6ec2d75efc6d55784c2ada9dcac8434a634f3a/9eadcc2cdf140addc9dae83648bb2c6611f5e4b66eaed7475fa5a0ca48eda371?response-content-disposition=attachment%3B+filename*%3DUTF-8%27%27queries.jsonl.gz%3B+filename%3D%22queries.jsonl.gz%22%3B&response-content-type=application%2Fgzip&Expires=1680702059&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9jZG4tbGZzLmh1Z2dpbmdmYWNlLmNvL3JlcG9zL2E4LzEwL2E4MTBlODhiMGU3YjIzM2JlODJiODljMWZhNmVjMmQ3NWVmYzZkNTU3ODRjMmFkYTlkY2FjODQzNGE2MzRmM2EvOWVhZGNjMmNkZjE0MGFkZGM5ZGFlODM2NDhiYjJjNjYxMWY1ZTRiNjZlYWVkNzQ3NWZhNWEwY2E0OGVkYTM3MT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE2ODA3MDIwNTl9fX1dfQ__&Signature=l%7ERAZJkZd3akgQJ5E2Qpj1lXSOXjWbdWaHgDWWYNGaKhWZfWcb7VGQH6FafvCH5PR2z4Ezw4QNwUrHKmoDSZjXjKREdrhxGi1oWCJTvnqVAQ1IRvarxJHxfEtXz2KgAJp1VVll9aDO-3G7idDfD6JlDhAUBBfirpnd2phemUcrtxFaGkgnueOtaQEMXET4VU8M2tVPZCLFsk1v7Mbr9VwMsnC8jqqI3ZjwDAG6IB2hEUTOvX4l2sBj7Xgw2OwrIBnyjDYLxkv4kd8WORui9b4%7E1H1g8JP6Dtxm-X-DN0VxkM46u-l8eEJNJGOn6BC0ozc%7E5D2E-GxQghgnVzuFtcAg__&Key-Pair-Id=KVTP0A1DKRTAX [following]\n",
      "--2023-04-02 10:40:58--  https://cdn-lfs.huggingface.co/repos/a8/10/a810e88b0e7b233be82b89c1fa6ec2d75efc6d55784c2ada9dcac8434a634f3a/9eadcc2cdf140addc9dae83648bb2c6611f5e4b66eaed7475fa5a0ca48eda371?response-content-disposition=attachment%3B+filename*%3DUTF-8%27%27queries.jsonl.gz%3B+filename%3D%22queries.jsonl.gz%22%3B&response-content-type=application%2Fgzip&Expires=1680702059&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9jZG4tbGZzLmh1Z2dpbmdmYWNlLmNvL3JlcG9zL2E4LzEwL2E4MTBlODhiMGU3YjIzM2JlODJiODljMWZhNmVjMmQ3NWVmYzZkNTU3ODRjMmFkYTlkY2FjODQzNGE2MzRmM2EvOWVhZGNjMmNkZjE0MGFkZGM5ZGFlODM2NDhiYjJjNjYxMWY1ZTRiNjZlYWVkNzQ3NWZhNWEwY2E0OGVkYTM3MT9yZXNwb25zZS1jb250ZW50LWRpc3Bvc2l0aW9uPSomcmVzcG9uc2UtY29udGVudC10eXBlPSoiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE2ODA3MDIwNTl9fX1dfQ__&Signature=l%7ERAZJkZd3akgQJ5E2Qpj1lXSOXjWbdWaHgDWWYNGaKhWZfWcb7VGQH6FafvCH5PR2z4Ezw4QNwUrHKmoDSZjXjKREdrhxGi1oWCJTvnqVAQ1IRvarxJHxfEtXz2KgAJp1VVll9aDO-3G7idDfD6JlDhAUBBfirpnd2phemUcrtxFaGkgnueOtaQEMXET4VU8M2tVPZCLFsk1v7Mbr9VwMsnC8jqqI3ZjwDAG6IB2hEUTOvX4l2sBj7Xgw2OwrIBnyjDYLxkv4kd8WORui9b4%7E1H1g8JP6Dtxm-X-DN0VxkM46u-l8eEJNJGOn6BC0ozc%7E5D2E-GxQghgnVzuFtcAg__&Key-Pair-Id=KVTP0A1DKRTAX\n",
      "Resolving cdn-lfs.huggingface.co (cdn-lfs.huggingface.co)... 2600:9000:24f5:d800:11:f807:5180:93a1, 2600:9000:24f5:ea00:11:f807:5180:93a1, 2600:9000:24f5:6600:11:f807:5180:93a1, ...\n",
      "Connecting to cdn-lfs.huggingface.co (cdn-lfs.huggingface.co)|2600:9000:24f5:d800:11:f807:5180:93a1|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 4702 (4,6K) [application/gzip]\n",
      "Saving to: ‘queries.jsonl.gz’\n",
      "\n",
      "queries.jsonl.gz    100%[===================>]   4,59K  --.-KB/s    in 0s      \n",
      "\n",
      "2023-04-02 10:40:59 (202 MB/s) - ‘queries.jsonl.gz’ saved [4702/4702]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget {TREC_COVID_QUERIES_URL}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "fddcafc3",
   "metadata": {},
   "outputs": [],
   "source": [
    "!gunzip {TREC_COVID_QUERIES_FILENAME}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "d3ec6ccb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--2023-04-02 10:47:23--  https://huggingface.co/datasets/BeIR/trec-covid-qrels/raw/main/test.tsv\n",
      "Resolving huggingface.co (huggingface.co)... 2600:1f18:147f:e850:4408:4de3:f379:c94c, 2600:1f18:147f:e800:fb72:485f:7796:fcdf, 2600:1f18:147f:e800:4313:62df:3a65:665f, ...\n",
      "Connecting to huggingface.co (huggingface.co)|2600:1f18:147f:e850:4408:4de3:f379:c94c|:443... connected.\n",
      "HTTP request sent, awaiting response... 200 OK\n",
      "Length: 980831 (958K) [text/plain]\n",
      "Saving to: ‘test.tsv’\n",
      "\n",
      "test.tsv            100%[===================>] 957,84K  1,30MB/s    in 0,7s    \n",
      "\n",
      "2023-04-02 10:47:25 (1,30 MB/s) - ‘test.tsv’ saved [980831/980831]\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!wget {TREC_COVID_QRELS_URL}"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "43db8b03",
   "metadata": {},
   "source": [
    "### Load the data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3c6aaff4",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_corpus =read_line_json(TREC_COVID_DATASET_FILENAME[:-3])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "b63a5d2c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "171332"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(trec_covid_corpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "579b6a24",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_corpus_df = pd.DataFrame(trec_covid_corpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "ca1cc632",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>_id</th>\n",
       "      <th>title</th>\n",
       "      <th>text</th>\n",
       "      <th>metadata</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ug7v899j</td>\n",
       "      <td>Clinical features of culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia</td>\n",
       "      <td>OBJECTIVE: This retrospective chart review describes the epidemiology and clinical features of 40 patients with culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia. METHODS: Patients with positive M. pneumoniae cultures from respiratory specimens from January 1997 through December 1998 were identified through the Microbiology records. Charts of patients were reviewed. RESULTS: 40 patients were identified, 33 (82.5%) of whom required admission. Most infections (92.5%) were community-acquired. The infection affected all age groups but was most common in infants (32.5%) and pre-school children (22.5%). It occurred year-round but was most common in the fall (35%) and spring (30%). More than three-quarters of patients (77.5%) had comorbidities. Twenty-four isolates (60%) were associated with pneumonia, 14 (35%) with upper respiratory tract infections, and 2 (5%) with bronchiolitis. Cough (82.5%), fever (75%), and malaise (58.8%) were the most common symptoms, and crepitations (60%), and wheezes (40%) were the most common signs. Most patients with pneumonia had crepitations (79.2%) but only 25% had bronchial breathing. Immunocompromised patients were more likely than non-immunocompromised patients to present with pneumonia (8/9 versus 16/31, P = 0.05). Of the 24 patients with pneumonia, 14 (58.3%) had uneventful recovery, 4 (16.7%) recovered following some complications, 3 (12.5%) died because of M pneumoniae infection, and 3 (12.5%) died due to underlying comorbidities. The 3 patients who died of M pneumoniae pneumonia had other comorbidities. CONCLUSION: our results were similar to published data except for the finding that infections were more common in infants and preschool children and that the mortality rate of pneumonia in patients with comorbidities was high.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC35282/', 'pubmed_id': '11472636'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>02tnwd4m</td>\n",
       "      <td>Nitric oxide: a pro-inflammatory mediator in lung disease?</td>\n",
       "      <td>Inflammatory diseases of the respiratory tract are commonly associated with elevated production of nitric oxide (NO•) and increased indices of NO• -dependent oxidative stress. Although NO• is known to have anti-microbial, anti-inflammatory and anti-oxidant properties, various lines of evidence support the contribution of NO• to lung injury in several disease models. On the basis of biochemical evidence, it is often presumed that such NO• -dependent oxidations are due to the formation of the oxidant peroxynitrite, although alternative mechanisms involving the phagocyte-derived heme proteins myeloperoxidase and eosinophil peroxidase might be operative during conditions of inflammation. Because of the overwhelming literature on NO• generation and activities in the respiratory tract, it would be beyond the scope of this commentary to review this area comprehensively. Instead, it focuses on recent evidence and concepts of the presumed contribution of NO• to inflammatory diseases of the lung.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59543/', 'pubmed_id': '11667967'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ejv2xln0</td>\n",
       "      <td>Surfactant protein-D and pulmonary host defense</td>\n",
       "      <td>Surfactant protein-D (SP-D) participates in the innate response to inhaled microorganisms and organic antigens, and contributes to immune and inflammatory regulation within the lung. SP-D is synthesized and secreted by alveolar and bronchiolar epithelial cells, but is also expressed by epithelial cells lining various exocrine ducts and the mucosa of the gastrointestinal and genitourinary tracts. SP-D, a collagenous calcium-dependent lectin (or collectin), binds to surface glycoconjugates expressed by a wide variety of microorganisms, and to oligosaccharides associated with the surface of various complex organic antigens. SP-D also specifically interacts with glycoconjugates and other molecules expressed on the surface of macrophages, neutrophils, and lymphocytes. In addition, SP-D binds to specific surfactant-associated lipids and can influence the organization of lipid mixtures containing phosphatidylinositol in vitro. Consistent with these diverse in vitro activities is the observation that SP-D-deficient transgenic mice show abnormal accumulations of surfactant lipids, and respond abnormally to challenge with respiratory viruses and bacterial lipopolysaccharides. The phenotype of macrophages isolated from the lungs of SP-D-deficient mice is altered, and there is circumstantial evidence that abnormal oxidant metabolism and/or increased metalloproteinase expression contributes to the development of emphysema. The expression of SP-D is increased in response to many forms of lung injury, and deficient accumulation of appropriately oligomerized SP-D might contribute to the pathogenesis of a variety of human lung diseases.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59549/', 'pubmed_id': '11667972'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2b73a28n</td>\n",
       "      <td>Role of endothelin-1 in lung disease</td>\n",
       "      <td>Endothelin-1 (ET-1) is a 21 amino acid peptide with diverse biological activity that has been implicated in numerous diseases. ET-1 is a potent mitogen regulator of smooth muscle tone, and inflammatory mediator that may play a key role in diseases of the airways, pulmonary circulation, and inflammatory lung diseases, both acute and chronic. This review will focus on the biology of ET-1 and its role in lung disease.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59574/', 'pubmed_id': '11686871'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>9785vg6d</td>\n",
       "      <td>Gene expression in epithelial cells in response to pneumovirus infection</td>\n",
       "      <td>Respiratory syncytial virus (RSV) and pneumonia virus of mice (PVM) are viruses of the family Paramyxoviridae, subfamily pneumovirus, which cause clinically important respiratory infections in humans and rodents, respectively. The respiratory epithelial target cells respond to viral infection with specific alterations in gene expression, including production of chemoattractant cytokines, adhesion molecules, elements that are related to the apoptosis response, and others that remain incompletely understood. Here we review our current understanding of these mucosal responses and discuss several genomic approaches, including differential display reverse transcription-polymerase chain reaction (PCR) and gene array strategies, that will permit us to unravel the nature of these responses in a more complete and systematic manner.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59580/', 'pubmed_id': '11686888'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171327</th>\n",
       "      <td>7e8r61e7</td>\n",
       "      <td>Can Pediatric COVID-19 Testing Sensitivity Be Improved With Sequential Tests?</td>\n",
       "      <td></td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32427657/; https://doi.org/10.1213/ane.0000000000004982', 'pubmed_id': '32427657'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171328</th>\n",
       "      <td>6jittbis</td>\n",
       "      <td>Heterogeneity and plasticity of porcine alveolar macrophage and pulmonary interstitial macrophage isolated from healthy pigs in vitro</td>\n",
       "      <td>This study investigated the heterogeneity and plasticity of porcine alveolar macrophages (PAM) and pulmonary interstitial macrophages (IM) isolated from healthy pigs, including phenotype, function and gene expression. Dynamic changes of nitric oxide (NO) levels secreted by PAM and IM with stimulation of different doses of lipopolysaccharide (LPS) were investigated by Griess method, and the viability of the PAM and IM cells was investigated by MTT assay. Flow cytometry, fluorescence quantitative PCR and ELISA techniques were used to measure cell phenotype, gene expression and cytokine secretion, respectively. The PAM and IM cells in normal healthy pigs showed heterogeneity with 95.42±1.51% and 31.99±5.84% of CD163+ macrophage, respectively. The NO level in IM was significantly higher versus PAM after LPS treatment. Consistently, the ratio of Arg I/iNOS in IM was much lower than that in PAM, suggesting that the PAM belong to M2 macrophages and the IM belong to M1 macrophages. The PAM and IM cells in normal healthy pigs also showed plasticity. The Arg I/iNOS ratio and TIMP1/MMP12 ratio were significantly decreased in LPS- or LPS+IFNγ-treated PAM and IM, suggesting that cells were polarized towards M1 macrophages under LPS or LPS+IFNγ stimulation. On the contrary, IL-4 and IL-13 stimulation on PAM and IM lead to M2 polarization. A similar result was found in IL-1β gene expression and TNFα secretion. In conclusion, porcine macrophages have shown heterogeneity and plasticity on polarization under the stimulation of LPS, IFNγ, IL-4 and IL-13.</td>\n",
       "      <td>{'url': 'https://doi.org/10.1242/bio.046342; https://www.ncbi.nlm.nih.gov/pubmed/31615770/', 'pubmed_id': '31615770'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171329</th>\n",
       "      <td>hi8k8wvb</td>\n",
       "      <td>SARS E protein in phospholipid bilayers: an anomalous X-ray reflectivity study</td>\n",
       "      <td>Abstract We report on an anomalous X-ray reflectivity study to locate a labelled residue of a membrane protein with respect to the lipid bilayer. From such experiments, important constraints on the protein or peptide conformation can be derived. Specifically, our aim is to localize an iodine-labelled phenylalanine in the SARS E protein, incorporated in DMPC phospholipid bilayers, which are deposited in the form of thick multilamellar stacks on silicon surfaces. Here, we discuss the experimental aspects and the difficulties associated with the Fourier synthesis analysis that gives the electron density profile of the membranes.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32288217/; https://doi.org/10.1016/j.physb.2004.11.015; https://www.sciencedirect.com/science/article/pii/S0921452604011354; https://api.elsevier.com/content/article/pii/S0921452604011354', 'pubmed_id': '32288217'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171330</th>\n",
       "      <td>ma3ndg41</td>\n",
       "      <td>Italian Society of Interventional Cardiology (GISE) position paper for Cath lab‐specific preparedness recommendations for healthcare providers in case of suspected, probable or confirmed cases of COVID‐19</td>\n",
       "      <td>COVID‐19 pandemic raised the issue to guarantee the proper level of care to patients with acute cardiovascular diseases and concomitant suspected or confirmed COVID‐19 and, in the meantime safety and protection of healthcare providers. The aim of this position paper is to provide standards to healthcare facilities and healthcare providers on infection prevention and control measures during the management of suspected and confirmed cases of 2019‐nCoV infection accessing in cath‐lab. The document represents the view of the Italian Society of Interventional Cardiology (GISE), and it is based on recommendations from the main World and European Health Organizations (WHO, and ECDC) as well as from the Italian Society of Anesthesia, Analgesia, Resuscitation and Intensive Care (SIAARTI).</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32223063/; https://doi.org/10.1002/ccd.28888', 'pubmed_id': '32223063'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171331</th>\n",
       "      <td>pnl9th2c</td>\n",
       "      <td>Vascular Life during the COVID-19 Pandemic Reminds Us to Prepare for the Unexpected</td>\n",
       "      <td></td>\n",
       "      <td>{'url': 'https://www.sciencedirect.com/science/article/pii/S1078588420303804?v=s5; https://www.ncbi.nlm.nih.gov/pubmed/32446539/; https://doi.org/10.1016/j.ejvs.2020.04.040; https://api.elsevier.com/content/article/pii/S1078588420303804', 'pubmed_id': '32446539'}</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>171332 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             _id  \\\n",
       "0       ug7v899j   \n",
       "1       02tnwd4m   \n",
       "2       ejv2xln0   \n",
       "3       2b73a28n   \n",
       "4       9785vg6d   \n",
       "...          ...   \n",
       "171327  7e8r61e7   \n",
       "171328  6jittbis   \n",
       "171329  hi8k8wvb   \n",
       "171330  ma3ndg41   \n",
       "171331  pnl9th2c   \n",
       "\n",
       "                                                                                                                                                                                                               title  \\\n",
       "0                                                                                   Clinical features of culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia   \n",
       "1                                                                                                                                                         Nitric oxide: a pro-inflammatory mediator in lung disease?   \n",
       "2                                                                                                                                                                    Surfactant protein-D and pulmonary host defense   \n",
       "3                                                                                                                                                                               Role of endothelin-1 in lung disease   \n",
       "4                                                                                                                                           Gene expression in epithelial cells in response to pneumovirus infection   \n",
       "...                                                                                                                                                                                                              ...   \n",
       "171327                                                                                                                                 Can Pediatric COVID-19 Testing Sensitivity Be Improved With Sequential Tests?   \n",
       "171328                                                                         Heterogeneity and plasticity of porcine alveolar macrophage and pulmonary interstitial macrophage isolated from healthy pigs in vitro   \n",
       "171329                                                                                                                                SARS E protein in phospholipid bilayers: an anomalous X-ray reflectivity study   \n",
       "171330  Italian Society of Interventional Cardiology (GISE) position paper for Cath lab‐specific preparedness recommendations for healthcare providers in case of suspected, probable or confirmed cases of COVID‐19   \n",
       "171331                                                                                                                           Vascular Life during the COVID-19 Pandemic Reminds Us to Prepare for the Unexpected   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           text  \\\n",
       "0       OBJECTIVE: This retrospective chart review describes the epidemiology and clinical features of 40 patients with culture-proven Mycoplasma pneumoniae infections at King Abdulaziz University Hospital, Jeddah, Saudi Arabia. METHODS: Patients with positive M. pneumoniae cultures from respiratory specimens from January 1997 through December 1998 were identified through the Microbiology records. Charts of patients were reviewed. RESULTS: 40 patients were identified, 33 (82.5%) of whom required admission. Most infections (92.5%) were community-acquired. The infection affected all age groups but was most common in infants (32.5%) and pre-school children (22.5%). It occurred year-round but was most common in the fall (35%) and spring (30%). More than three-quarters of patients (77.5%) had comorbidities. Twenty-four isolates (60%) were associated with pneumonia, 14 (35%) with upper respiratory tract infections, and 2 (5%) with bronchiolitis. Cough (82.5%), fever (75%), and malaise (58.8%) were the most common symptoms, and crepitations (60%), and wheezes (40%) were the most common signs. Most patients with pneumonia had crepitations (79.2%) but only 25% had bronchial breathing. Immunocompromised patients were more likely than non-immunocompromised patients to present with pneumonia (8/9 versus 16/31, P = 0.05). Of the 24 patients with pneumonia, 14 (58.3%) had uneventful recovery, 4 (16.7%) recovered following some complications, 3 (12.5%) died because of M pneumoniae infection, and 3 (12.5%) died due to underlying comorbidities. The 3 patients who died of M pneumoniae pneumonia had other comorbidities. CONCLUSION: our results were similar to published data except for the finding that infections were more common in infants and preschool children and that the mortality rate of pneumonia in patients with comorbidities was high.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Inflammatory diseases of the respiratory tract are commonly associated with elevated production of nitric oxide (NO•) and increased indices of NO• -dependent oxidative stress. Although NO• is known to have anti-microbial, anti-inflammatory and anti-oxidant properties, various lines of evidence support the contribution of NO• to lung injury in several disease models. On the basis of biochemical evidence, it is often presumed that such NO• -dependent oxidations are due to the formation of the oxidant peroxynitrite, although alternative mechanisms involving the phagocyte-derived heme proteins myeloperoxidase and eosinophil peroxidase might be operative during conditions of inflammation. Because of the overwhelming literature on NO• generation and activities in the respiratory tract, it would be beyond the scope of this commentary to review this area comprehensively. Instead, it focuses on recent evidence and concepts of the presumed contribution of NO• to inflammatory diseases of the lung.   \n",
       "2                                                                                                                                                                                                               Surfactant protein-D (SP-D) participates in the innate response to inhaled microorganisms and organic antigens, and contributes to immune and inflammatory regulation within the lung. SP-D is synthesized and secreted by alveolar and bronchiolar epithelial cells, but is also expressed by epithelial cells lining various exocrine ducts and the mucosa of the gastrointestinal and genitourinary tracts. SP-D, a collagenous calcium-dependent lectin (or collectin), binds to surface glycoconjugates expressed by a wide variety of microorganisms, and to oligosaccharides associated with the surface of various complex organic antigens. SP-D also specifically interacts with glycoconjugates and other molecules expressed on the surface of macrophages, neutrophils, and lymphocytes. In addition, SP-D binds to specific surfactant-associated lipids and can influence the organization of lipid mixtures containing phosphatidylinositol in vitro. Consistent with these diverse in vitro activities is the observation that SP-D-deficient transgenic mice show abnormal accumulations of surfactant lipids, and respond abnormally to challenge with respiratory viruses and bacterial lipopolysaccharides. The phenotype of macrophages isolated from the lungs of SP-D-deficient mice is altered, and there is circumstantial evidence that abnormal oxidant metabolism and/or increased metalloproteinase expression contributes to the development of emphysema. The expression of SP-D is increased in response to many forms of lung injury, and deficient accumulation of appropriately oligomerized SP-D might contribute to the pathogenesis of a variety of human lung diseases.   \n",
       "3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Endothelin-1 (ET-1) is a 21 amino acid peptide with diverse biological activity that has been implicated in numerous diseases. ET-1 is a potent mitogen regulator of smooth muscle tone, and inflammatory mediator that may play a key role in diseases of the airways, pulmonary circulation, and inflammatory lung diseases, both acute and chronic. This review will focus on the biology of ET-1 and its role in lung disease.   \n",
       "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Respiratory syncytial virus (RSV) and pneumonia virus of mice (PVM) are viruses of the family Paramyxoviridae, subfamily pneumovirus, which cause clinically important respiratory infections in humans and rodents, respectively. The respiratory epithelial target cells respond to viral infection with specific alterations in gene expression, including production of chemoattractant cytokines, adhesion molecules, elements that are related to the apoptosis response, and others that remain incompletely understood. Here we review our current understanding of these mucosal responses and discuss several genomic approaches, including differential display reverse transcription-polymerase chain reaction (PCR) and gene array strategies, that will permit us to unravel the nature of these responses in a more complete and systematic manner.   \n",
       "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ...   \n",
       "171327                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            \n",
       "171328                                                                                                                                                                                                                                                                                                This study investigated the heterogeneity and plasticity of porcine alveolar macrophages (PAM) and pulmonary interstitial macrophages (IM) isolated from healthy pigs, including phenotype, function and gene expression. Dynamic changes of nitric oxide (NO) levels secreted by PAM and IM with stimulation of different doses of lipopolysaccharide (LPS) were investigated by Griess method, and the viability of the PAM and IM cells was investigated by MTT assay. Flow cytometry, fluorescence quantitative PCR and ELISA techniques were used to measure cell phenotype, gene expression and cytokine secretion, respectively. The PAM and IM cells in normal healthy pigs showed heterogeneity with 95.42±1.51% and 31.99±5.84% of CD163+ macrophage, respectively. The NO level in IM was significantly higher versus PAM after LPS treatment. Consistently, the ratio of Arg I/iNOS in IM was much lower than that in PAM, suggesting that the PAM belong to M2 macrophages and the IM belong to M1 macrophages. The PAM and IM cells in normal healthy pigs also showed plasticity. The Arg I/iNOS ratio and TIMP1/MMP12 ratio were significantly decreased in LPS- or LPS+IFNγ-treated PAM and IM, suggesting that cells were polarized towards M1 macrophages under LPS or LPS+IFNγ stimulation. On the contrary, IL-4 and IL-13 stimulation on PAM and IM lead to M2 polarization. A similar result was found in IL-1β gene expression and TNFα secretion. In conclusion, porcine macrophages have shown heterogeneity and plasticity on polarization under the stimulation of LPS, IFNγ, IL-4 and IL-13.   \n",
       "171329                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                Abstract We report on an anomalous X-ray reflectivity study to locate a labelled residue of a membrane protein with respect to the lipid bilayer. From such experiments, important constraints on the protein or peptide conformation can be derived. Specifically, our aim is to localize an iodine-labelled phenylalanine in the SARS E protein, incorporated in DMPC phospholipid bilayers, which are deposited in the form of thick multilamellar stacks on silicon surfaces. Here, we discuss the experimental aspects and the difficulties associated with the Fourier synthesis analysis that gives the electron density profile of the membranes.   \n",
       "171330                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   COVID‐19 pandemic raised the issue to guarantee the proper level of care to patients with acute cardiovascular diseases and concomitant suspected or confirmed COVID‐19 and, in the meantime safety and protection of healthcare providers. The aim of this position paper is to provide standards to healthcare facilities and healthcare providers on infection prevention and control measures during the management of suspected and confirmed cases of 2019‐nCoV infection accessing in cath‐lab. The document represents the view of the Italian Society of Interventional Cardiology (GISE), and it is based on recommendations from the main World and European Health Organizations (WHO, and ECDC) as well as from the Italian Society of Anesthesia, Analgesia, Resuscitation and Intensive Care (SIAARTI).   \n",
       "171331                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            \n",
       "\n",
       "                                                                                                                                                                                                                                                                       metadata  \n",
       "0                                                                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC35282/', 'pubmed_id': '11472636'}  \n",
       "1                                                                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59543/', 'pubmed_id': '11667967'}  \n",
       "2                                                                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59549/', 'pubmed_id': '11667972'}  \n",
       "3                                                                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59574/', 'pubmed_id': '11686871'}  \n",
       "4                                                                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC59580/', 'pubmed_id': '11686888'}  \n",
       "...                                                                                                                                                                                                                                                                         ...  \n",
       "171327                                                                                                                                          {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32427657/; https://doi.org/10.1213/ane.0000000000004982', 'pubmed_id': '32427657'}  \n",
       "171328                                                                                                                                                    {'url': 'https://doi.org/10.1242/bio.046342; https://www.ncbi.nlm.nih.gov/pubmed/31615770/', 'pubmed_id': '31615770'}  \n",
       "171329      {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32288217/; https://doi.org/10.1016/j.physb.2004.11.015; https://www.sciencedirect.com/science/article/pii/S0921452604011354; https://api.elsevier.com/content/article/pii/S0921452604011354', 'pubmed_id': '32288217'}  \n",
       "171330                                                                                                                                                     {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/32223063/; https://doi.org/10.1002/ccd.28888', 'pubmed_id': '32223063'}  \n",
       "171331  {'url': 'https://www.sciencedirect.com/science/article/pii/S1078588420303804?v=s5; https://www.ncbi.nlm.nih.gov/pubmed/32446539/; https://doi.org/10.1016/j.ejvs.2020.04.040; https://api.elsevier.com/content/article/pii/S1078588420303804', 'pubmed_id': '32446539'}  \n",
       "\n",
       "[171332 rows x 4 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_corpus_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "fc6200ef",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_queries =read_line_json(TREC_COVID_QUERIES_FILENAME[:-3])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "0a982f9c",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_queries_df = pd.DataFrame(trec_covid_queries)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "af72c9f4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>_id</th>\n",
       "      <th>text</th>\n",
       "      <th>metadata</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>what is the origin of COVID-19</td>\n",
       "      <td>{'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>how does the coronavirus respond to changes in the weather</td>\n",
       "      <td>{'query': 'coronavirus response to weather changes', 'narrative': 'seeking range of information about the SARS-CoV-2 virus viability in different weather/climate conditions as well as information related to transmission of the virus in different climate conditions'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>3</td>\n",
       "      <td>will SARS-CoV2 infected people develop immunity? Is cross protection possible?</td>\n",
       "      <td>{'query': 'coronavirus immunity', 'narrative': 'seeking studies of immunity developed due to infection with SARS-CoV2 or cross protection gained due to infection with other coronavirus types'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>4</td>\n",
       "      <td>what causes death from Covid-19?</td>\n",
       "      <td>{'query': 'how do people die from the coronavirus', 'narrative': 'Studies looking at mechanisms of death from Covid-19.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5</td>\n",
       "      <td>what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?</td>\n",
       "      <td>{'query': 'animal models of COVID-19', 'narrative': 'Papers that describe the results  of testing drugs that bind to spike proteins of the virus or any other drugs in any animal models. Papers about SARS-CoV-2 infection in cell culture assays are also relevant.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>6</td>\n",
       "      <td>what types of rapid testing for Covid-19 have been developed?</td>\n",
       "      <td>{'query': 'coronavirus test rapid testing', 'narrative': 'Looking for studies identifying ways to diagnose Covid-19 more rapidly.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>7</td>\n",
       "      <td>are there serological tests that detect antibodies to coronavirus?</td>\n",
       "      <td>{'query': 'serological tests for coronavirus', 'narrative': 'Looking for assays that measure immune response to COVID-19 that will help determine past infection and subsequent possible immunity.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>8</td>\n",
       "      <td>how has lack of testing availability led to underreporting of true incidence of Covid-19?</td>\n",
       "      <td>{'query': 'coronavirus under reporting', 'narrative': 'Looking for studies answering questions of impact of lack of complete testing for Covid-19 on incidence and prevalence of Covid-19.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>9</td>\n",
       "      <td>how has COVID-19 affected Canada</td>\n",
       "      <td>{'query': 'coronavirus in Canada', 'narrative': 'seeking data related to infections (confirm, suspected, and projected) and health outcomes (symptoms, hospitalization, intensive care, mortality)'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>10</td>\n",
       "      <td>has social distancing had an impact on slowing the spread of COVID-19?</td>\n",
       "      <td>{'query': 'coronavirus social distancing impact', 'narrative': 'seeking specific information on studies that have measured COVID-19's transmission in one or more social distancing (or non-social distancing) approaches'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>11</td>\n",
       "      <td>what are the guidelines for triaging patients infected with coronavirus?</td>\n",
       "      <td>{'query': 'coronavirus hospital rationing', 'narrative': 'Seeking information on any guidelines for prioritizing COVID-19 patients infected with coronavirus based on demographics, clinical signs, serology and other tests.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>12</td>\n",
       "      <td>what are best practices in hospitals and at home in maintaining quarantine?</td>\n",
       "      <td>{'query': 'coronavirus quarantine', 'narrative': 'Seeking information on best practices for activities and duration of quarantine for those exposed and/ infected to COVID-19 virus.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>13</td>\n",
       "      <td>what are the transmission routes of coronavirus?</td>\n",
       "      <td>{'query': 'how does coronavirus spread', 'narrative': 'Looking for information on all possible ways to contract COVID-19 from people, animals and objects'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>14</td>\n",
       "      <td>what evidence is there related to COVID-19 super spreaders</td>\n",
       "      <td>{'query': 'coronavirus super spreaders', 'narrative': 'seeking range of information related to the number and proportion of super spreaders, their patterns of behavior that lead to spread, and potential prevention strategies targeted specifically toward super spreaders'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>15</td>\n",
       "      <td>how long can the coronavirus live outside the body</td>\n",
       "      <td>{'query': 'coronavirus outside body', 'narrative': 'seeking range of information on the SARS-CoV-2's virus's survival in different environments (surfaces, liquids, etc.) outside the human body while still being viable for transmission to another human'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>16</td>\n",
       "      <td>how long does coronavirus remain stable  on surfaces?</td>\n",
       "      <td>{'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>17</td>\n",
       "      <td>are there any clinical trials available for the coronavirus</td>\n",
       "      <td>{'query': 'coronavirus clinical trials', 'narrative': 'seeking specific COVID-19 clinical trials ranging from trials in recruitment to completed trials with results'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>18</td>\n",
       "      <td>what are the best masks for preventing infection by Covid-19?</td>\n",
       "      <td>{'query': 'masks prevent coronavirus', 'narrative': 'What types of masks should or should not be used to prevent infection by Covid-19?'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>19</td>\n",
       "      <td>what type of hand sanitizer is needed to destroy Covid-19?</td>\n",
       "      <td>{'query': 'what alcohol sanitizer kills coronavirus', 'narrative': 'Studies assessing chemicals and their concentrations needed to destroy the Covid-19 virus.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>20</td>\n",
       "      <td>are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?</td>\n",
       "      <td>{'query': 'coronavirus and ACE inhibitors', 'narrative': 'Looking for information on interactions between  coronavirus and  angiotensin converting enzyme 2 (ACE2) receptors, risk for patients taking these medications, and recommendations for these patients.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>21</td>\n",
       "      <td>what are the mortality rates overall and in specific populations</td>\n",
       "      <td>{'query': 'coronavirus mortality', 'narrative': 'Seeking  information on COVID-19 fatality rates in different countries and in different population groups based on gender, blood types, or other factors'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>22</td>\n",
       "      <td>are cardiac complications likely in patients with COVID-19?</td>\n",
       "      <td>{'query': 'coronavirus heart impacts', 'narrative': 'Seeking information on the types, frequency and mechanisms of cardiac complications caused by coronavirus.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>23</td>\n",
       "      <td>what kinds of complications related to COVID-19 are associated with hypertension?</td>\n",
       "      <td>{'query': 'coronavirus hypertension', 'narrative': 'seeking specific outcomes that hypertensive  (any type) patients are more/less likely to face if infected with the virus'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>24</td>\n",
       "      <td>what kinds of complications related to COVID-19 are associated with diabetes</td>\n",
       "      <td>{'query': 'coronavirus diabetes', 'narrative': 'seeking specific outcomes that diabetic (any type) patients are more/less likely to face if infected with the virus'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>25</td>\n",
       "      <td>which biomarkers predict the severe clinical course of 2019-nCOV infection?</td>\n",
       "      <td>{'query': 'coronavirus biomarkers', 'narrative': 'Looking for information on biomarkers that predict disease outcomes in people infected with coronavirus, specifically those that predict severe and fatal outcomes.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>26</td>\n",
       "      <td>what are the initial symptoms of Covid-19?</td>\n",
       "      <td>{'query': 'coronavirus early symptoms', 'narrative': 'Studies of patients and the first clinical manifestations they develop upon active infection?'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>27</td>\n",
       "      <td>what is known about those infected with Covid-19 but are asymptomatic?</td>\n",
       "      <td>{'query': 'coronavirus asymptomatic', 'narrative': 'Studies of people who are known to be infected with Covid-19 but show no symptoms?'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>28</td>\n",
       "      <td>what evidence is there for the value of hydroxychloroquine in treating Covid-19?</td>\n",
       "      <td>{'query': 'coronavirus hydroxychloroquine', 'narrative': 'Basic science or clinical studies assessing the benefit and harms of treating Covid-19 with hydroxychloroquine.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>29</td>\n",
       "      <td>which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?</td>\n",
       "      <td>{'query': 'coronavirus drug repurposing', 'narrative': 'Seeking information about protein-protein interactions for any of the SARS-CoV-2  structural proteins that represent a promising therapeutic target,  and the drug  molecules that may inhibit the virus and the host cell receptors at entry step.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>30</td>\n",
       "      <td>is remdesivir an effective treatment for COVID-19</td>\n",
       "      <td>{'query': 'coronavirus remdesivir', 'narrative': 'seeking specific information on clinical outcomes in COVID-19 patients treated with remdesivir'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>31</td>\n",
       "      <td>How does the coronavirus differ from seasonal flu?</td>\n",
       "      <td>{'query': 'difference between coronavirus and flu', 'narrative': 'Includes studies ranging from those focusing on genomic differences to global public health impacts, but must draw direct comparisons between COVID-19 and seasonal influenza.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>32</td>\n",
       "      <td>Does SARS-CoV-2 have any subtypes, and if so what are they?</td>\n",
       "      <td>{'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>33</td>\n",
       "      <td>What vaccine candidates are being tested for Covid-19?</td>\n",
       "      <td>{'query': 'coronavirus vaccine candidates', 'narrative': 'Seeking studies that discuss possible, but specific, COVID-19 vaccines. Includes articles from those describing the mechanisms of action of specific proposed vaccines to actual clinical trials, but excluding articles that do not name a specific vaccine candidate.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>34</td>\n",
       "      <td>What are the longer-term complications of those who recover from COVID-19?</td>\n",
       "      <td>{'query': 'coronavirus recovery', 'narrative': 'Seeking information on the health outcomes for those that recover from the virus. Excludes studies only focusing on adverse effects related to a particular COVID-19 drug.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>35</td>\n",
       "      <td>What new public datasets are available related to COVID-19?</td>\n",
       "      <td>{'query': 'coronavirus public datasets', 'narrative': 'Seeking articles that specifically release new data related to SARS-CoV-2 or COVID-19, including genomic data, patient data, public health data, etc. Articles that reference previously existing datasets are not relevant.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>36</td>\n",
       "      <td>What is the protein structure of the SARS-CoV-2 spike?</td>\n",
       "      <td>{'query': 'SARS-CoV-2 spike structure', 'narrative': 'Looking for studies of the structure of the spike protein on the virus using any methods, such as cryo-EM or crystallography'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>37</td>\n",
       "      <td>What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?</td>\n",
       "      <td>{'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>38</td>\n",
       "      <td>What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?</td>\n",
       "      <td>{'query': 'COVID inflammatory response', 'narrative': 'Looking for a range of studies which describes the inflammatory response cells and pathogenesis during the Coronavirus Disease 2019 (COVID-19) outbreak, including the mechanism of anti-inflammatory drugs, corticosteroids, and vitamin supplements'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>39</td>\n",
       "      <td>What is the mechanism of cytokine storm syndrome on the COVID-19?</td>\n",
       "      <td>{'query': 'COVID-19 cytokine storm', 'narrative': 'Looking for studies that describes mechanism of development of cytokine storm syndrome among COVID-19 cases and the range of drugs used for the therapy of cytokine storm'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>40</td>\n",
       "      <td>What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?</td>\n",
       "      <td>{'query': 'coronavirus mutations', 'narrative': 'Looking for studies that describes the emergence of genomic diversity of the coronavirus due to recurrent mutations which explore the potential genomic site of the mutation, mechanisms and its potential or observed clinical implications in the pathogenicity of the virus'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>41</td>\n",
       "      <td>What are the impacts of COVID-19 among African-Americans that differ from the rest of the U.S. population?</td>\n",
       "      <td>{'query': 'COVID-19 in African-Americans', 'narrative': 'Looking for studies that analyze burden of illness and death among the African-American/black racial/ethnic group. This includes potential reasons for transmission, morbidity, and mortality. This may include discussion of other minority groups, but all studies should contain specific information on the health disparities faced by African-Americans in this pandemic.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>42</td>\n",
       "      <td>Does Vitamin D impact COVID-19 prevention and treatment?</td>\n",
       "      <td>{'query': 'Vitamin D and COVID-19', 'narrative': 'This includes studies describing possible role of Vitamin D in prevention of COVID-19, suppression of cytokine storm, clinical outcomes, and associations between Vitamin D status and COVID-19 mortality.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>43</td>\n",
       "      <td>How has the COVID-19 pandemic impacted violence in society, including violent crimes?</td>\n",
       "      <td>{'query': 'violence during pandemic', 'narrative': 'Looking for analyses and data on how the pandemic is impacting rates of violence, including domestic/family violence related to quarantine.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>44</td>\n",
       "      <td>How much impact do masks have on preventing the spread of the COVID-19?</td>\n",
       "      <td>{'query': 'impact of masks on coronavirus transmission', 'narrative': 'Looking for studies of how masks slow SARS-CoV-2 transmission, including impact on R0. Studies can include both lab and population studies.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>45</td>\n",
       "      <td>How has the COVID-19 pandemic impacted mental health?</td>\n",
       "      <td>{'query': 'coronavirus mental health impact', 'narrative': 'Includes increasing/decreasing rates of depression, anxiety, panic disorder, and other psychiatric and mental health conditions.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>46</td>\n",
       "      <td>what evidence is there for dexamethasone as a treatment for COVID-19?</td>\n",
       "      <td>{'query': 'dexamethasone coronavirus', 'narrative': 'Looking for studies on the impact of dexamethasone treatment in COVID-19 patients, including health benefits as well as adverse effects. This also includes specific populations that are benefitted/harmed by dexamethasone.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>47</td>\n",
       "      <td>what are the health outcomes for children who contract COVID-19?</td>\n",
       "      <td>{'query': 'COVID-19 outcomes in children', 'narrative': 'Looking for studies on health outcomes in children related to COVID-19. This includes studies attempting to explain the underlying biological mechanisms for why children differ from adults in response to infection.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>48</td>\n",
       "      <td>what are the benefits and risks of re-opening schools in the midst of the COVID-19 pandemic?</td>\n",
       "      <td>{'query': 'school reopening coronavirus', 'narrative': 'With the possibility of schools re-opening while the COVID-19 pandemic is still ongoing, this topic is looking for evidence or projections on what the potential implications of this are in terms of COVID-19 cases, hospitalizations, or deaths, as well as other benefits or harms to re-opening schools. This includes both the impact on students, teachers, families, and the wider community.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>49</td>\n",
       "      <td>do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?</td>\n",
       "      <td>{'query': 'post-infection COVID-19 immunity', 'narrative': 'There is concern about re-infection for COVID-19, so this topic is looking for studies suggesting post-infection immunity, including post-infection antibody levels (over time) and evidence for individuals who have been infected more than once.'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   _id  \\\n",
       "0    1   \n",
       "1    2   \n",
       "2    3   \n",
       "3    4   \n",
       "4    5   \n",
       "5    6   \n",
       "6    7   \n",
       "7    8   \n",
       "8    9   \n",
       "9   10   \n",
       "10  11   \n",
       "11  12   \n",
       "12  13   \n",
       "13  14   \n",
       "14  15   \n",
       "15  16   \n",
       "16  17   \n",
       "17  18   \n",
       "18  19   \n",
       "19  20   \n",
       "20  21   \n",
       "21  22   \n",
       "22  23   \n",
       "23  24   \n",
       "24  25   \n",
       "25  26   \n",
       "26  27   \n",
       "27  28   \n",
       "28  29   \n",
       "29  30   \n",
       "30  31   \n",
       "31  32   \n",
       "32  33   \n",
       "33  34   \n",
       "34  35   \n",
       "35  36   \n",
       "36  37   \n",
       "37  38   \n",
       "38  39   \n",
       "39  40   \n",
       "40  41   \n",
       "41  42   \n",
       "42  43   \n",
       "43  44   \n",
       "44  45   \n",
       "45  46   \n",
       "46  47   \n",
       "47  48   \n",
       "48  49   \n",
       "49  50   \n",
       "\n",
       "                                                                                                                                                                     text  \\\n",
       "0                                                                                                                                          what is the origin of COVID-19   \n",
       "1                                                                                                              how does the coronavirus respond to changes in the weather   \n",
       "2                                                                                          will SARS-CoV2 infected people develop immunity? Is cross protection possible?   \n",
       "3                                                                                                                                        what causes death from Covid-19?   \n",
       "4                                                                                           what drugs have been active against SARS-CoV or SARS-CoV-2 in animal studies?   \n",
       "5                                                                                                           what types of rapid testing for Covid-19 have been developed?   \n",
       "6                                                                                                      are there serological tests that detect antibodies to coronavirus?   \n",
       "7                                                                               how has lack of testing availability led to underreporting of true incidence of Covid-19?   \n",
       "8                                                                                                                                        how has COVID-19 affected Canada   \n",
       "9                                                                                                  has social distancing had an impact on slowing the spread of COVID-19?   \n",
       "10                                                                                               what are the guidelines for triaging patients infected with coronavirus?   \n",
       "11                                                                                            what are best practices in hospitals and at home in maintaining quarantine?   \n",
       "12                                                                                                                       what are the transmission routes of coronavirus?   \n",
       "13                                                                                                             what evidence is there related to COVID-19 super spreaders   \n",
       "14                                                                                                                     how long can the coronavirus live outside the body   \n",
       "15                                                                                                                  how long does coronavirus remain stable  on surfaces?   \n",
       "16                                                                                                            are there any clinical trials available for the coronavirus   \n",
       "17                                                                                                          what are the best masks for preventing infection by Covid-19?   \n",
       "18                                                                                                             what type of hand sanitizer is needed to destroy Covid-19?   \n",
       "19                                                                     are patients taking Angiotensin-converting enzyme inhibitors (ACE) at increased risk for COVID-19?   \n",
       "20                                                                                                       what are the mortality rates overall and in specific populations   \n",
       "21                                                                                                            are cardiac complications likely in patients with COVID-19?   \n",
       "22                                                                                      what kinds of complications related to COVID-19 are associated with hypertension?   \n",
       "23                                                                                           what kinds of complications related to COVID-19 are associated with diabetes   \n",
       "24                                                                                            which biomarkers predict the severe clinical course of 2019-nCOV infection?   \n",
       "25                                                                                                                             what are the initial symptoms of Covid-19?   \n",
       "26                                                                                                 what is known about those infected with Covid-19 but are asymptomatic?   \n",
       "27                                                                                       what evidence is there for the value of hydroxychloroquine in treating Covid-19?   \n",
       "28  which SARS-CoV-2 proteins-human proteins interactions indicate potential for drug targets. Are there approved drugs that can be repurposed based on this information?   \n",
       "29                                                                                                                      is remdesivir an effective treatment for COVID-19   \n",
       "30                                                                                                                     How does the coronavirus differ from seasonal flu?   \n",
       "31                                                                                                            Does SARS-CoV-2 have any subtypes, and if so what are they?   \n",
       "32                                                                                                                 What vaccine candidates are being tested for Covid-19?   \n",
       "33                                                                                             What are the longer-term complications of those who recover from COVID-19?   \n",
       "34                                                                                                            What new public datasets are available related to COVID-19?   \n",
       "35                                                                                                                 What is the protein structure of the SARS-CoV-2 spike?   \n",
       "36                                                                                             What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?   \n",
       "37                                                                                     What is the mechanism of inflammatory response and pathogenesis of COVID-19 cases?   \n",
       "38                                                                                                      What is the mechanism of cytokine storm syndrome on the COVID-19?   \n",
       "39                                                                         What are the observed mutations in the SARS-CoV-2 genome and how often do the mutations occur?   \n",
       "40                                                             What are the impacts of COVID-19 among African-Americans that differ from the rest of the U.S. population?   \n",
       "41                                                                                                               Does Vitamin D impact COVID-19 prevention and treatment?   \n",
       "42                                                                                  How has the COVID-19 pandemic impacted violence in society, including violent crimes?   \n",
       "43                                                                                                How much impact do masks have on preventing the spread of the COVID-19?   \n",
       "44                                                                                                                  How has the COVID-19 pandemic impacted mental health?   \n",
       "45                                                                                                  what evidence is there for dexamethasone as a treatment for COVID-19?   \n",
       "46                                                                                                       what are the health outcomes for children who contract COVID-19?   \n",
       "47                                                                           what are the benefits and risks of re-opening schools in the midst of the COVID-19 pandemic?   \n",
       "48             do individuals who recover from COVID-19 show sufficient immune response, including antibody levels and T-cell mediated immunity, to prevent re-infection?   \n",
       "49                                                                                                          what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                          metadata  \n",
       "0                                                                                                                                                                                                                                                                     {'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}  \n",
       "1                                                                                                                                                                                       {'query': 'coronavirus response to weather changes', 'narrative': 'seeking range of information about the SARS-CoV-2 virus viability in different weather/climate conditions as well as information related to transmission of the virus in different climate conditions'}  \n",
       "2                                                                                                                                                                                                                                                                 {'query': 'coronavirus immunity', 'narrative': 'seeking studies of immunity developed due to infection with SARS-CoV2 or cross protection gained due to infection with other coronavirus types'}  \n",
       "3                                                                                                                                                                                                                                                                                                                                        {'query': 'how do people die from the coronavirus', 'narrative': 'Studies looking at mechanisms of death from Covid-19.'}  \n",
       "4                                                                                                                                                                                          {'query': 'animal models of COVID-19', 'narrative': 'Papers that describe the results  of testing drugs that bind to spike proteins of the virus or any other drugs in any animal models. Papers about SARS-CoV-2 infection in cell culture assays are also relevant.'}  \n",
       "5                                                                                                                                                                                                                                                                                                                              {'query': 'coronavirus test rapid testing', 'narrative': 'Looking for studies identifying ways to diagnose Covid-19 more rapidly.'}  \n",
       "6                                                                                                                                                                                                                                                             {'query': 'serological tests for coronavirus', 'narrative': 'Looking for assays that measure immune response to COVID-19 that will help determine past infection and subsequent possible immunity.'}  \n",
       "7                                                                                                                                                                                                                                                                     {'query': 'coronavirus under reporting', 'narrative': 'Looking for studies answering questions of impact of lack of complete testing for Covid-19 on incidence and prevalence of Covid-19.'}  \n",
       "8                                                                                                                                                                                                                                                             {'query': 'coronavirus in Canada', 'narrative': 'seeking data related to infections (confirm, suspected, and projected) and health outcomes (symptoms, hospitalization, intensive care, mortality)'}  \n",
       "9                                                                                                                                                                                                                                      {'query': 'coronavirus social distancing impact', 'narrative': 'seeking specific information on studies that have measured COVID-19's transmission in one or more social distancing (or non-social distancing) approaches'}  \n",
       "10                                                                                                                                                                                                                                 {'query': 'coronavirus hospital rationing', 'narrative': 'Seeking information on any guidelines for prioritizing COVID-19 patients infected with coronavirus based on demographics, clinical signs, serology and other tests.'}  \n",
       "11                                                                                                                                                                                                                                                                          {'query': 'coronavirus quarantine', 'narrative': 'Seeking information on best practices for activities and duration of quarantine for those exposed and/ infected to COVID-19 virus.'}  \n",
       "12                                                                                                                                                                                                                                                                                                     {'query': 'how does coronavirus spread', 'narrative': 'Looking for information on all possible ways to contract COVID-19 from people, animals and objects'}  \n",
       "13                                                                                                                                                                                 {'query': 'coronavirus super spreaders', 'narrative': 'seeking range of information related to the number and proportion of super spreaders, their patterns of behavior that lead to spread, and potential prevention strategies targeted specifically toward super spreaders'}  \n",
       "14                                                                                                                                                                                                   {'query': 'coronavirus outside body', 'narrative': 'seeking range of information on the SARS-CoV-2's virus's survival in different environments (surfaces, liquids, etc.) outside the human body while still being viable for transmission to another human'}  \n",
       "15                                                                                                                                                                                       {'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}  \n",
       "16                                                                                                                                                                                                                                                                                          {'query': 'coronavirus clinical trials', 'narrative': 'seeking specific COVID-19 clinical trials ranging from trials in recruitment to completed trials with results'}  \n",
       "17                                                                                                                                                                                                                                                                                                                       {'query': 'masks prevent coronavirus', 'narrative': 'What types of masks should or should not be used to prevent infection by Covid-19?'}  \n",
       "18                                                                                                                                                                                                                                                                                                {'query': 'what alcohol sanitizer kills coronavirus', 'narrative': 'Studies assessing chemicals and their concentrations needed to destroy the Covid-19 virus.'}  \n",
       "19                                                                                                                                                                                             {'query': 'coronavirus and ACE inhibitors', 'narrative': 'Looking for information on interactions between  coronavirus and  angiotensin converting enzyme 2 (ACE2) receptors, risk for patients taking these medications, and recommendations for these patients.'}  \n",
       "20                                                                                                                                                                                                                                                     {'query': 'coronavirus mortality', 'narrative': 'Seeking  information on COVID-19 fatality rates in different countries and in different population groups based on gender, blood types, or other factors'}  \n",
       "21                                                                                                                                                                                                                                                                                               {'query': 'coronavirus heart impacts', 'narrative': 'Seeking information on the types, frequency and mechanisms of cardiac complications caused by coronavirus.'}  \n",
       "22                                                                                                                                                                                                                                                                                  {'query': 'coronavirus hypertension', 'narrative': 'seeking specific outcomes that hypertensive  (any type) patients are more/less likely to face if infected with the virus'}  \n",
       "23                                                                                                                                                                                                                                                                                           {'query': 'coronavirus diabetes', 'narrative': 'seeking specific outcomes that diabetic (any type) patients are more/less likely to face if infected with the virus'}  \n",
       "24                                                                                                                                                                                                                                         {'query': 'coronavirus biomarkers', 'narrative': 'Looking for information on biomarkers that predict disease outcomes in people infected with coronavirus, specifically those that predict severe and fatal outcomes.'}  \n",
       "25                                                                                                                                                                                                                                                                                                           {'query': 'coronavirus early symptoms', 'narrative': 'Studies of patients and the first clinical manifestations they develop upon active infection?'}  \n",
       "26                                                                                                                                                                                                                                                                                                                        {'query': 'coronavirus asymptomatic', 'narrative': 'Studies of people who are known to be infected with Covid-19 but show no symptoms?'}  \n",
       "27                                                                                                                                                                                                                                                                                     {'query': 'coronavirus hydroxychloroquine', 'narrative': 'Basic science or clinical studies assessing the benefit and harms of treating Covid-19 with hydroxychloroquine.'}  \n",
       "28                                                                                                                                                   {'query': 'coronavirus drug repurposing', 'narrative': 'Seeking information about protein-protein interactions for any of the SARS-CoV-2  structural proteins that represent a promising therapeutic target,  and the drug  molecules that may inhibit the virus and the host cell receptors at entry step.'}  \n",
       "29                                                                                                                                                                                                                                                                                                              {'query': 'coronavirus remdesivir', 'narrative': 'seeking specific information on clinical outcomes in COVID-19 patients treated with remdesivir'}  \n",
       "30                                                                                                                                                                                                              {'query': 'difference between coronavirus and flu', 'narrative': 'Includes studies ranging from those focusing on genomic differences to global public health impacts, but must draw direct comparisons between COVID-19 and seasonal influenza.'}  \n",
       "31                                                                                                                                                                                                                                                                             {'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}  \n",
       "32                                                                                                                             {'query': 'coronavirus vaccine candidates', 'narrative': 'Seeking studies that discuss possible, but specific, COVID-19 vaccines. Includes articles from those describing the mechanisms of action of specific proposed vaccines to actual clinical trials, but excluding articles that do not name a specific vaccine candidate.'}  \n",
       "33                                                                                                                                                                                                                                    {'query': 'coronavirus recovery', 'narrative': 'Seeking information on the health outcomes for those that recover from the virus. Excludes studies only focusing on adverse effects related to a particular COVID-19 drug.'}  \n",
       "34                                                                                                                                                                           {'query': 'coronavirus public datasets', 'narrative': 'Seeking articles that specifically release new data related to SARS-CoV-2 or COVID-19, including genomic data, patient data, public health data, etc. Articles that reference previously existing datasets are not relevant.'}  \n",
       "35                                                                                                                                                                                                                                                                            {'query': 'SARS-CoV-2 spike structure', 'narrative': 'Looking for studies of the structure of the spike protein on the virus using any methods, such as cryo-EM or crystallography'}  \n",
       "36                                                                                                                                                                                                                                                                                {'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}  \n",
       "37                                                                                                                                                  {'query': 'COVID inflammatory response', 'narrative': 'Looking for a range of studies which describes the inflammatory response cells and pathogenesis during the Coronavirus Disease 2019 (COVID-19) outbreak, including the mechanism of anti-inflammatory drugs, corticosteroids, and vitamin supplements'}  \n",
       "38                                                                                                                                                                                                                                  {'query': 'COVID-19 cytokine storm', 'narrative': 'Looking for studies that describes mechanism of development of cytokine storm syndrome among COVID-19 cases and the range of drugs used for the therapy of cytokine storm'}  \n",
       "39                                                                                                                               {'query': 'coronavirus mutations', 'narrative': 'Looking for studies that describes the emergence of genomic diversity of the coronavirus due to recurrent mutations which explore the potential genomic site of the mutation, mechanisms and its potential or observed clinical implications in the pathogenicity of the virus'}  \n",
       "40                      {'query': 'COVID-19 in African-Americans', 'narrative': 'Looking for studies that analyze burden of illness and death among the African-American/black racial/ethnic group. This includes potential reasons for transmission, morbidity, and mortality. This may include discussion of other minority groups, but all studies should contain specific information on the health disparities faced by African-Americans in this pandemic.'}  \n",
       "41                                                                                                                                                                                                  {'query': 'Vitamin D and COVID-19', 'narrative': 'This includes studies describing possible role of Vitamin D in prevention of COVID-19, suppression of cytokine storm, clinical outcomes, and associations between Vitamin D status and COVID-19 mortality.'}  \n",
       "42                                                                                                                                                                                                                                                               {'query': 'violence during pandemic', 'narrative': 'Looking for analyses and data on how the pandemic is impacting rates of violence, including domestic/family violence related to quarantine.'}  \n",
       "43                                                                                                                                                                                                                                            {'query': 'impact of masks on coronavirus transmission', 'narrative': 'Looking for studies of how masks slow SARS-CoV-2 transmission, including impact on R0. Studies can include both lab and population studies.'}  \n",
       "44                                                                                                                                                                                                                                                                  {'query': 'coronavirus mental health impact', 'narrative': 'Includes increasing/decreasing rates of depression, anxiety, panic disorder, and other psychiatric and mental health conditions.'}  \n",
       "45                                                                                                                                                                            {'query': 'dexamethasone coronavirus', 'narrative': 'Looking for studies on the impact of dexamethasone treatment in COVID-19 patients, including health benefits as well as adverse effects. This also includes specific populations that are benefitted/harmed by dexamethasone.'}  \n",
       "46                                                                                                                                                                               {'query': 'COVID-19 outcomes in children', 'narrative': 'Looking for studies on health outcomes in children related to COVID-19. This includes studies attempting to explain the underlying biological mechanisms for why children differ from adults in response to infection.'}  \n",
       "47  {'query': 'school reopening coronavirus', 'narrative': 'With the possibility of schools re-opening while the COVID-19 pandemic is still ongoing, this topic is looking for evidence or projections on what the potential implications of this are in terms of COVID-19 cases, hospitalizations, or deaths, as well as other benefits or harms to re-opening schools. This includes both the impact on students, teachers, families, and the wider community.'}  \n",
       "48                                                                                                                                               {'query': 'post-infection COVID-19 immunity', 'narrative': 'There is concern about re-infection for COVID-19, so this topic is looking for studies suggesting post-infection immunity, including post-infection antibody levels (over time) and evidence for individuals who have been infected more than once.'}  \n",
       "49                                                                                                                                                                                                                          {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}  "
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_queries_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "cad8372b",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_qrels_df = pd.read_csv(TREC_COVID_QRELS_FILENAME, sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "f1bc5def",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>g7dhmyyo</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>0194oljo</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>021q9884</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66331</th>\n",
       "      <td>50</td>\n",
       "      <td>zvop8bxh</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66332</th>\n",
       "      <td>50</td>\n",
       "      <td>zwf26o63</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66333</th>\n",
       "      <td>50</td>\n",
       "      <td>zwsvlnwe</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66334</th>\n",
       "      <td>50</td>\n",
       "      <td>zxr01yln</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66335</th>\n",
       "      <td>50</td>\n",
       "      <td>zz8wvos9</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>66336 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       query-id corpus-id  score\n",
       "0             1  005b2j4b      2\n",
       "1             1  00fmeepz      1\n",
       "2             1  g7dhmyyo      2\n",
       "3             1  0194oljo      1\n",
       "4             1  021q9884      1\n",
       "...         ...       ...    ...\n",
       "66331        50  zvop8bxh      2\n",
       "66332        50  zwf26o63      1\n",
       "66333        50  zwsvlnwe      0\n",
       "66334        50  zxr01yln      1\n",
       "66335        50  zz8wvos9      1\n",
       "\n",
       "[66336 rows x 3 columns]"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_qrels_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "a34fa59d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1</td>\n",
       "      <td>0chuwvg6</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>957</th>\n",
       "      <td>1</td>\n",
       "      <td>0lywkro9</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>1</td>\n",
       "      <td>0nh58odf</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>1</td>\n",
       "      <td>0t2a5500</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66326</th>\n",
       "      <td>50</td>\n",
       "      <td>zstmdt4n</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66328</th>\n",
       "      <td>50</td>\n",
       "      <td>zth8ffy3</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66329</th>\n",
       "      <td>50</td>\n",
       "      <td>zv0ysi8m</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66333</th>\n",
       "      <td>50</td>\n",
       "      <td>zwsvlnwe</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66202</th>\n",
       "      <td>50</td>\n",
       "      <td>svo94kuo</td>\n",
       "      <td>-1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>66336 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       query-id corpus-id  score\n",
       "0             1  005b2j4b      2\n",
       "18            1  0chuwvg6      2\n",
       "957           1  0lywkro9      2\n",
       "29            1  0nh58odf      2\n",
       "35            1  0t2a5500      2\n",
       "...         ...       ...    ...\n",
       "66326        50  zstmdt4n      0\n",
       "66328        50  zth8ffy3      0\n",
       "66329        50  zv0ysi8m      0\n",
       "66333        50  zwsvlnwe      0\n",
       "66202        50  svo94kuo     -1\n",
       "\n",
       "[66336 rows x 3 columns]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_qrels_df.sort_values(['query-id', 'score', 'corpus-id'], ascending=[True, False, True])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "0497b382",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(35480,)"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_qrels_df['corpus-id'].unique().shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "554539d7",
   "metadata": {},
   "source": [
    "### Fix DF column types"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "a850881b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "query-id      int64\n",
       "corpus-id    object\n",
       "score         int64\n",
       "dtype: object"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_qrels_df.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "da36ad0b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "_id          int64\n",
       "text        object\n",
       "metadata    object\n",
       "dtype: object"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_queries_df.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "326a7a51",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "_id         object\n",
       "title       object\n",
       "text        object\n",
       "metadata    object\n",
       "dtype: object"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "trec_covid_corpus_df.dtypes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "e30e4e1b",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_queries_df['_id'] = trec_covid_queries_df['_id'].astype(int)\n",
    "trec_covid_queries_df['text'] = trec_covid_queries_df['text'].astype(str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "72864698",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_corpus_df['_id'] = trec_covid_corpus_df['_id'].astype(str)\n",
    "trec_covid_corpus_df['text'] = trec_covid_corpus_df['text'].astype(str)\n",
    "trec_covid_corpus_df['title'] = trec_covid_corpus_df['title'].astype(str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "b0bb10d1",
   "metadata": {},
   "outputs": [],
   "source": [
    "trec_covid_corpus_df.to_csv(\"trec_covid_corpus.tsv\", sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "601dbf53",
   "metadata": {},
   "source": [
    "### Merge the data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "b09cff59",
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = trec_covid_qrels_df.merge(trec_covid_queries_df, left_on=\"query-id\", right_on=\"_id\", how=\"inner\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "8184f4f3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(66336, 6)"
      ]
     },
     "execution_count": 52,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "d7aefdcf",
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = merged_df.merge(trec_covid_corpus_df, left_on=\"corpus-id\", right_on=\"_id\", how=\"inner\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "7a35dd7b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(66336, 10)"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "e9320f2d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "      <th>_id_x</th>\n",
       "      <th>text_x</th>\n",
       "      <th>metadata_x</th>\n",
       "      <th>_id_y</th>\n",
       "      <th>title</th>\n",
       "      <th>text_y</th>\n",
       "      <th>metadata_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>what is the origin of COVID-19</td>\n",
       "      <td>{'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>16</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>16</td>\n",
       "      <td>how long does coronavirus remain stable  on surfaces?</td>\n",
       "      <td>{'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>32</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>32</td>\n",
       "      <td>Does SARS-CoV-2 have any subtypes, and if so what are they?</td>\n",
       "      <td>{'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>37</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>37</td>\n",
       "      <td>What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?</td>\n",
       "      <td>{'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>what is the origin of COVID-19</td>\n",
       "      <td>{'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>Comprehensive overview of COVID-19 based on current evidence</td>\n",
       "      <td>In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.</td>\n",
       "      <td>{'url': '', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66331</th>\n",
       "      <td>50</td>\n",
       "      <td>zn10rnrm</td>\n",
       "      <td>1</td>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>zn10rnrm</td>\n",
       "      <td>Characterization of RNA in Saliva</td>\n",
       "      <td>Background: We have previously shown that human mRNAs are present in saliva and can be used as biomarkers of oral cancer. In this study, we analyzed the integrity, sources, and stability of salivary RNA. Methods: We measured the integrity of salivary RNA with reverse transcription followed by PCR (RT-PCR) or RT-quantitative PCR (RT-qPCR). To study RNA entry sites into the oral cavity, we used RT-PCR analysis of salivary RNA from the 3 major salivary glands, gingival crevice fluid, and desquamated oral epithelial cells. We measured stability of the salivary β-actin mRNA by RT-qPCR of salivary RNA incubated at room temperature for different periods of time. We measured RNA association with other macromolecules by filtering saliva through pores of different sizes before performing RT-qPCR. To assess RNA–macromolecule interaction, we incubated saliva with Triton X-100 for different periods of time before performing RT-qPCR. Results: In most cases, we detected partial- to full-length salivary mRNAs and smaller amounts of middle and 3′ gene amplicons compared with the 5′. RNA was present in all oral fluids examined. Endogenous salivary β-actin mRNA degraded more slowly than exogenous β-actin mRNA, with half-lives of 12.2 and 0.4 min, respectively (P &lt;0.001). Salivary RNA could not pass through 0.22 or 0.45 μm pores. Incubation of saliva with Triton X-100 accelerated degradation of salivary RNA. Conclusions: Saliva harbors both full-length and partially degraded forms of mRNA. RNA enters the oral cavity from different sources, and association with macromolecules may protect salivary RNA from degradation.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108156/', 'pubmed_id': '16601067'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66332</th>\n",
       "      <td>50</td>\n",
       "      <td>zstmdt4n</td>\n",
       "      <td>0</td>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>zstmdt4n</td>\n",
       "      <td>Coordinate induction of IFN-α and -γ by SARS-CoV also in the absence of virus replication</td>\n",
       "      <td>Abstract Background: Severe acute respiratory syndrome (SARS) is an emerging infection caused by a novel coronavirus known as SARS-CoV, characterized by an over-exuberant immune response with lung lymphomononuclear cells infiltration and proliferation that may account for tissue damage more than the direct effect of viral replication. This study is aimed at investigating the capability of SARS-CoV to activate IFN-α and -γ expression in lymphomonocytes (PBMC) from healthy donors, evaluating whether viral replication is necessary for this activation. Results: SARS-CoV virus is able to induce both IFN-α and -γ mRNA accumulation and protein release in a dose-dependent manner, MOI 10 being the most effective. The time course curve indicated that IFN-α mRNA induction peaked at 24 h.p.i,. whereas IFN-γ mRNA was still increasing at 48 h.p.i. Released IFN (both types) reached a plateau after 24–48 h.p.i. and remained rather stable over a 5-day period. A transient peak of negative strand viral RNA was detected after 1–2 days of infection, but neither infectious virus progeny yield nor newly produced viral genomic RNA could be evidenced in infected cultures, even after prolonged observation time (up to 13 days). Cocultivation of PBMC with fixed SARS-CoV-infected Vero cells was even more efficient than exposure to live virus in eliciting IFN-α and -γ induction. A combination of IFN-α and -γ strongly inhibited SARS-CoV replication in Vero cells, while the single cytokines were much less effective. Conclusions: This study provides evidence that SARS-CoV is able to induce in normal PBMC a coordinate induction of IFN-α and -γ gene expression. Virus replication is not necessary for IFN induction since efficient IFN expression could be obtained also by the cocultivation of normal PBMC with fixed SARS-CoV-infected cells. Concomitant activation of IFN-α and -γ gene expression by SARS-CoV in vivo may be relevant for the pathogenesis of the disease, both for the possible involvement in immunomediated damage of the tissues and for the strong inhibition of SARS-CoV replication as a result of combined cytokine action.</td>\n",
       "      <td>{'url': 'https://api.elsevier.com/content/article/pii/S0042682205004241; https://www.sciencedirect.com/science/article/pii/S0042682205004241; https://www.ncbi.nlm.nih.gov/pubmed/16095648/', 'pubmed_id': '16095648'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66333</th>\n",
       "      <td>50</td>\n",
       "      <td>zth8ffy3</td>\n",
       "      <td>0</td>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>zth8ffy3</td>\n",
       "      <td>Vasculopathy and Coagulopathy Associated with SARS-CoV-2 Infection</td>\n",
       "      <td>The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has resulted in &gt; 500,000 deaths worldwide, including &gt; 125,000 deaths in the U.S. since its emergence in late December 2019 and June 2020. Neither curative anti-viral drugs nor a protective vaccine is currently available for the treatment and prevention of COVID-19. Recently, new clinical syndromes associated with coagulopathy and vasculopathy have emerged as a cause of sudden death and other serious clinical manifestations in younger patients infected with SARS-CoV-2 infection. Angiotensin converting enzyme 2 (ACE2), the receptor for SARS-CoV-2 and other coronaviruses, is a transmembrane protein expressed by lung alveolar epithelial cells, enterocytes, and vascular endothelial cells, whose physiologic role is to induce the maturation of angiotensin I to generate angiotensin 1-7, a peptide hormone that controls vasoconstriction and blood pressure. In this review, we provide the general context of the molecular and cellular mechanisms of SARS-CoV-2 infection with a focus on endothelial cells, describe the vasculopathy and coagulopathy syndromes in patients with SARS-CoV-2, and outline current understanding of the underlying mechanistic aspects.</td>\n",
       "      <td>{'url': '', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66334</th>\n",
       "      <td>50</td>\n",
       "      <td>zv4nbz9p</td>\n",
       "      <td>2</td>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>zv4nbz9p</td>\n",
       "      <td>Emerging Technologies for Use in the Study, Diagnosis, and Treatment of Patients with COVID-19</td>\n",
       "      <td>INTRODUCTION: The COVID-19 pandemic has caused an unprecedented health and economic worldwide crisis. Innovative solutions are imperative given limited resources and immediate need for medical supplies, healthcare support and treatments. AIM: The purpose of this review is to summarize emerging technologies being implemented in the study, diagnosis, and treatment of COVID-19. RESULTS: Key focus areas include the applications of artificial intelligence, the use of Big Data and Internet of Things, the importance of mathematical modeling for predictions, utilization of technology for community screening, the use of nanotechnology for treatment and vaccine development, the utility of telemedicine, the implementation of 3D-printing to manage new demands and the potential of robotics. CONCLUSION: The review concludes by highlighting the need for collaboration in the scientific community with open sharing of knowledge, tools, and expertise.</td>\n",
       "      <td>{'url': 'https://doi.org/10.1007/s12195-020-00629-w', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66335</th>\n",
       "      <td>50</td>\n",
       "      <td>zvop8bxh</td>\n",
       "      <td>2</td>\n",
       "      <td>50</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>zvop8bxh</td>\n",
       "      <td>Antiviral RNAi therapy: emerging approaches for hitting a moving target</td>\n",
       "      <td>The field of directed RNA interference (RNAi) has rapidly developed into a highly promising approach for specifically downregulating genes to alleviate disease pathology. This technology is especially well-suited to treating viral infections, and numerous examples now illustrate that a wide range of viruses can be inhibited with RNAi, both in vitro and in vivo. One principle that has arisen from this work is that antiviral RNAi therapies must be tailored to the unique life cycle of each pathogen, including the choice of delivery vehicle, route of administration, gene(s) targeted and regulation and duration of RNAi induction. Although effective strategies will be customized to each virus, all such therapies must overcome similar challenges. Importantly, treatment strategies must compensate for the inevitable fact that viral genome sequences evolve extremely rapidly, and computational and bioinformatics approaches may aid in the development of therapies that resist viral escape. Furthermore, all RNAi strategies involve the delivery of nucleic acids to target cells, and all will therefore benefit from the development of enhanced gene design and delivery technologies. Here, we review the substantial progress that has been made towards identifying effective antiviral RNAi targets and discuss strategies for translating these findings into effective clinical therapies.</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/16177819/', 'pubmed_id': '16177819'}</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>66336 rows × 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       query-id corpus-id  score  _id_x  \\\n",
       "0             1  005b2j4b      2      1   \n",
       "1            16  005b2j4b      0     16   \n",
       "2            32  005b2j4b      0     32   \n",
       "3            37  005b2j4b      0     37   \n",
       "4             1  00fmeepz      1      1   \n",
       "...         ...       ...    ...    ...   \n",
       "66331        50  zn10rnrm      1     50   \n",
       "66332        50  zstmdt4n      0     50   \n",
       "66333        50  zth8ffy3      0     50   \n",
       "66334        50  zv4nbz9p      2     50   \n",
       "66335        50  zvop8bxh      2     50   \n",
       "\n",
       "                                                                           text_x  \\\n",
       "0                                                  what is the origin of COVID-19   \n",
       "1                           how long does coronavirus remain stable  on surfaces?   \n",
       "2                     Does SARS-CoV-2 have any subtypes, and if so what are they?   \n",
       "3      What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?   \n",
       "4                                                  what is the origin of COVID-19   \n",
       "...                                                                           ...   \n",
       "66331               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66332               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66333               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66334               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66335               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "\n",
       "                                                                                                                                                                                                                                                                      metadata_x  \\\n",
       "0                                                                                   {'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}   \n",
       "1      {'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}   \n",
       "2                                                                                            {'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}   \n",
       "3                                                                                               {'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}   \n",
       "4                                                                                   {'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}   \n",
       "...                                                                                                                                                                                                                                                                          ...   \n",
       "66331                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66332                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66333                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66334                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66335                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "\n",
       "          _id_y  \\\n",
       "0      005b2j4b   \n",
       "1      005b2j4b   \n",
       "2      005b2j4b   \n",
       "3      005b2j4b   \n",
       "4      00fmeepz   \n",
       "...         ...   \n",
       "66331  zn10rnrm   \n",
       "66332  zstmdt4n   \n",
       "66333  zth8ffy3   \n",
       "66334  zv4nbz9p   \n",
       "66335  zvop8bxh   \n",
       "\n",
       "                                                                                                           title  \\\n",
       "0      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "1      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "2      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "3      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "4                                                   Comprehensive overview of COVID-19 based on current evidence   \n",
       "...                                                                                                          ...   \n",
       "66331                                                                          Characterization of RNA in Saliva   \n",
       "66332                  Coordinate induction of IFN-α and -γ by SARS-CoV also in the absence of virus replication   \n",
       "66333                                         Vasculopathy and Coagulopathy Associated with SARS-CoV-2 Infection   \n",
       "66334             Emerging Technologies for Use in the Study, Diagnosis, and Treatment of Patients with COVID-19   \n",
       "66335                                    Antiviral RNAi therapy: emerging approaches for hitting a moving target   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   text_y  \\\n",
       "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.   \n",
       "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ...   \n",
       "66331                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Background: We have previously shown that human mRNAs are present in saliva and can be used as biomarkers of oral cancer. In this study, we analyzed the integrity, sources, and stability of salivary RNA. Methods: We measured the integrity of salivary RNA with reverse transcription followed by PCR (RT-PCR) or RT-quantitative PCR (RT-qPCR). To study RNA entry sites into the oral cavity, we used RT-PCR analysis of salivary RNA from the 3 major salivary glands, gingival crevice fluid, and desquamated oral epithelial cells. We measured stability of the salivary β-actin mRNA by RT-qPCR of salivary RNA incubated at room temperature for different periods of time. We measured RNA association with other macromolecules by filtering saliva through pores of different sizes before performing RT-qPCR. To assess RNA–macromolecule interaction, we incubated saliva with Triton X-100 for different periods of time before performing RT-qPCR. Results: In most cases, we detected partial- to full-length salivary mRNAs and smaller amounts of middle and 3′ gene amplicons compared with the 5′. RNA was present in all oral fluids examined. Endogenous salivary β-actin mRNA degraded more slowly than exogenous β-actin mRNA, with half-lives of 12.2 and 0.4 min, respectively (P <0.001). Salivary RNA could not pass through 0.22 or 0.45 μm pores. Incubation of saliva with Triton X-100 accelerated degradation of salivary RNA. Conclusions: Saliva harbors both full-length and partially degraded forms of mRNA. RNA enters the oral cavity from different sources, and association with macromolecules may protect salivary RNA from degradation.   \n",
       "66332  Abstract Background: Severe acute respiratory syndrome (SARS) is an emerging infection caused by a novel coronavirus known as SARS-CoV, characterized by an over-exuberant immune response with lung lymphomononuclear cells infiltration and proliferation that may account for tissue damage more than the direct effect of viral replication. This study is aimed at investigating the capability of SARS-CoV to activate IFN-α and -γ expression in lymphomonocytes (PBMC) from healthy donors, evaluating whether viral replication is necessary for this activation. Results: SARS-CoV virus is able to induce both IFN-α and -γ mRNA accumulation and protein release in a dose-dependent manner, MOI 10 being the most effective. The time course curve indicated that IFN-α mRNA induction peaked at 24 h.p.i,. whereas IFN-γ mRNA was still increasing at 48 h.p.i. Released IFN (both types) reached a plateau after 24–48 h.p.i. and remained rather stable over a 5-day period. A transient peak of negative strand viral RNA was detected after 1–2 days of infection, but neither infectious virus progeny yield nor newly produced viral genomic RNA could be evidenced in infected cultures, even after prolonged observation time (up to 13 days). Cocultivation of PBMC with fixed SARS-CoV-infected Vero cells was even more efficient than exposure to live virus in eliciting IFN-α and -γ induction. A combination of IFN-α and -γ strongly inhibited SARS-CoV replication in Vero cells, while the single cytokines were much less effective. Conclusions: This study provides evidence that SARS-CoV is able to induce in normal PBMC a coordinate induction of IFN-α and -γ gene expression. Virus replication is not necessary for IFN induction since efficient IFN expression could be obtained also by the cocultivation of normal PBMC with fixed SARS-CoV-infected cells. Concomitant activation of IFN-α and -γ gene expression by SARS-CoV in vivo may be relevant for the pathogenesis of the disease, both for the possible involvement in immunomediated damage of the tissues and for the strong inhibition of SARS-CoV replication as a result of combined cytokine action.   \n",
       "66333                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has resulted in > 500,000 deaths worldwide, including > 125,000 deaths in the U.S. since its emergence in late December 2019 and June 2020. Neither curative anti-viral drugs nor a protective vaccine is currently available for the treatment and prevention of COVID-19. Recently, new clinical syndromes associated with coagulopathy and vasculopathy have emerged as a cause of sudden death and other serious clinical manifestations in younger patients infected with SARS-CoV-2 infection. Angiotensin converting enzyme 2 (ACE2), the receptor for SARS-CoV-2 and other coronaviruses, is a transmembrane protein expressed by lung alveolar epithelial cells, enterocytes, and vascular endothelial cells, whose physiologic role is to induce the maturation of angiotensin I to generate angiotensin 1-7, a peptide hormone that controls vasoconstriction and blood pressure. In this review, we provide the general context of the molecular and cellular mechanisms of SARS-CoV-2 infection with a focus on endothelial cells, describe the vasculopathy and coagulopathy syndromes in patients with SARS-CoV-2, and outline current understanding of the underlying mechanistic aspects.   \n",
       "66334                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  INTRODUCTION: The COVID-19 pandemic has caused an unprecedented health and economic worldwide crisis. Innovative solutions are imperative given limited resources and immediate need for medical supplies, healthcare support and treatments. AIM: The purpose of this review is to summarize emerging technologies being implemented in the study, diagnosis, and treatment of COVID-19. RESULTS: Key focus areas include the applications of artificial intelligence, the use of Big Data and Internet of Things, the importance of mathematical modeling for predictions, utilization of technology for community screening, the use of nanotechnology for treatment and vaccine development, the utility of telemedicine, the implementation of 3D-printing to manage new demands and the potential of robotics. CONCLUSION: The review concludes by highlighting the need for collaboration in the scientific community with open sharing of knowledge, tools, and expertise.   \n",
       "66335                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            The field of directed RNA interference (RNAi) has rapidly developed into a highly promising approach for specifically downregulating genes to alleviate disease pathology. This technology is especially well-suited to treating viral infections, and numerous examples now illustrate that a wide range of viruses can be inhibited with RNAi, both in vitro and in vivo. One principle that has arisen from this work is that antiviral RNAi therapies must be tailored to the unique life cycle of each pathogen, including the choice of delivery vehicle, route of administration, gene(s) targeted and regulation and duration of RNAi induction. Although effective strategies will be customized to each virus, all such therapies must overcome similar challenges. Importantly, treatment strategies must compensate for the inevitable fact that viral genome sequences evolve extremely rapidly, and computational and bioinformatics approaches may aid in the development of therapies that resist viral escape. Furthermore, all RNAi strategies involve the delivery of nucleic acids to target cells, and all will therefore benefit from the development of enhanced gene design and delivery technologies. Here, we review the substantial progress that has been made towards identifying effective antiviral RNAi targets and discuss strategies for translating these findings into effective clinical therapies.   \n",
       "\n",
       "                                                                                                                                                                                                                   metadata_y  \n",
       "0                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "1                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "2                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "3                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "4                                                                                                                                                                                                {'url': '', 'pubmed_id': ''}  \n",
       "...                                                                                                                                                                                                                       ...  \n",
       "66331                                                                                                                               {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108156/', 'pubmed_id': '16601067'}  \n",
       "66332  {'url': 'https://api.elsevier.com/content/article/pii/S0042682205004241; https://www.sciencedirect.com/science/article/pii/S0042682205004241; https://www.ncbi.nlm.nih.gov/pubmed/16095648/', 'pubmed_id': '16095648'}  \n",
       "66333                                                                                                                                                                                            {'url': '', 'pubmed_id': ''}  \n",
       "66334                                                                                                                                                  {'url': 'https://doi.org/10.1007/s12195-020-00629-w', 'pubmed_id': ''}  \n",
       "66335                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/16177819/', 'pubmed_id': '16177819'}  \n",
       "\n",
       "[66336 rows x 10 columns]"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "2e17e679",
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = merged_df.rename(columns={'text_x': 'query-text',\n",
    "                                      'metadata_x': 'query-metadata',\n",
    "                                      'text_y': 'corpus-text', \n",
    "                                      'title': 'corpus-title',\n",
    "                                      'metadata_y': 'corpus-metadata'})[['query-id', 'corpus-id', 'score', 'query-text', 'corpus-title', 'corpus-text', 'query-metadata', 'corpus-metadata']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "57dc7392",
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df.to_csv(\"trec_covid_merged_data.tsv\", sep='\\t', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "6f312673",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>query-id</th>\n",
       "      <th>corpus-id</th>\n",
       "      <th>score</th>\n",
       "      <th>query-text</th>\n",
       "      <th>corpus-title</th>\n",
       "      <th>corpus-text</th>\n",
       "      <th>query-metadata</th>\n",
       "      <th>corpus-metadata</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>2</td>\n",
       "      <td>what is the origin of COVID-19</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>16</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>how long does coronavirus remain stable  on surfaces?</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>32</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>Does SARS-CoV-2 have any subtypes, and if so what are they?</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>37</td>\n",
       "      <td>005b2j4b</td>\n",
       "      <td>0</td>\n",
       "      <td>What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?</td>\n",
       "      <td>Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses</td>\n",
       "      <td>Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.</td>\n",
       "      <td>{'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>00fmeepz</td>\n",
       "      <td>1</td>\n",
       "      <td>what is the origin of COVID-19</td>\n",
       "      <td>Comprehensive overview of COVID-19 based on current evidence</td>\n",
       "      <td>In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.</td>\n",
       "      <td>{'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}</td>\n",
       "      <td>{'url': '', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66331</th>\n",
       "      <td>50</td>\n",
       "      <td>zn10rnrm</td>\n",
       "      <td>1</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>Characterization of RNA in Saliva</td>\n",
       "      <td>Background: We have previously shown that human mRNAs are present in saliva and can be used as biomarkers of oral cancer. In this study, we analyzed the integrity, sources, and stability of salivary RNA. Methods: We measured the integrity of salivary RNA with reverse transcription followed by PCR (RT-PCR) or RT-quantitative PCR (RT-qPCR). To study RNA entry sites into the oral cavity, we used RT-PCR analysis of salivary RNA from the 3 major salivary glands, gingival crevice fluid, and desquamated oral epithelial cells. We measured stability of the salivary β-actin mRNA by RT-qPCR of salivary RNA incubated at room temperature for different periods of time. We measured RNA association with other macromolecules by filtering saliva through pores of different sizes before performing RT-qPCR. To assess RNA–macromolecule interaction, we incubated saliva with Triton X-100 for different periods of time before performing RT-qPCR. Results: In most cases, we detected partial- to full-length salivary mRNAs and smaller amounts of middle and 3′ gene amplicons compared with the 5′. RNA was present in all oral fluids examined. Endogenous salivary β-actin mRNA degraded more slowly than exogenous β-actin mRNA, with half-lives of 12.2 and 0.4 min, respectively (P &lt;0.001). Salivary RNA could not pass through 0.22 or 0.45 μm pores. Incubation of saliva with Triton X-100 accelerated degradation of salivary RNA. Conclusions: Saliva harbors both full-length and partially degraded forms of mRNA. RNA enters the oral cavity from different sources, and association with macromolecules may protect salivary RNA from degradation.</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108156/', 'pubmed_id': '16601067'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66332</th>\n",
       "      <td>50</td>\n",
       "      <td>zstmdt4n</td>\n",
       "      <td>0</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>Coordinate induction of IFN-α and -γ by SARS-CoV also in the absence of virus replication</td>\n",
       "      <td>Abstract Background: Severe acute respiratory syndrome (SARS) is an emerging infection caused by a novel coronavirus known as SARS-CoV, characterized by an over-exuberant immune response with lung lymphomononuclear cells infiltration and proliferation that may account for tissue damage more than the direct effect of viral replication. This study is aimed at investigating the capability of SARS-CoV to activate IFN-α and -γ expression in lymphomonocytes (PBMC) from healthy donors, evaluating whether viral replication is necessary for this activation. Results: SARS-CoV virus is able to induce both IFN-α and -γ mRNA accumulation and protein release in a dose-dependent manner, MOI 10 being the most effective. The time course curve indicated that IFN-α mRNA induction peaked at 24 h.p.i,. whereas IFN-γ mRNA was still increasing at 48 h.p.i. Released IFN (both types) reached a plateau after 24–48 h.p.i. and remained rather stable over a 5-day period. A transient peak of negative strand viral RNA was detected after 1–2 days of infection, but neither infectious virus progeny yield nor newly produced viral genomic RNA could be evidenced in infected cultures, even after prolonged observation time (up to 13 days). Cocultivation of PBMC with fixed SARS-CoV-infected Vero cells was even more efficient than exposure to live virus in eliciting IFN-α and -γ induction. A combination of IFN-α and -γ strongly inhibited SARS-CoV replication in Vero cells, while the single cytokines were much less effective. Conclusions: This study provides evidence that SARS-CoV is able to induce in normal PBMC a coordinate induction of IFN-α and -γ gene expression. Virus replication is not necessary for IFN induction since efficient IFN expression could be obtained also by the cocultivation of normal PBMC with fixed SARS-CoV-infected cells. Concomitant activation of IFN-α and -γ gene expression by SARS-CoV in vivo may be relevant for the pathogenesis of the disease, both for the possible involvement in immunomediated damage of the tissues and for the strong inhibition of SARS-CoV replication as a result of combined cytokine action.</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>{'url': 'https://api.elsevier.com/content/article/pii/S0042682205004241; https://www.sciencedirect.com/science/article/pii/S0042682205004241; https://www.ncbi.nlm.nih.gov/pubmed/16095648/', 'pubmed_id': '16095648'}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66333</th>\n",
       "      <td>50</td>\n",
       "      <td>zth8ffy3</td>\n",
       "      <td>0</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>Vasculopathy and Coagulopathy Associated with SARS-CoV-2 Infection</td>\n",
       "      <td>The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has resulted in &gt; 500,000 deaths worldwide, including &gt; 125,000 deaths in the U.S. since its emergence in late December 2019 and June 2020. Neither curative anti-viral drugs nor a protective vaccine is currently available for the treatment and prevention of COVID-19. Recently, new clinical syndromes associated with coagulopathy and vasculopathy have emerged as a cause of sudden death and other serious clinical manifestations in younger patients infected with SARS-CoV-2 infection. Angiotensin converting enzyme 2 (ACE2), the receptor for SARS-CoV-2 and other coronaviruses, is a transmembrane protein expressed by lung alveolar epithelial cells, enterocytes, and vascular endothelial cells, whose physiologic role is to induce the maturation of angiotensin I to generate angiotensin 1-7, a peptide hormone that controls vasoconstriction and blood pressure. In this review, we provide the general context of the molecular and cellular mechanisms of SARS-CoV-2 infection with a focus on endothelial cells, describe the vasculopathy and coagulopathy syndromes in patients with SARS-CoV-2, and outline current understanding of the underlying mechanistic aspects.</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>{'url': '', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66334</th>\n",
       "      <td>50</td>\n",
       "      <td>zv4nbz9p</td>\n",
       "      <td>2</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>Emerging Technologies for Use in the Study, Diagnosis, and Treatment of Patients with COVID-19</td>\n",
       "      <td>INTRODUCTION: The COVID-19 pandemic has caused an unprecedented health and economic worldwide crisis. Innovative solutions are imperative given limited resources and immediate need for medical supplies, healthcare support and treatments. AIM: The purpose of this review is to summarize emerging technologies being implemented in the study, diagnosis, and treatment of COVID-19. RESULTS: Key focus areas include the applications of artificial intelligence, the use of Big Data and Internet of Things, the importance of mathematical modeling for predictions, utilization of technology for community screening, the use of nanotechnology for treatment and vaccine development, the utility of telemedicine, the implementation of 3D-printing to manage new demands and the potential of robotics. CONCLUSION: The review concludes by highlighting the need for collaboration in the scientific community with open sharing of knowledge, tools, and expertise.</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>{'url': 'https://doi.org/10.1007/s12195-020-00629-w', 'pubmed_id': ''}</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66335</th>\n",
       "      <td>50</td>\n",
       "      <td>zvop8bxh</td>\n",
       "      <td>2</td>\n",
       "      <td>what is known about an mRNA vaccine for the SARS-CoV-2 virus?</td>\n",
       "      <td>Antiviral RNAi therapy: emerging approaches for hitting a moving target</td>\n",
       "      <td>The field of directed RNA interference (RNAi) has rapidly developed into a highly promising approach for specifically downregulating genes to alleviate disease pathology. This technology is especially well-suited to treating viral infections, and numerous examples now illustrate that a wide range of viruses can be inhibited with RNAi, both in vitro and in vivo. One principle that has arisen from this work is that antiviral RNAi therapies must be tailored to the unique life cycle of each pathogen, including the choice of delivery vehicle, route of administration, gene(s) targeted and regulation and duration of RNAi induction. Although effective strategies will be customized to each virus, all such therapies must overcome similar challenges. Importantly, treatment strategies must compensate for the inevitable fact that viral genome sequences evolve extremely rapidly, and computational and bioinformatics approaches may aid in the development of therapies that resist viral escape. Furthermore, all RNAi strategies involve the delivery of nucleic acids to target cells, and all will therefore benefit from the development of enhanced gene design and delivery technologies. Here, we review the substantial progress that has been made towards identifying effective antiviral RNAi targets and discuss strategies for translating these findings into effective clinical therapies.</td>\n",
       "      <td>{'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}</td>\n",
       "      <td>{'url': 'https://www.ncbi.nlm.nih.gov/pubmed/16177819/', 'pubmed_id': '16177819'}</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>66336 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       query-id corpus-id  score  \\\n",
       "0             1  005b2j4b      2   \n",
       "1            16  005b2j4b      0   \n",
       "2            32  005b2j4b      0   \n",
       "3            37  005b2j4b      0   \n",
       "4             1  00fmeepz      1   \n",
       "...         ...       ...    ...   \n",
       "66331        50  zn10rnrm      1   \n",
       "66332        50  zstmdt4n      0   \n",
       "66333        50  zth8ffy3      0   \n",
       "66334        50  zv4nbz9p      2   \n",
       "66335        50  zvop8bxh      2   \n",
       "\n",
       "                                                                       query-text  \\\n",
       "0                                                  what is the origin of COVID-19   \n",
       "1                           how long does coronavirus remain stable  on surfaces?   \n",
       "2                     Does SARS-CoV-2 have any subtypes, and if so what are they?   \n",
       "3      What is the result of phylogenetic analysis of SARS-CoV-2 genome sequence?   \n",
       "4                                                  what is the origin of COVID-19   \n",
       "...                                                                           ...   \n",
       "66331               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66332               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66333               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66334               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "66335               what is known about an mRNA vaccine for the SARS-CoV-2 virus?   \n",
       "\n",
       "                                                                                                    corpus-title  \\\n",
       "0      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "1      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "2      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "3      Monophyletic Relationship between Severe Acute Respiratory Syndrome Coronavirus and Group 2 Coronaviruses   \n",
       "4                                                   Comprehensive overview of COVID-19 based on current evidence   \n",
       "...                                                                                                          ...   \n",
       "66331                                                                          Characterization of RNA in Saliva   \n",
       "66332                  Coordinate induction of IFN-α and -γ by SARS-CoV also in the absence of virus replication   \n",
       "66333                                         Vasculopathy and Coagulopathy Associated with SARS-CoV-2 Infection   \n",
       "66334             Emerging Technologies for Use in the Study, Diagnosis, and Treatment of Patients with COVID-19   \n",
       "66335                                    Antiviral RNAi therapy: emerging approaches for hitting a moving target   \n",
       "\n",
       "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              corpus-text  \\\n",
       "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.   \n",
       "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.   \n",
       "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ...   \n",
       "66331                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Background: We have previously shown that human mRNAs are present in saliva and can be used as biomarkers of oral cancer. In this study, we analyzed the integrity, sources, and stability of salivary RNA. Methods: We measured the integrity of salivary RNA with reverse transcription followed by PCR (RT-PCR) or RT-quantitative PCR (RT-qPCR). To study RNA entry sites into the oral cavity, we used RT-PCR analysis of salivary RNA from the 3 major salivary glands, gingival crevice fluid, and desquamated oral epithelial cells. We measured stability of the salivary β-actin mRNA by RT-qPCR of salivary RNA incubated at room temperature for different periods of time. We measured RNA association with other macromolecules by filtering saliva through pores of different sizes before performing RT-qPCR. To assess RNA–macromolecule interaction, we incubated saliva with Triton X-100 for different periods of time before performing RT-qPCR. Results: In most cases, we detected partial- to full-length salivary mRNAs and smaller amounts of middle and 3′ gene amplicons compared with the 5′. RNA was present in all oral fluids examined. Endogenous salivary β-actin mRNA degraded more slowly than exogenous β-actin mRNA, with half-lives of 12.2 and 0.4 min, respectively (P <0.001). Salivary RNA could not pass through 0.22 or 0.45 μm pores. Incubation of saliva with Triton X-100 accelerated degradation of salivary RNA. Conclusions: Saliva harbors both full-length and partially degraded forms of mRNA. RNA enters the oral cavity from different sources, and association with macromolecules may protect salivary RNA from degradation.   \n",
       "66332  Abstract Background: Severe acute respiratory syndrome (SARS) is an emerging infection caused by a novel coronavirus known as SARS-CoV, characterized by an over-exuberant immune response with lung lymphomononuclear cells infiltration and proliferation that may account for tissue damage more than the direct effect of viral replication. This study is aimed at investigating the capability of SARS-CoV to activate IFN-α and -γ expression in lymphomonocytes (PBMC) from healthy donors, evaluating whether viral replication is necessary for this activation. Results: SARS-CoV virus is able to induce both IFN-α and -γ mRNA accumulation and protein release in a dose-dependent manner, MOI 10 being the most effective. The time course curve indicated that IFN-α mRNA induction peaked at 24 h.p.i,. whereas IFN-γ mRNA was still increasing at 48 h.p.i. Released IFN (both types) reached a plateau after 24–48 h.p.i. and remained rather stable over a 5-day period. A transient peak of negative strand viral RNA was detected after 1–2 days of infection, but neither infectious virus progeny yield nor newly produced viral genomic RNA could be evidenced in infected cultures, even after prolonged observation time (up to 13 days). Cocultivation of PBMC with fixed SARS-CoV-infected Vero cells was even more efficient than exposure to live virus in eliciting IFN-α and -γ induction. A combination of IFN-α and -γ strongly inhibited SARS-CoV replication in Vero cells, while the single cytokines were much less effective. Conclusions: This study provides evidence that SARS-CoV is able to induce in normal PBMC a coordinate induction of IFN-α and -γ gene expression. Virus replication is not necessary for IFN induction since efficient IFN expression could be obtained also by the cocultivation of normal PBMC with fixed SARS-CoV-infected cells. Concomitant activation of IFN-α and -γ gene expression by SARS-CoV in vivo may be relevant for the pathogenesis of the disease, both for the possible involvement in immunomediated damage of the tissues and for the strong inhibition of SARS-CoV replication as a result of combined cytokine action.   \n",
       "66333                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the causative agent of coronavirus disease 2019 (COVID-19), has resulted in > 500,000 deaths worldwide, including > 125,000 deaths in the U.S. since its emergence in late December 2019 and June 2020. Neither curative anti-viral drugs nor a protective vaccine is currently available for the treatment and prevention of COVID-19. Recently, new clinical syndromes associated with coagulopathy and vasculopathy have emerged as a cause of sudden death and other serious clinical manifestations in younger patients infected with SARS-CoV-2 infection. Angiotensin converting enzyme 2 (ACE2), the receptor for SARS-CoV-2 and other coronaviruses, is a transmembrane protein expressed by lung alveolar epithelial cells, enterocytes, and vascular endothelial cells, whose physiologic role is to induce the maturation of angiotensin I to generate angiotensin 1-7, a peptide hormone that controls vasoconstriction and blood pressure. In this review, we provide the general context of the molecular and cellular mechanisms of SARS-CoV-2 infection with a focus on endothelial cells, describe the vasculopathy and coagulopathy syndromes in patients with SARS-CoV-2, and outline current understanding of the underlying mechanistic aspects.   \n",
       "66334                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  INTRODUCTION: The COVID-19 pandemic has caused an unprecedented health and economic worldwide crisis. Innovative solutions are imperative given limited resources and immediate need for medical supplies, healthcare support and treatments. AIM: The purpose of this review is to summarize emerging technologies being implemented in the study, diagnosis, and treatment of COVID-19. RESULTS: Key focus areas include the applications of artificial intelligence, the use of Big Data and Internet of Things, the importance of mathematical modeling for predictions, utilization of technology for community screening, the use of nanotechnology for treatment and vaccine development, the utility of telemedicine, the implementation of 3D-printing to manage new demands and the potential of robotics. CONCLUSION: The review concludes by highlighting the need for collaboration in the scientific community with open sharing of knowledge, tools, and expertise.   \n",
       "66335                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            The field of directed RNA interference (RNAi) has rapidly developed into a highly promising approach for specifically downregulating genes to alleviate disease pathology. This technology is especially well-suited to treating viral infections, and numerous examples now illustrate that a wide range of viruses can be inhibited with RNAi, both in vitro and in vivo. One principle that has arisen from this work is that antiviral RNAi therapies must be tailored to the unique life cycle of each pathogen, including the choice of delivery vehicle, route of administration, gene(s) targeted and regulation and duration of RNAi induction. Although effective strategies will be customized to each virus, all such therapies must overcome similar challenges. Importantly, treatment strategies must compensate for the inevitable fact that viral genome sequences evolve extremely rapidly, and computational and bioinformatics approaches may aid in the development of therapies that resist viral escape. Furthermore, all RNAi strategies involve the delivery of nucleic acids to target cells, and all will therefore benefit from the development of enhanced gene design and delivery technologies. Here, we review the substantial progress that has been made towards identifying effective antiviral RNAi targets and discuss strategies for translating these findings into effective clinical therapies.   \n",
       "\n",
       "                                                                                                                                                                                                                                                                  query-metadata  \\\n",
       "0                                                                                   {'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}   \n",
       "1      {'query': 'how long does coronavirus survive on surfaces', 'narrative': 'Studies of time SARS-CoV-2 remains stable after being deposited from an infected person on everyday surfaces in a household or hospital setting, such as through coughing or touching objects.'}   \n",
       "2                                                                                            {'query': 'coronavirus subtypes', 'narrative': 'Papers that discuss subtypes of the virus, from named subtypes to speculative subtypes based on genomic or geographic clustering.'}   \n",
       "3                                                                                               {'query': 'SARS-CoV-2 phylogenetic analysis', 'narrative': 'Looking for a range of studies which provide the results of phylogenetic network analysis on the SARS-CoV-2 genome'}   \n",
       "4                                                                                   {'query': 'coronavirus origin', 'narrative': 'seeking range of information about the SARS-CoV-2 virus's origin, including its evolution, animal source, and first transmission into humans'}   \n",
       "...                                                                                                                                                                                                                                                                          ...   \n",
       "66331                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66332                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66333                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66334                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "66335                                     {'query': 'mRNA vaccine coronavirus', 'narrative': 'Looking for studies specifically focusing on mRNA vaccines for COVID-19, including how mRNA vaccines work, why they are promising, and any results from actual clinical studies.'}   \n",
       "\n",
       "                                                                                                                                                                                                              corpus-metadata  \n",
       "0                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "1                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "2                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "3                                                                                                                                           {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15116304/', 'pubmed_id': '15116304'}  \n",
       "4                                                                                                                                                                                                {'url': '', 'pubmed_id': ''}  \n",
       "...                                                                                                                                                                                                                       ...  \n",
       "66331                                                                                                                               {'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108156/', 'pubmed_id': '16601067'}  \n",
       "66332  {'url': 'https://api.elsevier.com/content/article/pii/S0042682205004241; https://www.sciencedirect.com/science/article/pii/S0042682205004241; https://www.ncbi.nlm.nih.gov/pubmed/16095648/', 'pubmed_id': '16095648'}  \n",
       "66333                                                                                                                                                                                            {'url': '', 'pubmed_id': ''}  \n",
       "66334                                                                                                                                                  {'url': 'https://doi.org/10.1007/s12195-020-00629-w', 'pubmed_id': ''}  \n",
       "66335                                                                                                                                       {'url': 'https://www.ncbi.nlm.nih.gov/pubmed/16177819/', 'pubmed_id': '16177819'}  \n",
       "\n",
       "[66336 rows x 8 columns]"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "e5ddc53f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.',\n",
       " 'Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.',\n",
       " 'Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.',\n",
       " 'Although primary genomic analysis has revealed that severe acute respiratory syndrome coronavirus (SARS CoV) is a new type of coronavirus, the different protein trees published in previous reports have provided no conclusive evidence indicating the phylogenetic position of SARS CoV. To clarify the phylogenetic relationship between SARS CoV and other coronaviruses, we compiled a large data set composed of 7 concatenated protein sequences and performed comprehensive analyses, using the maximum-likelihood, Bayesian-inference, and maximum-parsimony methods. All resulting phylogenetic trees displayed an identical topology and supported the hypothesis that the relationship between SARS CoV and group 2 CoVs is monophyletic. Relationships among all major groups were well resolved and were supported by all statistical analyses.',\n",
       " 'In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.',\n",
       " 'In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.',\n",
       " 'In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.',\n",
       " 'In December 2019, twenty-seven pneumonia patients with unknown causes originated in South China seafood market in Wuhan. The virus infection spread rapidly and swept through China in less than a month. Subsequently, the virus was proven a novel coronavirus and named SARS-CoV-2. The outbreak of novel coronavirus has been determined as a Public Health Emergency of International Concern (PHEIC) by WHO on January 31, 2020. Similar to other coronaviruses like the Middle East Respiratory Syndrome (MERS) CoV and Severe Acute Respiratory Syndrome (SARS) CoV, the novel coronavirus was reported to spread via respiratory droplets and close contact from human to human, which means the virus is highly infectious and dangerous. Unfortunately, till now the virus has spread to over 200 countries/territories/areas around the world and the Coronavirus Disease 2019 (COVID-19) outbreak is continuing to grow. Currently, information sharing and transparency are essential for risk assessment and epidemic control in all endemic areas. In this article, we compared SARS-CoV-2 with SARS-CoV and influenza virus, discussed current researching progress of COVID-19, including clinical characteristics, pathological changes, treatment measures, and so on.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'OBJECTIVES: To provide an overview of the three major deadly coronaviruses and identify areas for improvement of future preparedness plans, as well as provide a critical assessment of the risk factors and actionable items for stopping their spread, utilizing lessons learned from the first two deadly coronavirus outbreaks, as well as initial reports from the current novel coronavirus (COVID-19) epidemic in Wuhan, China. METHODS: Utilizing the Centers for Disease Control and Prevention (CDC, USA) website, and a comprehensive review of PubMed literature, we obtained information regarding clinical signs and symptoms, treatment and diagnosis, transmission methods, protection methods and risk factors for Middle East Respiratory Syndrome (MERS), Severe Acute Respiratory Syndrome (SARS) and COVID-19. Comparisons between the viruses were made. RESULTS: Inadequate risk assessment regarding the urgency of the situation, and limited reporting on the virus within China has, in part, led to the rapid spread of COVID-19 throughout mainland China and into proximal and distant countries. Compared with SARS and MERS, COVID-19 has spread more rapidly, due in part to increased globalization and the focus of the epidemic. Wuhan, China is a large hub connecting the North, South, East and West of China via railways and a major international airport. The availability of connecting flights, the timing of the outbreak during the Chinese (Lunar) New Year, and the massive rail transit hub located in Wuhan has enabled the virus to perforate throughout China, and eventually, globally. CONCLUSIONS: We conclude that we did not learn from the two prior epidemics of coronavirus and were ill-prepared to deal with the challenges the COVID-19 epidemic has posed. Future research should attempt to address the uses and implications of internet of things (IoT) technologies for mapping the spread of infection.',\n",
       " 'Middle East respiratory syndrome (MERS) is an emerging infectious disease, caused by Middle East respiratory syndrome coronavirus (MERS-CoV) and is considered to be a zoonosis. However, the natural reservoirs of MERS-CoV remain obscure, with bats and camels as the most suspected sources. In this article, we review the evidence supporting a bat/camel origin of human MERS-CoV infection and current knowledge on the modes of camel-to-human transmission of MERS-CoV.',\n",
       " \"Genetic engineering helps reveal origin of deadly 'flu\",\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " 'A novel coronavirus SARS-CoV-2, also called novel coronavirus 2019 (nCoV-19), started to circulate among humans around December 2019, and it is now widespread as a global pandemic. The disease caused by SARS-CoV-2 virus is called COVID-19, which is highly contagious and has an overall mortality rate of 6.96% as of May 4, 2020. There is no vaccine or antiviral available for SARS-CoV-2. In this study, we report our discovery of inhibitors targeting the SARS-CoV-2 main protease (M(pro)). Using the FRET-based enzymatic assay, several inhibitors including boceprevir, GC-376, and calpain inhibitors II, and XII were identified to have potent activity with single-digit to submicromolar IC(50) values in the enzymatic assay. The mechanism of action of the hits was further characterized using enzyme kinetic studies, thermal shift binding assays, and native mass spectrometry. Significantly, four compounds (boceprevir, GC-376, calpain inhibitors II and XII) inhibit SARS-CoV-2 viral replication in cell culture with EC(50) values ranging from 0.49 to 3.37 μM. Notably, boceprevir, calpain inhibitors II and XII represent novel chemotypes that are distinct from known M(pro) inhibitors. A complex crystal structure of SARS-CoV-2 M(pro) with GC-376, determined at 2.15 Å resolution with three monomers per asymmetric unit, revealed two unique binding configurations, shedding light on the molecular interactions and protein conformational flexibility underlying substrate and inhibitor binding by M(pro). Overall, the compounds identified herein provide promising starting points for the further development of SARS-CoV-2 therapeutics.',\n",
       " \"There has been considerable growth in interest in the field of travel medicine and the intersection with Tourism Studies since the 1990s. Yet this interest from a medical perspective is not new as a review of The Lancet, one of the most well-established medical journals, shows. What is new is the way in which the interest in travel medicine has developed across the science–social science divide and has now become one strand of a wider practitioner and academic interest in tourist well-being. With the exception of studies on technology and tourism and environmental science and tourism (e.g. climate change), this science–social science intersection has been comparatively absent from research in Tourism Studies. For this reason, this current issue's paper seeks to broadly outline the evolution of this area of study and some of the influential studies published to date along with some of the research agendas now emerging in this new area of study.\",\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Coronavirus disease 2019 is an infectious disease caused by severe acute respiratory syndrome coronavirus 2. It has taken a toll of lots of lives since its outbreak. Infection prevention at present is an appropriate control measure in addition to other measure like hand hygiene and personal protective equipment (PPE). In our country with a large population, supplying PPE to all the health care workers of all hospitals definitely is an economic burden. Hence we have come up with an economic and simple solution for face mask. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s12070-020-01863-4) contains supplementary material, which is available to authorized users.',\n",
       " 'Abstract Many new viruses have emerged in the last five decades. These newer genetically active agents have a major impact on the public health systems worldwide. Most emerging infections appear to be caused by pathogens already present in the environment, brought out of obscurity or given a selective advantage by changing conditions and afforded an opportunity to infect new host populations. Also on rare occasions, a new variant may evolve and cause a new disease. Altered virus transmission because of deforestation and environment change, ecological changes and agricultural development, commerce, technology, microbial adaptation and change, breakdown of public health measures, and deficiencies in public health infrastructure are the reasons for emergence and reemergence of infectious diseases in general and viral infections in particular. Specific factors precipitating disease emergence can be identified in virtually all cases. Moreover, these factors are increasing in prevalence. This increase, together with the ongoing evolution of viral and microbial variants and selection for drug resistance, suggests that infections will continue to emerge and probably increase and emphasize the urgent need for effective surveillance and control. These viruses are rich source of emerging diseases due to the introduction of infections from other species in the zoonotic pool. In the near future, there is an urgent need to monitor collaboration between human–animal interface so that public health risks can be understood. A number of activities increase microbial traffic from animals to humans or disseminate microbes from isolated groups into new populations and as a result promote emergence and epidemics. In some cases, including many of the most novel infections, the agents are zoonotic crossing from their natural hosts into the human population because of many similarities. Vector-borne diseases also have a natural advantage of dissemination.',\n",
       " 'Abstract Many new viruses have emerged in the last five decades. These newer genetically active agents have a major impact on the public health systems worldwide. Most emerging infections appear to be caused by pathogens already present in the environment, brought out of obscurity or given a selective advantage by changing conditions and afforded an opportunity to infect new host populations. Also on rare occasions, a new variant may evolve and cause a new disease. Altered virus transmission because of deforestation and environment change, ecological changes and agricultural development, commerce, technology, microbial adaptation and change, breakdown of public health measures, and deficiencies in public health infrastructure are the reasons for emergence and reemergence of infectious diseases in general and viral infections in particular. Specific factors precipitating disease emergence can be identified in virtually all cases. Moreover, these factors are increasing in prevalence. This increase, together with the ongoing evolution of viral and microbial variants and selection for drug resistance, suggests that infections will continue to emerge and probably increase and emphasize the urgent need for effective surveillance and control. These viruses are rich source of emerging diseases due to the introduction of infections from other species in the zoonotic pool. In the near future, there is an urgent need to monitor collaboration between human–animal interface so that public health risks can be understood. A number of activities increase microbial traffic from animals to humans or disseminate microbes from isolated groups into new populations and as a result promote emergence and epidemics. In some cases, including many of the most novel infections, the agents are zoonotic crossing from their natural hosts into the human population because of many similarities. Vector-borne diseases also have a natural advantage of dissemination.',\n",
       " '',\n",
       " 'The pandemic of COVID-19 is seriously challenging the medical organization in many parts of the world. This novel corona virus SARS-CoV-2 has a specific tropism for the low respiratory airways, but causes severe pneumonia in a low percentage of patients. However, the rapid spread of the infection during this pandemic is causing the need to hospitalize a high number of patients. Pneumonia in COVID-19 has peculiar features and can be studied by lung ultrasound in the early approach to suspected patients. The sonographic signs are non-specific when considered alone, but observation of some aspects of vertical artifacts can enhance the diagnostic power of the ultrasound examination. Also, the combination of sonographic signs in patterns and their correlation with blood exams in different phenotypes of the disease may allow for a reliable characterization and be of help in triaging and admitting patients.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'High throughput, deep sequencing assays are powerful tools for gaining insights into virus–host interactions. Sequencing assays can discover novel viruses and describe the genomes of novel and known viruses. Genomic information can predict viral proteins that can be characterized, describe important genes in the host that control infections, and evaluate gene expression of viruses and hosts during infection. Sequencing can also describe variation and evolution of viruses during replication and transmission. This review recounts some of the major advances in the studies of virus–host interactions from the last two years, and discusses the uses of sequencing technologies relating to these studies.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " 'In 2012, a novel coronavirus, initially named as human coronavirus EMC (HCoV-EMC) but recently renamed as Middle East respiratory syndrome human coronavirus (MERS-CoV), was identified in patients who suffered severe acute respiratory infection and subsequent renal failure that resulted in death. Ongoing epidemiological investigations together with retrospective studies have found 61 laboratory-confirmed cases of infection with this novel coronavirus, including 34 deaths to date. This novel coronavirus is culturable and two complete genome sequences are now available. Furthermore, molecular detection and indirect immunofluorescence assay have been developed. The present paper summarises the limited recent advances of this novel human coronavirus, including its discovery, genomic characterisation and detection.',\n",
       " '',\n",
       " 'Based on the classical SIR model, we derive a simple modification for the dynamics of epidemics with a known incubation period of infection. The model is described by a system of integro-differential equations. Parameters of our model directly related to epidemiological data. We derive some analytical results, as well as perform numerical simulations. We use the proposed model to analyze COVID-19 epidemic data in Armenia. We propose a strategy: organize a quarantine, and then conduct extensive testing of risk groups during the quarantine, evaluating the percentage of the population among risk groups and people with symptoms.',\n",
       " 'Based on the classical SIR model, we derive a simple modification for the dynamics of epidemics with a known incubation period of infection. The model is described by a system of integro-differential equations. Parameters of our model directly related to epidemiological data. We derive some analytical results, as well as perform numerical simulations. We use the proposed model to analyze COVID-19 epidemic data in Armenia. We propose a strategy: organize a quarantine, and then conduct extensive testing of risk groups during the quarantine, evaluating the percentage of the population among risk groups and people with symptoms.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'RNA viruses are proficient at switching to novel host species due to their fast mutation rates. Implicit in this assumption is the need to evolve adaptations in the new host species to exploit their cells efficiently. However, SARS-CoV-2 has required no significant adaptation to humans since the pandemic began, with no observed selective sweeps to date. Here we contrast the role of positive selection and recombination in the Sarbecoviruses in horseshoe bats to SARS-CoV-2 evolution in humans. While methods can detect some evidence for positive selection in SARS-CoV-2, we demonstrate these are mostly due to recombination and sequencing artefacts. Purifying selection is also substantially weaker in SARS-CoV-2 than in the related bat Sarbecoviruses. In comparison, our results show evidence for positive, specifically episodic selection, acting on the bat virus lineage SARS-CoV-2 emerged from. This signature of selection can also be observed among synonymous substitutions, for example, linked to ancestral CpG depletion on this bat lineage. We show the bat virus RmYN02 has recombinant CpG content in Spike pointing to coinfection and evolution in bats without involvement of other species. Our results suggest the non-human progenitor of SARS-CoV-2 was capable of human-human transmission as a consequence of its natural evolution in bats.',\n",
       " 'RNA viruses are proficient at switching to novel host species due to their fast mutation rates. Implicit in this assumption is the need to evolve adaptations in the new host species to exploit their cells efficiently. However, SARS-CoV-2 has required no significant adaptation to humans since the pandemic began, with no observed selective sweeps to date. Here we contrast the role of positive selection and recombination in the Sarbecoviruses in horseshoe bats to SARS-CoV-2 evolution in humans. While methods can detect some evidence for positive selection in SARS-CoV-2, we demonstrate these are mostly due to recombination and sequencing artefacts. Purifying selection is also substantially weaker in SARS-CoV-2 than in the related bat Sarbecoviruses. In comparison, our results show evidence for positive, specifically episodic selection, acting on the bat virus lineage SARS-CoV-2 emerged from. This signature of selection can also be observed among synonymous substitutions, for example, linked to ancestral CpG depletion on this bat lineage. We show the bat virus RmYN02 has recombinant CpG content in Spike pointing to coinfection and evolution in bats without involvement of other species. Our results suggest the non-human progenitor of SARS-CoV-2 was capable of human-human transmission as a consequence of its natural evolution in bats.',\n",
       " 'RNA viruses are proficient at switching to novel host species due to their fast mutation rates. Implicit in this assumption is the need to evolve adaptations in the new host species to exploit their cells efficiently. However, SARS-CoV-2 has required no significant adaptation to humans since the pandemic began, with no observed selective sweeps to date. Here we contrast the role of positive selection and recombination in the Sarbecoviruses in horseshoe bats to SARS-CoV-2 evolution in humans. While methods can detect some evidence for positive selection in SARS-CoV-2, we demonstrate these are mostly due to recombination and sequencing artefacts. Purifying selection is also substantially weaker in SARS-CoV-2 than in the related bat Sarbecoviruses. In comparison, our results show evidence for positive, specifically episodic selection, acting on the bat virus lineage SARS-CoV-2 emerged from. This signature of selection can also be observed among synonymous substitutions, for example, linked to ancestral CpG depletion on this bat lineage. We show the bat virus RmYN02 has recombinant CpG content in Spike pointing to coinfection and evolution in bats without involvement of other species. Our results suggest the non-human progenitor of SARS-CoV-2 was capable of human-human transmission as a consequence of its natural evolution in bats.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'Abstract This study aimed to find novel information concerning pathogen detection and some probable coinfection factors in hand, foot, and mouth disease (HFMD). In this study, 1104 clinically diagnosed HFMD patients were included. Enterovirus 71 (EV71), coxsackievirus A16 (CA16), and 14 different respiratory pathogens were examined from nasopharyngeal swabs using polymerase chain reaction (PCR) or reverse transcriptase PCR (RT-PCR). To evaluate the immune activation in HFMD patients, 8 cytokines and IgM antibodies to EV71 and CA16 from mild and severe patients were detected. Our results indicated that the severity of HFMD may affect the pathogen detection. The lower positive rates of enterovirus and respiratory viruses in severe HFMD cases by RT-PCR were probably related to stronger immune response. Therefore, immunological tests such as ELISA are essential supplements to PCR or RT-PCR in order to increase pathogen diagnosis in HFMD, especially in severe cases.',\n",
       " 'The spectrum of human pathogens and the infectious diseases they cause is continuously changing through evolution and changes in the way human populations interact with their environment and each other. New human pathogens most often emerge from an animal reservoir, emphasizing the central role that non-human reservoirs play in human infectious diseases. Pathogens may also re-emerge with new characteristics, such as multidrug-resistance, or in different places, such as West Nile virus in the USA in 1999, to cause new epidemics. Most human pathogens have a history of evolution in which they first emerge and cause epidemics, become unstably adapted, re-emerge periodically, and eventually become endemic with the potential for future outbreaks.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'BACKGROUND: Three clusters of coronavirus disease 2019 (COVID-19) linked to a tour group from China, a company conference, and a church were identified in Singapore in February, 2020. METHODS: We gathered epidemiological and clinical data from individuals with confirmed COVID-19, via interviews and inpatient medical records, and we did field investigations to assess interactions and possible modes of transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Open source reports were obtained for overseas cases. We reported the median (IQR) incubation period of SARS-CoV-2. FINDINGS: As of Feb 15, 2020, 36 cases of COVID-19 were linked epidemiologically to the first three clusters of circumscribed local transmission in Singapore. 425 close contacts were quarantined. Direct or prolonged close contact was reported among affected individuals, although indirect transmission (eg, via fomites and shared food) could not be excluded. The median incubation period of SARS-CoV-2 was 4 days (IQR 3–6). The serial interval between transmission pairs ranged between 3 days and 8 days. INTERPRETATION: SARS-CoV-2 is transmissible in community settings, and local clusters of COVID-19 are expected in countries with high travel volume from China before the lockdown of Wuhan and institution of travel restrictions. Enhanced surveillance and contact tracing is essential to minimise the risk of widespread transmission in the community. FUNDING: None.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'The emergence of the SARS‐CoV‐2 strain of the human coronavirus has thrown the world into the midst of a new pandemic. In the human body, the virus causes COVID‐19, a disease characterized by shortness of breath, fever, and pneumonia, which can be fatal in vulnerable individuals. SARS‐CoV‐2 has characteristics of past human coronaviruses, with close genomic similarities to SARS‐CoV, the virus that causes the disease SARS. Like these related coronaviruses, SARS‐CoV‐2 is transmitted through the inhalation of droplets and interaction with contaminated surfaces. Across the world, laboratories are developing candidate vaccines for the virus – with vaccine trials underway in the US and the United Kingdom ‐ and considering various drugs for possible treatments and prophylaxis. Here, we provide an overview of SARS‐CoV‐2 by analyzing its virology, epidemiology, and modes of transmission while examining the current progress of testing procedures and possible treatments through drugs and vaccines.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Novel coronavirus-19 (2019-nCoV or COVID-19) is by far the most dangerous coronavirus ever identified for the third time in the three decades capable of infecting not only the animals but also the humans across the globe. Nearly 6000 deaths have been recorded due mainly to COVID-19 outbreak worldwide and more than 50% of these deaths appeared to have evolved from China where the virus was thought to originate. The endemicity of COVID-19 dramatically surpassed severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV) that were so far discovered in 2003 and 2012 respectively. Thus, the World Health Organization (WHO) has declared the 2019-nCoV outbreak not only a public health emergency but also pandemic in nature. Currently, over 120 countries including Nigeria were reported to have more than 157,844 confirmed cases and 5,846 deaths due mainly to COVID-19 outbreak as of March 15, 2020, 10:55 GMT. Artificial Intelligence (AI) is widely used to aid in the prediction, detection, response, recovery of disease and making clinical diagnosis. In this study, we highlighted the power of AI in the containment and mitigation of the spread of COVID-19 outbreak in African countries such as Nigeria where human to human contact is apparently inevitable.',\n",
       " 'Bats are responsible for the zoonotic transmission of several major viral diseases including the 2003 SARS outbreak and the ongoing COVID-19 pandemic. While bat genomic sequencing studies have revealed characteristic adaptations of the innate immune system, functional genomic studies are urgently needed to provide a foundation for the molecular dissection of the tolerance of viral infections in bats. Here we report the establishment and screening of genome-wide RNAi library and CRISPR library for the model megabat, Pteropus Alecto. We used the complementary RNAi and CRISPR libraries to interrogate Pteropus Alecto cells for infection with two different viruses, mumps virus and Influenza A virus, respectively. Screening results converged on the endocytosis pathway and the protein secretory pathway as required for both viral infections. Additionally, we revealed a general dependence of the C-1-tetrahydrofolate synthase gene, MTHFD1, for viral replication in bat cells as well as in human cells. MTHFD1 inhibitor carolacton potently blocked replication of several RNA viruses including SARS-CoV-2. Our studies provide a resource for systematic inquiry into the genetic underpinnings of bat biology and a potential target for developing broad spectrum antiviral therapy.',\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " \"Abstract Novel Corona Virus Disease (COVID-19) originating from China has rapidly crossed borders, infecting people throughout the whole world. This phenomenon has led to a massive public reaction; the media has been reporting continuously across borders to keep all informed about the pandemic situation. All these things are creating a lot of concern for people leading to heightened levels of anxiety. Pandemics can lead to heightened levels of stress; Anxiety is a common response to any stressful situation. This study attempted to assess the knowledge, attitude, anxiety experience, and perceived mental healthcare need among adult Indian population during the COVID-19 pandemic. An online survey was conducted using a semi-structured questionnaire using a non-probability snowball sampling technique. A total of 662 responses were received. The responders had a moderate level of knowledge about the COVID-19 infection and adequate knowledge about its preventive aspects. The attitude towards COVID-19 showed peoples' willingness to follow government guidelines on quarantine and social distancing. The anxiety levels identified in the study were high. More than 80 % of the people were preoccupied with the thoughts of COVID-19 and 72 % reported the need to use gloves, and sanitizers. In this study, sleep difficulties, paranoia about acquiring COVID-19 infection and distress related social media were reported in 12.5 %, 37.8 %, and 36.4 % participants respectively. The perceived mental healthcare need was seen in more than 80 % of participants. There is a need to intensify the awareness and address the mental health issues of people during this COVID-19 pandemic.\",\n",
       " 'The COVID-19 pandemic is a serious and global public health concern. It is now well known that COVID-19 cases may result in mild symptoms leading to patient recovery. However, severity of infection, fatality rates, and treatment responses across different countries, age groups, and demographic groups suggest that the nature of infection is diverse, and a timely investigation of the same is needed for evolving sound treatment and preventive strategies. This paper reports an the analysis of age distribution patterns in six groups of Indian COVID-19 patient populations based on their likely geographical origin of infection viz. the United Kingdom, North America, the European Union, the Middle East, and Asian countries. It was observed that patient groups stratified in this way had a distinct age profile and that some of these groups e.g. patient groups from Asia, the European Union, and the United Kingdom formed a different cluster than those from North America, the Middle East, and other regions. Patient age profiles of a population were found to be highly predictive of the group they belong to, and there are indications of their distinct recovery and fatality rates across gender. Altogether this study provides a scalable framework to estimate the source of infection in a new population of COVID-19 patients with unknown origin. It is also concluded that greater public availability of age and other demographic profile details of patients may be helpful in gaining robust insights into COVID-19 infection origins. Datasets and scripts used in this work are shared at http://covid.sciwhylab.org.',\n",
       " 'The COVID-19 pandemic is a serious and global public health concern. It is now well known that COVID-19 cases may result in mild symptoms leading to patient recovery. However, severity of infection, fatality rates, and treatment responses across different countries, age groups, and demographic groups suggest that the nature of infection is diverse, and a timely investigation of the same is needed for evolving sound treatment and preventive strategies. This paper reports an the analysis of age distribution patterns in six groups of Indian COVID-19 patient populations based on their likely geographical origin of infection viz. the United Kingdom, North America, the European Union, the Middle East, and Asian countries. It was observed that patient groups stratified in this way had a distinct age profile and that some of these groups e.g. patient groups from Asia, the European Union, and the United Kingdom formed a different cluster than those from North America, the Middle East, and other regions. Patient age profiles of a population were found to be highly predictive of the group they belong to, and there are indications of their distinct recovery and fatality rates across gender. Altogether this study provides a scalable framework to estimate the source of infection in a new population of COVID-19 patients with unknown origin. It is also concluded that greater public availability of age and other demographic profile details of patients may be helpful in gaining robust insights into COVID-19 infection origins. Datasets and scripts used in this work are shared at http://covid.sciwhylab.org.',\n",
       " 'The COVID-19 pandemic is a serious and global public health concern. It is now well known that COVID-19 cases may result in mild symptoms leading to patient recovery. However, severity of infection, fatality rates, and treatment responses across different countries, age groups, and demographic groups suggest that the nature of infection is diverse, and a timely investigation of the same is needed for evolving sound treatment and preventive strategies. This paper reports an the analysis of age distribution patterns in six groups of Indian COVID-19 patient populations based on their likely geographical origin of infection viz. the United Kingdom, North America, the European Union, the Middle East, and Asian countries. It was observed that patient groups stratified in this way had a distinct age profile and that some of these groups e.g. patient groups from Asia, the European Union, and the United Kingdom formed a different cluster than those from North America, the Middle East, and other regions. Patient age profiles of a population were found to be highly predictive of the group they belong to, and there are indications of their distinct recovery and fatality rates across gender. Altogether this study provides a scalable framework to estimate the source of infection in a new population of COVID-19 patients with unknown origin. It is also concluded that greater public availability of age and other demographic profile details of patients may be helpful in gaining robust insights into COVID-19 infection origins. Datasets and scripts used in this work are shared at http://covid.sciwhylab.org.',\n",
       " 'The COVID-19 pandemic is a serious and global public health concern. It is now well known that COVID-19 cases may result in mild symptoms leading to patient recovery. However, severity of infection, fatality rates, and treatment responses across different countries, age groups, and demographic groups suggest that the nature of infection is diverse, and a timely investigation of the same is needed for evolving sound treatment and preventive strategies. This paper reports an the analysis of age distribution patterns in six groups of Indian COVID-19 patient populations based on their likely geographical origin of infection viz. the United Kingdom, North America, the European Union, the Middle East, and Asian countries. It was observed that patient groups stratified in this way had a distinct age profile and that some of these groups e.g. patient groups from Asia, the European Union, and the United Kingdom formed a different cluster than those from North America, the Middle East, and other regions. Patient age profiles of a population were found to be highly predictive of the group they belong to, and there are indications of their distinct recovery and fatality rates across gender. Altogether this study provides a scalable framework to estimate the source of infection in a new population of COVID-19 patients with unknown origin. It is also concluded that greater public availability of age and other demographic profile details of patients may be helpful in gaining robust insights into COVID-19 infection origins. Datasets and scripts used in this work are shared at http://covid.sciwhylab.org.',\n",
       " 'The current outbreak of a novel severe acute respiratory syndrome-like coronavirus, 2019_nCoV (now named SARS-CoV-2), illustrated difficulties in identifying a novel coronavirus and its natural host, as the coding sequences of various Betacoronavirus species can be highly diverse. By means of whole-genome sequence comparisons, we demonstrate that the noncoding flanks of the viral genome can be used to correctly separate the recognized four betacoronavirus subspecies. The conservation would be sufficient to define target sequences that could, in theory, classify novel virus species into their subspecies. Only 253 upstream noncoding sequences of Sarbecovirus are sufficient to identify genetic similarities between species of this subgenus. Furthermore, it was investigated which bat species have commercial value in China, and would thus likely be handled for trading purposes. A number of coronavirus genomes have been published that were obtained from such bat species. These bats are used in Traditional Chinese Medicine, and their handling poses a potential risk to cause zoonotic coronavirus epidemics. SIGNIFICANCE AND IMPACT OF THE STUDY: The noncoding upstream and downstream flanks of coronavirus genomes allow for rapid classification of novel Betacoronavirus species and correct identification of genetic relationships. Although bats are the likely natural host of 2019_nCoV, the exact bat species that serves as the natural host of the virus remains as yet unknown. Chinese bat species with commercial value were identified as natural reservoirs of coronaviruses and are used in Traditional Chinese Medicine. Since their trading provides a potential risk for spreading zoonoses, a change in these practices is highly recommended.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Abstract COVID-19 is a viral respiratory illness caused by a new coronavirus called SARS-CoV-2. The World Health Organization declared the SARS-CoV-2 outbreak a global public health emergency. We performed genetic analyses of eighty-six complete or near-complete genomes of SARS-CoV-2 and revealed many mutations and deletions on coding and non-coding regions. These observations provided evidence of the genetic diversity and rapid evolution of this novel coronavirus.',\n",
       " 'Coronavirus disease 2019 (CoVID-19) is a new outbreak infectious disease caused by SARS-CoV-2, which was originated from Wuhan in China and has now spread to the whole world. At the meantime, dengue was endemic in the Southeast Asia and South America, and a part of the patients shared the same symptoms, so, we write this paper to alert the clinicians to distinguish these two diseases.',\n",
       " 'Coronavirus disease 2019 (COVID-19), caused by Severe Acute Respiratory Syndrome-Coronavirus-2 (SARS-CoV-2), was first reported in Wuhan, Hubei, China, and has spread to more than 200 other countries around the world. COVID-19 is a highly contagious disease with continuous human-to-human transmission. The origin of the virus is unknown. Airway manipulations and intubations, which are common during anesthesia procedures may increasingly expose anesthesia providers and intensive care unit team members to SARS-CoV-2. Through a comprehensive review of existing studies on COVID-19, this article presents the epidemiological and clinical characteristics of COVID-19, reviews current medical management, and suggests ways to improve the safety of anesthetic procedures. Owing to the highly contagious nature of the virus and the lack of therapeutic drugs or vaccines, precautions should be taken to prevent medical staff from COVID-19.',\n",
       " 'Many serious emerging zoonotic infections have recently arisen from bats, including Ebola, Marburg, SARS-coronavirus, Hendra, Nipah, and a number of rabies and rabies-related viruses, consistent with the overall observation that wildlife are an important source of emerging zoonoses for the human population. Mechanisms underlying the recognized association between ecosystem health and human health remain poorly understood and responding appropriately to the ecological, social and economic conditions that facilitate disease emergence and transmission represents a substantial societal challenge. In the context of disease emergence from wildlife, wildlife and habitat should be conserved, which in turn will preserve vital ecosystem structure and function, which has broader implications for human wellbeing and environmental sustainability, while simultaneously minimizing the spillover of pathogens from wild animals into human beings. In this review, we propose a novel framework for the holistic and interdisciplinary investigation of zoonotic disease emergence and its drivers, using the spillover of bat pathogens as a case study. This study has been developed to gain a detailed interdisciplinary understanding, and it combines cutting-edge perspectives from both natural and social sciences, linked to policy impacts on public health, land use and conservation.',\n",
       " 'Many serious emerging zoonotic infections have recently arisen from bats, including Ebola, Marburg, SARS-coronavirus, Hendra, Nipah, and a number of rabies and rabies-related viruses, consistent with the overall observation that wildlife are an important source of emerging zoonoses for the human population. Mechanisms underlying the recognized association between ecosystem health and human health remain poorly understood and responding appropriately to the ecological, social and economic conditions that facilitate disease emergence and transmission represents a substantial societal challenge. In the context of disease emergence from wildlife, wildlife and habitat should be conserved, which in turn will preserve vital ecosystem structure and function, which has broader implications for human wellbeing and environmental sustainability, while simultaneously minimizing the spillover of pathogens from wild animals into human beings. In this review, we propose a novel framework for the holistic and interdisciplinary investigation of zoonotic disease emergence and its drivers, using the spillover of bat pathogens as a case study. This study has been developed to gain a detailed interdisciplinary understanding, and it combines cutting-edge perspectives from both natural and social sciences, linked to policy impacts on public health, land use and conservation.',\n",
       " 'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior. It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior. RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread. We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region. We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses. This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS. CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS. Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected. We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients. Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic. The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.',\n",
       " 'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior. It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior. RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread. We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region. We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses. This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS. CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS. Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected. We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients. Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic. The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.',\n",
       " 'BACKGROUND: An epidemic of a Severe Acute Respiratory Syndrome (SARS) caused by a new coronavirus has spread from the Guangdong province to the rest of China and to the world, with a puzzling contagion behavior. It is important both for predicting the future of the present outbreak and for implementing effective prophylactic measures, to identify the causes of this behavior. RESULTS: In this report, we show first that the standard Susceptible-Infected-Removed (SIR) model cannot account for the patterns observed in various regions where the disease spread. We develop a model involving two superimposed epidemics to study the recent spread of the SARS in Hong Kong and in the region. We explore the situation where these epidemics may be caused either by a virus and one or several mutants that changed its tropism, or by two unrelated viruses. This has important consequences for the future: the innocuous epidemic might still be there and generate, from time to time, variants that would have properties similar to those of SARS. CONCLUSION: We find that, in order to reconcile the existing data and the spread of the disease, it is convenient to suggest that a first milder outbreak protected against the SARS. Regions that had not seen the first epidemic, or that were affected simultaneously with the SARS suffered much more, with a very high percentage of persons affected. We also find regions where the data appear to be inconsistent, suggesting that they are incomplete or do not reflect an appropriate identification of SARS patients. Finally, we could, within the framework of the model, fix limits to the future development of the epidemic, allowing us to identify landmarks that may be useful to set up a monitoring system to follow the evolution of the epidemic. The model also suggests that there might exist a SARS precursor in a large reservoir, prompting for implementation of precautionary measures when the weather cools down.',\n",
       " \"To investigate the effect of villagers' moral obligation and village cadres' public leadership on villagers' collective action for epidemic prevention and control, against the background of the corona virus disease 2019 (COVID-19) emergency in China, we constructed models based on the institutional analysis and development (IAD) framework and employed principal component analysis (PCA) and ordered probit regression, drawing on survey data from 533 villagers in Henan province adjacent to the COVID-19 origin province, Hubei, China. The results indicate that: (1) generally, both moral obligation and public leadership as well as their constituent indicators contributed positively to collective action for COVID-19 prevention and control; (2) moreover, moral obligation and public leadership can strengthen each other's positive role in collective action for COVID-19 prevention and control. Based on the above findings, this paper suggests that villagers' moral obligation can be perfected through internalizing epidemic prevention and control norms into the villagers' moral norms by the way of villagers mastering the rural public health governance scheme. In addition, public leadership can be improved through professional training of village cadres and by motivating village elites to run for village cadres. With improved villagers' moral obligation and village cadres' public leadership, collective action for epidemic prevention and control could be more likely to be realized.\",\n",
       " 'The Chinese rufous horseshoe bat (Rhinolophus sinicus) has been suggested to carry the direct ancestor of severe acute respiratory syndrome (SARS) coronavirus (SCoV), and the diversity of SARS-like CoVs (SLCoV) within this Rhinolophus species is therefore worth investigating. Here, we demonstrate the remarkable diversity of SLCoVs in R. sinicus and identify a strain with the same pattern of phylogenetic incongruence (i.e. an indication of recombination) as reported previously in another SLCoV strain. Moreover, this strain possesses a distinctive 579 nt deletion in the nsp3 region that was also found in a human SCoV from the late-phase epidemic. Phylogenetic analysis of the Orf1 region suggested that the human SCoVs are phylogenetically closer to SLCoVs in R. sinicus than to SLCoVs in other Rhinolophus species. These findings reveal a closer evolutionary linkage between SCoV in humans and SLCoVs in R. sinicus, defining the scope of surveillance to search for the direct ancestor of human SCoVs.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Background. As the world grapples with the COVID-19 pandemic, there is increasing global interest in the role of serological testing for population monitoring and to inform public policy. However, limitations in serological study designs and test standards raise concerns about the validity of seroprevalence estimates and their utility in decision-making. There is now a critical window of opportunity to learn from early SARS-CoV-2 serology studies. We aimed to synthesize the results of SARS-CoV-2 serosurveillance projects from around the world and provide recommendations to improve the coordination, strategy, and methodology of future serosurveillance efforts. Methods. This was a rapid systematic review of cross-sectional and cohort studies reporting seroprevalence outcomes for SARS-CoV 2. We included completed, ongoing, and proposed serosurveys. The search included electronic databases (PubMed, MedRXIV, BioRXIV, and WHO ICTPR); five medical journals (NEJM, BMJ, JAMA, The Lancet, Annals of Internal Medicine); reports by governments, NGOs, and health systems; and media reports (Google News) from December 1, 2019 to May 1, 2020. We extracted data on study characteristics and critically appraised prevalence estimates using Joanna Briggs Institute criteria. Results. Seventy records met inclusion criteria, describing 73 studies. Of these, 23 reported prevalence estimates: eight preprints, 14 news articles, and one government report. These studies had a total sample size of 35,784 and reported 42 prevalence estimates. Seroprevalence estimates ranged from 0.4% to 59.3%. No estimates were found to have a low risk of bias (43% high risk, 21% moderate risk, 36% unclear). Fifty records reported characteristics of ongoing or proposed serosurveys. Overall, twenty countries have completed, ongoing, or proposed serosurveys. Discussion. Study design, quality, and prevalence estimates of early SARS-CoV2 serosurveys are heterogeneous, suggesting that the urgency to examine seroprevalence may have compromised methodological rigour. Based on the limitations of included studies, future serosurvey investigators and stakeholders should ensure that: i) serological tests used undergo high-quality independent evaluations that include cross-reactivity; ii) all reports of serosurvey results, including media, describe the test used, sample size, and sampling method; and iii) initiatives are coordinated to prevent test fatigue, minimize redundant efforts, and encourage better study methodology. Other. PROSPERO: CRD42020183634. No third-party funding.',\n",
       " 'Background. As the world grapples with the COVID-19 pandemic, there is increasing global interest in the role of serological testing for population monitoring and to inform public policy. However, limitations in serological study designs and test standards raise concerns about the validity of seroprevalence estimates and their utility in decision-making. There is now a critical window of opportunity to learn from early SARS-CoV-2 serology studies. We aimed to synthesize the results of SARS-CoV-2 serosurveillance projects from around the world and provide recommendations to improve the coordination, strategy, and methodology of future serosurveillance efforts. Methods. This was a rapid systematic review of cross-sectional and cohort studies reporting seroprevalence outcomes for SARS-CoV 2. We included completed, ongoing, and proposed serosurveys. The search included electronic databases (PubMed, MedRXIV, BioRXIV, and WHO ICTPR); five medical journals (NEJM, BMJ, JAMA, The Lancet, Annals of Internal Medicine); reports by governments, NGOs, and health systems; and media reports (Google News) from December 1, 2019 to May 1, 2020. We extracted data on study characteristics and critically appraised prevalence estimates using Joanna Briggs Institute criteria. Results. Seventy records met inclusion criteria, describing 73 studies. Of these, 23 reported prevalence estimates: eight preprints, 14 news articles, and one government report. These studies had a total sample size of 35,784 and reported 42 prevalence estimates. Seroprevalence estimates ranged from 0.4% to 59.3%. No estimates were found to have a low risk of bias (43% high risk, 21% moderate risk, 36% unclear). Fifty records reported characteristics of ongoing or proposed serosurveys. Overall, twenty countries have completed, ongoing, or proposed serosurveys. Discussion. Study design, quality, and prevalence estimates of early SARS-CoV2 serosurveys are heterogeneous, suggesting that the urgency to examine seroprevalence may have compromised methodological rigour. Based on the limitations of included studies, future serosurvey investigators and stakeholders should ensure that: i) serological tests used undergo high-quality independent evaluations that include cross-reactivity; ii) all reports of serosurvey results, including media, describe the test used, sample size, and sampling method; and iii) initiatives are coordinated to prevent test fatigue, minimize redundant efforts, and encourage better study methodology. Other. PROSPERO: CRD42020183634. No third-party funding.',\n",
       " 'Background. As the world grapples with the COVID-19 pandemic, there is increasing global interest in the role of serological testing for population monitoring and to inform public policy. However, limitations in serological study designs and test standards raise concerns about the validity of seroprevalence estimates and their utility in decision-making. There is now a critical window of opportunity to learn from early SARS-CoV-2 serology studies. We aimed to synthesize the results of SARS-CoV-2 serosurveillance projects from around the world and provide recommendations to improve the coordination, strategy, and methodology of future serosurveillance efforts. Methods. This was a rapid systematic review of cross-sectional and cohort studies reporting seroprevalence outcomes for SARS-CoV 2. We included completed, ongoing, and proposed serosurveys. The search included electronic databases (PubMed, MedRXIV, BioRXIV, and WHO ICTPR); five medical journals (NEJM, BMJ, JAMA, The Lancet, Annals of Internal Medicine); reports by governments, NGOs, and health systems; and media reports (Google News) from December 1, 2019 to May 1, 2020. We extracted data on study characteristics and critically appraised prevalence estimates using Joanna Briggs Institute criteria. Results. Seventy records met inclusion criteria, describing 73 studies. Of these, 23 reported prevalence estimates: eight preprints, 14 news articles, and one government report. These studies had a total sample size of 35,784 and reported 42 prevalence estimates. Seroprevalence estimates ranged from 0.4% to 59.3%. No estimates were found to have a low risk of bias (43% high risk, 21% moderate risk, 36% unclear). Fifty records reported characteristics of ongoing or proposed serosurveys. Overall, twenty countries have completed, ongoing, or proposed serosurveys. Discussion. Study design, quality, and prevalence estimates of early SARS-CoV2 serosurveys are heterogeneous, suggesting that the urgency to examine seroprevalence may have compromised methodological rigour. Based on the limitations of included studies, future serosurvey investigators and stakeholders should ensure that: i) serological tests used undergo high-quality independent evaluations that include cross-reactivity; ii) all reports of serosurvey results, including media, describe the test used, sample size, and sampling method; and iii) initiatives are coordinated to prevent test fatigue, minimize redundant efforts, and encourage better study methodology. Other. PROSPERO: CRD42020183634. No third-party funding.',\n",
       " 'Background. As the world grapples with the COVID-19 pandemic, there is increasing global interest in the role of serological testing for population monitoring and to inform public policy. However, limitations in serological study designs and test standards raise concerns about the validity of seroprevalence estimates and their utility in decision-making. There is now a critical window of opportunity to learn from early SARS-CoV-2 serology studies. We aimed to synthesize the results of SARS-CoV-2 serosurveillance projects from around the world and provide recommendations to improve the coordination, strategy, and methodology of future serosurveillance efforts. Methods. This was a rapid systematic review of cross-sectional and cohort studies reporting seroprevalence outcomes for SARS-CoV 2. We included completed, ongoing, and proposed serosurveys. The search included electronic databases (PubMed, MedRXIV, BioRXIV, and WHO ICTPR); five medical journals (NEJM, BMJ, JAMA, The Lancet, Annals of Internal Medicine); reports by governments, NGOs, and health systems; and media reports (Google News) from December 1, 2019 to May 1, 2020. We extracted data on study characteristics and critically appraised prevalence estimates using Joanna Briggs Institute criteria. Results. Seventy records met inclusion criteria, describing 73 studies. Of these, 23 reported prevalence estimates: eight preprints, 14 news articles, and one government report. These studies had a total sample size of 35,784 and reported 42 prevalence estimates. Seroprevalence estimates ranged from 0.4% to 59.3%. No estimates were found to have a low risk of bias (43% high risk, 21% moderate risk, 36% unclear). Fifty records reported characteristics of ongoing or proposed serosurveys. Overall, twenty countries have completed, ongoing, or proposed serosurveys. Discussion. Study design, quality, and prevalence estimates of early SARS-CoV2 serosurveys are heterogeneous, suggesting that the urgency to examine seroprevalence may have compromised methodological rigour. Based on the limitations of included studies, future serosurvey investigators and stakeholders should ensure that: i) serological tests used undergo high-quality independent evaluations that include cross-reactivity; ii) all reports of serosurvey results, including media, describe the test used, sample size, and sampling method; and iii) initiatives are coordinated to prevent test fatigue, minimize redundant efforts, and encourage better study methodology. Other. PROSPERO: CRD42020183634. No third-party funding.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2; previously provisionally named 2019 novel coronavirus or 2019-nCoV) disease (COVID-19) in China at the end of 2019 has caused a large global outbreak and is a major public health issue. As of 11 February 2020, data from the World Health Organization (WHO) have shown that more than 43 000 confirmed cases have been identified in 28 countries/regions, with >99% of cases being detected in China. On 30 January 2020, the WHO declared COVID-19 as the sixth public health emergency of international concern. SARS-CoV-2 is closely related to two bat-derived severe acute respiratory syndrome-like coronaviruses, bat-SL-CoVZC45 and bat-SL-CoVZXC21. It is spread by human-to-human transmission via droplets or direct contact, and infection has been estimated to have mean incubation period of 6.4 days and a basic reproduction number of 2.24–3.58. Among patients with pneumonia caused by SARS-CoV-2 (novel coronavirus pneumonia or Wuhan pneumonia), fever was the most common symptom, followed by cough. Bilateral lung involvement with ground-glass opacity was the most common finding from computed tomography images of the chest. The one case of SARS-CoV-2 pneumonia in the USA is responding well to remdesivir, which is now undergoing a clinical trial in China. Currently, controlling infection to prevent the spread of SARS-CoV-2 is the primary intervention being used. However, public health authorities should keep monitoring the situation closely, as the more we can learn about this novel virus and its associated outbreak, the better we can respond.',\n",
       " 'Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive and GenBank repositories. Up until 27 March 2020, we downloaded 50 illumina datasets, mostly from China, USA (WA State) and Australia (VIC). A total of 30 datasets (60%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA next-generation sequencing samples. Sequencing samples from North America in GenBank (22 April 2020) present this signature with up to 39% allele frequencies among samples (n = 1,359). Australian variant signatures were more diverse than USA samples, but still, clonal events were found in these samples. Mutations in the helicase, encoded by the ORF1ab gene in SARS-CoV-2 were predominant, among others, suggesting that these regions are actively evolving. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.',\n",
       " 'Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive and GenBank repositories. Up until 27 March 2020, we downloaded 50 illumina datasets, mostly from China, USA (WA State) and Australia (VIC). A total of 30 datasets (60%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA next-generation sequencing samples. Sequencing samples from North America in GenBank (22 April 2020) present this signature with up to 39% allele frequencies among samples (n = 1,359). Australian variant signatures were more diverse than USA samples, but still, clonal events were found in these samples. Mutations in the helicase, encoded by the ORF1ab gene in SARS-CoV-2 were predominant, among others, suggesting that these regions are actively evolving. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.',\n",
       " 'Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive and GenBank repositories. Up until 27 March 2020, we downloaded 50 illumina datasets, mostly from China, USA (WA State) and Australia (VIC). A total of 30 datasets (60%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA next-generation sequencing samples. Sequencing samples from North America in GenBank (22 April 2020) present this signature with up to 39% allele frequencies among samples (n = 1,359). Australian variant signatures were more diverse than USA samples, but still, clonal events were found in these samples. Mutations in the helicase, encoded by the ORF1ab gene in SARS-CoV-2 were predominant, among others, suggesting that these regions are actively evolving. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.',\n",
       " 'Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive and GenBank repositories. Up until 27 March 2020, we downloaded 50 illumina datasets, mostly from China, USA (WA State) and Australia (VIC). A total of 30 datasets (60%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA next-generation sequencing samples. Sequencing samples from North America in GenBank (22 April 2020) present this signature with up to 39% allele frequencies among samples (n = 1,359). Australian variant signatures were more diverse than USA samples, but still, clonal events were found in these samples. Mutations in the helicase, encoded by the ORF1ab gene in SARS-CoV-2 were predominant, among others, suggesting that these regions are actively evolving. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.',\n",
       " 'Here we aim to describe early mutational events across samples from publicly available SARS-CoV-2 sequences from the sequence read archive and GenBank repositories. Up until 27 March 2020, we downloaded 50 illumina datasets, mostly from China, USA (WA State) and Australia (VIC). A total of 30 datasets (60%) contain at least a single founder mutation and most of the variants are missense (over 63%). Five-point mutations with clonal (founder) effect were found in USA next-generation sequencing samples. Sequencing samples from North America in GenBank (22 April 2020) present this signature with up to 39% allele frequencies among samples (n = 1,359). Australian variant signatures were more diverse than USA samples, but still, clonal events were found in these samples. Mutations in the helicase, encoded by the ORF1ab gene in SARS-CoV-2 were predominant, among others, suggesting that these regions are actively evolving. Finally, we firmly urge that primer sets for diagnosis be carefully designed, since rapidly occurring variants would affect the performance of the reverse transcribed quantitative PCR (RT-qPCR) based viral testing.',\n",
       " 'The new coronavirus covid-19 has spread very quickly worldwide, leading the World Health Organization (WHO) to declare a state of pandemic. One of the questions many policy makers, and governments are asking themselves is how the spread is going to evolve in time. In this study, I applied an adapted SIR model previously used in South Korea to model the MERS outbreak, which is also caused by a coronavirus, to estimate the evolution of the curve of active cases in the case of the Portuguese situation, using Italian dara. I then construct five different scenarios for the evolution of covid-19 in Portugal. In the out of control scenario, the number of active cases could reach as much as ~40,000 people on the 5th of April (out-of-control scenario). If the self-protective and control are taken in the same level as what was considered for the South Korean model, this number could have be reduced to about 800 cases (scenario 1). Considering that this scenario is now unrealistic, three other scenarios were devised. In all these scenarios, the government measures had a 50\\\\% effectiveness when compared to the measures in Korea. But in scenario 2 the transmission rate $\\\\beta$ was effectively reduced to 50\\\\%, In this scenario active cases could reach circa 7,000 people. In scenario 3, the transmission rate $\\\\beta$ was reduced to 70\\\\% of its initial value, in which the number of cases would reach a peak of ~11,000 people. And finally in scenario 4, $\\\\beta$ was reduced to 80\\\\%. In this scenario, the peak would be reached at about ~13,000 cases. This study shows the importance of control and self-protecting measure to bring down the number of affected people by following the recommendations of the WHO and health authorities. With the appropriate measures, this number can be brought down to ~7,000-13,000 people. Hopefully that will be the case not just in Portugal, but in the rest of the World.',\n",
       " 'The new coronavirus covid-19 has spread very quickly worldwide, leading the World Health Organization (WHO) to declare a state of pandemic. One of the questions many policy makers, and governments are asking themselves is how the spread is going to evolve in time. In this study, I applied an adapted SIR model previously used in South Korea to model the MERS outbreak, which is also caused by a coronavirus, to estimate the evolution of the curve of active cases in the case of the Portuguese situation, using Italian dara. I then construct five different scenarios for the evolution of covid-19 in Portugal. In the out of control scenario, the number of active cases could reach as much as ~40,000 people on the 5th of April (out-of-control scenario). If the self-protective and control are taken in the same level as what was considered for the South Korean model, this number could have be reduced to about 800 cases (scenario 1). Considering that this scenario is now unrealistic, three other scenarios were devised. In all these scenarios, the government measures had a 50\\\\% effectiveness when compared to the measures in Korea. But in scenario 2 the transmission rate $\\\\beta$ was effectively reduced to 50\\\\%, In this scenario active cases could reach circa 7,000 people. In scenario 3, the transmission rate $\\\\beta$ was reduced to 70\\\\% of its initial value, in which the number of cases would reach a peak of ~11,000 people. And finally in scenario 4, $\\\\beta$ was reduced to 80\\\\%. In this scenario, the peak would be reached at about ~13,000 cases. This study shows the importance of control and self-protecting measure to bring down the number of affected people by following the recommendations of the WHO and health authorities. With the appropriate measures, this number can be brought down to ~7,000-13,000 people. Hopefully that will be the case not just in Portugal, but in the rest of the World.',\n",
       " 'A number of virological, epidemiological and ethnographic arguments suggest that COVID-19 has a zoonotic origin. The pangolin, a species threatened with extinction due to poaching for both culinary purposes and traditional Chinese pharmacopoeia, is now suspected of being the “missing link” in the transmission to humans of a virus that probably originated in a species of bat. Our predation of wild fauna and the reduction in their habitats have thus ended up creating new interfaces that favour the transmission of pathogens (mainly viruses) to humans. Domesticated animals and wild fauna thus constitute a reservoir for almost 80% of emerging human diseases (SARS-CoV, MERS-CoV, Ebola). These diseases are all zoonotic in origin. As if out of a Chinese fairy tale, the bat and the pangolin have taught us a lesson: within an increasingly interdependent world, environmental crises will become ever more intertwined with health crises. Questions relating to public health will no longer be confined to the secrecy of the physician’s consulting room or the sanitised environment of the hospital. They are now being played out in the arena of international trade, ports and airports and distribution networks. Simply put, all human activity creates new interfaces that facilitate the transmission of pathogens from an animal reservoir to humans. This pluri-disciplinary article highlights that environmental changes, such as the reduction in habitats for wild fauna and the intemperate trade in fauna, are the biggest causes of the emergence of new diseases. Against this background, it reviews the different measures taken to control, eradicate and prevent the emergence of animal diseases in a globalised world.',\n",
       " 'The emergence of a novel coronavirus, SARS-CoV-2, resulted in a pandemic. Here, we used X-ray structures of human ACE2 bound to the receptor-binding domain (RBD) of the spike protein (S) from SARS-CoV-2 to predict its binding to ACE2 proteins from different animals, including pets, farm animals, and putative intermediate hosts of SARS-CoV-2. Comparing the interaction sites of ACE2 proteins known to serve or not serve as receptor allows to define residues important for binding. From the 20 amino acids in ACE2 that contact S up to seven can be replaced and ACE2 can still function as the SARS-CoV-2 receptor. These variable amino acids are clustered at certain positions, mostly at the periphery of the binding site, while changes of the invariable residues prevent S-binding or infection of the respective animal. Some ACE2 proteins even tolerate the loss or the acquisition of N-glycosylation sites located near the S-interface. Of note, Pigs and dogs, which are not or not effectively infected and have only a few changes in the binding site, exhibit relatively low levels of ACE2 in the respiratory tract. Comparison of the RBD of S of SARS-CoV-2 with viruses from Bat-CoV-RaTG13 and Pangolin-CoV revealed that the latter contains only one substitution, whereas the Bat-CoV-RaTG13 exhibits five. However, ACE2 of pangolin exhibit seven changes relative to human ACE2, a similar number of substitutions is present in ACE2 of bats, raccoon, and civet suggesting that SARS-CoV-2 may not especially adapted to ACE2 of any of its putative intermediate hosts. These analyses provide new insight into the receptor usage and animal source/origin of SARS-CoV-2.IMPORTANCE SARS-CoV-2 is threatening people worldwide and there are no drugs or vaccines available to mitigate its spread. The origin of the virus is still unclear and whether pets and livestock can be infected and transmit SARS-CoV-2 are important and unknown scientific questions. Effective binding to the host receptor ACE2 is the first prerequisite for infection of cells and determines the host range. Our analysis provides a framework for the prediction of potential hosts of SARS-CoV-2. We found that ACE2 from species known to support SARS-CoV-2 infection tolerate many amino acid changes indicating that the species barrier might be low. An exception are dogs and especially pigs, which, however, revealed relatively low ACE2 expression levels in the respiratory tract. Monitoring of animals is necessary to prevent the generation of a new coronavirus reservoir. Finally, our analysis also showed that SARS-CoV-2 may not be specifically adapted to any of its putative intermediate hosts.',\n",
       " 'The emergence of a novel coronavirus, SARS-CoV-2, resulted in a pandemic. Here, we used X-ray structures of human ACE2 bound to the receptor-binding domain (RBD) of the spike protein (S) from SARS-CoV-2 to predict its binding to ACE2 proteins from different animals, including pets, farm animals, and putative intermediate hosts of SARS-CoV-2. Comparing the interaction sites of ACE2 proteins known to serve or not serve as receptor allows to define residues important for binding. From the 20 amino acids in ACE2 that contact S up to seven can be replaced and ACE2 can still function as the SARS-CoV-2 receptor. These variable amino acids are clustered at certain positions, mostly at the periphery of the binding site, while changes of the invariable residues prevent S-binding or infection of the respective animal. Some ACE2 proteins even tolerate the loss or the acquisition of N-glycosylation sites located near the S-interface. Of note, Pigs and dogs, which are not or not effectively infected and have only a few changes in the binding site, exhibit relatively low levels of ACE2 in the respiratory tract. Comparison of the RBD of S of SARS-CoV-2 with viruses from Bat-CoV-RaTG13 and Pangolin-CoV revealed that the latter contains only one substitution, whereas the Bat-CoV-RaTG13 exhibits five. However, ACE2 of pangolin exhibit seven changes relative to human ACE2, a similar number of substitutions is present in ACE2 of bats, raccoon, and civet suggesting that SARS-CoV-2 may not especially adapted to ACE2 of any of its putative intermediate hosts. These analyses provide new insight into the receptor usage and animal source/origin of SARS-CoV-2.IMPORTANCE SARS-CoV-2 is threatening people worldwide and there are no drugs or vaccines available to mitigate its spread. The origin of the virus is still unclear and whether pets and livestock can be infected and transmit SARS-CoV-2 are important and unknown scientific questions. Effective binding to the host receptor ACE2 is the first prerequisite for infection of cells and determines the host range. Our analysis provides a framework for the prediction of potential hosts of SARS-CoV-2. We found that ACE2 from species known to support SARS-CoV-2 infection tolerate many amino acid changes indicating that the species barrier might be low. An exception are dogs and especially pigs, which, however, revealed relatively low ACE2 expression levels in the respiratory tract. Monitoring of animals is necessary to prevent the generation of a new coronavirus reservoir. Finally, our analysis also showed that SARS-CoV-2 may not be specifically adapted to any of its putative intermediate hosts.',\n",
       " 'The emergence of a novel coronavirus, SARS-CoV-2, resulted in a pandemic. Here, we used X-ray structures of human ACE2 bound to the receptor-binding domain (RBD) of the spike protein (S) from SARS-CoV-2 to predict its binding to ACE2 proteins from different animals, including pets, farm animals, and putative intermediate hosts of SARS-CoV-2. Comparing the interaction sites of ACE2 proteins known to serve or not serve as receptor allows to define residues important for binding. From the 20 amino acids in ACE2 that contact S up to seven can be replaced and ACE2 can still function as the SARS-CoV-2 receptor. These variable amino acids are clustered at certain positions, mostly at the periphery of the binding site, while changes of the invariable residues prevent S-binding or infection of the respective animal. Some ACE2 proteins even tolerate the loss or the acquisition of N-glycosylation sites located near the S-interface. Of note, Pigs and dogs, which are not or not effectively infected and have only a few changes in the binding site, exhibit relatively low levels of ACE2 in the respiratory tract. Comparison of the RBD of S of SARS-CoV-2 with viruses from Bat-CoV-RaTG13 and Pangolin-CoV revealed that the latter contains only one substitution, whereas the Bat-CoV-RaTG13 exhibits five. However, ACE2 of pangolin exhibit seven changes relative to human ACE2, a similar number of substitutions is present in ACE2 of bats, raccoon, and civet suggesting that SARS-CoV-2 may not especially adapted to ACE2 of any of its putative intermediate hosts. These analyses provide new insight into the receptor usage and animal source/origin of SARS-CoV-2.IMPORTANCE SARS-CoV-2 is threatening people worldwide and there are no drugs or vaccines available to mitigate its spread. The origin of the virus is still unclear and whether pets and livestock can be infected and transmit SARS-CoV-2 are important and unknown scientific questions. Effective binding to the host receptor ACE2 is the first prerequisite for infection of cells and determines the host range. Our analysis provides a framework for the prediction of potential hosts of SARS-CoV-2. We found that ACE2 from species known to support SARS-CoV-2 infection tolerate many amino acid changes indicating that the species barrier might be low. An exception are dogs and especially pigs, which, however, revealed relatively low ACE2 expression levels in the respiratory tract. Monitoring of animals is necessary to prevent the generation of a new coronavirus reservoir. Finally, our analysis also showed that SARS-CoV-2 may not be specifically adapted to any of its putative intermediate hosts.',\n",
       " 'Introduction: Recent events highlight how emerging and re-emerging pathogens are becoming global challenges for public health. In December 2019, a novel coronavirus has emerged. This has suddenly turned out into global health concern. Objectives: Aim of this research is to focus on the bibliometric aspects in order to measure what is published in the first 30-days of a global epidemic outbreak Methods: We searched PubMed database in order to find all relevant studies in the first 30-days from the first publication. Results: From the initial 442 identified articles, 234 were read in-extenso. The majority of papers come from China, UK and USA. 63.7% of the papers were commentaries, editorials and reported data and only 17.5% of the sources used data directly collected on the field. Topics mainly addressed were epidemiology, preparedness and generic discussion. NNR showed a reduction for both the objectives assessed from January to February. Conclusions: Diagnosis and effective preventive and therapeutic measures were the fields in which more research is still needed. The vast majority of scientific literature in the first 30-days of an epidemic outbreak is based on reported data rather than primary data. Nevertheless, the scientific statements and public health decisions rely on these data.',\n",
       " 'Introduction: Recent events highlight how emerging and re-emerging pathogens are becoming global challenges for public health. In December 2019, a novel coronavirus has emerged. This has suddenly turned out into global health concern. Objectives: Aim of this research is to focus on the bibliometric aspects in order to measure what is published in the first 30-days of a global epidemic outbreak Methods: We searched PubMed database in order to find all relevant studies in the first 30-days from the first publication. Results: From the initial 442 identified articles, 234 were read in-extenso. The majority of papers come from China, UK and USA. 63.7% of the papers were commentaries, editorials and reported data and only 17.5% of the sources used data directly collected on the field. Topics mainly addressed were epidemiology, preparedness and generic discussion. NNR showed a reduction for both the objectives assessed from January to February. Conclusions: Diagnosis and effective preventive and therapeutic measures were the fields in which more research is still needed. The vast majority of scientific literature in the first 30-days of an epidemic outbreak is based on reported data rather than primary data. Nevertheless, the scientific statements and public health decisions rely on these data.',\n",
       " 'Introduction: Recent events highlight how emerging and re-emerging pathogens are becoming global challenges for public health. In December 2019, a novel coronavirus has emerged. This has suddenly turned out into global health concern. Objectives: Aim of this research is to focus on the bibliometric aspects in order to measure what is published in the first 30-days of a global epidemic outbreak Methods: We searched PubMed database in order to find all relevant studies in the first 30-days from the first publication. Results: From the initial 442 identified articles, 234 were read in-extenso. The majority of papers come from China, UK and USA. 63.7% of the papers were commentaries, editorials and reported data and only 17.5% of the sources used data directly collected on the field. Topics mainly addressed were epidemiology, preparedness and generic discussion. NNR showed a reduction for both the objectives assessed from January to February. Conclusions: Diagnosis and effective preventive and therapeutic measures were the fields in which more research is still needed. The vast majority of scientific literature in the first 30-days of an epidemic outbreak is based on reported data rather than primary data. Nevertheless, the scientific statements and public health decisions rely on these data.',\n",
       " 'Introduction: Recent events highlight how emerging and re-emerging pathogens are becoming global challenges for public health. In December 2019, a novel coronavirus has emerged. This has suddenly turned out into global health concern. Objectives: Aim of this research is to focus on the bibliometric aspects in order to measure what is published in the first 30-days of a global epidemic outbreak Methods: We searched PubMed database in order to find all relevant studies in the first 30-days from the first publication. Results: From the initial 442 identified articles, 234 were read in-extenso. The majority of papers come from China, UK and USA. 63.7% of the papers were commentaries, editorials and reported data and only 17.5% of the sources used data directly collected on the field. Topics mainly addressed were epidemiology, preparedness and generic discussion. NNR showed a reduction for both the objectives assessed from January to February. Conclusions: Diagnosis and effective preventive and therapeutic measures were the fields in which more research is still needed. The vast majority of scientific literature in the first 30-days of an epidemic outbreak is based on reported data rather than primary data. Nevertheless, the scientific statements and public health decisions rely on these data.',\n",
       " 'Introduction: Recent events highlight how emerging and re-emerging pathogens are becoming global challenges for public health. In December 2019, a novel coronavirus has emerged. This has suddenly turned out into global health concern. Objectives: Aim of this research is to focus on the bibliometric aspects in order to measure what is published in the first 30-days of a global epidemic outbreak Methods: We searched PubMed database in order to find all relevant studies in the first 30-days from the first publication. Results: From the initial 442 identified articles, 234 were read in-extenso. The majority of papers come from China, UK and USA. 63.7% of the papers were commentaries, editorials and reported data and only 17.5% of the sources used data directly collected on the field. Topics mainly addressed were epidemiology, preparedness and generic discussion. NNR showed a reduction for both the objectives assessed from January to February. Conclusions: Diagnosis and effective preventive and therapeutic measures were the fields in which more research is still needed. The vast majority of scientific literature in the first 30-days of an epidemic outbreak is based on reported data rather than primary data. Nevertheless, the scientific statements and public health decisions rely on these data.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'The outbreak of COVID-19 started in mid-December 2019 in Wuhan, China. Up to 29 February 2020, SARS-CoV-2 (HCoV-19 / 2019-nCoV) had infected more than 85 000 people in the world. In this study, we used 93 complete genomes of SARS-CoV-2 from the GISAID EpiFlu(TM) database to investigate the evolution and human-to-human transmissions of SARS-CoV-2 in the first two months of the outbreak. We constructed haplotypes of the SARS-CoV-2 genomes, performed phylogenomic analyses and estimated the potential population size changes of the virus. The date of population expansion was calculated based on the expansion parameter tau (τ) using the formula t=τ/2u. A total of 120 substitution sites with 119 codons, including 79 non-synonymous and 40 synonymous substitutions, were found in eight coding-regions in the SARS-CoV-2 genomes. Forty non-synonymous substitutions are potentially associated with virus adaptation. No combinations were detected. The 58 haplotypes (31 found in samples from China and 31 from outside China) were identified in 93 viral genomes under study and could be classified into five groups. By applying the reported bat coronavirus genome (bat-RaTG13-CoV) as the outgroup, we found that haplotypes H13 and H38 might be considered as ancestral haplotypes, and later H1 was derived from the intermediate haplotype H3. The population size of the SARS-CoV-2 was estimated to have undergone a recent expansion on 06 January 2020, and an early expansion on 08 December 2019. Furthermore, phyloepidemiologic approaches have recovered specific directions of human-to-human transmissions and the potential sources for international infected cases.',\n",
       " 'We showed that severe acute respiratory syndrome coronavirus 2 is probably a novel recombinant virus. Its genome is closest to that of severe acute respiratory syndrome–related coronaviruses from horseshoe bats, and its receptor-binding domain is closest to that of pangolin viruses. Its origin and direct ancestral viruses have not been identified.',\n",
       " 'We showed that severe acute respiratory syndrome coronavirus 2 is probably a novel recombinant virus. Its genome is closest to that of severe acute respiratory syndrome–related coronaviruses from horseshoe bats, and its receptor-binding domain is closest to that of pangolin viruses. Its origin and direct ancestral viruses have not been identified.',\n",
       " 'Abstract Coronaviruses are responsible on respiratory diseases in animal and human. The combination of numerical encoding techniques and digital signal processing methods are becoming increasingly important in handling large genomic data. In this paper, we propose to analyze the SARS-CoV-2 genomic signature using the combination of different nucleotide representations and signal processing tools in the aim to identify its genetic origin. The sequence of SARS-CoV-2 was compared with 21 relevant sequences including bat, yak and pangolin coronavirus sequences. In addition, we developed a new algorithm to locate the nucleotide modifications. The results show that the Bat and Pangolin coronaviruses were the most related to SARS-CoV-2 with 96% and 86% of identity all along the genome. Within the S gene sequence, the Pangolin sequence presents local highest nucleotide identity. Those findings suggest genesis of SARS-Cov-2 through evolution from bat and pangolin strains. This study offers new ways to automatically characterize viruses.',\n",
       " 'Abstract Coronaviruses are responsible on respiratory diseases in animal and human. The combination of numerical encoding techniques and digital signal processing methods are becoming increasingly important in handling large genomic data. In this paper, we propose to analyze the SARS-CoV-2 genomic signature using the combination of different nucleotide representations and signal processing tools in the aim to identify its genetic origin. The sequence of SARS-CoV-2 was compared with 21 relevant sequences including bat, yak and pangolin coronavirus sequences. In addition, we developed a new algorithm to locate the nucleotide modifications. The results show that the Bat and Pangolin coronaviruses were the most related to SARS-CoV-2 with 96% and 86% of identity all along the genome. Within the S gene sequence, the Pangolin sequence presents local highest nucleotide identity. Those findings suggest genesis of SARS-Cov-2 through evolution from bat and pangolin strains. This study offers new ways to automatically characterize viruses.',\n",
       " 'Abstract Coronaviruses are responsible on respiratory diseases in animal and human. The combination of numerical encoding techniques and digital signal processing methods are becoming increasingly important in handling large genomic data. In this paper, we propose to analyze the SARS-CoV-2 genomic signature using the combination of different nucleotide representations and signal processing tools in the aim to identify its genetic origin. The sequence of SARS-CoV-2 was compared with 21 relevant sequences including bat, yak and pangolin coronavirus sequences. In addition, we developed a new algorithm to locate the nucleotide modifications. The results show that the Bat and Pangolin coronaviruses were the most related to SARS-CoV-2 with 96% and 86% of identity all along the genome. Within the S gene sequence, the Pangolin sequence presents local highest nucleotide identity. Those findings suggest genesis of SARS-Cov-2 through evolution from bat and pangolin strains. This study offers new ways to automatically characterize viruses.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Abstract This paper concerns study of the genome of the Wuhan Seafood Market isolate believed to represent the causative agent of the disease COVID-19. This is to find a short section or sections of viral protein sequence suitable for preliminary design proposal for a peptide synthetic vaccine and a peptidomimetic therapeutic, and to explore some design possibilities. The project was originally directed towards a use case for the Q-UEL language and its implementation in a knowledge management and automated inference system for medicine called the BioIngine, but focus here remains mostly on the virus itself. However, using Q-UEL systems to access relevant and emerging literature, and to interact with standard publically available bioinformatics tools on the Internet, did help quickly identify sequences of amino acids that are well conserved across many coronaviruses including 2019-nCoV. KRSFIEDLLFNKV was found to be particularly well conserved in this study and corresponds to the region around one of the known cleavage sites of the SARS virus that are believed to be required for virus activation for cell entry. This sequence motif and surrounding variations formed the basis for proposing a specific synthetic vaccine epitope and peptidomimetic agent. The work can, nonetheless, be described in traditional bioinformatics terms, and readily reproduced by others, albeit with the caveat that new data and research into 2019-nCoV is emerging and evolving at an explosive pace. Preliminary studies using molecular modeling and docking, and in that context the potential value of certain known herbal extracts, are also described.',\n",
       " 'Between January 24(th) and March 10(th), a total of 2,370 individuals had contact with the first 30 cases of COVID-19. There were 13 individuals who contracted COVID-19 resulting in a secondary attack rate of 0.55% (95% CI 0.31–0.96). There were 119 household contacts, of which 9 individuals developed COVID-19 resulting in a secondary attack rate of 7.56% (95% CI 3.7–14.26).',\n",
       " '',\n",
       " '',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'On March 11th, 2020 the World Health Organization declared COVID-19 a global pandemic. The infection, transmitted by 2019 novel coronavirus (2019-nCov), was first discovered in December 2019, in Wuhan, Hubei Province, and then rapidly spread worldwide. Italy was early and severely involved, with a critical spread of the infection and a very high number of victims. Person-to-person spread mainly occurs via respiratory droplets and contact. The median incubation period is 5 days. The spectrum of respiratory symptoms may range from mild to severe, strictly depending on the age of the patient and the underlying comorbidities. In children COVID-19 related disease is less frequent and less aggressive. In Italy 1% of positive cases are under 18 years of age, and no deaths have been recorded before 29 years of age. For patients affected by rheumatic disease, despite the concerns related to the imbalance of their immune response and the effect of immunosuppressive treatments, there are still few data to understand the real consequences of this infection. Major scientific societies have issued recommendations to help rheumatologists in caring their patients. Interestingly, some of the drugs mostly used by rheumatologists appear to be promising in critical COVID-19 infected patients, where the hyperinflammation and cytokine storm seem to drive to the multiorgan failure. Pediatric rheumatologists are expected to play a supporting role in this new front of COVID-19 pandemic, both as general pediatricians treating infected children, and as rheumatologists taking care of their rheumatic patients, as well as offering their experience in the possible alternative use of immunomodulatory drugs.',\n",
       " 'OBJECTIVE. The purpose of this article is to review new pediatric lung disorders-including disorders that have occurred in recent years years such as severe acute respiratory syndrome (SARS), swine-origin influenza A (H1N1), Middle East respiratory syndrome (MERS), e-cigarette or vaping product use-associated lung injury (EVALI), and coronavirus disease (COVID-19) pneumonia-to enhance understanding of the characteristic imaging findings. CONCLUSION. Although the clinical symptoms of SARS, H1N1, MERS, EVALI, and COVID-19 pneumonia in pediatric patients may be nonspecific, some characteristic imaging findings have emerged or are currently emerging. It is essential for radiologists to have a clear understanding of the characteristic imaging appearances of these lung disorders in pediatric patients to ensure optimal patient care.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'Abstract The pneumonia caused by novel coronavirus (SARS-CoV-2) in Wuhan, China in December 2019 is a highly contagious disease. The World Health Organization (WHO) has declared the ongoing outbreak as a global public health emergency. Currently, the research on novel coronavirus is still in the primary stage. Based on the current published evidence, we systematically summarizes the epidemiology, clinical characteristics, diagnosis, treatment and prevention of knowledge surrounding COVID-19. This review in the hope of helping the public effectively recognize and deal with the 2019 novel coronavirus (SARS-CoV-2), and providing a reference for future studies.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'The novel coronavirus disease (COVID-19) outbreak, caused by SARS-CoV-2, represents the greatest medical challenge in decades. We provide a comprehensive review of the clinical course of COVID-19, its comorbidities, and mechanistic considerations for future therapies. While COVID-19 primarily affects the lungs, causing interstitial pneumonitis and severe acute respiratory distress syndrome (ARDS), it also affects multiple organs, particularly the cardiovascular system. Risk of severe infection and mortality increase with advancing age and male sex. Mortality is increased by comorbidities: cardiovascular disease, hypertension, diabetes, chronic pulmonary disease, and cancer. The most common complications include arrhythmia (atrial fibrillation, ventricular tachyarrhythmia, and ventricular fibrillation), cardiac injury [elevated highly sensitive troponin I (hs-cTnI) and creatine kinase (CK) levels], fulminant myocarditis, heart failure, pulmonary embolism, and disseminated intravascular coagulation (DIC). Mechanistically, SARS-CoV-2, following proteolytic cleavage of its S protein by a serine protease, binds to the transmembrane angiotensin-converting enzyme 2 (ACE2) —a homologue of ACE—to enter type 2 pneumocytes, macrophages, perivascular pericytes, and cardiomyocytes. This may lead to myocardial dysfunction and damage, endothelial dysfunction, microvascular dysfunction, plaque instability, and myocardial infarction (MI). While ACE2 is essential for viral invasion, there is no evidence that ACE inhibitors or angiotensin receptor blockers (ARBs) worsen prognosis. Hence, patients should not discontinue their use. Moreover, renin–angiotensin–aldosterone system (RAAS) inhibitors might be beneficial in COVID-19. Initial immune and inflammatory responses induce a severe cytokine storm [interleukin (IL)-6, IL-7, IL-22, IL-17, etc.] during the rapid progression phase of COVID-19. Early evaluation and continued monitoring of cardiac damage (cTnI and NT-proBNP) and coagulation (D-dimer) after hospitalization may identify patients with cardiac injury and predict COVID-19 complications. Preventive measures (social distancing and social isolation) also increase cardiovascular risk. Cardiovascular considerations of therapies currently used, including remdesivir, chloroquine, hydroxychloroquine, tocilizumab, ribavirin, interferons, and lopinavir/ritonavir, as well as experimental therapies, such as human recombinant ACE2 (rhACE2), are discussed.',\n",
       " 'Abstract Taiwan experienced a large number of severe acute respiratory syndrome (SARS) viral infections between March and July 2003; by September of that year, 346 SARS cases were confirmed by RT-PCR or serological tests. In order to better understand evolutionary relationships among SARS coronaviruses (SCoVs) from different international regions, we performed phylogenetic comparisons of full-length genomic and protein sequences from 45 human SCoVs (including 12 from Taiwan) and two civet SCoVs. All the Taiwanese SARS-CoV strains which associated with nosocomial infection formed a monophyletic clade within the late phase of the SARS epidemic. This Taiwanese clade could be further divided into two epidemic waves. Taiwan SCoVs in the first wave clustered with three isolates from the Amoy Gardens housing complex in Hong Kong indicating their possible origin. Of the 45 human SCoVs, one isolate from Guangdong province, China, exhibited an extra 29-nucleotide fragment between Orf 10 and Orf 11—similar to the civet SCoV genome. Nucleotide and protein sequence comparisons suggested that all SCoVs of late epidemic came from human-to-human transmission, while certain SCoVs of early epidemic might have originated in animals.',\n",
       " 'Abstract Taiwan experienced a large number of severe acute respiratory syndrome (SARS) viral infections between March and July 2003; by September of that year, 346 SARS cases were confirmed by RT-PCR or serological tests. In order to better understand evolutionary relationships among SARS coronaviruses (SCoVs) from different international regions, we performed phylogenetic comparisons of full-length genomic and protein sequences from 45 human SCoVs (including 12 from Taiwan) and two civet SCoVs. All the Taiwanese SARS-CoV strains which associated with nosocomial infection formed a monophyletic clade within the late phase of the SARS epidemic. This Taiwanese clade could be further divided into two epidemic waves. Taiwan SCoVs in the first wave clustered with three isolates from the Amoy Gardens housing complex in Hong Kong indicating their possible origin. Of the 45 human SCoVs, one isolate from Guangdong province, China, exhibited an extra 29-nucleotide fragment between Orf 10 and Orf 11—similar to the civet SCoV genome. Nucleotide and protein sequence comparisons suggested that all SCoVs of late epidemic came from human-to-human transmission, while certain SCoVs of early epidemic might have originated in animals.',\n",
       " 'We miscalculated identities of several genes, and the corrected Table 1 should be as follows.',\n",
       " 'The outbreak of the COVID-19 pandemic has been accompanied by a bulk of scientific research and related Twitter discussions. To unravel the public concerns about the COVID-19 crisis reflected in the science-based Twitter conversations, this study tracked the Twitter attention around the COVID-19 research efforts during the first three months of 2020. On the basis of nearly 1.4 million Twitter mentions of 6,162 COVID-19-related scientific publications, we investigated the temporal tweeting dynamic and the Twitter users involved in the online discussions around COVID-19-related research. The results show that the quantity of Twitter mentions of COVID-19-related publications was on rising. Scholarly-oriented Twitter users played an influential role in disseminating research outputs on COVID-19, with their tweets being frequently retweeted. Over time, a change in the focus of the Twitter discussions can be observed, from the initial attention to virological and clinical research to more practical topics, such as the potential treatments, the countermeasures by the governments, the healthcare measures, and the influences on the economy and society, in more recent times.',\n",
       " 'The outbreak of the COVID-19 pandemic has been accompanied by a bulk of scientific research and related Twitter discussions. To unravel the public concerns about the COVID-19 crisis reflected in the science-based Twitter conversations, this study tracked the Twitter attention around the COVID-19 research efforts during the first three months of 2020. On the basis of nearly 1.4 million Twitter mentions of 6,162 COVID-19-related scientific publications, we investigated the temporal tweeting dynamic and the Twitter users involved in the online discussions around COVID-19-related research. The results show that the quantity of Twitter mentions of COVID-19-related publications was on rising. Scholarly-oriented Twitter users played an influential role in disseminating research outputs on COVID-19, with their tweets being frequently retweeted. Over time, a change in the focus of the Twitter discussions can be observed, from the initial attention to virological and clinical research to more practical topics, such as the potential treatments, the countermeasures by the governments, the healthcare measures, and the influences on the economy and society, in more recent times.',\n",
       " 'The outbreak of the COVID-19 pandemic has been accompanied by a bulk of scientific research and related Twitter discussions. To unravel the public concerns about the COVID-19 crisis reflected in the science-based Twitter conversations, this study tracked the Twitter attention around the COVID-19 research efforts during the first three months of 2020. On the basis of nearly 1.4 million Twitter mentions of 6,162 COVID-19-related scientific publications, we investigated the temporal tweeting dynamic and the Twitter users involved in the online discussions around COVID-19-related research. The results show that the quantity of Twitter mentions of COVID-19-related publications was on rising. Scholarly-oriented Twitter users played an influential role in disseminating research outputs on COVID-19, with their tweets being frequently retweeted. Over time, a change in the focus of the Twitter discussions can be observed, from the initial attention to virological and clinical research to more practical topics, such as the potential treatments, the countermeasures by the governments, the healthcare measures, and the influences on the economy and society, in more recent times.',\n",
       " \"The origin and evolution of polyploids have been studied extensively in angiosperms and ferns but very rarely in gymnosperms. With the exception of three species of conifers, all natural polyploid species of gymnosperms belong to Ephedra, in which more than half of the species show polyploid cytotypes. Here, we investigated the origin and evolution of polyploids of Ephedra distributed in the Qinghai–Tibetan Plateau (QTP) and neighbouring areas. Flow cytometry (FCM) was used to measure the ploidy levels of the sampled species that are represented by multiple individuals from different populations, and then, two single‐copy nuclear genes (LFY and DDB2) and two chloroplast DNA fragments were used to unravel the possible origins and maternal donors of the polyploids. The results indicate that the studied polyploid species are allopolyploids, and suggest that allotetraploidy is a dominant mode of speciation in Ephedra. The high percentage of polyploids in the genus could be related to some of its biological attributes such as vegetative propagation, a relatively high rate of unreduced gamete formation, and a small genome size relative to most other gymnosperms. Significant ecological divergences between allotetraploids and their putative progenitors were detected by PCAs and anova and Tukey's tests, with the exception of E. saxatilis. The overlap of geographical distributions and ecological niches of some diploid species could have provided opportunities for interspecific hybridization and allopolyploid speciation.\",\n",
       " \"The origin and evolution of polyploids have been studied extensively in angiosperms and ferns but very rarely in gymnosperms. With the exception of three species of conifers, all natural polyploid species of gymnosperms belong to Ephedra, in which more than half of the species show polyploid cytotypes. Here, we investigated the origin and evolution of polyploids of Ephedra distributed in the Qinghai–Tibetan Plateau (QTP) and neighbouring areas. Flow cytometry (FCM) was used to measure the ploidy levels of the sampled species that are represented by multiple individuals from different populations, and then, two single‐copy nuclear genes (LFY and DDB2) and two chloroplast DNA fragments were used to unravel the possible origins and maternal donors of the polyploids. The results indicate that the studied polyploid species are allopolyploids, and suggest that allotetraploidy is a dominant mode of speciation in Ephedra. The high percentage of polyploids in the genus could be related to some of its biological attributes such as vegetative propagation, a relatively high rate of unreduced gamete formation, and a small genome size relative to most other gymnosperms. Significant ecological divergences between allotetraploids and their putative progenitors were detected by PCAs and anova and Tukey's tests, with the exception of E. saxatilis. The overlap of geographical distributions and ecological niches of some diploid species could have provided opportunities for interspecific hybridization and allopolyploid speciation.\",\n",
       " 'A novel type of coronavirus (2019-nCoV) infecting humans appeared in Wuhan, China, at the end of December 2019. Since the identification of the outbreak the infection quickly spread involving in one month more than 31,000 confirmed cases with 638 death. Molecular analysis suggest that 2019-nCoV could be originated from bats after passaging in intermediate hosts, highlighting the high zoonotic potential of coronaviruses.',\n",
       " 'A novel type of coronavirus (2019-nCoV) infecting humans appeared in Wuhan, China, at the end of December 2019. Since the identification of the outbreak the infection quickly spread involving in one month more than 31,000 confirmed cases with 638 death. Molecular analysis suggest that 2019-nCoV could be originated from bats after passaging in intermediate hosts, highlighting the high zoonotic potential of coronaviruses.',\n",
       " 'A novel type of coronavirus (2019-nCoV) infecting humans appeared in Wuhan, China, at the end of December 2019. Since the identification of the outbreak the infection quickly spread involving in one month more than 31,000 confirmed cases with 638 death. Molecular analysis suggest that 2019-nCoV could be originated from bats after passaging in intermediate hosts, highlighting the high zoonotic potential of coronaviruses.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " 'The ongoing episode of coronavirus disease 19 (COVID-19) has imposed a serious threat to global health and the world economy. The disease has rapidly acquired a pandemic status affecting almost all populated areas of the planet. The causative agent of COVID-19 is a novel coronavirus known as SARS-CoV-2. The virus has an approximate 30 kb single-stranded positive-sense RNA genome, which is 74.5% to 99% identical to that of SARS-CoV, CoV-pangolin, and the coronavirus the from horseshoe bat. According to available information, SARS-CoV-2 is inferred to be a recombinant virus that originated from bats and was transmitted to humans, possibly using the pangolin as the intermediate host. The interaction of the SARS-CoV-2 spike protein with the human ACE2 (angiotensin-converting enzyme 2) receptor, and its subsequent cleavage by serine protease and fusion, are the main events in the pathophysiology. The serine protease inhibitors, spike protein-based vaccines, or ACE2 blockers may have therapeutic potential in the near future. At present, no vaccine is available against COVID-19. The disease is being treated with antiviral, antimalarial, anti-inflammatory, herbal medicines, and active plasma antibodies. In this context, the present review article provides a cumulative account of the recent information regarding the viral characteristics, potential therapeutic targets, treatment options, and prospective research questions.',\n",
       " \"Abstract Background and aim As a result of its rapid spread in various countries around the world, on March 11, 2020, WHO issued an announcement of the change in coronavirus disease 2019 status from epidemic to pandemic disease. The virus that causes this disease is indicated originating from animals traded in a live animal market in Wuhan, China. Severe Acute Respiratory Syndrome Coronavirus 2 can attack lung cells because there are many conserved receptor entries, namely Angiotensin Converting Enzyme-2. The presence of this virus in host cells will initiate various protective responses leading to pneumonia and Acute Respiratory Distress Syndrome. This review aimed to provide an overview related to this virus and examine the body's responses and possible therapies. Method We searched PubMed databases for Severe Acute Respiratory Syndrome Coronavirus-2, Middle East respiratory syndrome-related coronavirus and Severe Acute Respiratory Syndrome Coronavirus. Full texts were retrieved, analyzed and developed into an easy-to-understand review. Results We provide a complete review related to structure, origin, and how the body responds to this virus infection and explain the possibility of an immune system over-reaction or cytokine storm. We also include an explanation of how this virus creates modes of avoidance to evade immune system attacks. We further explain the therapeutic approaches that can be taken in the treatment and prevention of this viral infection. Conclusion In summary, based on the structural and immune-evasion system of coronavirus, we suggest several approaches to treat the disease.\",\n",
       " \"Abstract Background and aim As a result of its rapid spread in various countries around the world, on March 11, 2020, WHO issued an announcement of the change in coronavirus disease 2019 status from epidemic to pandemic disease. The virus that causes this disease is indicated originating from animals traded in a live animal market in Wuhan, China. Severe Acute Respiratory Syndrome Coronavirus 2 can attack lung cells because there are many conserved receptor entries, namely Angiotensin Converting Enzyme-2. The presence of this virus in host cells will initiate various protective responses leading to pneumonia and Acute Respiratory Distress Syndrome. This review aimed to provide an overview related to this virus and examine the body's responses and possible therapies. Method We searched PubMed databases for Severe Acute Respiratory Syndrome Coronavirus-2, Middle East respiratory syndrome-related coronavirus and Severe Acute Respiratory Syndrome Coronavirus. Full texts were retrieved, analyzed and developed into an easy-to-understand review. Results We provide a complete review related to structure, origin, and how the body responds to this virus infection and explain the possibility of an immune system over-reaction or cytokine storm. We also include an explanation of how this virus creates modes of avoidance to evade immune system attacks. We further explain the therapeutic approaches that can be taken in the treatment and prevention of this viral infection. Conclusion In summary, based on the structural and immune-evasion system of coronavirus, we suggest several approaches to treat the disease.\",\n",
       " \"Abstract Background and aim As a result of its rapid spread in various countries around the world, on March 11, 2020, WHO issued an announcement of the change in coronavirus disease 2019 status from epidemic to pandemic disease. The virus that causes this disease is indicated originating from animals traded in a live animal market in Wuhan, China. Severe Acute Respiratory Syndrome Coronavirus 2 can attack lung cells because there are many conserved receptor entries, namely Angiotensin Converting Enzyme-2. The presence of this virus in host cells will initiate various protective responses leading to pneumonia and Acute Respiratory Distress Syndrome. This review aimed to provide an overview related to this virus and examine the body's responses and possible therapies. Method We searched PubMed databases for Severe Acute Respiratory Syndrome Coronavirus-2, Middle East respiratory syndrome-related coronavirus and Severe Acute Respiratory Syndrome Coronavirus. Full texts were retrieved, analyzed and developed into an easy-to-understand review. Results We provide a complete review related to structure, origin, and how the body responds to this virus infection and explain the possibility of an immune system over-reaction or cytokine storm. We also include an explanation of how this virus creates modes of avoidance to evade immune system attacks. We further explain the therapeutic approaches that can be taken in the treatment and prevention of this viral infection. Conclusion In summary, based on the structural and immune-evasion system of coronavirus, we suggest several approaches to treat the disease.\",\n",
       " \"Abstract Background and aim As a result of its rapid spread in various countries around the world, on March 11, 2020, WHO issued an announcement of the change in coronavirus disease 2019 status from epidemic to pandemic disease. The virus that causes this disease is indicated originating from animals traded in a live animal market in Wuhan, China. Severe Acute Respiratory Syndrome Coronavirus 2 can attack lung cells because there are many conserved receptor entries, namely Angiotensin Converting Enzyme-2. The presence of this virus in host cells will initiate various protective responses leading to pneumonia and Acute Respiratory Distress Syndrome. This review aimed to provide an overview related to this virus and examine the body's responses and possible therapies. Method We searched PubMed databases for Severe Acute Respiratory Syndrome Coronavirus-2, Middle East respiratory syndrome-related coronavirus and Severe Acute Respiratory Syndrome Coronavirus. Full texts were retrieved, analyzed and developed into an easy-to-understand review. Results We provide a complete review related to structure, origin, and how the body responds to this virus infection and explain the possibility of an immune system over-reaction or cytokine storm. We also include an explanation of how this virus creates modes of avoidance to evade immune system attacks. We further explain the therapeutic approaches that can be taken in the treatment and prevention of this viral infection. Conclusion In summary, based on the structural and immune-evasion system of coronavirus, we suggest several approaches to treat the disease.\",\n",
       " \"Abstract Background and aim As a result of its rapid spread in various countries around the world, on March 11, 2020, WHO issued an announcement of the change in coronavirus disease 2019 status from epidemic to pandemic disease. The virus that causes this disease is indicated originating from animals traded in a live animal market in Wuhan, China. Severe Acute Respiratory Syndrome Coronavirus 2 can attack lung cells because there are many conserved receptor entries, namely Angiotensin Converting Enzyme-2. The presence of this virus in host cells will initiate various protective responses leading to pneumonia and Acute Respiratory Distress Syndrome. This review aimed to provide an overview related to this virus and examine the body's responses and possible therapies. Method We searched PubMed databases for Severe Acute Respiratory Syndrome Coronavirus-2, Middle East respiratory syndrome-related coronavirus and Severe Acute Respiratory Syndrome Coronavirus. Full texts were retrieved, analyzed and developed into an easy-to-understand review. Results We provide a complete review related to structure, origin, and how the body responds to this virus infection and explain the possibility of an immune system over-reaction or cytokine storm. We also include an explanation of how this virus creates modes of avoidance to evade immune system attacks. We further explain the therapeutic approaches that can be taken in the treatment and prevention of this viral infection. Conclusion In summary, based on the structural and immune-evasion system of coronavirus, we suggest several approaches to treat the disease.\",\n",
       " 'At the beginning of the 21st century, a new deadly infectious disease known as severe acute respiratory syndrome (SARS) was recognized as a global public health threat. Subsequently, ten years after the initial SARS cases occurred in 2002, new cases of another atypical respiratory disease caused worldwide concern. This disease became known as Middle East respiratory syndrome (MERS) and was even more lethal than SARS. Currently, history has repeated itself with the emergence of a new Chinese epidemic at the end of 2019. For this respiratory disease, called COVID-19, a novel coronavirus (SARS-CoV-2) was identified as the etiologic agent. In sum, SARS, MERS and COVID-19 are caused by recently discovered coronaviruses that cause flu-like illnesses, but with a clinical outcome that tends to be more severe. As a result of the current importance of coronaviruses in global public health, we conducted a review to summarize and update, above all, the epidemiological historical aspects of the three major diseases in humans caused by coronaviral infection. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1007/s00705-020-04628-0) contains supplementary material, which is available to authorized users.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " 'Background The ongoing epidemics of coronavirus disease 2019 (COVID-19) have caused serious concerns about its potential adverse effects on pregnancy. There are limited data on maternal and neonatal outcomes of pregnant women with COVID-19 pneumonia. Methods We conducted a case-control study to compare clinical characteristics, maternal and neonatal outcomes of pregnant women with and without COVID-19 pneumonia. Results During January 24 to February 29, 2020, there were sixteen pregnant women with confirmed COVID-19 pneumonia and eighteen suspected cases who were admitted to labor in the third trimester. Two had vaginal delivery and the rest took cesarean section. Few patients presented respiratory symptoms (fever and cough) on admission, but most had typical chest CT images of COVID-19 pneumonia. Compared to the controls, COVID-19 pneumonia patients had lower counts of white blood cells (WBC), neutrophils, C-reactive protein (CRP), and alanine aminotransferase (ALT) on admission. Increased levels of WBC, neutrophils, eosinophils, and CRP were found in postpartum blood tests of pneumonia patients. There were three (18.8%) and two (10.5%) of the mothers with confirmed or suspected COVID-19 pneumonia had preterm delivery due to maternal complications, which were significantly higher than the control group. None experienced respiratory failure during hospital stay. COVID-19 infection was not found in the newborns and none developed severe neonatal complications. Conclusion Severe maternal and neonatal complications were not observed in pregnant women with COVID-19 pneumonia who had vaginal delivery or caesarean section. Mild respiratory symptoms of pregnant women with COVID-19 pneumonia highlight the need of effective screening on admission.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS‐CoV‐2) cause china epidemics with high morbidity and mortality, the infection has been transmitted to other countries. About three neonates and more than 230 children cases are reported. The disease condition of the main children was mild. There is currently no evidence that SARS‐CoV‐2 can be transmitted transplacentally from mother to the newborn. The treatment strategy for children with Coronavirus disease (COVID‐19) is based on adult experience. Thus far, no deaths have been reported in the pediatric age group. This review describes the current understanding of COVID‐19 infection in newborns and children.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " '• Intravenous vinorelbine is a standard treatment for advanced NSCLC; • Oral vinorelbine is equally effective as compared to intravenous vinorelbine; • Metronomic oral formulation of vinorelbine is a safe and active treatment for advanced NSCLC; • No established treatment is a “gold standard” during COVID19 era; • Metronomic oral vinorelbine permitted to carry on active treatment during COVID Emergency; • Metronomic oral vinorelbine reduce side effects and hospital visits to lung cancer patients; • Management of lung cancer patients may be improved by metronomic oral vinorelbine.',\n",
       " 'SUMMARYIn recent decades, several new diseases have emerged in different geographical areas, with pathogens including Ebola virus, Zika virus, Nipah virus, and coronaviruses (CoVs). Recently, a new type of viral infection emerged in Wuhan City, China, and initial genomic sequencing data of this virus do not match with previously sequenced CoVs, suggesting a novel CoV strain (2019-nCoV), which has now been termed severe acute respiratory syndrome CoV-2 (SARS-CoV-2). Although coronavirus disease 2019 (COVID-19) is suspected to originate from an animal host (zoonotic origin) followed by human-to-human transmission, the possibility of other routes should not be ruled out. Compared to diseases caused by previously known human CoVs, COVID-19 shows less severe pathogenesis but higher transmission competence, as is evident from the continuously increasing number of confirmed cases globally. Compared to other emerging viruses, such as Ebola virus, avian H7N9, SARS-CoV, and Middle East respiratory syndrome coronavirus (MERS-CoV), SARS-CoV-2 has shown relatively low pathogenicity and moderate transmissibility. Codon usage studies suggest that this novel virus has been transferred from an animal source, such as bats. Early diagnosis by real-time PCR and next-generation sequencing has facilitated the identification of the pathogen at an early stage. Since no antiviral drug or vaccine exists to treat or prevent SARS-CoV-2, potential therapeutic strategies that are currently being evaluated predominantly stem from previous experience with treating SARS-CoV, MERS-CoV, and other emerging viral diseases. In this review, we address epidemiological, diagnostic, clinical, and therapeutic aspects, including perspectives of vaccines and preventive measures that have already been globally recommended to counter this pandemic virus.',\n",
       " 'Abstract Zoonotic disease emergence is not a purely biological process mediated only by ecologic factors; opportunities for transmission of zoonoses from animals to humans also depend on how people interact with animals. While exposure is conditioned by the type of animal and the location in which interactions occur, these in turn are influenced by human activity. The activities people engage in are determined by social as well as contextual factors including gender, age, socio-economic status, occupation, social norms, settlement patterns and livelihood systems, family and community dynamics, as well as national and global influences. This paper proposes an expanded “One Health” conceptual model for human-animal exposure that accounts for social as well as epidemiologic factors. The expanded model informed a new study approach to document the extent of human exposure to animals and explore the interplay of social and environmental factors that influence risk of transmission at the individual and community level. The approach includes a formative phase using qualitative and participatory methods, and a representative, random sample survey to quantify exposure to animals in a variety of settings. The paper discusses the different factors that were considered in developing the approach, including the range of animals asked about and the parameters of exposure that are included, as well as factors to be considered in local adaptation of the generic instruments. Illustrative results from research using this approach in Lao PDR are presented to demonstrate the effect of social factors on how people interact with animals. We believe that the expanded model can be similarly operationalized to explore the interactions of other social and policy-level determinants that may influence transmission of zoonoses.',\n",
       " 'Abstract Zoonotic disease emergence is not a purely biological process mediated only by ecologic factors; opportunities for transmission of zoonoses from animals to humans also depend on how people interact with animals. While exposure is conditioned by the type of animal and the location in which interactions occur, these in turn are influenced by human activity. The activities people engage in are determined by social as well as contextual factors including gender, age, socio-economic status, occupation, social norms, settlement patterns and livelihood systems, family and community dynamics, as well as national and global influences. This paper proposes an expanded “One Health” conceptual model for human-animal exposure that accounts for social as well as epidemiologic factors. The expanded model informed a new study approach to document the extent of human exposure to animals and explore the interplay of social and environmental factors that influence risk of transmission at the individual and community level. The approach includes a formative phase using qualitative and participatory methods, and a representative, random sample survey to quantify exposure to animals in a variety of settings. The paper discusses the different factors that were considered in developing the approach, including the range of animals asked about and the parameters of exposure that are included, as well as factors to be considered in local adaptation of the generic instruments. Illustrative results from research using this approach in Lao PDR are presented to demonstrate the effect of social factors on how people interact with animals. We believe that the expanded model can be similarly operationalized to explore the interactions of other social and policy-level determinants that may influence transmission of zoonoses.',\n",
       " 'Abstract Zoonotic disease emergence is not a purely biological process mediated only by ecologic factors; opportunities for transmission of zoonoses from animals to humans also depend on how people interact with animals. While exposure is conditioned by the type of animal and the location in which interactions occur, these in turn are influenced by human activity. The activities people engage in are determined by social as well as contextual factors including gender, age, socio-economic status, occupation, social norms, settlement patterns and livelihood systems, family and community dynamics, as well as national and global influences. This paper proposes an expanded “One Health” conceptual model for human-animal exposure that accounts for social as well as epidemiologic factors. The expanded model informed a new study approach to document the extent of human exposure to animals and explore the interplay of social and environmental factors that influence risk of transmission at the individual and community level. The approach includes a formative phase using qualitative and participatory methods, and a representative, random sample survey to quantify exposure to animals in a variety of settings. The paper discusses the different factors that were considered in developing the approach, including the range of animals asked about and the parameters of exposure that are included, as well as factors to be considered in local adaptation of the generic instruments. Illustrative results from research using this approach in Lao PDR are presented to demonstrate the effect of social factors on how people interact with animals. We believe that the expanded model can be similarly operationalized to explore the interactions of other social and policy-level determinants that may influence transmission of zoonoses.',\n",
       " 'Since the outbreak of the coronavirus disease 2019 (COVID-19) in December of 2019 in China, the estimation of the pandemic case fatality rate (CFR) has been the focus and interest of many stakeholders. In this manuscript, we prove that the method of using the cumulative CFR is static and does not reflect the trend according to the daily change per unit of time. A proportion meta-analysis was carried out on CFR in every country reporting COVID-19 cases. Based on the results, we performed a meta-analysis for global COVID-19 CFR. Each analysis was performed on two different calculations of CFR: according to calendar date and according to days since the outbreak of the first confirmed case. We thus explored an innovative and original calculation of CFR concurrently based on the date of the first confirmed case as well as on a daily basis. For the first time, we showed that using meta-analyses, according to calendar date and days since the outbreak of the first confirmed case were different. We propose that CFR according to days since the outbreak of the first confirmed case might be a better predictor of the current CFR of COVID-19 and its kinetics. Keywords: COVID-19, case fatality rate, proportion meta-analysis, calendar date, days since first confirmed case.',\n",
       " '',\n",
       " '',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'The coronavirus disease 19 (COVID-19) is a highly transmittable and pathogenic viral infection caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which emerged in Wuhan, China and spread around the world. Genomic analysis revealed that SARS-CoV-2 is phylogenetically related to severe acute respiratory syndrome-like (SARS-like) bat viruses, therefore bats could be the possible primary reservoir. The intermediate source of origin and transfer to humans is not known, however, the rapid human to human transfer has been confirmed widely. There is no clinically approved antiviral drug or vaccine available to be used against COVID-19. However, few broad-spectrum antiviral drugs have been evaluated against COVID-19 in clinical trials, resulted in clinical recovery. In the current review, we summarize and comparatively analyze the emergence and pathogenicity of COVID-19 infection and previous human coronaviruses severe acute respiratory syndrome coronavirus (SARS-CoV) and middle east respiratory syndrome coronavirus (MERS-CoV). We also discuss the approaches for developing effective vaccines and therapeutic combinations to cope with this viral outbreak.',\n",
       " 'Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and represents a potentially fatal disease of great global public health importance. As of March 26, 2020, the outbreak of COVID-19 has resulted in 462,801 confirmed cases and 20,839 deaths globally, which is more than those caused by SARS and Middle East respiratory syndrome (MERS) in 2003 and 2013, respectively. The epidemic has posed considerable challenges worldwide. Under a strict mechanism of massive prevention and control, China has seen a rapid decrease in new cases of coronavirus; however, the global situation remains serious. Additionally, the origin of COVID-19 has not been determined and no specific antiviral treatment or vaccine is currently available. Based on the published data, this review systematically discusses the etiology, epidemiology, clinical characteristics, and current intervention measures related to COVID-19 in the hope that it may provide a reference for future studies and aid in the prevention and control of the COVID-19 epidemic.',\n",
       " 'Coronavirus disease 2019 (COVID-19) is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and represents a potentially fatal disease of great global public health importance. As of March 26, 2020, the outbreak of COVID-19 has resulted in 462,801 confirmed cases and 20,839 deaths globally, which is more than those caused by SARS and Middle East respiratory syndrome (MERS) in 2003 and 2013, respectively. The epidemic has posed considerable challenges worldwide. Under a strict mechanism of massive prevention and control, China has seen a rapid decrease in new cases of coronavirus; however, the global situation remains serious. Additionally, the origin of COVID-19 has not been determined and no specific antiviral treatment or vaccine is currently available. Based on the published data, this review systematically discusses the etiology, epidemiology, clinical characteristics, and current intervention measures related to COVID-19 in the hope that it may provide a reference for future studies and aid in the prevention and control of the COVID-19 epidemic.',\n",
       " '',\n",
       " 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese provinces (in Hubei from December 1, 2019 to February 11, 2020 and in other provinces from January 20, 2020 to Februarys 11, 2020). A Generalized Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China.',\n",
       " 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese provinces (in Hubei from December 1, 2019 to February 11, 2020 and in other provinces from January 20, 2020 to Februarys 11, 2020). A Generalized Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China.',\n",
       " 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese provinces (in Hubei from December 1, 2019 to February 11, 2020 and in other provinces from January 20, 2020 to Februarys 11, 2020). A Generalized Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China.',\n",
       " 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese provinces (in Hubei from December 1, 2019 to February 11, 2020 and in other provinces from January 20, 2020 to Februarys 11, 2020). A Generalized Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China.',\n",
       " 'COVID-19 has become a pandemic. The influence of meteorological factors on the transmission and spread of COVID-19 if of interest. This study sought to examine the associations of daily average temperature (AT) and relative humidity (ARH) with the daily count of COVID-19 cases in 30 Chinese provinces (in Hubei from December 1, 2019 to February 11, 2020 and in other provinces from January 20, 2020 to Februarys 11, 2020). A Generalized Additive Model (GAM) was fitted to quantify the province-specific associations between meteorological variables and the daily cases of COVID-19 during the study periods. In the model, the 14-day exponential moving averages (EMAs) of AT and ARH, and their interaction were included with time trend and health-seeking behavior adjusted. Their spatial distributions were visualized. AT and ARH showed significantly negative associations with COVID-19 with a significant interaction between them (0.04, 95% confidence interval: 0.004-0.07) in Hubei. Every 1°C increase in the AT led to a decrease in the daily confirmed cases by 36% to 57% when ARH was in the range from 67% to 85.5%. Every 1% increase in ARH led to a decrease in the daily confirmed cases by 11% to 22% when AT was in the range from 5.04°C to 8.2°C. However, these associations were not consistent throughout Mainland China.',\n",
       " \"BACKGROUND Since the outbreak of the novel coronavirus disease 2019 (COVID-19) in December 2019, the coronavirus has spread all over the world at an unprecedented rate. The transmissibility of coronavirus from asymptomatic patients to healthy individuals has received enormous attention. An important study using the COVID-19 data from the city Ningbo, China, was carried out to estimate and compare the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. However, in the original analysis, the usual chi-squared tests were unduly used for some contingency tables with small cell counts including zero, which may violate the assumptions for the chi-squared test. OBJECTIVE We reanalyze the data from the city Ningbo with more appropriate statistical methods to draw more reliable and sound conclusions on the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. METHODS We exclude the cases associated with the super-spreader and adopt more appropriate statistical methods, including the permutation test and Fisher's exact test, to reanalyze the COVID-19 data from the city Ningbo. RESULTS After excluding the cases related to the super-spreader, Fisher's exact test yields p-value=.84, which indicates stronger evidence of no difference in the transmission rates compared with the original analysis. The odds ratio of transmission rates of coronavirus between the symptomatic and asymptomatic patients is 1.2 with a 95% confidence interval [0.5, 2.8]. CONCLUSIONS Through a more in-depth and comprehensive statistical analysis of the Ningbo data, we conclude that there is no difference in the transmission rates of coronavirus between the symptomatic and asymptomatic patients.\",\n",
       " \"BACKGROUND Since the outbreak of the novel coronavirus disease 2019 (COVID-19) in December 2019, the coronavirus has spread all over the world at an unprecedented rate. The transmissibility of coronavirus from asymptomatic patients to healthy individuals has received enormous attention. An important study using the COVID-19 data from the city Ningbo, China, was carried out to estimate and compare the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. However, in the original analysis, the usual chi-squared tests were unduly used for some contingency tables with small cell counts including zero, which may violate the assumptions for the chi-squared test. OBJECTIVE We reanalyze the data from the city Ningbo with more appropriate statistical methods to draw more reliable and sound conclusions on the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. METHODS We exclude the cases associated with the super-spreader and adopt more appropriate statistical methods, including the permutation test and Fisher's exact test, to reanalyze the COVID-19 data from the city Ningbo. RESULTS After excluding the cases related to the super-spreader, Fisher's exact test yields p-value=.84, which indicates stronger evidence of no difference in the transmission rates compared with the original analysis. The odds ratio of transmission rates of coronavirus between the symptomatic and asymptomatic patients is 1.2 with a 95% confidence interval [0.5, 2.8]. CONCLUSIONS Through a more in-depth and comprehensive statistical analysis of the Ningbo data, we conclude that there is no difference in the transmission rates of coronavirus between the symptomatic and asymptomatic patients.\",\n",
       " \"BACKGROUND Since the outbreak of the novel coronavirus disease 2019 (COVID-19) in December 2019, the coronavirus has spread all over the world at an unprecedented rate. The transmissibility of coronavirus from asymptomatic patients to healthy individuals has received enormous attention. An important study using the COVID-19 data from the city Ningbo, China, was carried out to estimate and compare the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. However, in the original analysis, the usual chi-squared tests were unduly used for some contingency tables with small cell counts including zero, which may violate the assumptions for the chi-squared test. OBJECTIVE We reanalyze the data from the city Ningbo with more appropriate statistical methods to draw more reliable and sound conclusions on the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. METHODS We exclude the cases associated with the super-spreader and adopt more appropriate statistical methods, including the permutation test and Fisher's exact test, to reanalyze the COVID-19 data from the city Ningbo. RESULTS After excluding the cases related to the super-spreader, Fisher's exact test yields p-value=.84, which indicates stronger evidence of no difference in the transmission rates compared with the original analysis. The odds ratio of transmission rates of coronavirus between the symptomatic and asymptomatic patients is 1.2 with a 95% confidence interval [0.5, 2.8]. CONCLUSIONS Through a more in-depth and comprehensive statistical analysis of the Ningbo data, we conclude that there is no difference in the transmission rates of coronavirus between the symptomatic and asymptomatic patients.\",\n",
       " \"BACKGROUND Since the outbreak of the novel coronavirus disease 2019 (COVID-19) in December 2019, the coronavirus has spread all over the world at an unprecedented rate. The transmissibility of coronavirus from asymptomatic patients to healthy individuals has received enormous attention. An important study using the COVID-19 data from the city Ningbo, China, was carried out to estimate and compare the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. However, in the original analysis, the usual chi-squared tests were unduly used for some contingency tables with small cell counts including zero, which may violate the assumptions for the chi-squared test. OBJECTIVE We reanalyze the data from the city Ningbo with more appropriate statistical methods to draw more reliable and sound conclusions on the transmission rates of the coronavirus by the symptomatic and asymptomatic patients. METHODS We exclude the cases associated with the super-spreader and adopt more appropriate statistical methods, including the permutation test and Fisher's exact test, to reanalyze the COVID-19 data from the city Ningbo. RESULTS After excluding the cases related to the super-spreader, Fisher's exact test yields p-value=.84, which indicates stronger evidence of no difference in the transmission rates compared with the original analysis. The odds ratio of transmission rates of coronavirus between the symptomatic and asymptomatic patients is 1.2 with a 95% confidence interval [0.5, 2.8]. CONCLUSIONS Through a more in-depth and comprehensive statistical analysis of the Ningbo data, we conclude that there is no difference in the transmission rates of coronavirus between the symptomatic and asymptomatic patients.\",\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade’s first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as ‘Public Health Emergency of International Concern’ on January 30(th), 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'In April 2009, a new influenza A (H1N1 2009) virus emerged that rapidly spread around the world. While current variants of this virus have caused widespread disease, particularly in vulnerable groups, there remains the possibility that future variants may cause increased virulence, drug resistance or vaccine escape. Early detection of these virus variants may offer the chance for increased containment and potentially prevention of the virus spread. We have developed and field-tested a resequencing kit that is capable of interrogating all eight segments of the 2009 influenza A(H1N1) virus genome and its variants, with added focus on critical regions such as drug-binding sites, structural components and mutation hotspots. The accompanying base-calling software (EvolSTAR) introduces novel methods that utilize neighbourhood hybridization intensity profiles and substitution bias of probes on the microarray for mutation confirmation and recovery of ambiguous base queries. Our results demonstrate that EvolSTAR is highly accurate and has a much improved call rate. The high throughput and short turn-around time from sample to sequence and analysis results (30 h for 24 samples) makes this kit an efficient large-scale evolutionary biosurveillance tool.',\n",
       " 'The Feline coronavirus (FCoV) can lead to Feline infectious peritonitis (FIP), which the precise cause is still unknown. The theory of internal mutation suggests that a less virulent biotype of FCoV (FECV) would lead to another more pathogenic biotype (FIPV) capable of causing FIP. In this work, the 7b gene was amplified from 51 domestic cat plasma samples by semi-nested PCR and tested through phylogenetic and phylogeographical approaches. The 7b gene of Brazilian isolates displayed high conservation, a strong correlation between the geographic origin of the viral isolates and their genealogy, and its evolution was possibly shaped by a combination of high rates of nucleotide substitution and purifying selection.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'From the beginning of 2002 and 2012, severe respiratory syndrome coronavirus (SARS‐CoV) and Middle East respiratory syndrome coronavirus (MERS‐CoV) crossed the species barriers to infect humans, causing thousands of infections and hundreds of deaths, respectively. Currently, a novel coronavirus (SARS‐CoV‐2), which has become the cause of the outbreak of Coronavirus Disease 2019 (COVID‐19), was discovered. Until 18 February 2020, there were 72 533 confirmed COVID‐19 cases (including 10 644 severe cases) and 1872 deaths in China. SARS‐CoV‐2 is spreading among the public and causing substantial burden due to its human‐to‐human transmission. However, the intermediate host of SARS‐CoV‐2 is still unclear. Finding the possible intermediate host of SARS‐CoV‐2 is imperative to prevent further spread of the epidemic. In this study, we used systematic comparison and analysis to predict the interaction between the receptor‐binding domain (RBD) of coronavirus spike protein and the host receptor, angiotensin‐converting enzyme 2 (ACE2). The interaction between the key amino acids of S protein RBD and ACE2 indicated that, other than pangolins and snakes, as previously suggested, turtles (Chrysemys picta bellii, Chelonia mydas, and Pelodiscus sinensis) may act as the potential intermediate hosts transmitting SARS‐CoV‐2 to humans.',\n",
       " 'Many new and emerging RNA and DNA viruses are zoonotic or have zoonotic origins in an animal reservoir that is usually mammalian and sometimes avian. Not all zoonotic viruses are transmissible (directly or by an arthropod vector) between human hosts. Virus genome sequence data provide the best evidence of transmission. Of human transmissible virus, 37 species have so far been restricted to self-limiting outbreaks. These viruses are priorities for surveillance because relatively minor changes in their epidemiologies can potentially lead to major changes in the threat they pose to public health. On the basis of comparisons across all recognized human viruses, we consider the characteristics of these priority viruses and assess the likelihood that they will further emerge in human populations. We also assess the likelihood that a virus that can infect humans but is not capable of transmission (directly or by a vector) between human hosts can acquire that capability.',\n",
       " 'Middle East respiratory syndrome coronavirus (MERS-CoV) is a betacoronavirus which can cause acute respiratory distress in humans and is associated with a relatively high mortality rate. Since it was first identified in a patient who died in a Jeddah hospital in 2012, the World Health Organization has been notified of 1735 laboratory-confirmed cases from 27 countries, including 628 deaths. Most cases have occurred in Saudi Arabia. MERS-CoVancestors may be found in OldWorld bats of the Vespertilionidae family. After a proposed bat to camel switching event, transmission of MERS-CoV to humans is likely to have been the result of multiple zoonotic transfers from dromedary camels. Human-to-human transmission appears to require close contact with infected persons, with outbreaks mainly occurring in hospital environments. Outbreaks have been associated with inadequate infection prevention and control implementation, resulting in recommendations on basic and more advanced infection prevention and control measures by the World Health Organization, and issuing of government guidelines based on these recommendations in affected countries including Saudi Arabia. Evolutionary changes in the virus, particularly in the viral spike protein which mediates virus-host cell contact may potentially increase transmission of this virus. Efforts are on-going to identify specific evidence-based therapies or vaccines. The broad-spectrum antiviral nitazoxanide has been shown to have in vitro activity against MERS-CoV. Synthetic peptides and candidate vaccines based on regions of the spike protein have shown promise in rodent and non-human primate models. GLS-5300, a prophylactic DNA-plasmid vaccine encoding S protein, is the first MERS-CoV vaccine to be tested in humans, while monoclonal antibody, m336 has given promising results in animal models and has potential for use in outbreak situations.',\n",
       " 'While dromedary camels are the immediate animal source of MERS coronavirus (MERS-CoV) infection, the evolutionary origin of MERS-CoV remains obscure. We analyzed 219 camel and human MERS-CoV genome sequences available in GenBank. Phylogenetic analysis showed that 5 and 214 strains belong to clade A and B, respectively, with clade A further divided into lineage A1 (3 human strains) and lineage A2 (2 camel strains), and clade B divided into B1 to B6 (each containing both human and camel strains). Recombination analysis showed potential recombination events in five strains from dromedaries in Saudi Arabia, with recombination between lineage B5 and B3 in four strains, and between lineage B3 and B4 in one strain. The spike protein showed the highest number of amino acid substitutions, especially between A2 and other lineages, and contained positively selected codons. Notably, codon 1020 was positively selected among B and B5 strains, and can distinguish between clade A (Q1020) and B (R1020/H1020) strains, suggesting that this residue may play a role in the evolution of S protein during divergence of different lineages. The time of the most recent common ancestor of all MERS-CoV was dated to approximately 2010. The implications on the role of camels in the evolution of MERS-CoV are discussed.',\n",
       " 'While dromedary camels are the immediate animal source of MERS coronavirus (MERS-CoV) infection, the evolutionary origin of MERS-CoV remains obscure. We analyzed 219 camel and human MERS-CoV genome sequences available in GenBank. Phylogenetic analysis showed that 5 and 214 strains belong to clade A and B, respectively, with clade A further divided into lineage A1 (3 human strains) and lineage A2 (2 camel strains), and clade B divided into B1 to B6 (each containing both human and camel strains). Recombination analysis showed potential recombination events in five strains from dromedaries in Saudi Arabia, with recombination between lineage B5 and B3 in four strains, and between lineage B3 and B4 in one strain. The spike protein showed the highest number of amino acid substitutions, especially between A2 and other lineages, and contained positively selected codons. Notably, codon 1020 was positively selected among B and B5 strains, and can distinguish between clade A (Q1020) and B (R1020/H1020) strains, suggesting that this residue may play a role in the evolution of S protein during divergence of different lineages. The time of the most recent common ancestor of all MERS-CoV was dated to approximately 2010. The implications on the role of camels in the evolution of MERS-CoV are discussed.',\n",
       " 'Abstract Since the first case of human infection by the Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia in June 2012, more than 2260 cases of confirmed MERS-CoV infection and 803 related deaths have been reported since the 16th of October 2018. The vast majority of these cases (71%) were reported in Saudi Arabia but the epidemic has now spread to 27 countries and has not ceased 6 years later, unlike SARS-CoV that disappeared a little less than 2 years after emerging. Due to the high fatality rate observed in MERS-CoV infected patients (36%), much effort has been put into understanding the origin and pathophysiology of this novel coronavirus to prevent it from becoming endemic in humans. This review focuses in particular on the origin, epidemiology and clinical manifestations of MERS-CoV, as well as the diagnosis and treatment of infected patients. The experience gained over recent years on how to manage the different risks related to this kind of epidemic will be key to being prepared for future outbreaks of communicable disease.',\n",
       " 'Abstract Since the first case of human infection by the Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia in June 2012, more than 2260 cases of confirmed MERS-CoV infection and 803 related deaths have been reported since the 16th of October 2018. The vast majority of these cases (71%) were reported in Saudi Arabia but the epidemic has now spread to 27 countries and has not ceased 6 years later, unlike SARS-CoV that disappeared a little less than 2 years after emerging. Due to the high fatality rate observed in MERS-CoV infected patients (36%), much effort has been put into understanding the origin and pathophysiology of this novel coronavirus to prevent it from becoming endemic in humans. This review focuses in particular on the origin, epidemiology and clinical manifestations of MERS-CoV, as well as the diagnosis and treatment of infected patients. The experience gained over recent years on how to manage the different risks related to this kind of epidemic will be key to being prepared for future outbreaks of communicable disease.',\n",
       " 'Abstract Since the first case of human infection by the Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia in June 2012, more than 2260 cases of confirmed MERS-CoV infection and 803 related deaths have been reported since the 16th of October 2018. The vast majority of these cases (71%) were reported in Saudi Arabia but the epidemic has now spread to 27 countries and has not ceased 6 years later, unlike SARS-CoV that disappeared a little less than 2 years after emerging. Due to the high fatality rate observed in MERS-CoV infected patients (36%), much effort has been put into understanding the origin and pathophysiology of this novel coronavirus to prevent it from becoming endemic in humans. This review focuses in particular on the origin, epidemiology and clinical manifestations of MERS-CoV, as well as the diagnosis and treatment of infected patients. The experience gained over recent years on how to manage the different risks related to this kind of epidemic will be key to being prepared for future outbreaks of communicable disease.',\n",
       " 'Abstract Since the first case of human infection by the Middle East respiratory syndrome coronavirus (MERS-CoV) in Saudi Arabia in June 2012, more than 2260 cases of confirmed MERS-CoV infection and 803 related deaths have been reported since the 16th of October 2018. The vast majority of these cases (71%) were reported in Saudi Arabia but the epidemic has now spread to 27 countries and has not ceased 6 years later, unlike SARS-CoV that disappeared a little less than 2 years after emerging. Due to the high fatality rate observed in MERS-CoV infected patients (36%), much effort has been put into understanding the origin and pathophysiology of this novel coronavirus to prevent it from becoming endemic in humans. This review focuses in particular on the origin, epidemiology and clinical manifestations of MERS-CoV, as well as the diagnosis and treatment of infected patients. The experience gained over recent years on how to manage the different risks related to this kind of epidemic will be key to being prepared for future outbreaks of communicable disease.',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'The novel Coronavirus (2019-nCoV) was identified in Wuhan, Hubei Province, China, in December 2019 and has created a medical emergency worldwide. It has spread rapidly to multiple countries and has been declared a pandemic by the World Health Organization. In India, it is already reported more than 18 thousand cases and more than 600 deaths due to Coronavirus disease 2019 (COVID-19) till April 20, 2020. Previous studies on various viral infections like influenza have supported an epidemiological hypothesis that the cold and dry (low absolute humidity) environments favor the survival and spread of droplet-mediated viral diseases. These viral transmissions found attenuated in warm and humid (high absolute humidity) environments. However, the role of temperature, humidity, and absolute humidity in the transmission of COVID-19 has not yet been well established. Therefore the study to investigate the meteorological condition for incidence and spread of COVID-19 infection, to predict the epidemiology of the infectious disease, and to provide a scientific basis for prevention and control measures against the new disease is required for India. In this work, we analyze the local weather patterns of the Indian region affected by the COVID-19 virus for March and April months, 2020. We have investigated the effect of meteorological parameters like Temperature, relative humidity, and absolute humidity on the rate of spread of COVID-19 using daily confirm cases in India. We have used daily averaged meteorological data for the last three years (2017-2019) for March and April month and the same for the year 2020 for March 1 to April 15. We found a positive association (Pearsons r=0.56) between temperature and daily COVID-19 cases over India. We found a negative association of humidity (RH and AH) with daily COVID-19 Cases (Persons r=-0.62, -0.37). We have also investigated the role of aerosol in spreading the pandemic across India because its possible airborne nature. For this, we have investigated the association of aerosols (AOD) and other pollutions (NO2) with COVID-19 cases during the study period and also during the first lockdown period (25 March-15 April) in India. We found a negative association in March when there were few cases, but in April, it shows positive association when the number of cases is more (for AOD it was r=-0.41 and r=0.28 respectively). During the lockdown period, aerosols (AOD) and other pollutants (NO2; an indicator of PM2.5) reduced sharply with a percentage drop of about 36 and 37, respectively. This reduction may have reduced the risk for COVID-19 through air transmission due to the unavailability of aerosol particles as a base. HYSPLIT forward trajectory model also shows that surface aerosols may travel up to 4 km according to wind and direction within three h of its generation. If coronavirus becomes airborne as suggested by many studies, then it may have a higher risk of transmission by aerosols particles. So relaxing in the lockdown and environmental rules in terms of pollutant emissions from power plants, factories, and other facilities would be a wrong choice and could result in more COVID-19 incidences and deaths in India. Therefore the current study, although limited, suggests that it is doubtful that the spread of COVID-19 would slow down in India due to meteorological factors, like high temperature and high humidity. Because a large number of cases have already been reported in the range of high Tem, high Relative, and high absolute humidity regions of India. Thus our results in no way suggest that COVID-19 would not spread in warm, humid regions or during summer/monsoon. So effective public health interventions should be implemented across India to slow down the transmission of COVID-19. If COVID-19 is indeed sensitive to environmental factors, it could be tested in the coming summer-monsoon for India. So the only summer is not going to help India until monsoon is coming. Only government mitigations strategies would be helpful, whether its lockdown, aggressive and strategic testing, medical facilities, imposing social distancing, encouraging to use face mask or monitoring by a mobile application (Aarogya Setu).',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'SUMMARY The ongoing outbreak of the novel coronavirus pneumonia COVID-19 has caused great number of cases and deaths, but our understanding about the pathogen SARS-CoV-2 remains largely unclear. The attachment of the virus with the cell-surface receptor and a co-factor is the first step for the infection. Here, bioinformatics approaches combining human-virus protein interaction prediction and protein docking based on crystal structures have revealed the high affinity between human dipeptidyl peptidase 4 (DPP4) and the spike (S) receptor-binding domain of SARS-CoV-2. Intriguingly, the crucial binding residues of DPP4 are identical to those as bound to the MERS-CoV-S. Moreover, E484 insertion and adjacent substitutions should be most essential for this DPP4-binding ability acquirement of SARS-CoV-2-S compared with SARS-CoV-S. This potential utilization of DPP4 as a binding target for SARS-CoV-2 may offer novel insight into the viral pathogenesis, and help the surveillance and therapeutics strategy for meeting the challenge of COVID-19.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract According to data compiled by researchers at Johns Hopkins University in Baltimore, Maryland, more than two and half million cases of coronavirus disease 2019 (COVID-19), caused by a newly discovered virus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), have been confirmed on April 20, 2020 (Nature, 2020b). Since the emergence of this infectious disease in Asia (Wuhan, China) late last year, it has been subsequently span to every continent of the world except Antarctica (Rodríguez-Morales et al., 2020). Along with a foothold in every country, the current disease pandemic is disrupting practically every aspect of life all over the world. As the outbreak are continuing to evolve, several research activities have been conducted for better understanding the origin, functions, treatments, and preventions of this novel coronavirus. This review will be a summa of the key features of novel coronavirus (nCoV), the virus causing disease 2019 and the present epidemic situation worldwide up to April 20, 2020. It is expected that this record will play an important role to take more preventive measures for overcoming the challenges faced during this current pandemic.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'Abstract In late December 2019, the world woke to a reality of a pandemic of Coronavirus Disease (COVID-19), elicited by the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), which belongs to a group of β-coronavirus. The potential to cause life-threatening respiratory failure and rapid transmission puts COVID-19 in the list of Public Health Emergency of International Concern (PHEIC). In the last two decades, this is the 3rd deadliest Coronavirus pandemic, following SARS which lasted between 2002 and 2003 and Middle East Respiratory Syndrome (MERS) from 2012 till date. Globally and as of April 1st 2020, COVID-19 has affected 924,688 individuals in over 200 countries, causing 46,368 fatalities. While the SARS-CoV-2 originated in China with over 82,724 confirmed cases and more than 3000 deaths as at the time of writing this review, the rapid transmission of SARS-CoV-2 has resulted in exponential increase in the number of cases outside of China to about 10 times the report case and death in mainland China. SARS-CoV-2 is suspected to be zoonotic in nature as genetic studies have shown sequence similarity to viruses originating from bats. Extreme precautionary measures, such as curfew, shutting of borders and quarantining of individuals suspected to be infected have been instituted with immediate effect; however, due to individuals that are asymptomatic, uncontrolled human-to-human transmission has resulted in exponential infection rate and numerous loss of lives even with this lockdown measures. This review article summarizes the developing situation surrounding the SARS-CoV-2 pandemic with respect to its epidemiology, unique genomic structure, possible origins, transmission, pathogenesis, comparison with other deadly species of Coronaviruses (CoV) and emerging treatment strategies built on informed literature.',\n",
       " 'In late December 2019, some cases of atypical pneumonia, at that time of unknown origin, were reported in Wuhan, China. Days later, the etiologic agent was identified as a new coronavirus. This new coronavirus was called SARS-CoV-2 and the disease it produces was named COVID-19. The origin of this new virus is presumed zoonotic, with bats being its probable vector. Due to the rapid number of infections and deaths that occurred first in China and later around the world, the infection of this virus quickly went from being an isolated outbreak in a Chinese region to becoming a health emergency of international concern and later, a pandemic. The purpose of this review is to study the most relevant and current information on the pathogen, as well as epidemiology, pathology, clinical features, transmission, prevention, and treatment of the disease.',\n",
       " 'As the COVID-19 pandemic continues its march around the world, an unprecedented amount of open data is being generated for genetics and epidemiological research. The unparalleled rate at which many research groups around the world are releasing data and publications on the ongoing pandemic is allowing other scientists to learn from local experiences and data generated in the front lines of the COVID-19 pandemic. However, there is a need to integrate additional data sources that map and measure the role of social dynamics of such a unique world-wide event into biomedical, biological, and epidemiological analyses. For this purpose, we present a large-scale curated dataset of over 152 million tweets, growing daily, related to COVID-19 chatter generated from January 1st to April 4th at the time of writing. This open dataset will allow researchers to conduct a number of research projects relating to the emotional and mental responses to social distancing measures, the identification of sources of misinformation, and the stratified measurement of sentiment towards the pandemic in near real time.',\n",
       " 'As the COVID-19 pandemic continues its march around the world, an unprecedented amount of open data is being generated for genetics and epidemiological research. The unparalleled rate at which many research groups around the world are releasing data and publications on the ongoing pandemic is allowing other scientists to learn from local experiences and data generated in the front lines of the COVID-19 pandemic. However, there is a need to integrate additional data sources that map and measure the role of social dynamics of such a unique world-wide event into biomedical, biological, and epidemiological analyses. For this purpose, we present a large-scale curated dataset of over 152 million tweets, growing daily, related to COVID-19 chatter generated from January 1st to April 4th at the time of writing. This open dataset will allow researchers to conduct a number of research projects relating to the emotional and mental responses to social distancing measures, the identification of sources of misinformation, and the stratified measurement of sentiment towards the pandemic in near real time.',\n",
       " '',\n",
       " \"Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade's first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as 'Public Health Emergency of International Concern' on January 30th, 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.\",\n",
       " \"Coronaviruses are the well-known cause of severe respiratory, enteric and systemic infections in a wide range of hosts including man, mammals, fish, and avian. The scientific interest on coronaviruses increased after the emergence of Severe Acute Respiratory Syndrome coronavirus (SARS-CoV) outbreaks in 2002-2003 followed by Middle East Respiratory Syndrome CoV (MERS-CoV). This decade's first CoV, named 2019-nCoV, emerged from Wuhan, China, and declared as 'Public Health Emergency of International Concern' on January 30th, 2020 by the World Health Organization (WHO). As on February 4, 2020, 425 deaths reported in China only and one death outside China (Philippines). In a short span of time, the virus spread has been noted in 24 countries. The zoonotic transmission (animal-to-human) is suspected as the route of disease origin. The genetic analyses predict bats as the most probable source of 2019-nCoV though further investigations needed to confirm the origin of the novel virus. The ongoing nCoV outbreak highlights the hidden wild animal reservoir of the deadly viruses and possible threat of spillover zoonoses as well. The successful virus isolation attempts have made doors open for developing better diagnostics and effective vaccines helping in combating the spread of the virus to newer areas.\",\n",
       " 'Abstract Objective: We aim to provide the first evidence of belief in conspiracy theory regarding the COVID-19 virus as a predictor of the mental health and well-being of healthcare workers. Methods: We conducted a survey of 252 healthcare workers in Ecuador from April 10 to May 2, 2020. Results: In Ecuador, 32.54% of the sampled healthcare workers experienced distress disorder, and 28.17% had anxiety disorder. Compared to healthcare workers who were not sure where the virus originated, those who believed the virus was developed intentionally in a lab reported higher levels of distress and anxiety, and lower levels of job satisfaction and life satisfaction. Older healthcare workers and those who exercise more reported higher job satisfaction. Married healthcare workers, those who exercise more, and those not infected reported higher life satisfaction. Conclusion: This paper identifies belief in a COVID-19 conspiracy theory as an important predictor of distress, anxiety, and job and life satisfaction of healthcare workers. It enables mental health services to better target and help mentally vulnerable healthcare workers during the ongoing COVID-19 pandemic.',\n",
       " 'Researchers routinely adopt molecular clock assumptions in conducting sequence analyses to estimate dates for viral origins in humans. We used computational methods to examine the extent to which this practice can result in inaccurate ‘retrodiction.’ Failing to account for dynamic molecular evolution can affect greatly estimating index case dates, resulting in an overestimated age for the SARS-CoV-human infection, for instance.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'The rapid spread of SARS-CoV-2, the causative agent of Coronavirus disease 2019 (COVID-19), has been accompanied by the emergence of distinct viral clades, though their clinical significance remains unclear. Here, we examined the genome sequences of 88 SARS-CoV-2 viruses from COVID-19 patients in Chicago, USA and identified three distinct phylogenetic clades. Clade 1 was most closely related to clades centered in New York, and showed evidence of rapid expansion across the USA, while Clade 3 was most closely related to those in Washington. Clade 2 was localized primarily to the Chicago area with limited evidence of expansion elsewhere. Average viral loads in the airways of patients infected with the rapidly spreading Clade 1 viruses were significantly higher than those of the poorly spreading Clade 2. These results show that multiple variants of SARS-CoV-2 are circulating in the USA that differ in their relative airway viral loads and potential for expansion.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'Mutation and adaptation have driven the co-evolution of coronaviruses (CoVs) and their hosts, including human beings, for thousands of years. Before 2003, two human CoVs (HCoVs) were known to cause mild illness, such as common cold. The outbreaks of severe acute respiratory syndrome (SARS) and the Middle East respiratory syndrome (MERS) have flipped the coin to reveal how devastating and life-threatening an HCoV infection could be. The emergence of SARS-CoV-2 in central China at the end of 2019 has thrusted CoVs into the spotlight again and surprised us with its high transmissibility but reduced pathogenicity compared to its sister SARS-CoV. HCoV infection is a zoonosis and understanding the zoonotic origins of HCoVs would serve us well. Most HCoVs originated from bats where they are non-pathogenic. The intermediate reservoir hosts of some HCoVs are also known. Identifying the animal hosts has direct implications in the prevention of human diseases. Investigating CoV-host interactions in animals might also derive important insight on CoV pathogenesis in humans. In this review, we present an overview of the existing knowledge about the seven HCoVs, with a focus on the history of their discovery as well as their zoonotic origins and interspecies transmission. Importantly, we compare and contrast the different HCoVs from a perspective of virus evolution and genome recombination. The current CoV disease 2019 (COVID-19) epidemic is discussed in this context. In addition, the requirements for successful host switches and the implications of virus evolution on disease severity are also highlighted.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'A new porcine coronavirus SADS-CoV was recently identified from suckling piglets with severe diarrhea in southern China and its genome sequence is most identical (~95% identity) to that of bat α-coronavirus HKU2. It again indicates bats are the natural reservoir of many coronaviruses that have great potential for cross-species transmission to animals and humans by recombination and/or mutation. Here we report the cryo-EM structures of HKU2 and SADS-CoV spike glycoprotein trimers at 2.38 Å and 2.83 Å resolution, respectively. HKU2 and SADS-CoV spikes exhibit very high structural similarity, with subtle differences mainly distributed in the NTD and CTD of the S1 subunit responsible for cell attachment and receptor binding. We systematically analyzed and compared the NTD, CTD, SD1 and SD2 domains of the S1 subunit and the S2 subunit of HKU2 spike with those of α-, β-, γ-, and δ-coronavirus spikes. The results show that the NTD and CTD of HKU2/SADS-CoV are probably the most ancestral in the evolution of spike. Although the S2 subunit mediating membrane fusion is highly conserved, the connecting region after fusion peptide in HKU2/SADS-CoV S2 subunit also adopts a conformation distinct from other coronaviruses. These results structurally demonstrate a close evolutionary relationship between HKU2 /SADS-CoV and β-coronavirus spikes and provide new insights into the evolution and cross-species transmission of coronaviruses.',\n",
       " 'The current outbreak of viral pneumonia in the city of Wuhan, China, was caused by a novel coronavirus designated 2019-nCoV by the World Health Organization, as determined by sequencing the viral RNA genome. Many initial patients were exposed to wildlife animals at the Huanan seafood wholesale market, where poultry, snake, bats, and other farm animals were also sold. To investigate possible virus reservoir, we have carried out comprehensive sequence analysis and comparison in conjunction with relative synonymous codon usage (RSCU) bias among different animal species based on the 2019-nCoV sequence. Results obtained from our analyses suggest that the 2019-nCoV may appear to be a recombinant virus between the bat coronavirus and an origin-unknown coronavirus. The recombination may occurred within the viral spike glycoprotein, which recognizes a cell surface receptor. Additionally, our findings suggest that 2019-nCoV has most similar genetic information with bat coronovirus and most similar codon usage bias with snake. Taken together, our results suggest that homologous recombination may occur and contribute to the 2019-nCoV cross-species transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'The 2019 novel coronavirus (2019-nCoV/SARS-CoV-2) originally arose as part of a major outbreak of respiratory disease centered on Hubei province, China. It is now a global pandemic and is a major public health concern. Taxonomically, SARS-CoV-2 was shown to be a Betacoronavirus (lineage B) closely related to SARS-CoV and SARS-related bat coronaviruses, and it has been reported to share a common receptor with SARS-CoV (ACE-2). Subsequently, betacoronaviruses from pangolins were identified as close relatives to SARS-CoV-2. Here, we perform structural modeling of the SARS-CoV-2 spike glycoprotein. Our data provide support for the similar receptor utilization between SARS-CoV-2 and SARS-CoV, despite a relatively low amino acid similarity in the receptor binding module. Compared to SARS-CoV and all other coronaviruses in Betacoronavirus lineage B, we identify an extended structural loop containing basic amino acids at the interface of the receptor binding (S1) and fusion (S2) domains. We suggest this loop confers fusion activation and entry properties more in line with betacoronaviruses in lineages A and C, and be a key component in the evolution of SARS-CoV-2 with this structural loop affecting virus stability and transmission.',\n",
       " 'AIM: The COVID pandemic is caused by infection with the SARS‐CoV‐2 virus. The major mutation detected to date in the SARS‐CoV‐2 viral envelope spike protein, which is responsible for virus attachment to the host and is also the main target for host antibodies, is a mutation of an aspartate (D) at position 614 found frequently in Chinese strains to a glycine (G). We sought to infer health impact of this mutation. RESULT: Increased case fatality rate correlated strongly with the proportion of viruses bearing G614 on a country by country basis. The amino acid at position 614 occurs at an internal protein interface of the viral spike, and the presence of G at this position was calculated to destabilize a specific conformation of the viral spike, within which the key host receptor binding site is more accessible. CONCLUSION: These results imply that G614 is a more pathogenic strain of SARS‐CoV‐2, which may influence vaccine design. The prevalence of this form of the virus should also be included in epidemiologic models predicting the COVID‐19 health burden and fatality over time in specific regions. Physicians should be aware of this characteristic of the virus to anticipate the clinical course of infection.',\n",
       " 'AIM: The COVID pandemic is caused by infection with the SARS‐CoV‐2 virus. The major mutation detected to date in the SARS‐CoV‐2 viral envelope spike protein, which is responsible for virus attachment to the host and is also the main target for host antibodies, is a mutation of an aspartate (D) at position 614 found frequently in Chinese strains to a glycine (G). We sought to infer health impact of this mutation. RESULT: Increased case fatality rate correlated strongly with the proportion of viruses bearing G614 on a country by country basis. The amino acid at position 614 occurs at an internal protein interface of the viral spike, and the presence of G at this position was calculated to destabilize a specific conformation of the viral spike, within which the key host receptor binding site is more accessible. CONCLUSION: These results imply that G614 is a more pathogenic strain of SARS‐CoV‐2, which may influence vaccine design. The prevalence of this form of the virus should also be included in epidemiologic models predicting the COVID‐19 health burden and fatality over time in specific regions. Physicians should be aware of this characteristic of the virus to anticipate the clinical course of infection.',\n",
       " 'AIM: The COVID pandemic is caused by infection with the SARS‐CoV‐2 virus. The major mutation detected to date in the SARS‐CoV‐2 viral envelope spike protein, which is responsible for virus attachment to the host and is also the main target for host antibodies, is a mutation of an aspartate (D) at position 614 found frequently in Chinese strains to a glycine (G). We sought to infer health impact of this mutation. RESULT: Increased case fatality rate correlated strongly with the proportion of viruses bearing G614 on a country by country basis. The amino acid at position 614 occurs at an internal protein interface of the viral spike, and the presence of G at this position was calculated to destabilize a specific conformation of the viral spike, within which the key host receptor binding site is more accessible. CONCLUSION: These results imply that G614 is a more pathogenic strain of SARS‐CoV‐2, which may influence vaccine design. The prevalence of this form of the virus should also be included in epidemiologic models predicting the COVID‐19 health burden and fatality over time in specific regions. Physicians should be aware of this characteristic of the virus to anticipate the clinical course of infection.',\n",
       " 'AIM: The COVID pandemic is caused by infection with the SARS‐CoV‐2 virus. The major mutation detected to date in the SARS‐CoV‐2 viral envelope spike protein, which is responsible for virus attachment to the host and is also the main target for host antibodies, is a mutation of an aspartate (D) at position 614 found frequently in Chinese strains to a glycine (G). We sought to infer health impact of this mutation. RESULT: Increased case fatality rate correlated strongly with the proportion of viruses bearing G614 on a country by country basis. The amino acid at position 614 occurs at an internal protein interface of the viral spike, and the presence of G at this position was calculated to destabilize a specific conformation of the viral spike, within which the key host receptor binding site is more accessible. CONCLUSION: These results imply that G614 is a more pathogenic strain of SARS‐CoV‐2, which may influence vaccine design. The prevalence of this form of the virus should also be included in epidemiologic models predicting the COVID‐19 health burden and fatality over time in specific regions. Physicians should be aware of this characteristic of the virus to anticipate the clinical course of infection.',\n",
       " 'AIM: The COVID pandemic is caused by infection with the SARS‐CoV‐2 virus. The major mutation detected to date in the SARS‐CoV‐2 viral envelope spike protein, which is responsible for virus attachment to the host and is also the main target for host antibodies, is a mutation of an aspartate (D) at position 614 found frequently in Chinese strains to a glycine (G). We sought to infer health impact of this mutation. RESULT: Increased case fatality rate correlated strongly with the proportion of viruses bearing G614 on a country by country basis. The amino acid at position 614 occurs at an internal protein interface of the viral spike, and the presence of G at this position was calculated to destabilize a specific conformation of the viral spike, within which the key host receptor binding site is more accessible. CONCLUSION: These results imply that G614 is a more pathogenic strain of SARS‐CoV‐2, which may influence vaccine design. The prevalence of this form of the virus should also be included in epidemiologic models predicting the COVID‐19 health burden and fatality over time in specific regions. Physicians should be aware of this characteristic of the virus to anticipate the clinical course of infection.',\n",
       " 'Most recent emerging infectious diseases have been zoonotic in origin. It is our contention that one of the factors responsible for such emergence is the trade in wildlife and bushmeat in particular. This article considers the effect of increasing diversity in the species hunted on the probability of global epidemics such as SARS. In particular, we develop a mathematical model of the probability of such an outbreak in terms of the number of species hunted, the number of susceptibles, and the rate of contact. Hence, we postulate that local biodiversity loss and increasing rates of animal trafficking, and trade and transportation of animals to large cities—where there is a greater potential for person-to-person transmission—may increase the probability of such outbreaks dramatically.',\n",
       " 'A global emergency due to the COVID-19 pandemic demands various studies related to genes and genomes of the SARS-CoV2. Among other important proteins, the role of accessory proteins are of immense importance in replication, regulation of infections of the coronavirus in the hosts. The largest accessory proteins in the SARS-CoV2 genome is ORF3a which modulates the host response to the virus infection and consequently it plays an important role in pathogenesis. In this study, an attempt is made to decipher the conservation of nucleotides, dimers, codons and amino acids in the ORF3a genes across thirty two genomes of Indian patients. ORF3a gene possesses single and double point mutations in Indian SARS-CoV2 genomes suggesting the change of SARS-CoV2’s virulence property in Indian patients. We find that the parental origin of the ORF3a gene over the genomes of SARS-CoV2 and Pangolin-CoV is same from the phylogenetic analysis based on conservations of nucleotides and so on. This study highlights the accumulation of mutation on ORF3a in Indian SARS-CoV2 genomes which may provide the designing therapeutic approach against SARS-CoV2.',\n",
       " 'A global emergency due to the COVID-19 pandemic demands various studies related to genes and genomes of the SARS-CoV2. Among other important proteins, the role of accessory proteins are of immense importance in replication, regulation of infections of the coronavirus in the hosts. The largest accessory proteins in the SARS-CoV2 genome is ORF3a which modulates the host response to the virus infection and consequently it plays an important role in pathogenesis. In this study, an attempt is made to decipher the conservation of nucleotides, dimers, codons and amino acids in the ORF3a genes across thirty two genomes of Indian patients. ORF3a gene possesses single and double point mutations in Indian SARS-CoV2 genomes suggesting the change of SARS-CoV2’s virulence property in Indian patients. We find that the parental origin of the ORF3a gene over the genomes of SARS-CoV2 and Pangolin-CoV is same from the phylogenetic analysis based on conservations of nucleotides and so on. This study highlights the accumulation of mutation on ORF3a in Indian SARS-CoV2 genomes which may provide the designing therapeutic approach against SARS-CoV2.',\n",
       " 'A global emergency due to the COVID-19 pandemic demands various studies related to genes and genomes of the SARS-CoV2. Among other important proteins, the role of accessory proteins are of immense importance in replication, regulation of infections of the coronavirus in the hosts. The largest accessory proteins in the SARS-CoV2 genome is ORF3a which modulates the host response to the virus infection and consequently it plays an important role in pathogenesis. In this study, an attempt is made to decipher the conservation of nucleotides, dimers, codons and amino acids in the ORF3a genes across thirty two genomes of Indian patients. ORF3a gene possesses single and double point mutations in Indian SARS-CoV2 genomes suggesting the change of SARS-CoV2’s virulence property in Indian patients. We find that the parental origin of the ORF3a gene over the genomes of SARS-CoV2 and Pangolin-CoV is same from the phylogenetic analysis based on conservations of nucleotides and so on. This study highlights the accumulation of mutation on ORF3a in Indian SARS-CoV2 genomes which may provide the designing therapeutic approach against SARS-CoV2.',\n",
       " 'A global emergency due to the COVID-19 pandemic demands various studies related to genes and genomes of the SARS-CoV2. Among other important proteins, the role of accessory proteins are of immense importance in replication, regulation of infections of the coronavirus in the hosts. The largest accessory proteins in the SARS-CoV2 genome is ORF3a which modulates the host response to the virus infection and consequently it plays an important role in pathogenesis. In this study, an attempt is made to decipher the conservation of nucleotides, dimers, codons and amino acids in the ORF3a genes across thirty two genomes of Indian patients. ORF3a gene possesses single and double point mutations in Indian SARS-CoV2 genomes suggesting the change of SARS-CoV2’s virulence property in Indian patients. We find that the parental origin of the ORF3a gene over the genomes of SARS-CoV2 and Pangolin-CoV is same from the phylogenetic analysis based on conservations of nucleotides and so on. This study highlights the accumulation of mutation on ORF3a in Indian SARS-CoV2 genomes which may provide the designing therapeutic approach against SARS-CoV2.',\n",
       " 'A global emergency due to the COVID-19 pandemic demands various studies related to genes and genomes of the SARS-CoV2. Among other important proteins, the role of accessory proteins are of immense importance in replication, regulation of infections of the coronavirus in the hosts. The largest accessory proteins in the SARS-CoV2 genome is ORF3a which modulates the host response to the virus infection and consequently it plays an important role in pathogenesis. In this study, an attempt is made to decipher the conservation of nucleotides, dimers, codons and amino acids in the ORF3a genes across thirty two genomes of Indian patients. ORF3a gene possesses single and double point mutations in Indian SARS-CoV2 genomes suggesting the change of SARS-CoV2’s virulence property in Indian patients. We find that the parental origin of the ORF3a gene over the genomes of SARS-CoV2 and Pangolin-CoV is same from the phylogenetic analysis based on conservations of nucleotides and so on. This study highlights the accumulation of mutation on ORF3a in Indian SARS-CoV2 genomes which may provide the designing therapeutic approach against SARS-CoV2.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " \"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is rapidly spreading across the world to cause thousands of mortalities each day. Poor responses from the authorities to the spread of infection, lack of effective measures for prevention, unavailability of promising treatment options, and sufficient diagnostic options have created an alarming for the world. The transmission routes from human to human of SARS-CoV-2 can be the direct transmission, droplet inhalation transmission, contact transmission, transmission through saliva, and transmission via fecal–oral routes. Due to the asymptomatic spread of SARS-CoV-2's, developing control and prevention measures is challenging. Implementing proper strategies addressing the infection control and clinical supplies, understanding the mechanism associated with pathogenesis, advancing in preventive measures and effective treatment and diagnostic options are necessary to control the ongoing pandemic. In this article, we briefly discuss the features, entry mechanism, infectiousness, and health consequences related to the COVID-19 outbreak.\",\n",
       " \"Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is rapidly spreading across the world to cause thousands of mortalities each day. Poor responses from the authorities to the spread of infection, lack of effective measures for prevention, unavailability of promising treatment options, and sufficient diagnostic options have created an alarming for the world. The transmission routes from human to human of SARS-CoV-2 can be the direct transmission, droplet inhalation transmission, contact transmission, transmission through saliva, and transmission via fecal–oral routes. Due to the asymptomatic spread of SARS-CoV-2's, developing control and prevention measures is challenging. Implementing proper strategies addressing the infection control and clinical supplies, understanding the mechanism associated with pathogenesis, advancing in preventive measures and effective treatment and diagnostic options are necessary to control the ongoing pandemic. In this article, we briefly discuss the features, entry mechanism, infectiousness, and health consequences related to the COVID-19 outbreak.\",\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Since end of December 2019, a cluster of patients with pneumonia of unknown origin was reported from Wuhan, Hubei province, China. As of Feb 17th, 2020, statistical data show that the outbreak constitutes an epidemic threat in China, where the exponential increase in patients has reached 75114 confirmed cases, with 2239 deaths. Different from SARS-CoV (severe acute respiratory syndrome coronavirus) and MERS-CoV (Middle East respiratory syndrome coronavirus) infection, the initial presentations or the chief complain of some patients with the 2019 novel coronavirus (COVID-19) were gastrointestinal symptoms. So we call upon all the first-line medical staff to be cautious and pay more attention to those untypical patients especially from the epidemic area. Besides, as the viral nucleic acids could be found in the fecal samples and anal swabs of some patients with COVID-19 infection, the possibility of fecal-oral transmission need to be took into account. Based on the previously and recently studies, we speculate that COVID-19 may have some relationship with the gut microbiota through angiotensin-converting enzyme 2 (ACE2) receptor, thus targeting gut microbiota might be a new therapeutic option for the treatment of virus-related pneumonia. This article is protected by copyright. All rights reserved.',\n",
       " 'Since end of December 2019, a cluster of patients with pneumonia of unknown origin was reported from Wuhan, Hubei province, China. As of Feb 17th, 2020, statistical data show that the outbreak constitutes an epidemic threat in China, where the exponential increase in patients has reached 75114 confirmed cases, with 2239 deaths. Different from SARS-CoV (severe acute respiratory syndrome coronavirus) and MERS-CoV (Middle East respiratory syndrome coronavirus) infection, the initial presentations or the chief complain of some patients with the 2019 novel coronavirus (COVID-19) were gastrointestinal symptoms. So we call upon all the first-line medical staff to be cautious and pay more attention to those untypical patients especially from the epidemic area. Besides, as the viral nucleic acids could be found in the fecal samples and anal swabs of some patients with COVID-19 infection, the possibility of fecal-oral transmission need to be took into account. Based on the previously and recently studies, we speculate that COVID-19 may have some relationship with the gut microbiota through angiotensin-converting enzyme 2 (ACE2) receptor, thus targeting gut microbiota might be a new therapeutic option for the treatment of virus-related pneumonia. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'A novel coronavirus, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), emerged in Wuhan, Hubei Province in China in December 2019 and caused a serious type of pneumonia called coronavirus disease 2019 or COVID-19. This epidemic quickly spread across China and extended to more than 20 other countries. This commentary discusses the reasons for the fast spread of SARS-CoV-2 in three aspects: the infectious sources, including the biological nature of the virus; the susceptible population; and the transmission routes. The current situations and suggestions regarding the control of the disease are summarized. This article is protected by copyright. All rights reserved.',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) emerges in Wuhan City, Hubei Province, spreads worldwide, and threats the human life. The detection of SARS-CoV-2 is important for the prevention of the outbreak and management of patients. Real-time reverse-transcription polymerase chain reaction (RT-PCR) assay detected the virus in clinical laboratory. Methods: This study utilized primers and single-quencher probes in accordance with the Centers for Disease Control and Prevention (CDC) in the USA and the National Institute of Infectious Diseases (NIID) in Japan. Moreover, we designed the double-quencher probes (YCH assay) according to the oligonucleotide sequence established by NIID. Using these assays, we conducted a one-step real-time RT-PCR with serial DNA positive control to assess the detection sensitivity. Results: The threshold cycle (Ct) value of RT-PCR was relatively low in CDC and YCH assays compared to NIID assay. Serial dilution assay showed that both CDC and YCH assays could detect a low-copy number of DNA positive control. The background fluorescent signal at the baseline was lower in YCH than that of NIID. Conclusion: Double-quencher probes decreased background fluorescent signal and improved detection sensitivity of SARS-CoV-2.',\n",
       " 'Although the original US porcine epidemic diarrhea virus (PEDV) was confirmed as highly virulent by multiple studies, the virulence of spike-insertion deletion (S-INDEL) PEDV strains is undefined. In this study, 3–4 day-old conventional suckling piglets were inoculated with S-INDEL PEDV Iowa106 (4 pig litters) to study its virulence. Two litters of age-matched piglets were inoculated with either the original US PEDV PC21A or mock as positive and negative controls, respectively. Subsequently, all pigs were challenged with the original US PEDV PC21A on 21–29 days post-inoculation (dpi) to assess cross-protection. All S-INDEL Iowa106- and the original US PC21A-inoculated piglets developed diarrhea. However, the severity of clinical signs, mortality (0–75%) and fecal PEDV RNA shedding titers varied among the four S-INDEL Iowa106-inoculated litters. Compared with the original PC21A, piglets euthanized/died acutely from S-INDEL Iowa106 infection had relatively milder villous atrophy, lower antigen scores and more limited intestinal infection. Two of four S-INDEL Iowa106-infected sows and the original PC21A-infected sow showed anorexia and watery diarrhea for 1–4 days. After the original PC21A challenge, a subset (13/16) of S-INDEL Iowa106-inoculated piglets developed diarrhea, whereas all (5/5) and no (0/4) pigs in the mock and original PC21A-inoculated pigs had diarrhea, respectively. Our results suggest that the virulence of S-INDEL PEDV Iowa106 was less than the original US PEDV PC21A in suckling pigs, with 100% morbidity and 18% (6/33) overall (0–75%) mortality in suckling pigs depending on factors such as the sow’s health and lactation and the piglets’ birth weight. Prior infection by S-INDEL Iowa106 provided partial cross-protection to piglets against the original PC21A challenge at 21–29 dpi.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'Effective response to the Coronavirus Disease 2019 (COVID-19) pandemic requires genomic resources and bioinformatics tools for genomic epidemiology and surveillance studies that involve characterizing full-length viral genomes, identifying origins of infections, determining the relatedness of viral infections, performing phylogenetic analyses, and monitoring the continuous evolution of the SARS-CoV-2 viral genomes. The Children’s Hospital, Los Angeles (CHLA) COVID-19 Analysis Research Database (CARD) (https://covid19.cpmbiodev.net/) is a comprehensive genomic resource that provides access to full-length SARS-CoV-2 viral genomes and associated meta-data for over 30,000 (as of May 20, 2020) isolates collected from global sequencing repositories and the sequencing performed at the Center for Personalized Medicine (CPM) at CHLA. Reference phylogenetic trees of global and USA viral isolates were constructed and are periodically updated using selected high quality SARS-CoV-2 genome sequences. These provide the baseline and analytical context for identifying the origin of a viral infection, as well as the relatedness of SARS-CoV-2 genomes of interest. A web-based and interactive Phylogenetic Tree Browser supports flexible tree manipulation and advanced analysis based on keyword search while highlighting time series animation, as well as subtree export for graphical representation or offline exploration. A Virus Genome Tracker accepts complete or partial SARS-CoV-2 genome sequence, compares it against all available sequences in the database (>30,000 at time of writing), detects and annotates the variants, and places the new viral isolate within the global or USA phylogenetic contexts based upon variant profiles and haplotype comparisons, in a few seconds. The generated analysis can potentially aid in genomic surveillance to trace the transmission of any new infection. Using CHLA CARD, we demonstrate the identification of a candidate outbreak point where 13 of 31 CHLA internal isolates may have originated. We also discovered multiple indels of unknown clinical significance in the orf3a gene, and revealed a number of USA-specific variants and haplotypes.',\n",
       " 'Effective response to the Coronavirus Disease 2019 (COVID-19) pandemic requires genomic resources and bioinformatics tools for genomic epidemiology and surveillance studies that involve characterizing full-length viral genomes, identifying origins of infections, determining the relatedness of viral infections, performing phylogenetic analyses, and monitoring the continuous evolution of the SARS-CoV-2 viral genomes. The Children’s Hospital, Los Angeles (CHLA) COVID-19 Analysis Research Database (CARD) (https://covid19.cpmbiodev.net/) is a comprehensive genomic resource that provides access to full-length SARS-CoV-2 viral genomes and associated meta-data for over 30,000 (as of May 20, 2020) isolates collected from global sequencing repositories and the sequencing performed at the Center for Personalized Medicine (CPM) at CHLA. Reference phylogenetic trees of global and USA viral isolates were constructed and are periodically updated using selected high quality SARS-CoV-2 genome sequences. These provide the baseline and analytical context for identifying the origin of a viral infection, as well as the relatedness of SARS-CoV-2 genomes of interest. A web-based and interactive Phylogenetic Tree Browser supports flexible tree manipulation and advanced analysis based on keyword search while highlighting time series animation, as well as subtree export for graphical representation or offline exploration. A Virus Genome Tracker accepts complete or partial SARS-CoV-2 genome sequence, compares it against all available sequences in the database (>30,000 at time of writing), detects and annotates the variants, and places the new viral isolate within the global or USA phylogenetic contexts based upon variant profiles and haplotype comparisons, in a few seconds. The generated analysis can potentially aid in genomic surveillance to trace the transmission of any new infection. Using CHLA CARD, we demonstrate the identification of a candidate outbreak point where 13 of 31 CHLA internal isolates may have originated. We also discovered multiple indels of unknown clinical significance in the orf3a gene, and revealed a number of USA-specific variants and haplotypes.',\n",
       " 'Effective response to the Coronavirus Disease 2019 (COVID-19) pandemic requires genomic resources and bioinformatics tools for genomic epidemiology and surveillance studies that involve characterizing full-length viral genomes, identifying origins of infections, determining the relatedness of viral infections, performing phylogenetic analyses, and monitoring the continuous evolution of the SARS-CoV-2 viral genomes. The Children’s Hospital, Los Angeles (CHLA) COVID-19 Analysis Research Database (CARD) (https://covid19.cpmbiodev.net/) is a comprehensive genomic resource that provides access to full-length SARS-CoV-2 viral genomes and associated meta-data for over 30,000 (as of May 20, 2020) isolates collected from global sequencing repositories and the sequencing performed at the Center for Personalized Medicine (CPM) at CHLA. Reference phylogenetic trees of global and USA viral isolates were constructed and are periodically updated using selected high quality SARS-CoV-2 genome sequences. These provide the baseline and analytical context for identifying the origin of a viral infection, as well as the relatedness of SARS-CoV-2 genomes of interest. A web-based and interactive Phylogenetic Tree Browser supports flexible tree manipulation and advanced analysis based on keyword search while highlighting time series animation, as well as subtree export for graphical representation or offline exploration. A Virus Genome Tracker accepts complete or partial SARS-CoV-2 genome sequence, compares it against all available sequences in the database (>30,000 at time of writing), detects and annotates the variants, and places the new viral isolate within the global or USA phylogenetic contexts based upon variant profiles and haplotype comparisons, in a few seconds. The generated analysis can potentially aid in genomic surveillance to trace the transmission of any new infection. Using CHLA CARD, we demonstrate the identification of a candidate outbreak point where 13 of 31 CHLA internal isolates may have originated. We also discovered multiple indels of unknown clinical significance in the orf3a gene, and revealed a number of USA-specific variants and haplotypes.',\n",
       " 'Effective response to the Coronavirus Disease 2019 (COVID-19) pandemic requires genomic resources and bioinformatics tools for genomic epidemiology and surveillance studies that involve characterizing full-length viral genomes, identifying origins of infections, determining the relatedness of viral infections, performing phylogenetic analyses, and monitoring the continuous evolution of the SARS-CoV-2 viral genomes. The Children’s Hospital, Los Angeles (CHLA) COVID-19 Analysis Research Database (CARD) (https://covid19.cpmbiodev.net/) is a comprehensive genomic resource that provides access to full-length SARS-CoV-2 viral genomes and associated meta-data for over 30,000 (as of May 20, 2020) isolates collected from global sequencing repositories and the sequencing performed at the Center for Personalized Medicine (CPM) at CHLA. Reference phylogenetic trees of global and USA viral isolates were constructed and are periodically updated using selected high quality SARS-CoV-2 genome sequences. These provide the baseline and analytical context for identifying the origin of a viral infection, as well as the relatedness of SARS-CoV-2 genomes of interest. A web-based and interactive Phylogenetic Tree Browser supports flexible tree manipulation and advanced analysis based on keyword search while highlighting time series animation, as well as subtree export for graphical representation or offline exploration. A Virus Genome Tracker accepts complete or partial SARS-CoV-2 genome sequence, compares it against all available sequences in the database (>30,000 at time of writing), detects and annotates the variants, and places the new viral isolate within the global or USA phylogenetic contexts based upon variant profiles and haplotype comparisons, in a few seconds. The generated analysis can potentially aid in genomic surveillance to trace the transmission of any new infection. Using CHLA CARD, we demonstrate the identification of a candidate outbreak point where 13 of 31 CHLA internal isolates may have originated. We also discovered multiple indels of unknown clinical significance in the orf3a gene, and revealed a number of USA-specific variants and haplotypes.',\n",
       " 'Background. The Coronavirus disease 19 (COVID-19) pandemic has led to widespread concerns about the risk of infection in patients with rheumatic diseases (RD) receiving disease modifying ant-rheumatic drugs (DMARDs) and other immunosuppressants (IS). Methods. A SurveyMonkey based electronic survey was conducted amongst members of the Indian Rheumatology Association to understand the need for changes in prevailing practices. Results. Of the 861 invitees, 221 responded. In the wake of the pandemic, 47.5% would reduce biological DMARDs (bDMARDs) while only 12.2% would reduce the use of conventional synthetic DMARDs. 64.2% were likely to defer change in IS, the reluctance being most with rituximab (58.3%) followed by cyclophosphamide (53.3%), anti-tumor necrosis factor alpha agents (52.4%) and Janus kinase inhibitors (34.39%). Hydroxychloroquine was the preferred choice (81.9%) for the treatment of COVID-19 followed by protease inhibitors (22.1%) and intravenous immunoglobulin (8.1%). Chloroquine was less preferred (19%). More than two-thirds (70.5%) believed that COVID-19 might trigger macrophage activation syndrome. Social distancing (98.1%) and hand hygiene (74.6%) were recommended by majority. 62.8% would avoid touch for clinical examination whenever feasible. Conclusion. Most rheumatologists perceived the need to change treatment of RDs during the COVID-19 pandemic; reduce immunosuppression and defer the usage of rituximab and bDMARDs.',\n",
       " 'Background. The Coronavirus disease 19 (COVID-19) pandemic has led to widespread concerns about the risk of infection in patients with rheumatic diseases (RD) receiving disease modifying ant-rheumatic drugs (DMARDs) and other immunosuppressants (IS). Methods. A SurveyMonkey based electronic survey was conducted amongst members of the Indian Rheumatology Association to understand the need for changes in prevailing practices. Results. Of the 861 invitees, 221 responded. In the wake of the pandemic, 47.5% would reduce biological DMARDs (bDMARDs) while only 12.2% would reduce the use of conventional synthetic DMARDs. 64.2% were likely to defer change in IS, the reluctance being most with rituximab (58.3%) followed by cyclophosphamide (53.3%), anti-tumor necrosis factor alpha agents (52.4%) and Janus kinase inhibitors (34.39%). Hydroxychloroquine was the preferred choice (81.9%) for the treatment of COVID-19 followed by protease inhibitors (22.1%) and intravenous immunoglobulin (8.1%). Chloroquine was less preferred (19%). More than two-thirds (70.5%) believed that COVID-19 might trigger macrophage activation syndrome. Social distancing (98.1%) and hand hygiene (74.6%) were recommended by majority. 62.8% would avoid touch for clinical examination whenever feasible. Conclusion. Most rheumatologists perceived the need to change treatment of RDs during the COVID-19 pandemic; reduce immunosuppression and defer the usage of rituximab and bDMARDs.',\n",
       " 'Background. The Coronavirus disease 19 (COVID-19) pandemic has led to widespread concerns about the risk of infection in patients with rheumatic diseases (RD) receiving disease modifying ant-rheumatic drugs (DMARDs) and other immunosuppressants (IS). Methods. A SurveyMonkey based electronic survey was conducted amongst members of the Indian Rheumatology Association to understand the need for changes in prevailing practices. Results. Of the 861 invitees, 221 responded. In the wake of the pandemic, 47.5% would reduce biological DMARDs (bDMARDs) while only 12.2% would reduce the use of conventional synthetic DMARDs. 64.2% were likely to defer change in IS, the reluctance being most with rituximab (58.3%) followed by cyclophosphamide (53.3%), anti-tumor necrosis factor alpha agents (52.4%) and Janus kinase inhibitors (34.39%). Hydroxychloroquine was the preferred choice (81.9%) for the treatment of COVID-19 followed by protease inhibitors (22.1%) and intravenous immunoglobulin (8.1%). Chloroquine was less preferred (19%). More than two-thirds (70.5%) believed that COVID-19 might trigger macrophage activation syndrome. Social distancing (98.1%) and hand hygiene (74.6%) were recommended by majority. 62.8% would avoid touch for clinical examination whenever feasible. Conclusion. Most rheumatologists perceived the need to change treatment of RDs during the COVID-19 pandemic; reduce immunosuppression and defer the usage of rituximab and bDMARDs.',\n",
       " 'Background. The Coronavirus disease 19 (COVID-19) pandemic has led to widespread concerns about the risk of infection in patients with rheumatic diseases (RD) receiving disease modifying ant-rheumatic drugs (DMARDs) and other immunosuppressants (IS). Methods. A SurveyMonkey based electronic survey was conducted amongst members of the Indian Rheumatology Association to understand the need for changes in prevailing practices. Results. Of the 861 invitees, 221 responded. In the wake of the pandemic, 47.5% would reduce biological DMARDs (bDMARDs) while only 12.2% would reduce the use of conventional synthetic DMARDs. 64.2% were likely to defer change in IS, the reluctance being most with rituximab (58.3%) followed by cyclophosphamide (53.3%), anti-tumor necrosis factor alpha agents (52.4%) and Janus kinase inhibitors (34.39%). Hydroxychloroquine was the preferred choice (81.9%) for the treatment of COVID-19 followed by protease inhibitors (22.1%) and intravenous immunoglobulin (8.1%). Chloroquine was less preferred (19%). More than two-thirds (70.5%) believed that COVID-19 might trigger macrophage activation syndrome. Social distancing (98.1%) and hand hygiene (74.6%) were recommended by majority. 62.8% would avoid touch for clinical examination whenever feasible. Conclusion. Most rheumatologists perceived the need to change treatment of RDs during the COVID-19 pandemic; reduce immunosuppression and defer the usage of rituximab and bDMARDs.',\n",
       " 'Abstract 5′ and 3′ UTR sequences on the coronavirus genome are known to carry cis-acting elements for DI RNA replication and presumably also virus genome replication. 5′ UTR-adjacent coding sequences are also thought to harbor cis-acting elements. Here we have determined the 5′ UTR and adjacent 289-nt sequences, and 3′ UTR sequences, for six group 2 coronaviruses and have compared them to each other and to three previously reported group 2 members. Extensive regions of highly similar UTR sequences were found but small regions of divergence were also found indicating group 2 coronaviruses could be subdivided into those that are bovine coronavirus (BCoV)-like (BCoV, human respiratory coronavirus-OC43, human enteric coronavirus, porcine hemagglutinating encephalomyelitis virus, and equine coronavirus) and those that are murine hepatitis virus (MHV)-like (A59, 2, and JHM strains of MHV, puffinosis virus, and rat sialodacryoadenitis virus). The 3′ UTRs of BCoV and MHV have been previously shown to be interchangeable. Here, a reporter-containing BCoV DI RNA was shown to be replicated by all five BCoV-like helper viruses and by MHV-H2 (a human cell-adapted MHV strain), a representative of the MHV-like subgroup, demonstrating group 2 common 5′ and 3′ replication signaling elements. BCoV DI RNA, furthermore, acquired the leader of HCoV-OC43 by leader switching, demonstrating for the first time in vivo recombination between animal and human coronavirus molecules. These results indicate that common replication signaling elements exist among group 2 coronaviruses despite a two-cluster pattern within the group and imply there could exist a high potential for recombination among group members.',\n",
       " 'BACKGROUND: The recent coronavirus disease (COVID-19) pandemic is taking a toll on the world’s health care infrastructure as well as the social, economic, and psychological well-being of humanity. Individuals, organizations, and governments are using social media to communicate with each other on a number of issues relating to the COVID-19 pandemic. Not much is known about the topics being shared on social media platforms relating to COVID-19. Analyzing such information can help policy makers and health care organizations assess the needs of their stakeholders and address them appropriately. OBJECTIVE: This study aims to identify the main topics posted by Twitter users related to the COVID-19 pandemic. METHODS: Leveraging a set of tools (Twitter’s search application programming interface (API), Tweepy Python library, and PostgreSQL database) and using a set of predefined search terms (“corona,” “2019-nCov,” and “COVID-19”), we extracted the text and metadata (number of likes and retweets, and user profile information including the number of followers) of public English language tweets from February 2, 2020, to March 15, 2020. We analyzed the collected tweets using word frequencies of single (unigrams) and double words (bigrams). We leveraged latent Dirichlet allocation for topic modeling to identify topics discussed in the tweets. We also performed sentiment analysis and extracted the mean number of retweets, likes, and followers for each topic and calculated the interaction rate per topic. RESULTS: Out of approximately 2.8 million tweets included, 167,073 unique tweets from 160,829 unique users met the inclusion criteria. Our analysis identified 12 topics, which were grouped into four main themes: origin of the virus; its sources; its impact on people, countries, and the economy; and ways of mitigating the risk of infection. The mean sentiment was positive for 10 topics and negative for 2 topics (deaths caused by COVID-19 and increased racism). The mean for tweet topics of account followers ranged from 2722 (increased racism) to 13,413 (economic losses). The highest mean of likes for the tweets was 15.4 (economic loss), while the lowest was 3.94 (travel bans and warnings). CONCLUSIONS: Public health crisis response activities on the ground and online are becoming increasingly simultaneous and intertwined. Social media provides an opportunity to directly communicate health information to the public. Health systems should work on building national and international disease detection and surveillance systems through monitoring social media. There is also a need for a more proactive and agile public health presence on social media to combat the spread of fake news.',\n",
       " 'BACKGROUND: The recent coronavirus disease (COVID-19) pandemic is taking a toll on the world’s health care infrastructure as well as the social, economic, and psychological well-being of humanity. Individuals, organizations, and governments are using social media to communicate with each other on a number of issues relating to the COVID-19 pandemic. Not much is known about the topics being shared on social media platforms relating to COVID-19. Analyzing such information can help policy makers and health care organizations assess the needs of their stakeholders and address them appropriately. OBJECTIVE: This study aims to identify the main topics posted by Twitter users related to the COVID-19 pandemic. METHODS: Leveraging a set of tools (Twitter’s search application programming interface (API), Tweepy Python library, and PostgreSQL database) and using a set of predefined search terms (“corona,” “2019-nCov,” and “COVID-19”), we extracted the text and metadata (number of likes and retweets, and user profile information including the number of followers) of public English language tweets from February 2, 2020, to March 15, 2020. We analyzed the collected tweets using word frequencies of single (unigrams) and double words (bigrams). We leveraged latent Dirichlet allocation for topic modeling to identify topics discussed in the tweets. We also performed sentiment analysis and extracted the mean number of retweets, likes, and followers for each topic and calculated the interaction rate per topic. RESULTS: Out of approximately 2.8 million tweets included, 167,073 unique tweets from 160,829 unique users met the inclusion criteria. Our analysis identified 12 topics, which were grouped into four main themes: origin of the virus; its sources; its impact on people, countries, and the economy; and ways of mitigating the risk of infection. The mean sentiment was positive for 10 topics and negative for 2 topics (deaths caused by COVID-19 and increased racism). The mean for tweet topics of account followers ranged from 2722 (increased racism) to 13,413 (economic losses). The highest mean of likes for the tweets was 15.4 (economic loss), while the lowest was 3.94 (travel bans and warnings). CONCLUSIONS: Public health crisis response activities on the ground and online are becoming increasingly simultaneous and intertwined. Social media provides an opportunity to directly communicate health information to the public. Health systems should work on building national and international disease detection and surveillance systems through monitoring social media. There is also a need for a more proactive and agile public health presence on social media to combat the spread of fake news.',\n",
       " 'BACKGROUND: The recent coronavirus disease (COVID-19) pandemic is taking a toll on the world’s health care infrastructure as well as the social, economic, and psychological well-being of humanity. Individuals, organizations, and governments are using social media to communicate with each other on a number of issues relating to the COVID-19 pandemic. Not much is known about the topics being shared on social media platforms relating to COVID-19. Analyzing such information can help policy makers and health care organizations assess the needs of their stakeholders and address them appropriately. OBJECTIVE: This study aims to identify the main topics posted by Twitter users related to the COVID-19 pandemic. METHODS: Leveraging a set of tools (Twitter’s search application programming interface (API), Tweepy Python library, and PostgreSQL database) and using a set of predefined search terms (“corona,” “2019-nCov,” and “COVID-19”), we extracted the text and metadata (number of likes and retweets, and user profile information including the number of followers) of public English language tweets from February 2, 2020, to March 15, 2020. We analyzed the collected tweets using word frequencies of single (unigrams) and double words (bigrams). We leveraged latent Dirichlet allocation for topic modeling to identify topics discussed in the tweets. We also performed sentiment analysis and extracted the mean number of retweets, likes, and followers for each topic and calculated the interaction rate per topic. RESULTS: Out of approximately 2.8 million tweets included, 167,073 unique tweets from 160,829 unique users met the inclusion criteria. Our analysis identified 12 topics, which were grouped into four main themes: origin of the virus; its sources; its impact on people, countries, and the economy; and ways of mitigating the risk of infection. The mean sentiment was positive for 10 topics and negative for 2 topics (deaths caused by COVID-19 and increased racism). The mean for tweet topics of account followers ranged from 2722 (increased racism) to 13,413 (economic losses). The highest mean of likes for the tweets was 15.4 (economic loss), while the lowest was 3.94 (travel bans and warnings). CONCLUSIONS: Public health crisis response activities on the ground and online are becoming increasingly simultaneous and intertwined. Social media provides an opportunity to directly communicate health information to the public. Health systems should work on building national and international disease detection and surveillance systems through monitoring social media. There is also a need for a more proactive and agile public health presence on social media to combat the spread of fake news.',\n",
       " 'BACKGROUND: The recent coronavirus disease (COVID-19) pandemic is taking a toll on the world’s health care infrastructure as well as the social, economic, and psychological well-being of humanity. Individuals, organizations, and governments are using social media to communicate with each other on a number of issues relating to the COVID-19 pandemic. Not much is known about the topics being shared on social media platforms relating to COVID-19. Analyzing such information can help policy makers and health care organizations assess the needs of their stakeholders and address them appropriately. OBJECTIVE: This study aims to identify the main topics posted by Twitter users related to the COVID-19 pandemic. METHODS: Leveraging a set of tools (Twitter’s search application programming interface (API), Tweepy Python library, and PostgreSQL database) and using a set of predefined search terms (“corona,” “2019-nCov,” and “COVID-19”), we extracted the text and metadata (number of likes and retweets, and user profile information including the number of followers) of public English language tweets from February 2, 2020, to March 15, 2020. We analyzed the collected tweets using word frequencies of single (unigrams) and double words (bigrams). We leveraged latent Dirichlet allocation for topic modeling to identify topics discussed in the tweets. We also performed sentiment analysis and extracted the mean number of retweets, likes, and followers for each topic and calculated the interaction rate per topic. RESULTS: Out of approximately 2.8 million tweets included, 167,073 unique tweets from 160,829 unique users met the inclusion criteria. Our analysis identified 12 topics, which were grouped into four main themes: origin of the virus; its sources; its impact on people, countries, and the economy; and ways of mitigating the risk of infection. The mean sentiment was positive for 10 topics and negative for 2 topics (deaths caused by COVID-19 and increased racism). The mean for tweet topics of account followers ranged from 2722 (increased racism) to 13,413 (economic losses). The highest mean of likes for the tweets was 15.4 (economic loss), while the lowest was 3.94 (travel bans and warnings). CONCLUSIONS: Public health crisis response activities on the ground and online are becoming increasingly simultaneous and intertwined. Social media provides an opportunity to directly communicate health information to the public. Health systems should work on building national and international disease detection and surveillance systems through monitoring social media. There is also a need for a more proactive and agile public health presence on social media to combat the spread of fake news.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'Abstract The outbreak of Coronavirus Disease 2019 (COVID-19) causing novel coronavirus-infected pneumonia (NCIP), has affected the lives of 71,429 people globally. Originating in China, the disease has a rapid progression to other countries. Research suggests remarkable genomic resemblance of 2019-nCoV with Severe Acute Respiratory Syndrome (SARS) which has a history of a pandemic in 2002. With evidence of nosocomial spread, a number of diligent measures are being employed to constrain its propagation. Hence, the Public Health Emergency of International Concern (PHEIC) has been established by the World Health Organization (WHO) with strategic objectives for public health to curtail its impact on global health and economy.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " 'The novel Coronavirus disease has increased rapidly in the Wuhan city of China in December 2019. This fatal virus has spread across the whole world like a fire in different stages and affecting millions of population and thousands of deaths worldwide. Therefore, it is essential to classify the infected people, so that they can take the precaution in the earlier stages. Also, due to the increasing cases spread of Coronavirus, there are only limited numbers of polymerase change reaction kits available in the hospitals for testing Coronavirus patients. That why it is extremely important to develop artificial intelligence-based automatic diagnostic tools to classify the Coronavirus outbreak. The objective of this paper is to know the novel disease epidemiology, major prevention from spreading of Coronavirus Severe Acute Respiratory Syndrome, and to assess the machine and deep learning-based architectures performance that is proposed in the present year for classification of Coronavirus images such as, X-Ray and computed tomography. Specifically, advanced deep learning-based algorithms known as the Convolutional neural network, which plays a great effect on extracting highly essential features, mostly in terms of medical images. This technique, with using CT and X-Ray image scans, has been adopted in most of the recently published articles on the Coronavirus with remarkable results. Furthermore, according to this paper, this can be noted and said that deep learning technology has potential clinical applications.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'DNA replication in metazoan cells initiates at multiple discrete chromosomal sites called replication origins. Recent genome-wide studies have mapped thousands of origins in animal and plant cells, but without yielding a distinct and universal consensus sequence. However, origin-associated regions with particular base composition features have been identified, such as the G-rich OGRE motif, predicted to form G-quadruplexes. Epigenetic marks such as histone modifications that promote open chromatin also favor origin formation. Before DNA replication can initiate at origins, they need to be “licensed” by the binding of the origin recognition complex (ORC) and other proteins to form pre-replication and pre-initiation complexes. Origin-associated proteins are themselves subject to multilayered regulation, notably by posttranslational modification and proteasomal degradation. Origins are organized into replicons and replicon clusters, whose firing occurs at discrete subnuclear bodies known as replication foci. A still poorly characterized nuclear matrix structure might be involved in the attachment of replication units and the formation of replication foci. Preferential genomic sequences responsible for these attachments have been reported, as well as factors playing roles in their regulation. There appears to be great flexibility in the choice of origins used in each S-phase, being affected by factors including cell identity and replication stresses. Origin selection also changes dramatically during embryogenesis in concert with developmental signaling pathways. Dysregulation of origin positioning and recognition are implicated in human disease, as mutations in origin-binding proteins have been found in developmental disorders, and expansions in repeat-containing genomic regions promote genome instability. This chapter summarizes current understanding about replication origins, the most recent discoveries, and outlines key unanswered questions in this exciting field.',\n",
       " 'Abstract No Abstract',\n",
       " 'Abstract No Abstract',\n",
       " \"COVID-19 was originated from Wuhan city of Hubei Province in China in December 2019. Since then it has spread in more than 210 countries and territories. It is a viral disease due to the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) virus. The patients show flu-like symptoms with a dry cough, sore throat, high fever, and breathing problems. The disease due to SARS-CoV-2 was named as COVID-19. About 2.2 million people have been infected with more than 0.15 million deaths globally. The United States of America is the most affected country with the highest patients of about 0.7 million. Despite great efforts, there is no treatment of this disease. However, prevention and management are the best options. This article describes SARS-CoV-2, disease, prevention and management, treatment and social impact on society. It was analyzed that a combination of antiviral drugs with hydroxyl-chloroquine and azithromycin (with the consultation of a medical practitioner) may be the best option to treat the patients, depending on the patient's conditions and symptoms. However, Unani therapy may be useful along with allopathic treatment. It is urgently advised and requested that all the persons should follow the preventive measures, managements and quarantine strictly without any religious discrepancy otherwise the situation may be the worst. Also, there is an urgent requirement to educate our new generation for science and technology to fight against any such disaster in future; if any. There is no need to be panic and proper prevention and management are essential to combat this disease. This article may be useful to create awareness among the public, to prevent, manage and treat COVID-19.\",\n",
       " \"COVID-19 was originated from Wuhan city of Hubei Province in China in December 2019. Since then it has spread in more than 210 countries and territories. It is a viral disease due to the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) virus. The patients show flu-like symptoms with a dry cough, sore throat, high fever, and breathing problems. The disease due to SARS-CoV-2 was named as COVID-19. About 2.2 million people have been infected with more than 0.15 million deaths globally. The United States of America is the most affected country with the highest patients of about 0.7 million. Despite great efforts, there is no treatment of this disease. However, prevention and management are the best options. This article describes SARS-CoV-2, disease, prevention and management, treatment and social impact on society. It was analyzed that a combination of antiviral drugs with hydroxyl-chloroquine and azithromycin (with the consultation of a medical practitioner) may be the best option to treat the patients, depending on the patient's conditions and symptoms. However, Unani therapy may be useful along with allopathic treatment. It is urgently advised and requested that all the persons should follow the preventive measures, managements and quarantine strictly without any religious discrepancy otherwise the situation may be the worst. Also, there is an urgent requirement to educate our new generation for science and technology to fight against any such disaster in future; if any. There is no need to be panic and proper prevention and management are essential to combat this disease. This article may be useful to create awareness among the public, to prevent, manage and treat COVID-19.\",\n",
       " 'Coronavirus disease strain (SARS-CoV-2) was discovered in 2019, and it is spreading very fast around the world causing the disease Covid-19. Currently, more than 1.6 million individuals are infected, and several thousand are dead across the globe because of Covid-19. Here, we utilized the in-silico approaches to identify possible protease inhibitors against SARS-CoV-2. Potential compounds were screened from the CHEMBL database, ZINC database, FDA approved drugs and molecules under clinical trials. Our study is based on 6Y2F and 6W63 co-crystallized structures available in the protein data bank (PDB). Seven hundred compounds from ZINC/CHEMBL databases and fourteen hundred compounds from drug-bank were selected based on positive interactions with the reported binding site. All the selected compounds were subjected to standard-precision (SP) and extra-precision (XP) mode of docking. Generated docked poses were carefully visualized for known interactions within the binding site. Molecular mechanics-generalized born surface area (MM-GBSA) calculations were performed to screen the best compounds based on docking scores and binding energy values. Molecular dynamics (MD) simulations were carried out on four selected compounds from the CHEMBL database to validate the stability and interactions. MD simulations were also performed on the PDB structure 6YF2F to understand the differences between screened molecules and co-crystallized ligand. We screened 300 potential compounds from various databases, and 66 potential compounds from FDA approved drugs. Cobicistat, ritonavir, lopinavir, and darunavir are in the top screened molecules from FDA approved drugs. The screened drugs and molecules may be helpful in fighting with SARS-CoV-2 after further studies.',\n",
       " 'Coronavirus disease strain (SARS-CoV-2) was discovered in 2019, and it is spreading very fast around the world causing the disease Covid-19. Currently, more than 1.6 million individuals are infected, and several thousand are dead across the globe because of Covid-19. Here, we utilized the in-silico approaches to identify possible protease inhibitors against SARS-CoV-2. Potential compounds were screened from the CHEMBL database, ZINC database, FDA approved drugs and molecules under clinical trials. Our study is based on 6Y2F and 6W63 co-crystallized structures available in the protein data bank (PDB). Seven hundred compounds from ZINC/CHEMBL databases and fourteen hundred compounds from drug-bank were selected based on positive interactions with the reported binding site. All the selected compounds were subjected to standard-precision (SP) and extra-precision (XP) mode of docking. Generated docked poses were carefully visualized for known interactions within the binding site. Molecular mechanics-generalized born surface area (MM-GBSA) calculations were performed to screen the best compounds based on docking scores and binding energy values. Molecular dynamics (MD) simulations were carried out on four selected compounds from the CHEMBL database to validate the stability and interactions. MD simulations were also performed on the PDB structure 6YF2F to understand the differences between screened molecules and co-crystallized ligand. We screened 300 potential compounds from various databases, and 66 potential compounds from FDA approved drugs. Cobicistat, ritonavir, lopinavir, and darunavir are in the top screened molecules from FDA approved drugs. The screened drugs and molecules may be helpful in fighting with SARS-CoV-2 after further studies.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " \"An outbreak of a novel coronavirus (COVID‐19 or 2019‐CoV) infection has posed significant threats to international health and the economy. In the absence of treatment for this virus, there is an urgent need to find alternative methods to control the spread of disease. Here, we have conducted an online search for all treatment options related to coronavirus infections as well as some RNA‐virus infection and we have found that general treatments, coronavirus‐specific treatments, and antiviral treatments should be useful in fighting COVID‐19. We suggest that the nutritional status of each infected patient should be evaluated before the administration of general treatments and the current children's RNA‐virus vaccines including influenza vaccine should be immunized for uninfected people and health care workers. In addition, convalescent plasma should be given to COVID‐19 patients if it is available. In conclusion, we suggest that all the potential interventions be implemented to control the emerging COVID‐19 if the infection is uncontrollable.\",\n",
       " 'In December 2019, an outbreak of pneumonia of unknown origin was reported in Wuhan, Hubei Province, China. Pneumonia cases were epidemiologically linked to the Huanan Seafood Wholesale Market. Inoculation of respiratory samples into human airway epithelial cells, Vero E6 and Huh7 cell lines, led to the isolation of a novel respiratory virus whose genome analysis showed it to be a novel coronavirus related to SARS-CoV, and therefore named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). SARS-CoV-2 is a betacoronavirus belonging to the subgenus Sarbecovirus. The global spread of SARS-CoV-2 and the thousands of deaths caused by coronavirus disease (COVID-19) led the World Health Organization to declare a pandemic on 12 March 2020. To date, the world has paid a high toll in this pandemic in terms of human lives lost, economic repercussions and increased poverty. In this review, we provide information regarding the epidemiology, serological and molecular diagnosis, origin of SARS-CoV-2 and its ability to infect human cells, and safety issues. Then we focus on the available therapies to fight COVID-19, the development of vaccines, the role of artificial intelligence in the management of the pandemic and limiting the spread of the virus, the impact of the COVID-19 epidemic on our lifestyle, and preparation for a possible second wave.',\n",
       " 'Abstract Introduction The World Health Organization has declared the coronavirus disease (COVID-19) as a pandemic on 11 March 2020, after the number of confirmed cases outside China increased 13-fold. As the epicentre of the initial outbreak, China has been updating the National COVID-19 Diagnostic and Treatment Guideline with up-to-date information about the disease. To facilitate the implementation of integrative Chinese–Western Medicine in COVID-19 management, Traditional Chinese medicine (TCM) has been recommended in recent editions of the national guideline. Methods The national guideline summarised the opinions and frontline experience of medical experts across the country to provide by far the best management for COVID-19. We extracted the case definition and clinical classifications of COVID-19 in China along with relevant TCM treatments cited in the seventh edition of the guideline, with an intend to disseminate practical information to TCM practitioners and researchers around the world. Results We presented the most recent case definition, clinical classifications, and relevant TCM treatments of COVID-19 in accordance to recommendations of the Chinese guideline. TCM treatments are stratified into two groups based on patients’ disease status. Four types of Chinese patent medicines are recommended for suspected COVID-19 cases. Several herbal formulae are recommended for confirmed COVID-19 cases according to their clinical classification and TCM pattern diagnoses. Two herbal formulae are also recommended for rehabilitation of recovering cases. Conclusion To control the waves of COVID-19 outbreak, countries must ensure the adherence of their citizens to local public health measures. Medical professionals should diagnose and treat patients according to up-to-date guidelines. Future evaluation of the outcomes of implementing TCM recommendations will strengthen the evidence base for COVID-19 management for the sake of public health and the internationalisation of TCM.',\n",
       " 'Abstract Introduction The World Health Organization has declared the coronavirus disease (COVID-19) as a pandemic on 11 March 2020, after the number of confirmed cases outside China increased 13-fold. As the epicentre of the initial outbreak, China has been updating the National COVID-19 Diagnostic and Treatment Guideline with up-to-date information about the disease. To facilitate the implementation of integrative Chinese–Western Medicine in COVID-19 management, Traditional Chinese medicine (TCM) has been recommended in recent editions of the national guideline. Methods The national guideline summarised the opinions and frontline experience of medical experts across the country to provide by far the best management for COVID-19. We extracted the case definition and clinical classifications of COVID-19 in China along with relevant TCM treatments cited in the seventh edition of the guideline, with an intend to disseminate practical information to TCM practitioners and researchers around the world. Results We presented the most recent case definition, clinical classifications, and relevant TCM treatments of COVID-19 in accordance to recommendations of the Chinese guideline. TCM treatments are stratified into two groups based on patients’ disease status. Four types of Chinese patent medicines are recommended for suspected COVID-19 cases. Several herbal formulae are recommended for confirmed COVID-19 cases according to their clinical classification and TCM pattern diagnoses. Two herbal formulae are also recommended for rehabilitation of recovering cases. Conclusion To control the waves of COVID-19 outbreak, countries must ensure the adherence of their citizens to local public health measures. Medical professionals should diagnose and treat patients according to up-to-date guidelines. Future evaluation of the outcomes of implementing TCM recommendations will strengthen the evidence base for COVID-19 management for the sake of public health and the internationalisation of TCM.',\n",
       " 'Abstract Introduction The World Health Organization has declared the coronavirus disease (COVID-19) as a pandemic on 11 March 2020, after the number of confirmed cases outside China increased 13-fold. As the epicentre of the initial outbreak, China has been updating the National COVID-19 Diagnostic and Treatment Guideline with up-to-date information about the disease. To facilitate the implementation of integrative Chinese–Western Medicine in COVID-19 management, Traditional Chinese medicine (TCM) has been recommended in recent editions of the national guideline. Methods The national guideline summarised the opinions and frontline experience of medical experts across the country to provide by far the best management for COVID-19. We extracted the case definition and clinical classifications of COVID-19 in China along with relevant TCM treatments cited in the seventh edition of the guideline, with an intend to disseminate practical information to TCM practitioners and researchers around the world. Results We presented the most recent case definition, clinical classifications, and relevant TCM treatments of COVID-19 in accordance to recommendations of the Chinese guideline. TCM treatments are stratified into two groups based on patients’ disease status. Four types of Chinese patent medicines are recommended for suspected COVID-19 cases. Several herbal formulae are recommended for confirmed COVID-19 cases according to their clinical classification and TCM pattern diagnoses. Two herbal formulae are also recommended for rehabilitation of recovering cases. Conclusion To control the waves of COVID-19 outbreak, countries must ensure the adherence of their citizens to local public health measures. Medical professionals should diagnose and treat patients according to up-to-date guidelines. Future evaluation of the outcomes of implementing TCM recommendations will strengthen the evidence base for COVID-19 management for the sake of public health and the internationalisation of TCM.',\n",
       " 'Abstract Introduction The World Health Organization has declared the coronavirus disease (COVID-19) as a pandemic on 11 March 2020, after the number of confirmed cases outside China increased 13-fold. As the epicentre of the initial outbreak, China has been updating the National COVID-19 Diagnostic and Treatment Guideline with up-to-date information about the disease. To facilitate the implementation of integrative Chinese–Western Medicine in COVID-19 management, Traditional Chinese medicine (TCM) has been recommended in recent editions of the national guideline. Methods The national guideline summarised the opinions and frontline experience of medical experts across the country to provide by far the best management for COVID-19. We extracted the case definition and clinical classifications of COVID-19 in China along with relevant TCM treatments cited in the seventh edition of the guideline, with an intend to disseminate practical information to TCM practitioners and researchers around the world. Results We presented the most recent case definition, clinical classifications, and relevant TCM treatments of COVID-19 in accordance to recommendations of the Chinese guideline. TCM treatments are stratified into two groups based on patients’ disease status. Four types of Chinese patent medicines are recommended for suspected COVID-19 cases. Several herbal formulae are recommended for confirmed COVID-19 cases according to their clinical classification and TCM pattern diagnoses. Two herbal formulae are also recommended for rehabilitation of recovering cases. Conclusion To control the waves of COVID-19 outbreak, countries must ensure the adherence of their citizens to local public health measures. Medical professionals should diagnose and treat patients according to up-to-date guidelines. Future evaluation of the outcomes of implementing TCM recommendations will strengthen the evidence base for COVID-19 management for the sake of public health and the internationalisation of TCM.',\n",
       " 'Abstract Introduction The World Health Organization has declared the coronavirus disease (COVID-19) as a pandemic on 11 March 2020, after the number of confirmed cases outside China increased 13-fold. As the epicentre of the initial outbreak, China has been updating the National COVID-19 Diagnostic and Treatment Guideline with up-to-date information about the disease. To facilitate the implementation of integrative Chinese–Western Medicine in COVID-19 management, Traditional Chinese medicine (TCM) has been recommended in recent editions of the national guideline. Methods The national guideline summarised the opinions and frontline experience of medical experts across the country to provide by far the best management for COVID-19. We extracted the case definition and clinical classifications of COVID-19 in China along with relevant TCM treatments cited in the seventh edition of the guideline, with an intend to disseminate practical information to TCM practitioners and researchers around the world. Results We presented the most recent case definition, clinical classifications, and relevant TCM treatments of COVID-19 in accordance to recommendations of the Chinese guideline. TCM treatments are stratified into two groups based on patients’ disease status. Four types of Chinese patent medicines are recommended for suspected COVID-19 cases. Several herbal formulae are recommended for confirmed COVID-19 cases according to their clinical classification and TCM pattern diagnoses. Two herbal formulae are also recommended for rehabilitation of recovering cases. Conclusion To control the waves of COVID-19 outbreak, countries must ensure the adherence of their citizens to local public health measures. Medical professionals should diagnose and treat patients according to up-to-date guidelines. Future evaluation of the outcomes of implementing TCM recommendations will strengthen the evidence base for COVID-19 management for the sake of public health and the internationalisation of TCM.',\n",
       " 'The novel coronavirus disease 2019 (COVID-19) pandemic poses a substantial threat to the health of health care personnel on the front line of caring for patients with COVID-19. The Centers for Medicare and Medicaid Services have announced that all nonessential planned surgeries and procedures should be postponed until further notice and only urgent procedures should proceed. Neurologic surgeries and procedures should not be delayed under the circumstance in which it is essential at saving a life or preserving functioning of the central nervous system. With the intent to advise the neurosurgery team on how to adequately prepare and safely perform neurosurgical procedures on confirmed and suspected patients with COVID-19, we discuss considerations and recommendations based on the lessons and experience shared by neurosurgeons in China. Perioperative and intraoperative strategies, considerations, as well as challenges arisen under the specific circumstance have been discussed. In addition, a case of a ruptured aneurysm in a suspected patient with COVID-19 is reported. It is advised that all health care personnel who immediately participate in neurosurgical surgeries and procedures for confirmed and suspected patients with COVID-19 should take airborne precautions and wear enhanced personal protective equipment. Following the proposed guidance, urgent neurosurgical surgeries and procedures can be safely performed for the benefit of critical patients with or suspected for COVID-19.',\n",
       " 'A comprehensive review of the neurological disorders reported during the current COVID-19 pandemic demonstrates that infection with SARS-CoV-2 affects the central nervous system (CNS), the peripheral nervous system (PNS) and the muscle. CNS manifestations include: headache and decreased responsiveness considered initial indicators of potential neurological involvement; anosmia, hyposmia, hypogeusia, and dysgeusia are frequent early symptoms of coronavirus infection. Respiratory failure, the lethal manifestation of COVID-19, responsible for 215,461 deaths worldwide, is probably neurogenic in origin and may result from the viral invasion of cranial nerve I, progressing into rhinencephalon and brainstem respiratory centers. Cerebrovascular disease, in particular large-vessel ischemic strokes, and less frequently cerebral venous thrombosis, intracerebral hemorrhage and subarachnoid hemorrhage, usually occur as part of a thrombotic state induced by viral attachment to ACE2 receptors in endothelium causing widespread endotheliitis, coagulopathy, arterial and venous thromboses. Acute hemorrhagic necrotizing encephalopathy is associated to the cytokine storm. A frontal hypoperfusion syndrome has been identified. There are isolated reports of seizures, encephalopathy, meningitis, encephalitis, and myelitis. The neurological diseases affecting the PNS and muscle in COVID-19 are less frequent and include Guillain-Barré syndrome; Miller Fisher syndrome; polyneuritis cranialis; and rare instances of viral myopathy with rhabdomyolysis. The main conclusion of this review is the pressing need to define the neurology of COVID-19, its frequency, manifestations, neuropathology and pathogenesis. On behalf of the World Federation of Neurology we invite national and regional neurological associations to create local databases to report cases with neurological manifestations observed during the on-going pandemic. International neuroepidemiological collaboration may help define the natural history of this worldwide problem.',\n",
       " 'A comprehensive review of the neurological disorders reported during the current COVID-19 pandemic demonstrates that infection with SARS-CoV-2 affects the central nervous system (CNS), the peripheral nervous system (PNS) and the muscle. CNS manifestations include: headache and decreased responsiveness considered initial indicators of potential neurological involvement; anosmia, hyposmia, hypogeusia, and dysgeusia are frequent early symptoms of coronavirus infection. Respiratory failure, the lethal manifestation of COVID-19, responsible for 215,461 deaths worldwide, is probably neurogenic in origin and may result from the viral invasion of cranial nerve I, progressing into rhinencephalon and brainstem respiratory centers. Cerebrovascular disease, in particular large-vessel ischemic strokes, and less frequently cerebral venous thrombosis, intracerebral hemorrhage and subarachnoid hemorrhage, usually occur as part of a thrombotic state induced by viral attachment to ACE2 receptors in endothelium causing widespread endotheliitis, coagulopathy, arterial and venous thromboses. Acute hemorrhagic necrotizing encephalopathy is associated to the cytokine storm. A frontal hypoperfusion syndrome has been identified. There are isolated reports of seizures, encephalopathy, meningitis, encephalitis, and myelitis. The neurological diseases affecting the PNS and muscle in COVID-19 are less frequent and include Guillain-Barré syndrome; Miller Fisher syndrome; polyneuritis cranialis; and rare instances of viral myopathy with rhabdomyolysis. The main conclusion of this review is the pressing need to define the neurology of COVID-19, its frequency, manifestations, neuropathology and pathogenesis. On behalf of the World Federation of Neurology we invite national and regional neurological associations to create local databases to report cases with neurological manifestations observed during the on-going pandemic. International neuroepidemiological collaboration may help define the natural history of this worldwide problem.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'The beginning of 2020 has seen the emergence of COVID-19 outbreak caused by a novel coronavirus, Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). There is an imminent need to better understand this new virus and to develop ways to control its spread. In this study, we sought to gain insights for vaccine design against SARS-CoV-2 by considering the high genetic similarity between SARS-CoV-2 and SARS-CoV, which caused the outbreak in 2003, and leveraging existing immunological studies of SARS-CoV. By screening the experimentally-determined SARS-CoV-derived B cell and T cell epitopes in the immunogenic structural proteins of SARS-CoV, we identified a set of B cell and T cell epitopes derived from the spike (S) and nucleocapsid (N) proteins that map identically to SARS-CoV-2 proteins. As no mutation has been observed in these identified epitopes among the 120 available SARS-CoV-2 sequences (as of 21 February 2020), immune targeting of these epitopes may potentially offer protection against this novel virus. For the T cell epitopes, we performed a population coverage analysis of the associated MHC alleles and proposed a set of epitopes that is estimated to provide broad coverage globally, as well as in China. Our findings provide a screened set of epitopes that can help guide experimental efforts towards the development of vaccines against SARS-CoV-2.',\n",
       " 'Human coronaviruses are known as respiratory pathogens mainly associated with a range of respiratory diseases. In the past 18 years, the beginning of severe acute respiratory syndrome human coronavirus (SARS-HCoV), Middle East respiratory syndrome coronavirus (MERS-CoV) and now the (COVID-19) have driven the HCoV into the platform of the scientific community due to their high pathogenicity in humans. Our study about the HCoV has contributed extensively to the understanding of HCoV pathogenesis its outbreak and precautionary measurements mainly COVID-19. By the 16-February 2020, total existing diagnosis cases of COVID-19, in china now numbered around 57,255, total cumulative diagnosis is about 68,592, total existing suspicion is about 8228, a total severe existing disease 11,272, the total cumulative cure is about 9671 with more than 1666 cumulative deaths have been reported. The purpose of the review is to summarize the present knowledge on the viral diversity, reservoir hosts, and geographical distributions of Human coronavirus (COVID-19) in China.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'SARS-CoV-2, a novel coronavirus (CoV), has recently emerged causing an ongoing outbreak of viral pneumonia around the world. While genetically distinct from the original SARS-CoV, both group 2B CoVs share similar genome organization and origins to coronaviruses harbored in bats. Importantly, initial guidance has used insights from SARS-CoV infection to inform treatment and public health strategies. In this report, we evaluate type-I Interferon (IFN-I) sensitivity of SARS-CoV-2 relative to the original SARS-CoV. Our results indicate that while SARS-CoV-2 maintains similar viral replication kinetics to SARS-CoV in Vero cell, the novel CoV is much more sensitive to IFN-I pretreatment. Examining transcriptional factor activation and interferon stimulated gene (ISG) induction, SARS-CoV-2 in the context of type I IFN induces phosphorylation of STAT1 and increased ISG proteins. In contrast, the original SARS-CoV has no evidence for STAT1 phosphorylation or ISG protein increases even in the presence of type I IFN pretreatment. Next, we examined IFN competent Calu3 2B4 cells finding SARS-CoV-2 had reduced viral replication relative to SARS-CoV and induced STAT1 phosphorylation late during infection. Finally, we examined homology between SARS-CoV and SARS-CoV-2 in viral proteins shown to be interferon antagonist. The absence of open reading frame (ORF) 3b and significant changes to ORF6 suggest the two key IFN antagonists may not maintain equivalent function in SARS-CoV-2. Together, the results identify key differences in susceptibility to the IFN-I response between SARS-CoV and SARS-CoV-2. that could help inform disease progression, treatment options, and animal model development.',\n",
       " 'Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.',\n",
       " 'Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.',\n",
       " 'Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.',\n",
       " 'Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.',\n",
       " 'Background The emergence of a novel coronavirus (SARS-CoV-2) in Wuhan, China in early December 2019 has caused widespread transmission within the country, with over 1,000 deaths reported to date. Other countries have since reported coronavirus disease 2019 (COVID-19) importation from China, with some experiencing local transmission and even case importation from countries outside China. We aim to estimate the number of cases imported from Wuhan to each country or territory outside mainland China, and with these estimates assess the risk of onward local transmission and the relative potential of case importation between countries outside China. Methods We used the reported number of cases imported from Wuhan and flight data to generate an uncertainty distribution for the estimated number of imported cases from Wuhan to each location outside mainland China. This uncertainty was propagated to quantify the local outbreak risk using a branching process model. A COVID-19 introduction index was derived for each pair of donor and recipient countries, accounting for the local outbreak risk in the donor country and the between-country connectivity. Results We identified 13 countries or territories outside mainland China that may have under-detected COVID-19 importation from Wuhan, such as Thailand and Indonesia. In addition, 16 countries had a local outbreak risk estimate exceeding 50%, including four outside Asia. The COVID-19 introduction index highlights potential locations outside mainland China from which cases may be imported to each recipient country. Conclusions As SARS-CoV-2 continues to spread globally, more epicentres may emerge outside China. Hence, it is important for countries to remain alert for the possibilities of viral introduction from other countries outside China, even before local transmission in a source country becomes known.',\n",
       " 'As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.',\n",
       " 'As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.',\n",
       " 'As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.',\n",
       " 'As the number of novel coronavirus cases grows both inside and outside of China, public health authorities require evidence on the effectiveness of control measures such as thermal screening of arrivals at airports. We evaluated the effectiveness of exit and entry screening for 2019-nCoV infection. In our baseline scenario, we estimated that 46.5% (95%CI: 35.9 to 57.7) of infected travellers would not be detected, depending on the incubation period, sensitivity of exit and entry screening, and the proportion of cases which are asymptomatic. Airport screening is unlikely to detect a sufficient proportion of 2019-nCoV infected travellers to avoid entry of infected travellers. We developed an online tool so that results can be updated as new information becomes available.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'A novel coronavirus strain 2019-nCoV has caused a rapid global pandemic-COVID-19. Scientists have taken onto the task of characterizing this new virus and understanding how this virus has transmitted to humans. All preliminary studies have found some striking similarities between this new virus and the SARS-CoV that caused a similar kind of epidemic in 2002–2003. Through bioinformatics tools, a great deal of information has been gathered about the origin, evolution and zoonosis of this virus. We, in this review, report the symptoms, mode of transmission and available and putative treatments to tackle 2019-nCoV infections. We also comprehensively summarize all the information so far made available regarding the genome, evolution and zoonosis of this virus.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " 'About one month after the COVID-19 epidemic peaked in Mainland China and SARS-CoV-2 migrated from China westward to Europe and then the U.S., the epidemiological data begin to provide important insights into the risks associated with the disease and the effectiveness of intervention strategies. Like other respiratory diseases, including the 2003 SARS epidemic, the virus remains only about two months in any given population, yet the peak incidence and the lethality can vary. The data suggest that at least two strains of the 2020 SARS-CoV-2 virus have evolved during its migration from Mainland China to Europe. South Korea, Iran, and Italy were hit by the more dangerous \"SKII\" variant. While the epidemic in China is about to end, and in Europe about to level off, the course of the epidemic in the younger US population is still increasing and the peak level will likely depend on which of the strains has entered the US first. The same models that help us to understand the epidemic also help us to choose prevention strategies. While containment merely prolongs the time the disease circulates until the proportion of immune people is high enough for \"herd immunity\", reducing disease severity, either by vaccination or by early treatment of complications, is the best strategy against a respiratory virus disease.',\n",
       " \"Coronaviruses, seven of which are known to infect humans, can cause a spectrum of clinical presentations ranging from asymptomatic infection to severe illness and death. Four human coronaviruses (hCoVs)-229E, HKU1, NL63 and OC43-circulate globally, commonly infect children and typically cause mild upper respiratory tract infections. Three novel coronaviruses of zoonotic origin have emerged during the past two decades: severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV) and the recently discovered severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which is the cause of the ongoing coronavirus disease 2019 (COVID-19) pandemic. These novel coronaviruses are known to cause severe illness and death predominantly in older adults and those with underlying comorbidities. Consistent with what has been observed during the outbreaks of SARS and MERS, children with COVID-19 are more likely to be asymptomatic or to have mild-to-moderate illness, with few deaths reported in children globally thus far. Clinical symptoms and laboratory and radiological abnormalities in children have been similar to those reported in adults but are generally less severe. A rare multisystem inflammatory syndrome in children (MIS-C) which has resulted in critical illness and some deaths has recently been described. Clinical trials for therapeutics and vaccine development should include paediatric considerations. Children may play an important role in the transmission of infection and outbreak dynamics and could be a key target population for effective measures to control outbreaks. The unintended consequences of the unprecedented scale and duration of pandemic control measures for children and families around the world should be carefully examined. ABBREVIATIONS: 2019-nCoV, 2019 novel coronavirus; ADEM, acute demyelinating encephalomyelitis; AAP, American Academy of Pediatrics; ACE-2, angiotensin-converting enzyme 2; ARDS, acute respiratory distress syndrome; BCG, bacillus Calmette-Guérin; BNP, brain natriuretic peptide; CDC, Centers for Disease Control and Prevention; CRP, C-reactive protein; CSF, cerebrospinal fluid; COVID-19, coronavirus disease 2019; CT, computed tomography; CXR, chest X-ray; DOL, day of life; hCoV, human coronavirus; ICU, intensive care unit; IL, interleukin; IVIG, intravenous immunoglobulin; KD, Kawasaki disease; LDH, lactate dehydrogenase; MERS, Middle East respiratory syndrome; MERS-CoV, Middle East respiratory syndrome coronavirus; MEURI, monitored emergency use of unregistered and experimental interventions; MIS-C, multi-system inflammatory syndrome in children; PCR, polymerase chain reaction; PICU, paediatric intensive care unit; RNA, ribonucleic acid; RCT, randomised-controlled trial; RSV, respiratory syncytial virus; SARS, severe acute respiratory syndrome; SARS-CoV-1, severe acute respiratory syndrome coronavirus 1; SARS-CoV-2, severe acute respiratory syndrome coronavirus 2; TNF-alpha, tumour necrosis factor alpha; UK United Kingdom; UNICEF, United Nations Children's Fund; USA, United States of America; WHO, World Health Organization.\",\n",
       " \"Coronaviruses, seven of which are known to infect humans, can cause a spectrum of clinical presentations ranging from asymptomatic infection to severe illness and death. Four human coronaviruses (hCoVs)-229E, HKU1, NL63 and OC43-circulate globally, commonly infect children and typically cause mild upper respiratory tract infections. Three novel coronaviruses of zoonotic origin have emerged during the past two decades: severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV) and the recently discovered severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which is the cause of the ongoing coronavirus disease 2019 (COVID-19) pandemic. These novel coronaviruses are known to cause severe illness and death predominantly in older adults and those with underlying comorbidities. Consistent with what has been observed during the outbreaks of SARS and MERS, children with COVID-19 are more likely to be asymptomatic or to have mild-to-moderate illness, with few deaths reported in children globally thus far. Clinical symptoms and laboratory and radiological abnormalities in children have been similar to those reported in adults but are generally less severe. A rare multisystem inflammatory syndrome in children (MIS-C) which has resulted in critical illness and some deaths has recently been described. Clinical trials for therapeutics and vaccine development should include paediatric considerations. Children may play an important role in the transmission of infection and outbreak dynamics and could be a key target population for effective measures to control outbreaks. The unintended consequences of the unprecedented scale and duration of pandemic control measures for children and families around the world should be carefully examined. ABBREVIATIONS: 2019-nCoV, 2019 novel coronavirus; ADEM, acute demyelinating encephalomyelitis; AAP, American Academy of Pediatrics; ACE-2, angiotensin-converting enzyme 2; ARDS, acute respiratory distress syndrome; BCG, bacillus Calmette-Guérin; BNP, brain natriuretic peptide; CDC, Centers for Disease Control and Prevention; CRP, C-reactive protein; CSF, cerebrospinal fluid; COVID-19, coronavirus disease 2019; CT, computed tomography; CXR, chest X-ray; DOL, day of life; hCoV, human coronavirus; ICU, intensive care unit; IL, interleukin; IVIG, intravenous immunoglobulin; KD, Kawasaki disease; LDH, lactate dehydrogenase; MERS, Middle East respiratory syndrome; MERS-CoV, Middle East respiratory syndrome coronavirus; MEURI, monitored emergency use of unregistered and experimental interventions; MIS-C, multi-system inflammatory syndrome in children; PCR, polymerase chain reaction; PICU, paediatric intensive care unit; RNA, ribonucleic acid; RCT, randomised-controlled trial; RSV, respiratory syncytial virus; SARS, severe acute respiratory syndrome; SARS-CoV-1, severe acute respiratory syndrome coronavirus 1; SARS-CoV-2, severe acute respiratory syndrome coronavirus 2; TNF-alpha, tumour necrosis factor alpha; UK United Kingdom; UNICEF, United Nations Children's Fund; USA, United States of America; WHO, World Health Organization.\",\n",
       " \"Coronaviruses, seven of which are known to infect humans, can cause a spectrum of clinical presentations ranging from asymptomatic infection to severe illness and death. Four human coronaviruses (hCoVs)-229E, HKU1, NL63 and OC43-circulate globally, commonly infect children and typically cause mild upper respiratory tract infections. Three novel coronaviruses of zoonotic origin have emerged during the past two decades: severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV) and the recently discovered severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which is the cause of the ongoing coronavirus disease 2019 (COVID-19) pandemic. These novel coronaviruses are known to cause severe illness and death predominantly in older adults and those with underlying comorbidities. Consistent with what has been observed during the outbreaks of SARS and MERS, children with COVID-19 are more likely to be asymptomatic or to have mild-to-moderate illness, with few deaths reported in children globally thus far. Clinical symptoms and laboratory and radiological abnormalities in children have been similar to those reported in adults but are generally less severe. A rare multisystem inflammatory syndrome in children (MIS-C) which has resulted in critical illness and some deaths has recently been described. Clinical trials for therapeutics and vaccine development should include paediatric considerations. Children may play an important role in the transmission of infection and outbreak dynamics and could be a key target population for effective measures to control outbreaks. The unintended consequences of the unprecedented scale and duration of pandemic control measures for children and families around the world should be carefully examined. ABBREVIATIONS: 2019-nCoV, 2019 novel coronavirus; ADEM, acute demyelinating encephalomyelitis; AAP, American Academy of Pediatrics; ACE-2, angiotensin-converting enzyme 2; ARDS, acute respiratory distress syndrome; BCG, bacillus Calmette-Guérin; BNP, brain natriuretic peptide; CDC, Centers for Disease Control and Prevention; CRP, C-reactive protein; CSF, cerebrospinal fluid; COVID-19, coronavirus disease 2019; CT, computed tomography; CXR, chest X-ray; DOL, day of life; hCoV, human coronavirus; ICU, intensive care unit; IL, interleukin; IVIG, intravenous immunoglobulin; KD, Kawasaki disease; LDH, lactate dehydrogenase; MERS, Middle East respiratory syndrome; MERS-CoV, Middle East respiratory syndrome coronavirus; MEURI, monitored emergency use of unregistered and experimental interventions; MIS-C, multi-system inflammatory syndrome in children; PCR, polymerase chain reaction; PICU, paediatric intensive care unit; RNA, ribonucleic acid; RCT, randomised-controlled trial; RSV, respiratory syncytial virus; SARS, severe acute respiratory syndrome; SARS-CoV-1, severe acute respiratory syndrome coronavirus 1; SARS-CoV-2, severe acute respiratory syndrome coronavirus 2; TNF-alpha, tumour necrosis factor alpha; UK United Kingdom; UNICEF, United Nations Children's Fund; USA, United States of America; WHO, World Health Organization.\",\n",
       " \"Coronaviruses, seven of which are known to infect humans, can cause a spectrum of clinical presentations ranging from asymptomatic infection to severe illness and death. Four human coronaviruses (hCoVs)-229E, HKU1, NL63 and OC43-circulate globally, commonly infect children and typically cause mild upper respiratory tract infections. Three novel coronaviruses of zoonotic origin have emerged during the past two decades: severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV) and the recently discovered severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) which is the cause of the ongoing coronavirus disease 2019 (COVID-19) pandemic. These novel coronaviruses are known to cause severe illness and death predominantly in older adults and those with underlying comorbidities. Consistent with what has been observed during the outbreaks of SARS and MERS, children with COVID-19 are more likely to be asymptomatic or to have mild-to-moderate illness, with few deaths reported in children globally thus far. Clinical symptoms and laboratory and radiological abnormalities in children have been similar to those reported in adults but are generally less severe. A rare multisystem inflammatory syndrome in children (MIS-C) which has resulted in critical illness and some deaths has recently been described. Clinical trials for therapeutics and vaccine development should include paediatric considerations. Children may play an important role in the transmission of infection and outbreak dynamics and could be a key target population for effective measures to control outbreaks. The unintended consequences of the unprecedented scale and duration of pandemic control measures for children and families around the world should be carefully examined. ABBREVIATIONS: 2019-nCoV, 2019 novel coronavirus; ADEM, acute demyelinating encephalomyelitis; AAP, American Academy of Pediatrics; ACE-2, angiotensin-converting enzyme 2; ARDS, acute respiratory distress syndrome; BCG, bacillus Calmette-Guérin; BNP, brain natriuretic peptide; CDC, Centers for Disease Control and Prevention; CRP, C-reactive protein; CSF, cerebrospinal fluid; COVID-19, coronavirus disease 2019; CT, computed tomography; CXR, chest X-ray; DOL, day of life; hCoV, human coronavirus; ICU, intensive care unit; IL, interleukin; IVIG, intravenous immunoglobulin; KD, Kawasaki disease; LDH, lactate dehydrogenase; MERS, Middle East respiratory syndrome; MERS-CoV, Middle East respiratory syndrome coronavirus; MEURI, monitored emergency use of unregistered and experimental interventions; MIS-C, multi-system inflammatory syndrome in children; PCR, polymerase chain reaction; PICU, paediatric intensive care unit; RNA, ribonucleic acid; RCT, randomised-controlled trial; RSV, respiratory syncytial virus; SARS, severe acute respiratory syndrome; SARS-CoV-1, severe acute respiratory syndrome coronavirus 1; SARS-CoV-2, severe acute respiratory syndrome coronavirus 2; TNF-alpha, tumour necrosis factor alpha; UK United Kingdom; UNICEF, United Nations Children's Fund; USA, United States of America; WHO, World Health Organization.\",\n",
       " 'Severe acute respiratory syndrome (SARS) first appeared in Guangdong Province, China, in November 2002. Although virus isolation and serology were useful early in the SARS outbreak for diagnosing new cases, these tests are not generally useful because virus culture requires a BSL-3 laboratory and seroconversion is often delayed until 2 to 3 weeks after infection. The first qualitative reverse transcriptase-polymerase chain reaction tests for SARS-coronavirus (CoV) were sensitive and capable of detecting 1 to 10 genome equivalents. These assays were quickly supplemented with quantitative real-time assays that helped elucidate the natural history of SARS, particularly the initial presence of low viral loads in the upper respiratory tract and high viral loads in the lower respiratory tract. The unique natural history of SARS-CoV infection dictates the testing of both respiratory and nonrespiratory specimens, the testing of multiple specimens from the same patient, and sending out positives to be confirmed by a reference laboratory. Commercially available reverse transcriptase-polymerase chain reaction tests for SARS have recently appeared; however, meaningful evaluations of these assays have not yet been performed and their true performance has not been determined. These and other issues related to diagnosis of SARS-CoV infection are discussed in this review.',\n",
       " 'Severe acute respiratory syndrome (SARS) first appeared in Guangdong Province, China, in November 2002. Although virus isolation and serology were useful early in the SARS outbreak for diagnosing new cases, these tests are not generally useful because virus culture requires a BSL-3 laboratory and seroconversion is often delayed until 2 to 3 weeks after infection. The first qualitative reverse transcriptase-polymerase chain reaction tests for SARS-coronavirus (CoV) were sensitive and capable of detecting 1 to 10 genome equivalents. These assays were quickly supplemented with quantitative real-time assays that helped elucidate the natural history of SARS, particularly the initial presence of low viral loads in the upper respiratory tract and high viral loads in the lower respiratory tract. The unique natural history of SARS-CoV infection dictates the testing of both respiratory and nonrespiratory specimens, the testing of multiple specimens from the same patient, and sending out positives to be confirmed by a reference laboratory. Commercially available reverse transcriptase-polymerase chain reaction tests for SARS have recently appeared; however, meaningful evaluations of these assays have not yet been performed and their true performance has not been determined. These and other issues related to diagnosis of SARS-CoV infection are discussed in this review.',\n",
       " 'Severe acute respiratory syndrome (SARS) first appeared in Guangdong Province, China, in November 2002. Although virus isolation and serology were useful early in the SARS outbreak for diagnosing new cases, these tests are not generally useful because virus culture requires a BSL-3 laboratory and seroconversion is often delayed until 2 to 3 weeks after infection. The first qualitative reverse transcriptase-polymerase chain reaction tests for SARS-coronavirus (CoV) were sensitive and capable of detecting 1 to 10 genome equivalents. These assays were quickly supplemented with quantitative real-time assays that helped elucidate the natural history of SARS, particularly the initial presence of low viral loads in the upper respiratory tract and high viral loads in the lower respiratory tract. The unique natural history of SARS-CoV infection dictates the testing of both respiratory and nonrespiratory specimens, the testing of multiple specimens from the same patient, and sending out positives to be confirmed by a reference laboratory. Commercially available reverse transcriptase-polymerase chain reaction tests for SARS have recently appeared; however, meaningful evaluations of these assays have not yet been performed and their true performance has not been determined. These and other issues related to diagnosis of SARS-CoV infection are discussed in this review.',\n",
       " 'Severe acute respiratory syndrome (SARS) first appeared in Guangdong Province, China, in November 2002. Although virus isolation and serology were useful early in the SARS outbreak for diagnosing new cases, these tests are not generally useful because virus culture requires a BSL-3 laboratory and seroconversion is often delayed until 2 to 3 weeks after infection. The first qualitative reverse transcriptase-polymerase chain reaction tests for SARS-coronavirus (CoV) were sensitive and capable of detecting 1 to 10 genome equivalents. These assays were quickly supplemented with quantitative real-time assays that helped elucidate the natural history of SARS, particularly the initial presence of low viral loads in the upper respiratory tract and high viral loads in the lower respiratory tract. The unique natural history of SARS-CoV infection dictates the testing of both respiratory and nonrespiratory specimens, the testing of multiple specimens from the same patient, and sending out positives to be confirmed by a reference laboratory. Commercially available reverse transcriptase-polymerase chain reaction tests for SARS have recently appeared; however, meaningful evaluations of these assays have not yet been performed and their true performance has not been determined. These and other issues related to diagnosis of SARS-CoV infection are discussed in this review.',\n",
       " 'Severe acute respiratory syndrome (SARS) first appeared in Guangdong Province, China, in November 2002. Although virus isolation and serology were useful early in the SARS outbreak for diagnosing new cases, these tests are not generally useful because virus culture requires a BSL-3 laboratory and seroconversion is often delayed until 2 to 3 weeks after infection. The first qualitative reverse transcriptase-polymerase chain reaction tests for SARS-coronavirus (CoV) were sensitive and capable of detecting 1 to 10 genome equivalents. These assays were quickly supplemented with quantitative real-time assays that helped elucidate the natural history of SARS, particularly the initial presence of low viral loads in the upper respiratory tract and high viral loads in the lower respiratory tract. The unique natural history of SARS-CoV infection dictates the testing of both respiratory and nonrespiratory specimens, the testing of multiple specimens from the same patient, and sending out positives to be confirmed by a reference laboratory. Commercially available reverse transcriptase-polymerase chain reaction tests for SARS have recently appeared; however, meaningful evaluations of these assays have not yet been performed and their true performance has not been determined. These and other issues related to diagnosis of SARS-CoV infection are discussed in this review.',\n",
       " 'Coronaviridae (CoVs) are single stranded RNA viruses. βeta‐CoVs encompasses Bat coronavirus (BCoV), the human Severe Acute Respiratory Syndrome (SARS) virus and the Middle Eastern Respiratory Syndrome (MERS) virus. In December 2019, a new βeta‐CoVs was identified in Wuhan, China, SARS‐CoV‐2. The genome sequencing demonstrated a 79.5% and 96.2% identity to SARS‐CoV and a bat CoV genome, respectively, supporting the interspecies transmission and the origin of this new pandemia.',\n",
       " 'The first major pandemic of the new millennium that arose from southern China in 2002 was of zoonotic origin from wild game animals, called severe acute respiratory syndrome [SARS]. The culprit was determined to be a coronavirus of animal origin [SARS-CoV]. The discovery of the SARS-CoV, which caused an outbreak of >8000 people in >30 countries with fatality of about 10%, resulted in intense search for novel coronaviruses with potentially high pathogenicity. Ten years later after the SARS pandemic, another novel coronavirus crossed the species barrier from bats to humans through an intermediate camel host, to produce a severe lower respiratory infection labeled the Middle East respiratory syndrome [MERS]. A novel coronavirus [MERS-CoV] was first identified in September 2012, from patients who resided or traveled to Saudi Arabia. The MERS-CoV spread through contacts with camel and subsequently from human to human via droplet transmission. MERS cases occurred in several other countries including in Europe and the United States, mainly from residence or travel to the Arabian Peninsula, but was not of pandemic potential. However, in the spring of 2015, a MERS outbreak started in South Korea which was initiated by a returning traveler from Saudi Arabia, and subsequently secondary infection of over 186 local residents occurred. Recent estimate in May 2015 indicates that the MERS-CoV have afflicted 1167 patients with MERS worldwide with 479 deaths [41% fatality]. Thus MERS is more deadly than SARS but appears to be less contagious. However, unlike SARS which has not reappeared since 2002–2003, MERS-CoV have the potential to cause sporadic or local outbreaks for many years as the virus may now be entrenched endemically in dromedary camels of the Middle East.',\n",
       " 'The first major pandemic of the new millennium that arose from southern China in 2002 was of zoonotic origin from wild game animals, called severe acute respiratory syndrome [SARS]. The culprit was determined to be a coronavirus of animal origin [SARS-CoV]. The discovery of the SARS-CoV, which caused an outbreak of >8000 people in >30 countries with fatality of about 10%, resulted in intense search for novel coronaviruses with potentially high pathogenicity. Ten years later after the SARS pandemic, another novel coronavirus crossed the species barrier from bats to humans through an intermediate camel host, to produce a severe lower respiratory infection labeled the Middle East respiratory syndrome [MERS]. A novel coronavirus [MERS-CoV] was first identified in September 2012, from patients who resided or traveled to Saudi Arabia. The MERS-CoV spread through contacts with camel and subsequently from human to human via droplet transmission. MERS cases occurred in several other countries including in Europe and the United States, mainly from residence or travel to the Arabian Peninsula, but was not of pandemic potential. However, in the spring of 2015, a MERS outbreak started in South Korea which was initiated by a returning traveler from Saudi Arabia, and subsequently secondary infection of over 186 local residents occurred. Recent estimate in May 2015 indicates that the MERS-CoV have afflicted 1167 patients with MERS worldwide with 479 deaths [41% fatality]. Thus MERS is more deadly than SARS but appears to be less contagious. However, unlike SARS which has not reappeared since 2002–2003, MERS-CoV have the potential to cause sporadic or local outbreaks for many years as the virus may now be entrenched endemically in dromedary camels of the Middle East.',\n",
       " 'The first major pandemic of the new millennium that arose from southern China in 2002 was of zoonotic origin from wild game animals, called severe acute respiratory syndrome [SARS]. The culprit was determined to be a coronavirus of animal origin [SARS-CoV]. The discovery of the SARS-CoV, which caused an outbreak of >8000 people in >30 countries with fatality of about 10%, resulted in intense search for novel coronaviruses with potentially high pathogenicity. Ten years later after the SARS pandemic, another novel coronavirus crossed the species barrier from bats to humans through an intermediate camel host, to produce a severe lower respiratory infection labeled the Middle East respiratory syndrome [MERS]. A novel coronavirus [MERS-CoV] was first identified in September 2012, from patients who resided or traveled to Saudi Arabia. The MERS-CoV spread through contacts with camel and subsequently from human to human via droplet transmission. MERS cases occurred in several other countries including in Europe and the United States, mainly from residence or travel to the Arabian Peninsula, but was not of pandemic potential. However, in the spring of 2015, a MERS outbreak started in South Korea which was initiated by a returning traveler from Saudi Arabia, and subsequently secondary infection of over 186 local residents occurred. Recent estimate in May 2015 indicates that the MERS-CoV have afflicted 1167 patients with MERS worldwide with 479 deaths [41% fatality]. Thus MERS is more deadly than SARS but appears to be less contagious. However, unlike SARS which has not reappeared since 2002–2003, MERS-CoV have the potential to cause sporadic or local outbreaks for many years as the virus may now be entrenched endemically in dromedary camels of the Middle East.',\n",
       " 'The first major pandemic of the new millennium that arose from southern China in 2002 was of zoonotic origin from wild game animals, called severe acute respiratory syndrome [SARS]. The culprit was determined to be a coronavirus of animal origin [SARS-CoV]. The discovery of the SARS-CoV, which caused an outbreak of >8000 people in >30 countries with fatality of about 10%, resulted in intense search for novel coronaviruses with potentially high pathogenicity. Ten years later after the SARS pandemic, another novel coronavirus crossed the species barrier from bats to humans through an intermediate camel host, to produce a severe lower respiratory infection labeled the Middle East respiratory syndrome [MERS]. A novel coronavirus [MERS-CoV] was first identified in September 2012, from patients who resided or traveled to Saudi Arabia. The MERS-CoV spread through contacts with camel and subsequently from human to human via droplet transmission. MERS cases occurred in several other countries including in Europe and the United States, mainly from residence or travel to the Arabian Peninsula, but was not of pandemic potential. However, in the spring of 2015, a MERS outbreak started in South Korea which was initiated by a returning traveler from Saudi Arabia, and subsequently secondary infection of over 186 local residents occurred. Recent estimate in May 2015 indicates that the MERS-CoV have afflicted 1167 patients with MERS worldwide with 479 deaths [41% fatality]. Thus MERS is more deadly than SARS but appears to be less contagious. However, unlike SARS which has not reappeared since 2002–2003, MERS-CoV have the potential to cause sporadic or local outbreaks for many years as the virus may now be entrenched endemically in dromedary camels of the Middle East.',\n",
       " 'The first major pandemic of the new millennium that arose from southern China in 2002 was of zoonotic origin from wild game animals, called severe acute respiratory syndrome [SARS]. The culprit was determined to be a coronavirus of animal origin [SARS-CoV]. The discovery of the SARS-CoV, which caused an outbreak of >8000 people in >30 countries with fatality of about 10%, resulted in intense search for novel coronaviruses with potentially high pathogenicity. Ten years later after the SARS pandemic, another novel coronavirus crossed the species barrier from bats to humans through an intermediate camel host, to produce a severe lower respiratory infection labeled the Middle East respiratory syndrome [MERS]. A novel coronavirus [MERS-CoV] was first identified in September 2012, from patients who resided or traveled to Saudi Arabia. The MERS-CoV spread through contacts with camel and subsequently from human to human via droplet transmission. MERS cases occurred in several other countries including in Europe and the United States, mainly from residence or travel to the Arabian Peninsula, but was not of pandemic potential. However, in the spring of 2015, a MERS outbreak started in South Korea which was initiated by a returning traveler from Saudi Arabia, and subsequently secondary infection of over 186 local residents occurred. Recent estimate in May 2015 indicates that the MERS-CoV have afflicted 1167 patients with MERS worldwide with 479 deaths [41% fatality]. Thus MERS is more deadly than SARS but appears to be less contagious. However, unlike SARS which has not reappeared since 2002–2003, MERS-CoV have the potential to cause sporadic or local outbreaks for many years as the virus may now be entrenched endemically in dromedary camels of the Middle East.',\n",
       " \"Recent outbreak of 2019-nCoV in Wuhan raised serious public health concerns. By February 15, 2020 in Wuhan, the total number of confirmed infection cases has reached 37,914, and the number of deaths has reached 1123, accounting for 56.9% of the total confirmed cases and 73.7% of the total deaths in China. People are eager to know when the epidemic will be completely controlled and when people's work and life will be on the right track. In this study we analyzed the epidemic dynamics and trend of 2019-nCoV in Wuhan by using the data after the closure of Wuhan city till February 12, 2020 based on the SEIR modeling method. The optimal parameters were estimated as R0=1.44 (interquartile range: 1.40-1.47)，TI=14 (interquartile range: 14-14) and TE=3.0 (interquartile range: 2.8-3.1). Based on these parameters, the number of infected individuals in Wuhan city may reach the peak around February 19 at about 45,000 people. Once entering March, the epidemic would gradually decline, and end around the late March. It is worth noting that the above prediction is based on the assumption that the number of susceptible population N = 200,000 will not increase. If the epidemic situation is not properly controlled, the peak of infected number can be further increased and the peak time will be a little postponed. It was expected that the epidemic would subside in early March, and disappear gradually towards the late March.\",\n",
       " \"Recent outbreak of 2019-nCoV in Wuhan raised serious public health concerns. By February 15, 2020 in Wuhan, the total number of confirmed infection cases has reached 37,914, and the number of deaths has reached 1123, accounting for 56.9% of the total confirmed cases and 73.7% of the total deaths in China. People are eager to know when the epidemic will be completely controlled and when people's work and life will be on the right track. In this study we analyzed the epidemic dynamics and trend of 2019-nCoV in Wuhan by using the data after the closure of Wuhan city till February 12, 2020 based on the SEIR modeling method. The optimal parameters were estimated as R0=1.44 (interquartile range: 1.40-1.47)，TI=14 (interquartile range: 14-14) and TE=3.0 (interquartile range: 2.8-3.1). Based on these parameters, the number of infected individuals in Wuhan city may reach the peak around February 19 at about 45,000 people. Once entering March, the epidemic would gradually decline, and end around the late March. It is worth noting that the above prediction is based on the assumption that the number of susceptible population N = 200,000 will not increase. If the epidemic situation is not properly controlled, the peak of infected number can be further increased and the peak time will be a little postponed. It was expected that the epidemic would subside in early March, and disappear gradually towards the late March.\",\n",
       " \"This current volume is, in many ways, a 2020 update to the important 1999–2000 compendium by Sir Fed Hoyle and Professor N. Chandra Wickramasinghe's “Astronomical Origins of life: Steps towards Panspermia.” The emerging new paradigm of biology that connects life on Earth with the wider cosmos is covered in considerable depth showing that terrestrial biological evolution is best understood as a cosmically derived habitat and an interconnected genetic system. The various chapters here discuss all aspects of this interconnectedness, particularly relevant now in this time of the coronavirus pandemic (COVID-19) as the human race reacts to the many microbes and viral pathogens that arrive regularly from space.\",\n",
       " 'Viruses have been infecting their host cells since the dawn of life, and this extremely long-term coevolution gave rise to some surprising consequences for the entire tree of life. It is hypothesised that viruses might have contributed to the formation of the first cellular life form, or that even the eukaryotic cell nucleus originates from an infection by a coated virus. The continuous struggle between viruses and their hosts to maintain at least a constant fitness level led to the development of an unceasing arms race, where weapons are often shuttled between the participants. In this literature review we try to give a short insight into some general consequences or traits of virus–host coevolution, and after this we zoom in to the viral clades of adenoviruses, herpesviruses, nucleo-cytoplasmic large DNA viruses, polyomaviruses and, finally, circoviruses.',\n",
       " 'Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is one of the major threats to the healthcare systems in some countries, especially in the Arabian Peninsula. MERS-CoV is considered an ideal example of the One Health concept. This is due to the animals, especially dromedary camels, play important roles in the transmission and sustainability of the virus, and the virus can be transmitted through aerosols of infected patients into the environment. However, there is some debate regarding the origin of MERS-CoV either from bats or other unknown reservoirs. The dromedary camel is the only identified animal reservoir to date. These animals play important roles in sustaining the virus in certain communities and may act as an amplifier of the virus by secreting it in their body fluids, especially in nasal and rectal discharges. MERS-CoV has been detected in the nasal and rectal secretions of infected camels, and MERS-CoV of this origin has full capacity to infect human airway epithelium in both in vitro and in vivo models. Other evidence confirms the direct transmission of MERS-CoV from camels to humans, though the role of camel meat and milk products has yet to be well studied. Human-to-human transmission is well documented through contact with an active infected patient or some silently infected persons. Furthermore, there are some significant risk factors of individuals in close contact with a positive MERS-CoV patient, including sleeping in the same patient room, removing patient waste (urine, stool, and sputum), and touching respiratory secretions from the index case. Outbreaks within family clusters have been reported, whereby some blood relative patients were infected through their wives in the same house were not infected. Some predisposing genetic factors favor MERS-CoV infection in some patients, which is worth investigating in the near future. The presence of other comorbidities may be another factor. Overall, there are many unknown/confirmed aspects of the virus/human/animal network. Here, the most recent advances in this context are discussed, and the possible reasons behind the emergence and sustainability of MERS-CoV in certain regions are presented. Identification of the exact mechanism of transmission of MERS-CoV from camels to humans and searching for new reservoir/s are of high priority. This will reduce the shedding of the virus into the environment, and thus the risk of human infection can be mitigated.',\n",
       " 'Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is one of the major threats to the healthcare systems in some countries, especially in the Arabian Peninsula. MERS-CoV is considered an ideal example of the One Health concept. This is due to the animals, especially dromedary camels, play important roles in the transmission and sustainability of the virus, and the virus can be transmitted through aerosols of infected patients into the environment. However, there is some debate regarding the origin of MERS-CoV either from bats or other unknown reservoirs. The dromedary camel is the only identified animal reservoir to date. These animals play important roles in sustaining the virus in certain communities and may act as an amplifier of the virus by secreting it in their body fluids, especially in nasal and rectal discharges. MERS-CoV has been detected in the nasal and rectal secretions of infected camels, and MERS-CoV of this origin has full capacity to infect human airway epithelium in both in vitro and in vivo models. Other evidence confirms the direct transmission of MERS-CoV from camels to humans, though the role of camel meat and milk products has yet to be well studied. Human-to-human transmission is well documented through contact with an active infected patient or some silently infected persons. Furthermore, there are some significant risk factors of individuals in close contact with a positive MERS-CoV patient, including sleeping in the same patient room, removing patient waste (urine, stool, and sputum), and touching respiratory secretions from the index case. Outbreaks within family clusters have been reported, whereby some blood relative patients were infected through their wives in the same house were not infected. Some predisposing genetic factors favor MERS-CoV infection in some patients, which is worth investigating in the near future. The presence of other comorbidities may be another factor. Overall, there are many unknown/confirmed aspects of the virus/human/animal network. Here, the most recent advances in this context are discussed, and the possible reasons behind the emergence and sustainability of MERS-CoV in certain regions are presented. Identification of the exact mechanism of transmission of MERS-CoV from camels to humans and searching for new reservoir/s are of high priority. This will reduce the shedding of the virus into the environment, and thus the risk of human infection can be mitigated.',\n",
       " 'The recent emergence of a new coronavirus, COVID-19, has gained extensive coverage in public media and global news. As of 24 March 2020, the virus has caused viral pneumonia in tens of thousands of people in Wuhan, China, and thousands of cases in 184 other countries and territories. This study explores the potential use of Google Trends (GT) to monitor worldwide interest in this COVID-19 epidemic. GT was chosen as a source of reverse engineering data, given the interest in the topic. Current data on COVID-19 is retrieved from (GT) using one main search topic: Coronavirus. Geographical settings for GT are worldwide, China, South Korea, Italy and Iran. The reported period is 15 January 2020 to 24 March 2020. The results show that the highest worldwide peak in the first wave of demand for information was on 31 January 2020. After the first peak, the number of new cases reported daily rose for 6 days. A second wave started on 21 February 2020 after the outbreaks were reported in Italy, with the highest peak on 16 March 2020. The second wave is six times as big as the first wave. The number of new cases reported daily is rising day by day. This short communication gives a brief introduction to how the demand for information on coronavirus epidemic is reported through GT.',\n",
       " 'The recent emergence of a new coronavirus, COVID-19, has gained extensive coverage in public media and global news. As of 24 March 2020, the virus has caused viral pneumonia in tens of thousands of people in Wuhan, China, and thousands of cases in 184 other countries and territories. This study explores the potential use of Google Trends (GT) to monitor worldwide interest in this COVID-19 epidemic. GT was chosen as a source of reverse engineering data, given the interest in the topic. Current data on COVID-19 is retrieved from (GT) using one main search topic: Coronavirus. Geographical settings for GT are worldwide, China, South Korea, Italy and Iran. The reported period is 15 January 2020 to 24 March 2020. The results show that the highest worldwide peak in the first wave of demand for information was on 31 January 2020. After the first peak, the number of new cases reported daily rose for 6 days. A second wave started on 21 February 2020 after the outbreaks were reported in Italy, with the highest peak on 16 March 2020. The second wave is six times as big as the first wave. The number of new cases reported daily is rising day by day. This short communication gives a brief introduction to how the demand for information on coronavirus epidemic is reported through GT.',\n",
       " 'The recent emergence of a new coronavirus, COVID-19, has gained extensive coverage in public media and global news. As of 24 March 2020, the virus has caused viral pneumonia in tens of thousands of people in Wuhan, China, and thousands of cases in 184 other countries and territories. This study explores the potential use of Google Trends (GT) to monitor worldwide interest in this COVID-19 epidemic. GT was chosen as a source of reverse engineering data, given the interest in the topic. Current data on COVID-19 is retrieved from (GT) using one main search topic: Coronavirus. Geographical settings for GT are worldwide, China, South Korea, Italy and Iran. The reported period is 15 January 2020 to 24 March 2020. The results show that the highest worldwide peak in the first wave of demand for information was on 31 January 2020. After the first peak, the number of new cases reported daily rose for 6 days. A second wave started on 21 February 2020 after the outbreaks were reported in Italy, with the highest peak on 16 March 2020. The second wave is six times as big as the first wave. The number of new cases reported daily is rising day by day. This short communication gives a brief introduction to how the demand for information on coronavirus epidemic is reported through GT.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'Summary Background Whether the patients with COVID-19 infected by SARS-CoV-2 would commonly develop acute renal function damage is a problem worthy of clinical attention. This study aimed to explore the effects of SARS-CoV-2 infection on renal function through analyzing the clinical data of 116 hospitalized COVID-19-confirmed patients. Methods 116 hospitalized COVID-19-confirmed patients enrolled in this study were hospitalized in the Department of Infectious Diseases, Renmin Hospital of Wuhan University from January 14 to February 13, 2020. The recorded information includes demographic data, medical history, contact history, potential comorbidities, symptoms, signs, laboratory test results, chest computer tomography (CT) scans, and treatment measures. SARS-CoV-2 RNA in 53 urine sediments of enrolled patients was examined by real-time RT-PCR. Findings 12 (10.8%) and 8 (7.2%) patients showed mild elevation of blood urea nitrogen or creatinine, and trace or 1+ albuminuria respectively in 111 COVID-19-confirmed patients without basic kidney disease. In addition, 5 patients with chronic renal failure (CRF) were undergone regular continuous renal replacement therapy (CRRT) were confirmed infection of SARS-CoV-2, and diagnosed as COVID-19. Beside the treatment of COVID-19, CRRT was also applied three times weekly. The course of treatment, the renal function indicators showed stable, without exacerbation of CRF, and pulmonary inflammation was gradually absorbed. All 5 patients with CRF were survived. Moreover, SARS-CoV-2 RNA in urine sediments was positive only in 3 patients from 48 cases without renal illness before, and one patient had a positive for SARS-CoV-2 ORF 1ab from 5 cases with CRF. Interpretation Acute renal impairment was uncommon in COVID-19. SARS-CoV-2 infection does not significantly cause obvious acute renal injury, or aggravate CRF in the COVID-19 patients.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'BACKGROUND: Similar to outbreaks of many other infectious diseases, success in controlling the novel 2019 coronavirus infection requires a timely and accurate monitoring of the epidemic, particularly during its early period with rather limited data while the need for information increases explosively. METHODS: In this study, we used a second derivative model to characterize the coronavirus epidemic in China with cumulatively diagnosed cases during the first 2 months. The analysis was further enhanced by an exponential model with a close-population assumption. This model was built with the data and used to assess the detection rate during the study period, considering the differences between the true infections, detectable and detected cases. RESULTS: Results from the second derivative modeling suggest the coronavirus epidemic as nonlinear and chaotic in nature. Although it emerged gradually, the epidemic was highly responsive to massive interventions initiated on January 21, 2020, as indicated by results from both second derivative and exponential modeling analyses. The epidemic started to decelerate immediately after the massive actions. The results derived from our analysis signaled the decline of the epidemic 14 days before it eventually occurred on February 4, 2020. Study findings further signaled an accelerated decline in the epidemic starting in 14 days on February 18, 2020. CONCLUSIONS: The coronavirus epidemic appeared to be nonlinear and chaotic, and was responsive to effective interventions. The methods used in this study can be applied in surveillance to inform and encourage the general public, public health professionals, clinicians and decision-makers to take coordinative and collaborative efforts to control the epidemic.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'COVID-19 is an acute respiratory disease that has been classified as a pandemic by the World Health Organization. Information regarding this particular disease is limited, however, it is known to have high mortality rates, particularly among individuals with preexisting medical conditions. Creating models to identify individuals who are at the greatest risk for severe complications due to COVID-19 will be useful to help for outreach campaigns in mitigating the diseases worst effects. While information specific to COVID-19 is limited, a model using complications due to other upper respiratory infections can be used as a proxy to help identify those individuals who are at the greatest risk. We present the results for three models predicting such complications, with each model having varying levels of predictive effectiveness at the expense of ease of implementation.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'Abstract This chapter discusses infections of rats with viruses in the following 14 virus families: Adenoviridae, Arenaviridae, Coronaviridae, Flaviviridae, Hantaviridae, Hepeviridae, Herpesviridae, Paramyxoviridae, Parvoviridae, Picornaviridae, Pneumoviridae, Polyomaviridae, Poxviridae, and Reoviridae. Serological surveys indicate that parvoviruses, coronaviruses, cardioviruses, and pneumoviruses are the most prevalent in laboratory rats. A new polyomavirus and a new cardiovirus that cause disease in laboratory rats are described. Metagenomic analyses of feces or intestinal contents from wild rats have detected viruses from an additional nine virus families that could potentially cause infections in laboratory rats.',\n",
       " 'The effect of COVID-19 on biomedical publishing (BP) (i.e. scientific biomedical periodicals continuously published by research communities or commercial publishers) has not been deeply explored. To estimate the immediate COVID-19 impact on BP, we have assessed PubMed-indexed articles about COVID-19 (PMIAC) from December 2019 to April 2020. PMIAC have been classified according to publication date, country, and journals for evaluation of time-, region- and scientometric-dependant impact of COVID-19 on BP and have been curated manually (i.e. each entry has been individually analyzed). PMIAC analysis reflects geographic and temporal parameters of outbreak spread. A major BP problem is related to the fact that only 40% of articles report/review/analyze data. Another BP weakness is the clusterization of \"highly-trusted\" publications according to countries of origin and \"highly impacting\" journals. Finally, a problem highlighted by COVID-19 crisis is the increased specification of biomedical research. To solve the problem, analytical reviews integrating data from different areas of biology and medicine are required. The data on PMIAC suggest priority of \"what is published\" over \"where it is published\" and \"who are the authors\". We believe that our brief analysis may help to shape forthcoming BP to become more effective in solving immediate problems resulted from global threats.',\n",
       " '',\n",
       " \"2019 novel coronavirus (SARS-CoV-2), which originated in Wuhan, China, has attracted the world's attention over the last month. The Chinese government has taken emergency measures to control the outbreak and has undertaken initial steps in the diagnosis and treatment of 2019 novel coronavirus infection disease (COVID-19). However, SARS-CoV-2 possesses powerful pathogenicity as well as transmissibility and still holds many mysteries that are yet to be solved, such as whether the virus can be transmitted by asymptomatic patients or by mothers to their infants. Our research presents selected available cases of COVID-19 in China to better understand the transmission and diagnosis regarding this infectious disease.\",\n",
       " \"2019 novel coronavirus (SARS-CoV-2), which originated in Wuhan, China, has attracted the world's attention over the last month. The Chinese government has taken emergency measures to control the outbreak and has undertaken initial steps in the diagnosis and treatment of 2019 novel coronavirus infection disease (COVID-19). However, SARS-CoV-2 possesses powerful pathogenicity as well as transmissibility and still holds many mysteries that are yet to be solved, such as whether the virus can be transmitted by asymptomatic patients or by mothers to their infants. Our research presents selected available cases of COVID-19 in China to better understand the transmission and diagnosis regarding this infectious disease.\",\n",
       " '',\n",
       " 'Human viral hepatitis, a major cause of morbidity and mortality worldwide, is caused by highly diverse viruses with different genetic, ecological, and pathogenetic features. Technological advances that allow throughput sequencing of viral genomes, as well as the development of computational tools to analyze such genome data, have largely expanded our knowledge on the host range and evolutionary history of human hepatitis viruses. Thus, with the exclusion of hepatitis D virus, close or distant relatives of these human pathogens were identified in a number of domestic and wild mammals. Also, sequences of human viral strains isolated from different geographic locations and over different time-spans have allowed the application of phylogeographic and molecular dating approaches to large viral phylogenies. In this review, we summarize the most recent insights into our understanding of the evolutionary events and ecological contexts that determined the origin and spread of human hepatitis viruses.',\n",
       " 'The novel coronavirus, SARS-CoV-2, causes the unfathomable pandemic in the history of humankind. Bangladesh is also a victim of this critical situation. To investigate the genomic features of the pathogen, the first complete genome of the virus has very recently been published. Therefore, the long awaiting questions regarding the possible origin and typing of the strain(s) can now be answered. Here, we endeavor to mainly discuss the published reports or online-accessed data (results) regarding those issues and presented a comprehensive picture of the typing of the virus alongside the probable origin of the sub-clade containing Bangladeshi strain. Our observation suggested that this strain might have originated from the United Kingdom (UK) or other European countries epidemiologically linked to the UK. According to different genotyping classification schemes, this strain belongs to A2a clade under G major clade, is of B and/or L type, and is a SARS-CoV-2a sub-strain. In the forwarding days, randomized genome data will certainly voluminate in Bangladesh, however because of globalization and immigrant movements, we urgently need a mass regional sequencing approach targeting the partial or complete genome that can link the epidemiological data and may help in further clinical interventions. This article is protected by copyright. All rights reserved.',\n",
       " 'The novel coronavirus, SARS-CoV-2, causes the unfathomable pandemic in the history of humankind. Bangladesh is also a victim of this critical situation. To investigate the genomic features of the pathogen, the first complete genome of the virus has very recently been published. Therefore, the long awaiting questions regarding the possible origin and typing of the strain(s) can now be answered. Here, we endeavor to mainly discuss the published reports or online-accessed data (results) regarding those issues and presented a comprehensive picture of the typing of the virus alongside the probable origin of the sub-clade containing Bangladeshi strain. Our observation suggested that this strain might have originated from the United Kingdom (UK) or other European countries epidemiologically linked to the UK. According to different genotyping classification schemes, this strain belongs to A2a clade under G major clade, is of B and/or L type, and is a SARS-CoV-2a sub-strain. In the forwarding days, randomized genome data will certainly voluminate in Bangladesh, however because of globalization and immigrant movements, we urgently need a mass regional sequencing approach targeting the partial or complete genome that can link the epidemiological data and may help in further clinical interventions. This article is protected by copyright. All rights reserved.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " 'Background Since December 29, 2019, pneumonia infection with 2019-nCoV has rapidly spread out from Wuhan, Hubei Province, China to most others provinces and other counties. However, the transmission dynamics of 2019-nCoV remain unclear. Methods Data of confirmed 2019-nCoV cases before January 23, 2020 were collected from medical records, epidemiological investigations or official websites. Data of severe acute respiratory syndrome (SARS) cases in Guangdong Province during 2002-2003 were obtained from Guangdong Provincial Center for Disease Control and Prevention (GDCDC). Exponential Growth (EG) and maximum likelihood estimation (ML) were applied to estimate the reproductive number (R) of 2019-nCoV and SARS. Findings As of January 23, 2020, a total of 830 confirmed 2019-nCoV cases were identified across China, and 9 cases were reported overseas. The average incubation duration of 2019-nCoV infection was 4.8days. The average period from onset of symptoms to isolation of 2019-nCoV and SARS cases were 2.9 and 4.2 days, respectively. The R values of 2019-nCoV were 2.90 (95%CI: 2.32-3.63) and 2.92 (95%CI: 2.28-3.67) estimated using EG and ML respectively, while the corresponding R values of SARS-CoV were 1.77 (95%CI: 1.37-2.27) and 1.85 (95%CI: 1.32-2.49). We observe a decreasing trend of the period from onset to isolation and R values of both 2019-nCoV and SARS-CoV. Interpretation The 2019-nCoV may have a higher pandemic risk than SARS broken out in 2003. The implemented public-health efforts have significantly decreased the pandemic risk of 2019-nCoV. However, more rigorous control and prevention strategies and measures to contain its further spread. Funding National Key Research and Development Program of China, Science and Technology Program of Guangdong Province, and Guangzhou Science and technology Plan Project.',\n",
       " '',\n",
       " '',\n",
       " 'The novel human coronavirus disease COVID-19 has become the fifth documented pandemic since the 1918 flu pandemic. COVID-19 was first reported in Wuhan, China, and subsequently spread worldwide. The coronavirus was officially named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses based on phylogenetic analysis. SARS-CoV-2 is believed to be a spillover of an animal coronavirus and later adapted the ability of human-to-human transmission. Because the virus is highly contagious, it rapidly spreads and continuously evolves in the human population. In this review article, we discuss the basic properties, potential origin, and evolution of the novel human coronavirus. These factors may be critical for studies of pathogenicity, antiviral designs, and vaccine development against the virus.',\n",
       " 'The novel human coronavirus disease COVID-19 has become the fifth documented pandemic since the 1918 flu pandemic. COVID-19 was first reported in Wuhan, China, and subsequently spread worldwide. The coronavirus was officially named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses based on phylogenetic analysis. SARS-CoV-2 is believed to be a spillover of an animal coronavirus and later adapted the ability of human-to-human transmission. Because the virus is highly contagious, it rapidly spreads and continuously evolves in the human population. In this review article, we discuss the basic properties, potential origin, and evolution of the novel human coronavirus. These factors may be critical for studies of pathogenicity, antiviral designs, and vaccine development against the virus.',\n",
       " 'The novel human coronavirus disease COVID-19 has become the fifth documented pandemic since the 1918 flu pandemic. COVID-19 was first reported in Wuhan, China, and subsequently spread worldwide. The coronavirus was officially named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses based on phylogenetic analysis. SARS-CoV-2 is believed to be a spillover of an animal coronavirus and later adapted the ability of human-to-human transmission. Because the virus is highly contagious, it rapidly spreads and continuously evolves in the human population. In this review article, we discuss the basic properties, potential origin, and evolution of the novel human coronavirus. These factors may be critical for studies of pathogenicity, antiviral designs, and vaccine development against the virus.',\n",
       " 'In December 2019, China diagnosed the first patient with 2019 novel coronavirus disease (COVID-19), and the following development of the epidemic had a huge impact on China and the whole world. For patients with lung occupying lesions, the whole process of diagnosis and treatment can not be carried out as usual due to the epidemic. For thoracic surgeons, the timing of surgical intervention should be very carefully considered. All thoracic surgeons in China should work together to develop the proper procedures for the diagnosis and treatment in this special situation, and continuously update the recommendations based on epidemic changes and further understanding of COVID-19. Here, we only offer some preliminary suggestions based on our own knowledge for further reference and discussion.',\n",
       " 'In December 2019, China diagnosed the first patient with 2019 novel coronavirus disease (COVID-19), and the following development of the epidemic had a huge impact on China and the whole world. For patients with lung occupying lesions, the whole process of diagnosis and treatment can not be carried out as usual due to the epidemic. For thoracic surgeons, the timing of surgical intervention should be very carefully considered. All thoracic surgeons in China should work together to develop the proper procedures for the diagnosis and treatment in this special situation, and continuously update the recommendations based on epidemic changes and further understanding of COVID-19. Here, we only offer some preliminary suggestions based on our own knowledge for further reference and discussion.',\n",
       " 'In December 2019, China diagnosed the first patient with 2019 novel coronavirus disease (COVID-19), and the following development of the epidemic had a huge impact on China and the whole world. For patients with lung occupying lesions, the whole process of diagnosis and treatment can not be carried out as usual due to the epidemic. For thoracic surgeons, the timing of surgical intervention should be very carefully considered. All thoracic surgeons in China should work together to develop the proper procedures for the diagnosis and treatment in this special situation, and continuously update the recommendations based on epidemic changes and further understanding of COVID-19. Here, we only offer some preliminary suggestions based on our own knowledge for further reference and discussion.',\n",
       " 'OBJECTIVES: The novel coronavirus (SARS-CoV-2) originating from Wuhan spread rapidly throughout China. While its origin remains uncertain, accumulating evidence links a wet market with the early spread of SARS-CoV-2 in Wuhan. Similarly, the influence of the marketplace on the early transmission dynamics is yet to be investigated. METHODS: Using the daily series of COVID-19 incidence, stratified according to contact history with the market, we have conducted quantitative modeling analyses to estimate the reproduction numbers (R) for market-to-human and human-to-human transmission, the reporting probability, and the early effects of public health interventions. RESULTS: We estimated R at 0.24 (95% CrI: 0.01-1.38) for market-to-human transmission and 2.37 (95% CrI: 2.08-2.71) for human-to-human transmission during the early spread in China (2019-2020). Moreover, we estimated that the reporting rate for cases stemming from market-to-human transmission was 2-34 fold higher than that for cases stemming from human-to-human transmission, suggesting that contact history with the wet market played a key role in identifying COVID-19 cases. CONCLUSIONS: Our R estimate tied to market-to-human transmission had substantial uncertainty, but it was significantly lower compared with the reproduction number driving human-to-human transmission. Our results also suggest that asymptomatic and subclinical infections constitute a substantial component of the COVID-19 morbidity burden.',\n",
       " 'OBJECTIVES: The novel coronavirus (SARS-CoV-2) originating from Wuhan spread rapidly throughout China. While its origin remains uncertain, accumulating evidence links a wet market with the early spread of SARS-CoV-2 in Wuhan. Similarly, the influence of the marketplace on the early transmission dynamics is yet to be investigated. METHODS: Using the daily series of COVID-19 incidence, stratified according to contact history with the market, we have conducted quantitative modeling analyses to estimate the reproduction numbers (R) for market-to-human and human-to-human transmission, the reporting probability, and the early effects of public health interventions. RESULTS: We estimated R at 0.24 (95% CrI: 0.01-1.38) for market-to-human transmission and 2.37 (95% CrI: 2.08-2.71) for human-to-human transmission during the early spread in China (2019-2020). Moreover, we estimated that the reporting rate for cases stemming from market-to-human transmission was 2-34 fold higher than that for cases stemming from human-to-human transmission, suggesting that contact history with the wet market played a key role in identifying COVID-19 cases. CONCLUSIONS: Our R estimate tied to market-to-human transmission had substantial uncertainty, but it was significantly lower compared with the reproduction number driving human-to-human transmission. Our results also suggest that asymptomatic and subclinical infections constitute a substantial component of the COVID-19 morbidity burden.',\n",
       " 'The use of Chinese-origin rhesus macaques (Macaca mulatta) for infectious disease immunity research is increasing despite the relative lack of major histocompatibility complex (MHC) class I immunogenetics information available for this population. We determined transcript-based MHC class I haplotypes for 385 Chinese rhesus macaques from five different experimental cohorts, providing a concise representation of the full complement of MHC class I major alleles expressed by each animal. In total, 123 Mamu-A and Mamu-B haplotypes were defined in the full Chinese rhesus macaque cohort. We then performed an analysis of haplotype frequencies across the experimental cohorts of Chinese rhesus macaques, as well as a comparison against a group of 96 Indian rhesus macaques. Notably, 35 of the 51 Mamu-A and Mamu-B haplotypes observed in Indian rhesus macaques were also detected in the Chinese population, with 85% of the 385 Chinese-origin rhesus macaques expressing at least one of these class I haplotypes. This unexpected conservation of Indian rhesus macaque MHC class I haplotypes in the Chinese rhesus macaque population suggests that immunologic insights originally gleaned from studies using Indian rhesus macaques may be more applicable to Chinese rhesus macaques than previously appreciated and may provide an opportunity for studies of CD8(+) T-cell responses between populations. It may also be possible to extend these studies across multiple species of macaques, as we found evidence of shared ancestral haplotypes between Chinese rhesus and Mauritian cynomolgus macaques.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'The novel coronavirus SARS-CoV-2 (2019-nCoV) is a member of the family coronaviridae and contains a single-stranded RNA genome with positive-polarity. To reveal the evolution mechanism of SARS-CoV-2 genome, we performed comprehensive genomic analysis with newly sequenced SARS-CoV-2 strains and 20 closely related coronavirus strains. Among 98 nucleotide mutations at 93 sites of the genome among different SARS-CoV-2 strains, 58 of them caused amino acid change, indicating a result of neutral evolution. However, the ratio of nucleotide substitutions to amino acid substitutions of spike gene (9.07) between SARS-CoV-2 WIV04 and Bat-SARSr-CoV RaTG13 was extensively higher than those from comparisons between other coronaviruses (range 1.29 - 4.81). The elevated synonymous mutations between SARS-CoV-2 and RaTG13, suggesting they underwent stronger purifying selection. Moreover, their nucleotide substitutions are enriched with T:C transition, which is consistent with the mutation signature caused by deactivity of RNA 3’-to-5’ exoribonuclease (ExoN). The codon usage was similar between SARS-CoV-2 and other strains in beta-coronavirus lineage B, suggesting it had small impact on the mutation pattern. In comparison of SARS-CoV-2 WIV04 with Bat-SARSr-CoV RaTG13, the ratios of non-synonymous to synonymous substitution rates (dN/dS) was the lowest among all performed comparisons, reconfirming the evolution of SARS-CoV-2 under stringent selective pressure. Moreover, some sites of spike protein might be subjected to positive selection. Therefore, our results will help understanding the evolutionary mechanisms contribute to viral pathogenicity and its adaptation with hosts.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'Since the 2019 novel coronavirus disease (COVID-19) outbreak originated from Wuhan, Hubei Province, China, at the end of 2019, it has become a clinical threat to the general population worldwide. Among people infected with the novel coronavirus (2019-nCoV), the intensive management of the critically ill patients in intensive care unit (ICU) needs substantial medical resource. In the present article, we have summarized the promising drugs, adjunctive agents, respiratory supportive strategies, as well as circulation management, multiple organ function monitoring and appropriate nutritional strategies for the treatment of COVID-19 in the ICU based on the previous experience of treating other viral infections and influenza. These treatments are referable before the vaccine and specific drugs are available for COVID-19.',\n",
       " 'A novel coronavirus emerged in human populations and spread rapidly to cause the global coronavirus disease 2019 pandemic. Although the origin of the associated virus (severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]) remains unclear, genetic evidence suggests that bats are a reservoir host of the virus, and pangolins are a probable intermediate. SARS-CoV-2 has crossed the species barrier to infect humans and other animal species, and infected humans can facilitate reverse-zoonotic transmission to animals. Considering the rapidly changing interconnections among people, animals, and ecosystems, traditional roles of veterinarians should evolve to include transdisciplinary roles.',\n",
       " 'A novel coronavirus emerged in human populations and spread rapidly to cause the global coronavirus disease 2019 pandemic. Although the origin of the associated virus (severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]) remains unclear, genetic evidence suggests that bats are a reservoir host of the virus, and pangolins are a probable intermediate. SARS-CoV-2 has crossed the species barrier to infect humans and other animal species, and infected humans can facilitate reverse-zoonotic transmission to animals. Considering the rapidly changing interconnections among people, animals, and ecosystems, traditional roles of veterinarians should evolve to include transdisciplinary roles.',\n",
       " 'A novel coronavirus emerged in human populations and spread rapidly to cause the global coronavirus disease 2019 pandemic. Although the origin of the associated virus (severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]) remains unclear, genetic evidence suggests that bats are a reservoir host of the virus, and pangolins are a probable intermediate. SARS-CoV-2 has crossed the species barrier to infect humans and other animal species, and infected humans can facilitate reverse-zoonotic transmission to animals. Considering the rapidly changing interconnections among people, animals, and ecosystems, traditional roles of veterinarians should evolve to include transdisciplinary roles.',\n",
       " 'A novel coronavirus emerged in human populations and spread rapidly to cause the global coronavirus disease 2019 pandemic. Although the origin of the associated virus (severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]) remains unclear, genetic evidence suggests that bats are a reservoir host of the virus, and pangolins are a probable intermediate. SARS-CoV-2 has crossed the species barrier to infect humans and other animal species, and infected humans can facilitate reverse-zoonotic transmission to animals. Considering the rapidly changing interconnections among people, animals, and ecosystems, traditional roles of veterinarians should evolve to include transdisciplinary roles.',\n",
       " 'Researchers have reported some useful information about the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) leading to CoV disease 2019 (COVID-19). Several studies have been performed in order to develop antiviral drugs, from which a few have been prescribed to patients. Also, several diagnostic tests have been designed to accelerate the process of identifying and treating COVID-19. It has been well-documented that the surface of host cells is covered by some receptors, known as angiotensin-converting enzyme 2 (ACE2), which mediates the binding and entry of CoV. After entering, the viral RNA interrupts the cell proliferation system to activate self-proliferation. However, having all the information about the outbreakof the SARS-COV-2, it is not still clear which factors determine the severity of lung and heart function impairment induced by COVID-19. A major step in exploring SARS-COV-2 pathogenesis is to determine the distribution of ACE2 in different tissues . In this review, the structure and origin of CoV, the role of ACE2 as a receptor of SARS-COV-2 on the surface of host cells, and the ACE2 distribution in different tissues with a focus on lung and cardiovascular system have been discussed. It was also revealed that acute and chronic cardiovascular diseases (CVDs) may result in the clinical severity of COVID-19. In conclusion, this review may provide useful information in developing some promising strategies to end up with a worldwide COVID-19 pandemic. Communicated by Ramaswamy H. Sarma',\n",
       " 'Researchers have reported some useful information about the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) leading to CoV disease 2019 (COVID-19). Several studies have been performed in order to develop antiviral drugs, from which a few have been prescribed to patients. Also, several diagnostic tests have been designed to accelerate the process of identifying and treating COVID-19. It has been well-documented that the surface of host cells is covered by some receptors, known as angiotensin-converting enzyme 2 (ACE2), which mediates the binding and entry of CoV. After entering, the viral RNA interrupts the cell proliferation system to activate self-proliferation. However, having all the information about the outbreakof the SARS-COV-2, it is not still clear which factors determine the severity of lung and heart function impairment induced by COVID-19. A major step in exploring SARS-COV-2 pathogenesis is to determine the distribution of ACE2 in different tissues . In this review, the structure and origin of CoV, the role of ACE2 as a receptor of SARS-COV-2 on the surface of host cells, and the ACE2 distribution in different tissues with a focus on lung and cardiovascular system have been discussed. It was also revealed that acute and chronic cardiovascular diseases (CVDs) may result in the clinical severity of COVID-19. In conclusion, this review may provide useful information in developing some promising strategies to end up with a worldwide COVID-19 pandemic. Communicated by Ramaswamy H. Sarma',\n",
       " 'Researchers have reported some useful information about the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) leading to CoV disease 2019 (COVID-19). Several studies have been performed in order to develop antiviral drugs, from which a few have been prescribed to patients. Also, several diagnostic tests have been designed to accelerate the process of identifying and treating COVID-19. It has been well-documented that the surface of host cells is covered by some receptors, known as angiotensin-converting enzyme 2 (ACE2), which mediates the binding and entry of CoV. After entering, the viral RNA interrupts the cell proliferation system to activate self-proliferation. However, having all the information about the outbreakof the SARS-COV-2, it is not still clear which factors determine the severity of lung and heart function impairment induced by COVID-19. A major step in exploring SARS-COV-2 pathogenesis is to determine the distribution of ACE2 in different tissues . In this review, the structure and origin of CoV, the role of ACE2 as a receptor of SARS-COV-2 on the surface of host cells, and the ACE2 distribution in different tissues with a focus on lung and cardiovascular system have been discussed. It was also revealed that acute and chronic cardiovascular diseases (CVDs) may result in the clinical severity of COVID-19. In conclusion, this review may provide useful information in developing some promising strategies to end up with a worldwide COVID-19 pandemic. Communicated by Ramaswamy H. Sarma',\n",
       " \"Coronavirus disease 2019 (COVID-19) is expanding rapidly, which made it as one of top priorities for scientists to develop novel treatment strategies. Researchers are racing to develop treatments based on antibodies to block and/or neutralize the coronavirus in affected patients. Initially, the genetic and structural similarity of the virus to severe acute respiratory syndrome coronavirus (SARS-CoV) created the potential for understanding disease pathogenesis. Researchers have published reports of specific monoclonal antibodies against to COVID-19 (B38, H4, 47D11) and hope that this method is effective. As well as studies on patients who are plasma therapy, the patient's condition shows improvement. The evidence for these studies is very promising and demonstrates the potential of monoclonal antibody therapy as a therapeutic approach and prevention of covid-19 infection.\",\n",
       " \"Coronavirus disease 2019 (COVID-19) is expanding rapidly, which made it as one of top priorities for scientists to develop novel treatment strategies. Researchers are racing to develop treatments based on antibodies to block and/or neutralize the coronavirus in affected patients. Initially, the genetic and structural similarity of the virus to severe acute respiratory syndrome coronavirus (SARS-CoV) created the potential for understanding disease pathogenesis. Researchers have published reports of specific monoclonal antibodies against to COVID-19 (B38, H4, 47D11) and hope that this method is effective. As well as studies on patients who are plasma therapy, the patient's condition shows improvement. The evidence for these studies is very promising and demonstrates the potential of monoclonal antibody therapy as a therapeutic approach and prevention of covid-19 infection.\",\n",
       " \"Coronavirus disease 2019 (COVID-19) is expanding rapidly, which made it as one of top priorities for scientists to develop novel treatment strategies. Researchers are racing to develop treatments based on antibodies to block and/or neutralize the coronavirus in affected patients. Initially, the genetic and structural similarity of the virus to severe acute respiratory syndrome coronavirus (SARS-CoV) created the potential for understanding disease pathogenesis. Researchers have published reports of specific monoclonal antibodies against to COVID-19 (B38, H4, 47D11) and hope that this method is effective. As well as studies on patients who are plasma therapy, the patient's condition shows improvement. The evidence for these studies is very promising and demonstrates the potential of monoclonal antibody therapy as a therapeutic approach and prevention of covid-19 infection.\",\n",
       " 'Coronavirus has emerged as a global health threat due to its accelerated geographic spread over the last two decades. This article reviews the current state of knowledge concerning the origin, transmission, diagnosis and management of coronavirus disease 2019 (COVID-19). Historically, it has caused two pandemics: severe acute respiratory syndrome and Middle East respiratory syndrome followed by the present COVID-19 that emerged from China. The virus is believed to be acquired from zoonotic source and spreads through direct and contact transmission. The symptomatic phase manifests with fever, cough and myalgia to severe respiratory failure. The diagnosis is confirmed using reverse transcriptase PCR. Management of COVID-19 is mainly by supportive therapy along with mechanical ventilation in severe cases. Preventive strategies form the major role in reducing the public spread of virus along with successful disease isolation and community containment. Development of a vaccine to eliminate the virus from the host still remains an ongoing challenge.',\n",
       " 'Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an emerging zoonotic virus considered as one of the major public threat with a total number of 2 298 laboratory-confirmed cases and 811 associated deaths reported by World Health Organization as of January 2019. The transmission of the virus was expected to be from the camels found in Middle Eastern countries via the animal and human interaction. The genome structure provided information about the pathogenicity and associated virulent factors present in the virus. Recent studies suggested that there were limited insight available on the development of novel therapeutic strategies to induce immunity against the virus. The severities of MERS-CoV infection highlight the necessity of effective approaches for the development of various therapeutic remedies. Thus, the present review comprehensively and critically illustrates the recent aspects on the epidemiology of the virus, the structural and functional features of the viral genome, viral entry and transmission, major mechanisms of pathogenesis and associated virulent factors, current animal models, detection methods and novel strategies for the development of vaccines against MERS-CoV. The review further illustrates the molecular and computational virtual screening platforms which provide insights for the identification of putative drug targets and novel lead molecules toward the development of therapeutic remedies.',\n",
       " 'Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an emerging zoonotic virus considered as one of the major public threat with a total number of 2 298 laboratory-confirmed cases and 811 associated deaths reported by World Health Organization as of January 2019. The transmission of the virus was expected to be from the camels found in Middle Eastern countries via the animal and human interaction. The genome structure provided information about the pathogenicity and associated virulent factors present in the virus. Recent studies suggested that there were limited insight available on the development of novel therapeutic strategies to induce immunity against the virus. The severities of MERS-CoV infection highlight the necessity of effective approaches for the development of various therapeutic remedies. Thus, the present review comprehensively and critically illustrates the recent aspects on the epidemiology of the virus, the structural and functional features of the viral genome, viral entry and transmission, major mechanisms of pathogenesis and associated virulent factors, current animal models, detection methods and novel strategies for the development of vaccines against MERS-CoV. The review further illustrates the molecular and computational virtual screening platforms which provide insights for the identification of putative drug targets and novel lead molecules toward the development of therapeutic remedies.',\n",
       " 'Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is an emerging zoonotic virus considered as one of the major public threat with a total number of 2 298 laboratory-confirmed cases and 811 associated deaths reported by World Health Organization as of January 2019. The transmission of the virus was expected to be from the camels found in Middle Eastern countries via the animal and human interaction. The genome structure provided information about the pathogenicity and associated virulent factors present in the virus. Recent studies suggested that there were limited insight available on the development of novel therapeutic strategies to induce immunity against the virus. The severities of MERS-CoV infection highlight the necessity of effective approaches for the development of various therapeutic remedies. Thus, the present review comprehensively and critically illustrates the recent aspects on the epidemiology of the virus, the structural and functional features of the viral genome, viral entry and transmission, major mechanisms of pathogenesis and associated virulent factors, current animal models, detection methods and novel strategies for the development of vaccines against MERS-CoV. The review further illustrates the molecular and computational virtual screening platforms which provide insights for the identification of putative drug targets and novel lead molecules toward the development of therapeutic remedies.',\n",
       " 'In the original version of the article, the article title was processed incorrectly. The correct article title is \"Venous Thromboembolism and Heparin Use in COVID-19 Patients: Juggling between Pragmatic Choices, Suggestions of Medical Societies and the Lack of Guidelines\". This has been corrected with this erratum and the original article has also been updated to reflect the change in article title.',\n",
       " 'In the original version of the article, the article title was processed incorrectly. The correct article title is \"Venous Thromboembolism and Heparin Use in COVID-19 Patients: Juggling between Pragmatic Choices, Suggestions of Medical Societies and the Lack of Guidelines\". This has been corrected with this erratum and the original article has also been updated to reflect the change in article title.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new strain that was discovered in 2019 and has not been previously identified in humans. On December 31st 2019 World Health Organization (WHO) was informed of a cluster of cases with pneumonia of unknown origin from Wuhan City, Hubei province of China. The WHO announced in February 2020 that COVID-19 is the official name of the coronavirus diseases. A total of 519,899 confirmed cases with 23,592 deaths linked to this pathogen as on March 27, 2020 have been reported. Due to increasing number of infected people across the continents and huge loss to human life, the WHO has declared the novel COVID-19 outbreak a pandemic. A pandemic is defined as the \"worldwide spread\" of a new disease. Currently, no COVID-19 specific treatments have been approved by the United States Food and Drug Administration (US-FDA). However, the current treatment options include hydroxychloroquine, tocilizumab, remdesivir, lopinavir-ritonavir (Kaletra®), and nitazoxanide. In recent past, some natural herbal compounds have demonstrated encouraging anti-viral properties. This article attempted to summarize available information on the reported anti-viral activity of some natural products.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new strain that was discovered in 2019 and has not been previously identified in humans. On December 31st 2019 World Health Organization (WHO) was informed of a cluster of cases with pneumonia of unknown origin from Wuhan City, Hubei province of China. The WHO announced in February 2020 that COVID-19 is the official name of the coronavirus diseases. A total of 519,899 confirmed cases with 23,592 deaths linked to this pathogen as on March 27, 2020 have been reported. Due to increasing number of infected people across the continents and huge loss to human life, the WHO has declared the novel COVID-19 outbreak a pandemic. A pandemic is defined as the \"worldwide spread\" of a new disease. Currently, no COVID-19 specific treatments have been approved by the United States Food and Drug Administration (US-FDA). However, the current treatment options include hydroxychloroquine, tocilizumab, remdesivir, lopinavir-ritonavir (Kaletra®), and nitazoxanide. In recent past, some natural herbal compounds have demonstrated encouraging anti-viral properties. This article attempted to summarize available information on the reported anti-viral activity of some natural products.',\n",
       " 'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a new strain that was discovered in 2019 and has not been previously identified in humans. On December 31st 2019 World Health Organization (WHO) was informed of a cluster of cases with pneumonia of unknown origin from Wuhan City, Hubei province of China. The WHO announced in February 2020 that COVID-19 is the official name of the coronavirus diseases. A total of 519,899 confirmed cases with 23,592 deaths linked to this pathogen as on March 27, 2020 have been reported. Due to increasing number of infected people across the continents and huge loss to human life, the WHO has declared the novel COVID-19 outbreak a pandemic. A pandemic is defined as the \"worldwide spread\" of a new disease. Currently, no COVID-19 specific treatments have been approved by the United States Food and Drug Administration (US-FDA). However, the current treatment options include hydroxychloroquine, tocilizumab, remdesivir, lopinavir-ritonavir (Kaletra®), and nitazoxanide. In recent past, some natural herbal compounds have demonstrated encouraging anti-viral properties. This article attempted to summarize available information on the reported anti-viral activity of some natural products.',\n",
       " 'Protein microbicides containing neutralizing antibodies and antiviral lectins may help to reduce the rate of infection with human immunodeficiency virus (HIV) if it is possible to manufacture the components in large quantities at a cost affordable in HIV‐endemic regions such as sub‐Saharan Africa. We expressed the antiviral lectin griffithsin (GRFT), which shows potent neutralizing activity against HIV, in the endosperm of transgenic rice plants (Oryza sativa), to determine whether rice can be used to produce inexpensive GRFT as a microbicide ingredient. The yield of (OS)GRFT in the best‐performing plants was 223 μg/g dry seed weight. We also established a one‐step purification protocol, achieving a recovery of 74% and a purity of 80%, which potentially could be developed into a larger‐scale process to facilitate inexpensive downstream processing. (OS)GRFT bound to HIV glycans with similar efficiency to GRFT produced in Escherichia coli. Whole‐cell assays using purified (OS)GRFT and infectivity assays using crude extracts of transgenic rice endosperm confirmed that both crude and pure (OS)GRFT showed potent activity against HIV and the crude extracts were not toxic towards human cell lines, suggesting they could be administered as a microbicide with only minimal processing. A freedom‐to‐operate analysis confirmed that GRFT produced in rice is suitable for commercial development, and an economic evaluation suggested that 1.8 kg/ha of pure GRFT could be produced from rice seeds. Our data therefore indicate that rice could be developed as an inexpensive production platform for GRFT as a microbicide component.',\n",
       " '• The majority of cases were in the age group of 50 to 60 years of old. • A total of 2964 cases of COVID-19 were investigated. • The male-to-female ratio was 1.93:1. • A significant effect of age, gender and comorbidities on the mortality. • The Case Fatality Rate among understudy cases was 8.06%.',\n",
       " '• The majority of cases were in the age group of 50 to 60 years of old. • A total of 2964 cases of COVID-19 were investigated. • The male-to-female ratio was 1.93:1. • A significant effect of age, gender and comorbidities on the mortality. • The Case Fatality Rate among understudy cases was 8.06%.',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " 'The human coronaviruses have been shown to be a major player in clinical microbiology and frequently occur as pathogens responsible for mild to severe respiratory infections. Moreover, two of the most dangerous viral respiratory infections are caused by novel coronaviruses, namely, the SARS and the MERS coronavirus. This chapter briefly summarizes the most important facts and knowledge required for the appropriate laboratory diagnostics of infections caused by the human coronaviruses.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " '[Image: see text] As the infection of 2019-nCoV coronavirus is quickly developing into a global pneumonia epidemic, the careful analysis of its transmission and cellular mechanisms is sorely needed. In this Communication, we first analyzed two recent studies that concluded that snakes are the intermediate hosts of 2019-nCoV and that the 2019-nCoV spike protein insertions share a unique similarity to HIV-1. However, the reimplementation of the analyses, built on larger scale data sets using state-of-the-art bioinformatics methods and databases, presents clear evidence that rebuts these conclusions. Next, using metagenomic samples from Manis javanica, we assembled a draft genome of the 2019-nCoV-like coronavirus, which shows 73% coverage and 91% sequence identity to the 2019-nCoV genome. In particular, the alignments of the spike surface glycoprotein receptor binding domain revealed four times more variations in the bat coronavirus RaTG13 than in the Manis coronavirus compared with 2019-nCoV, suggesting the pangolin as a missing link in the transmission of 2019-nCoV from bats to human.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'The COVID-19 pandemic has deeply impacted people\\'s lives around the globe. During the extended lockdowns caused by the pandemic, online communities are crucial for people to access information and share experiences. In particular, two\"new\"communities have emerged on Reddit: /r/China_flu and /r/Coronavirus. By studying activities and users in these two communities, we provide a characterization of people\\'s responses to COVID-19 on Reddit. First, we find that user activity peaks around March 17, when the World Health Organization (WHO) announced COVID-19 as a pandemic. Shortly after that, the activity levels of both communities have been declining week by week. We further illustrate the central role of these two communities in the emergence of COVID-related communities. Second, we study the differences between these two communities. /r/Coronavirus is recommended as the official community for COVID-19 on Reddit, while /r/China_flu adopts a loose moderation practice. As a result, we observe that these two communities are gradually growing apart and more extremism is being found in /r/China_flu. Finally, we examine the spillover effect of the COVID-19 pandemic on user activity across the entire Reddit platform. Our results show significant changes in user activities outside COVID-related communities. In subreddits related to finance, food, and countries/cities, user activity is recovering to the pre-pandemic level in late April and May as countries reopen, but subreddits related to travel and sports remain highly impacted and show lower activity levels than the pre-pandemic period. Our work highlights the strength of Reddit as a source for understanding public reactions to COVID-19 and the importance of content moderation on the Internet during a pandemic.',\n",
       " 'This introduction provides a telegraphic overview of the processes of zoonotic viral emergence, the intricacies of host–virus interactions, and the distinct role of biological transitions and modifying factors. The process of emergence is conceptualized as two transition stages which are common and required for all disease emergence, (1) human contact with the infectious agent and (2) cross-species transmission of the agent, and two transition stages which are not required for emergence and appear unavailable to many zoonotic pathogens, (3) sustained human-to-human transmission and (4) genetic adaptation to the human host. The latter two transitions are presumably prerequisites for the pandemic emergence of a pathogen. The themes introduced herein are amplified and explored in detail by the contributors to this volume. Each author explores the mechanisms and unique circumstances by which evolution, biology, history, and current context have contrived to drive the emergence of different zoonotic agents by a series of related events; although recognizable similarities exist among the events leading to emergence the details and circumstances are never repetitive.',\n",
       " '',\n",
       " '',\n",
       " 'Coronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus, which also includes severe acute respiratory syndrome-related coronavirus (SARSr-CoV) and Middle East respiratory syndrome-related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses helped identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, we compared the codon usage bias using global correspondence analysis (CA), within-group CA and between-group CA. We found that the bat RaTG13 virus best matched the overall codon usage pattern of SARS-CoV-2 in orf1ab, spike and nucleocapsid genes, while the pangolin P1E virus had a more similar codon usage in membrane gene. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.',\n",
       " 'Coronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus, which also includes severe acute respiratory syndrome-related coronavirus (SARSr-CoV) and Middle East respiratory syndrome-related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses helped identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, we compared the codon usage bias using global correspondence analysis (CA), within-group CA and between-group CA. We found that the bat RaTG13 virus best matched the overall codon usage pattern of SARS-CoV-2 in orf1ab, spike and nucleocapsid genes, while the pangolin P1E virus had a more similar codon usage in membrane gene. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.',\n",
       " 'Coronavirus disease 2019 (COVID-19) is a global health concern as it continues to spread within China and beyond. The causative agent of this disease, severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), belongs to the genus Betacoronavirus, which also includes severe acute respiratory syndrome-related coronavirus (SARSr-CoV) and Middle East respiratory syndrome-related coronavirus (MERSr-CoV). Codon usage of viral genes are believed to be subjected to different selection pressures in different host environments. Previous studies on codon usage of influenza A viruses helped identify viral host origins and evolution trends, however, similar studies on coronaviruses are lacking. In this study, we compared the codon usage bias using global correspondence analysis (CA), within-group CA and between-group CA. We found that the bat RaTG13 virus best matched the overall codon usage pattern of SARS-CoV-2 in orf1ab, spike and nucleocapsid genes, while the pangolin P1E virus had a more similar codon usage in membrane gene. The amino acid usage pattern of SARS-CoV-2 was generally found similar to bat and human SARSr-CoVs. However, we found greater synonymous codon usage differences between SARS-CoV-2 and its phylogenetic relatives on spike and membrane genes, suggesting these two genes of SARS-CoV-2 are subjected to different evolutionary pressures.',\n",
       " 'Coronavirus disease 2019 (COVID-19), has spread over 210 countries and territories beyond China shortly. On February 29, 2020, the World Health Organization (WHO) denoted it in a high-risk category, and on March 11, 2020, this virus was designated pandemic, after its declaration being a Public Health International Emergency on January 30, 2020. World over high efforts are being made to counter and contain this virus. The COVID-19 outbreak once again proves the potential of the animal-human interface to act as the primary source of emerging zoonotic diseases. Even though the circumstantial evidence suggests the possibility of an initial zoonotic emergence, it is too early to confirm the role of intermediate hosts such as snakes, pangolins, turtles, and other wild animals in the origin of SARS-CoV-2, in addition to bats, the natural hosts of multiple coronaviruses such as SARS-CoV and MERS-CoV. The lessons learned from past episodes of MERS-CoV and SARS-CoV are being exploited to retort this virus. Best efforts are being taken up by worldwide nations to implement effective diagnosis, strict vigilance, heightened surveillance, and monitoring, along with adopting appropriate preventive and control strategies. Identifying the possible zoonotic emergence and the exact mechanism responsible for its initial transmission will help us to design and implement appropriate preventive barriers against the further transmission of SARS-CoV-2. This review discusses in brief about the COVID-19/SARS-CoV-2 with a particular focus on the role of animals, the veterinary and associated zoonotic links along with prevention and control strategies based on One-health approaches.',\n",
       " 'Middle East respiratory syndrome coronavirus (MERS-CoV) still causes outbreaks despite public awareness and implementation of health care measures, such as rapid viral diagnosis and patient quarantine. Here we describe the current epidemiological picture of MERS-CoV, focusing on humans and animals affected by this virus and propose specific intervention strategies that would be appropriate to control MERS-CoV. One-third of MERS-CoV patients develop severe lower respiratory tract infection and succumb to a fatal outcome; these patients would require effective therapeutic antiviral therapy. Because of the lack of such intervention strategies, supportive care is the best that can be offered at the moment. Limiting viral spread from symptomatic human cases to health care workers and family members, on the other hand, could be achieved through prophylactic administration of MERS-CoV neutralizing antibodies and vaccines. To ultimately prevent spread of the virus into the human population, however, vaccination of dromedary camels – currently the only confirmed animal host for MERS-CoV – may be the best option to achieve a sustained drop in human MERS cases in time. In the end, a One Health approach combining all these different efforts is needed to tackle this zoonotic outbreak.',\n",
       " 'Middle East respiratory syndrome coronavirus (MERS-CoV) still causes outbreaks despite public awareness and implementation of health care measures, such as rapid viral diagnosis and patient quarantine. Here we describe the current epidemiological picture of MERS-CoV, focusing on humans and animals affected by this virus and propose specific intervention strategies that would be appropriate to control MERS-CoV. One-third of MERS-CoV patients develop severe lower respiratory tract infection and succumb to a fatal outcome; these patients would require effective therapeutic antiviral therapy. Because of the lack of such intervention strategies, supportive care is the best that can be offered at the moment. Limiting viral spread from symptomatic human cases to health care workers and family members, on the other hand, could be achieved through prophylactic administration of MERS-CoV neutralizing antibodies and vaccines. To ultimately prevent spread of the virus into the human population, however, vaccination of dromedary camels – currently the only confirmed animal host for MERS-CoV – may be the best option to achieve a sustained drop in human MERS cases in time. In the end, a One Health approach combining all these different efforts is needed to tackle this zoonotic outbreak.',\n",
       " 'Background: Understanding the extent of virus transmission that can occur before symptom onset is vital for targeting control measures against the global pandemic of COVID-19. Objective: Estimation of (1) the proportion of pre-symptomatic transmission of COVID-19 that can occur and (2) timing of transmission relative to symptom onset. Design: Secondary analysis of published data Data sources: Meta-analysis of COVID-19 incubation period and a rapid systematic review of serial interval and generation time, which are published separately. Methods: Simulations were generated of incubation period and of serial interval or generation time. From these, transmission times relative to symptom onset were calculated and the proportion of pre-symptomatic transmission was estimated. Results: A total of 23 estimates of serial interval and five estimates of generation time from 17 publications were included. These came from nine different data source categories (presented here in descending order of the proportion of pre-symptomatic transmission): Hong Kong, Tianjin, pooled data from Hong Kong and Shenzhen, Singapore, Mainland China excluding Hubei, mixed sources, Shenzhen, northern Italy and Wuhan. Transmission time relative to symptom onset ranged from a mean of 2.05 days before symptom onset for Hong Kong to 1.72 days after symptom onset for Wuhan. Proportion of pre-symptomatic transmission ranged from 33.7% in Wuhan to 72.7% in Hong Kong. Based on individual estimates, transmission time relative to symptom onset ranged from mean of 2.95 days before symptom onset to 1.72 days after symptom onset and proportion of pre-symptomatic transmission ranged from 33.7% to 79.9%. Simple unweighted pooling of estimates based on serial intervals resulted in a mean time of transmission of 0.67 days before symptoms, and an estimated 56.1% of transmission occurring in the pre-symptomatic period. Conclusions: Contact rates between symptomatic infectious and susceptible people are likely to influence the proportion of pre-symptomatic transmission. There is substantial potential for pre-symptomatic transmission of COVID-19 in a range of different contexts. Our work suggests that transmission of SARS-CoV-2 is most likely in the day before symptom onset whereas estimates suggesting most pre-symptomatic transmission highlighted a mean transmission times almost 3 days before symptom onset. These findings highlight the urgent need for extremely rapid and effective case detection, contact tracing and quarantine measures if strict social distancing measures are to be eased.',\n",
       " 'Background: Understanding the extent of virus transmission that can occur before symptom onset is vital for targeting control measures against the global pandemic of COVID-19. Objective: Estimation of (1) the proportion of pre-symptomatic transmission of COVID-19 that can occur and (2) timing of transmission relative to symptom onset. Design: Secondary analysis of published data Data sources: Meta-analysis of COVID-19 incubation period and a rapid systematic review of serial interval and generation time, which are published separately. Methods: Simulations were generated of incubation period and of serial interval or generation time. From these, transmission times relative to symptom onset were calculated and the proportion of pre-symptomatic transmission was estimated. Results: A total of 23 estimates of serial interval and five estimates of generation time from 17 publications were included. These came from nine different data source categories (presented here in descending order of the proportion of pre-symptomatic transmission): Hong Kong, Tianjin, pooled data from Hong Kong and Shenzhen, Singapore, Mainland China excluding Hubei, mixed sources, Shenzhen, northern Italy and Wuhan. Transmission time relative to symptom onset ranged from a mean of 2.05 days before symptom onset for Hong Kong to 1.72 days after symptom onset for Wuhan. Proportion of pre-symptomatic transmission ranged from 33.7% in Wuhan to 72.7% in Hong Kong. Based on individual estimates, transmission time relative to symptom onset ranged from mean of 2.95 days before symptom onset to 1.72 days after symptom onset and proportion of pre-symptomatic transmission ranged from 33.7% to 79.9%. Simple unweighted pooling of estimates based on serial intervals resulted in a mean time of transmission of 0.67 days before symptoms, and an estimated 56.1% of transmission occurring in the pre-symptomatic period. Conclusions: Contact rates between symptomatic infectious and susceptible people are likely to influence the proportion of pre-symptomatic transmission. There is substantial potential for pre-symptomatic transmission of COVID-19 in a range of different contexts. Our work suggests that transmission of SARS-CoV-2 is most likely in the day before symptom onset whereas estimates suggesting most pre-symptomatic transmission highlighted a mean transmission times almost 3 days before symptom onset. These findings highlight the urgent need for extremely rapid and effective case detection, contact tracing and quarantine measures if strict social distancing measures are to be eased.',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'Background: Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is a virus that causes severe respiratory illness in humans, which eventually results in the current outbreak of novel coronavirus disease (COVID-19) around the world. The research community is interested to know what are the hotspots in coronavirus (CoV) research and how much is known about COVID-19. This study aimed to evaluate the characteristics of publications involving coronaviruses as well as COVID-19 by using a topic modeling analysis. Methods: We extracted all abstracts and retained the most informative words from the COVID-19 Open Research Dataset, which contains all the 35,092 pieces of coronavirus related literature published up to March 20, 2020. Using Latent Dirichlet Allocation modeling, we trained an eight-topic model from the corpus. We then analyzed the semantic relationships between topics and compared the topic distribution between COVID-19 and other CoV infections. Results: Eight topics emerged overall: clinical characterization, pathogenesis research, therapeutics research, epidemiological study, virus transmission, vaccines research, virus diagnostics, and viral genomics. It was observed that COVID-19 research puts more emphasis on clinical characterization, epidemiological study, and virus transmission at present. In contrast, topics about diagnostics, therapeutics, vaccines, genomics and pathogenesis only accounted for less than 10% or even 4% of all the COVID-19 publications, much lower than those of other CoV infections. Conclusions: These results identified knowledge gaps in the area of COVID-19 and offered directions for future research. Keywords: COVID-19, coronavirus, topic modeling, hotspots, text mining',\n",
       " 'We have recently reported for the first time that SARS-CoV-2 maybe a bat-originated coronavirus with a recombination occurred within the spike (S) protein gene based on phylogenetic and simplot analyses1 . These two conclusions are supported by findings recently reported by others and are well accepted in the field of SARS-CoV-2 research2-4 . This article is protected by copyright. All rights reserved.',\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " \"BACKGROUND: Whole genome sequence data is a step towards generating the 'parts list' of life to understand the underlying principles of Biocomplexity. Genome sequencing initiatives of human and model organisms are targeted efforts towards understanding principles of evolution with an application envisaged to improve human health. These efforts culminated in the development of dedicated resources. Whereas a large number of viral genomes have been sequenced by groups or individuals with an interest to study antigenic variation amongst strains and species. These independent efforts enabled viruses to attain the status of 'best-represented taxa' with the highest number of genomes. However, due to lack of concerted efforts, viral genomic sequences merely remained as entries in the public repositories until recently. RESULTS: VirGen is a curated resource of viral genomes and their analyses. Since its first release, it has grown both in terms of coverage of viral families and development of new modules for annotation and analysis. The current release (2.0) includes data for twenty-five families with broad host range as against eight in the first release. The taxonomic description of viruses in VirGen is in accordance with the ICTV nomenclature. A well-characterised strain is identified as a 'representative entry' for every viral species. This non-redundant dataset is used for subsequent annotation and analyses using sequenced-based Bioinformatics approaches. VirGen archives precomputed data on genome and proteome comparisons. A new data module that provides structures of viral proteins available in PDB has been incorporated recently. One of the unique features of VirGen is predicted conformational and sequential epitopes of known antigenic proteins using in-house developed algorithms, a step towards reverse vaccinology. CONCLUSION: Structured organization of genomic data facilitates use of data mining tools, which provides opportunities for knowledge discovery. One of the approaches to achieve this goal is to carry out functional annotations using comparative genomics. VirGen, a comprehensive viral genome resource that serves as an annotation and analysis pipeline has been developed for the curation of public domain viral genome data . Various steps in the curation and annotation of the genomic data and applications of the value-added derived data are substantiated with case studies.\",\n",
       " 'COVID-19 is one of the most important topic these days, specifically on search engines and news. While fake news are easily shared, scientific papers are reliable sources where information can be extracted. With about 24,000 scientific publications on COVID-19 and related research on PUBMED, automatic computer-assisted analysis is required. In this paper, we develop two methodologies to get insights on specific sub-topics of interest and latest research sub-topics. They rely on natural language processing and graph-based visualizations. We run these methodologies on two cases: the virus origin and the uses of existing drugs.',\n",
       " 'COVID-19 is one of the most important topic these days, specifically on search engines and news. While fake news are easily shared, scientific papers are reliable sources where information can be extracted. With about 24,000 scientific publications on COVID-19 and related research on PUBMED, automatic computer-assisted analysis is required. In this paper, we develop two methodologies to get insights on specific sub-topics of interest and latest research sub-topics. They rely on natural language processing and graph-based visualizations. We run these methodologies on two cases: the virus origin and the uses of existing drugs.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " 'World Health Organization has declared the ongoing outbreak of coronavirus disease 2019 (COVID-19) a Public Health Emergency of International Concern. The virus was named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by the International Committee on Taxonomy of Viruses. Human infection with SARS-CoV-2 leads to a wide range of clinical manifestations ranging from asymptomatic, mild, moderate to severe. The severe cases present with pneumonia, which can progress to acute respiratory distress syndrome. The outbreak provides an opportunity for real-time tracking of an animal coronavirus that has just crossed species barrier to infect humans. The outcome of SARS-CoV-2 infection is largely determined by virus-host interaction. Here, we review the discovery, zoonotic origin, animal hosts, transmissibility and pathogenicity of SARS-CoV-2 in relation to its interplay with host antiviral defense. A comparison with SARS-CoV, Middle East respiratory syndrome coronavirus, community-acquired human coronaviruses and other pathogenic viruses including human immunodeficiency viruses is made. We summarize current understanding of the induction of a proinflammatory cytokine storm by other highly pathogenic human coronaviruses, their adaptation to humans and their usurpation of the cell death programmes. Important questions concerning the interaction between SARS-CoV-2 and host antiviral defence, including asymptomatic and presymptomatic virus shedding, are also discussed.',\n",
       " \"Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is a strain of coronavirus that causes coronavirus disease 2019 (Covid-19) and has been declared a global pandemic by the World Health Organization. Total cases of SARS-CoV-2 worldwide exceed 4.8 million, with over 320,000 deaths recorded. Little is known about the body's immune response to SARS-CoV-2 infection. In this paper, we describe SARS-CoV-2 IgG antibody responses in 11,092 patients from the New York City metropolitan area and report a SARS-CoV-2 IgG positivity rate of nearly 50%, indicating the widespread nature of the pandemic in the city and state of New York. Additionally, we report on the correlation between SARS-CoV-2 patient symptom severity and level of SARS-CoV-2 IgG antibody found in the patient sample.\",\n",
       " \"Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is a strain of coronavirus that causes coronavirus disease 2019 (Covid-19) and has been declared a global pandemic by the World Health Organization. Total cases of SARS-CoV-2 worldwide exceed 4.8 million, with over 320,000 deaths recorded. Little is known about the body's immune response to SARS-CoV-2 infection. In this paper, we describe SARS-CoV-2 IgG antibody responses in 11,092 patients from the New York City metropolitan area and report a SARS-CoV-2 IgG positivity rate of nearly 50%, indicating the widespread nature of the pandemic in the city and state of New York. Additionally, we report on the correlation between SARS-CoV-2 patient symptom severity and level of SARS-CoV-2 IgG antibody found in the patient sample.\",\n",
       " \"Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is a strain of coronavirus that causes coronavirus disease 2019 (Covid-19) and has been declared a global pandemic by the World Health Organization. Total cases of SARS-CoV-2 worldwide exceed 4.8 million, with over 320,000 deaths recorded. Little is known about the body's immune response to SARS-CoV-2 infection. In this paper, we describe SARS-CoV-2 IgG antibody responses in 11,092 patients from the New York City metropolitan area and report a SARS-CoV-2 IgG positivity rate of nearly 50%, indicating the widespread nature of the pandemic in the city and state of New York. Additionally, we report on the correlation between SARS-CoV-2 patient symptom severity and level of SARS-CoV-2 IgG antibody found in the patient sample.\",\n",
       " \"Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is a strain of coronavirus that causes coronavirus disease 2019 (Covid-19) and has been declared a global pandemic by the World Health Organization. Total cases of SARS-CoV-2 worldwide exceed 4.8 million, with over 320,000 deaths recorded. Little is known about the body's immune response to SARS-CoV-2 infection. In this paper, we describe SARS-CoV-2 IgG antibody responses in 11,092 patients from the New York City metropolitan area and report a SARS-CoV-2 IgG positivity rate of nearly 50%, indicating the widespread nature of the pandemic in the city and state of New York. Additionally, we report on the correlation between SARS-CoV-2 patient symptom severity and level of SARS-CoV-2 IgG antibody found in the patient sample.\",\n",
       " \"Abstract The ongoing wreaking global outbreak of the novel human beta coronavirus (CoV) pathogen was presumed to be from a seafood wholesale market in Wuhan, China, belongs to the Coronaviridae family in the Nidovirales order. The virus is highly contagious with potential human-human transmission which was named as the severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), has spread across six continents and emerged as a global pandemic in short span with alarming levels of spread and severity. This virus associated symptoms and infectious respiratory illness is designated as coronavirus disease 19 (COVID-19). The SARS-CoV-2 possesses enveloped club-like spike protein projections with positive-sense large RNA genome and has a unique replication strategy. This virus was believed to have zoonotic origin with genetical identity to bat and pangolin CoV. In the current review, we introduce a general overview about the human CoVs and the associated diseases, the origin, structure, replication and key clinical events that occur in the COVID-19 pathogenicity. Furthermore, we focused on possible therapeutic options such as repurposing drugs including antimalarials, antivirals, antiparasitic drugs, and anti-HIV drugs, as well as monoclonal antibodies, vaccines as potential treatment options. Also we have summarized the latest research progress on the usage of stem cell therapy, human convalescent serum, interferon's, in the treatment of COVID-19.\",\n",
       " ...]"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_df['corpus-text'].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "80781e4d",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
